{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "s3_prefix = \"s3://aegovan-data/pubmed_asbtract/predictions_multi_00/\"\n",
    "s3_data =\"s3://aegovan-data/human_output/human_interactions_ppi_v2.json\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "local_temp = \"temp\"\n",
    "local_temp_pred_dir = os.path.join( local_temp, \"pred_results\")\n",
    "local_temp_wk_dir = os.path.join( local_temp, \"wk\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "!rm -rf $local_temp\n",
    "!mkdir -p $local_temp_pred_dir\n",
    "!mkdir -p $local_temp_wk_dir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!aws s3 cp s3://aegovan-data/pubmed_asbtract/predictions_multi_95/pubmed19n0538.json.txt.json.prediction.json ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "import boto3\n",
    "import glob\n",
    "from multiprocessing.dummy import Pool as ThreadPool\n",
    "import argparse\n",
    "import datetime \n",
    "import os\n",
    "\n",
    "\n",
    "def uploadfile(localpath, s3path):\n",
    "        \"\"\"\n",
    "Uploads a file to s3\n",
    "        :param localpath: The local path\n",
    "        :param s3path: The s3 path in format s3://mybucket/mydir/mysample.txt\n",
    "        \"\"\"\n",
    "\n",
    "        bucket, key = get_bucketname_key(s3path)\n",
    "\n",
    "        if key.endswith(\"/\"):\n",
    "            key = \"{}{}\".format(key, os.path.basename(localpath))\n",
    "        \n",
    "        s3 = boto3.client('s3')\n",
    "        \n",
    "        s3.upload_file(localpath, bucket, key)\n",
    "\n",
    "def get_bucketname_key(uripath):\n",
    "    assert uripath.startswith(\"s3://\")\n",
    "\n",
    "    path_without_scheme = uripath[5:]\n",
    "    bucket_end_index = path_without_scheme.find(\"/\")\n",
    "\n",
    "    bucket_name = path_without_scheme\n",
    "    key = \"/\"\n",
    "    if bucket_end_index > -1:\n",
    "        bucket_name = path_without_scheme[0:bucket_end_index]\n",
    "        key = path_without_scheme[bucket_end_index + 1:]\n",
    "\n",
    "    return bucket_name, key\n",
    "\n",
    "\n",
    "def download_file(s3path, local_dir):\n",
    "    bucket, key = get_bucketname_key(s3path)\n",
    "    \n",
    "    s3 = boto3.client('s3')\n",
    "    \n",
    "    local_file = os.path.join(local_dir, s3path.split(\"/\")[-1])\n",
    "    \n",
    "\n",
    "    s3.download_file(bucket, key, local_file)\n",
    "    \n",
    "def download_object(s3path):\n",
    "    bucket, key = get_bucketname_key(s3path)\n",
    "    \n",
    "    s3 = boto3.client('s3')    \n",
    "\n",
    "    s3_response_object = s3.get_object(Bucket=bucket, Key=key)\n",
    "    object_content = s3_response_object['Body'].read()\n",
    "    \n",
    "    return len(object_content)\n",
    "\n",
    "\n",
    "\n",
    "def list_files(s3path_prefix):\n",
    "    assert s3path_prefix.startswith(\"s3://\")\n",
    "    assert s3path_prefix.endswith(\"/\")\n",
    "    \n",
    "    bucket, key = get_bucketname_key(s3path_prefix)\n",
    "    \n",
    "   \n",
    "   \n",
    "    s3 = boto3.resource('s3')\n",
    "    \n",
    "    bucket = s3.Bucket(name=bucket)\n",
    "\n",
    "    return ( (o.bucket_name, o.key) for o in bucket.objects.filter(Prefix=key))\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "def upload_files(local_dir, s3_prefix, num_threads=20):    \n",
    "    input_tuples = ( (f,  s3_prefix) for f in glob.glob(\"{}/*\".format(local_dir)))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        pool.starmap(uploadfile, input_tuples)\n",
    "    \n",
    "\n",
    "\n",
    "def download_files(s3_prefix, local_dir, num_threads=20):    \n",
    "    input_tuples = ( (\"s3://{}/{}\".format(s3_bucket,s3_key),  local_dir) for s3_bucket, s3_key in list_files(s3_prefix))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.starmap(download_file, input_tuples)\n",
    "        \n",
    "        \n",
    "\n",
    "def download_objects(s3_prefix, num_threads=20):    \n",
    "    s3_files = ( \"s3://{}/{}\".format(s3_bucket,s3_key) for s3_bucket, s3_key in list_files(s3_prefix))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.map(download_object, s3_files)\n",
    "        \n",
    "    return sum(results)/1024\n",
    "        \n",
    "\n",
    "def get_directory_size(start_path):\n",
    "    total_size = 0\n",
    "    for dirpath, dirnames, filenames in os.walk(start_path):\n",
    "        for f in filenames:\n",
    "            fp = os.path.join(dirpath, f)\n",
    "            # skip if it is symbolic link\n",
    "            if not os.path.islink(fp):\n",
    "                total_size += os.path.getsize(fp)\n",
    "    return total_size\n",
    "\n",
    "def get_s3file_size(bucket, key):\n",
    "    s3 = boto3.client('s3')\n",
    "    response = s3.head_object(Bucket=bucket, Key=key)\n",
    "    size = response['ContentLength']\n",
    "    return size\n",
    "    \n",
    "def download_files_min_files(s3_prefix, local_dir, min_file_size=310, num_threads=20):    \n",
    "    input_tuples = ( (\"s3://{}/{}\".format(s3_bucket,s3_key),  local_dir) for s3_bucket, s3_key in list_files(s3_prefix) if get_s3file_size(s3_bucket, s3_key) > min_file_size )\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.starmap(download_file, input_tuples)\n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 3min 53s, sys: 1min 52s, total: 5min 46s\n",
      "Wall time: 4min 31s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "download_files(s3_prefix, local_temp_pred_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31\r\n"
     ]
    }
   ],
   "source": [
    "!ls -l $local_temp_dir | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "threshold_config = {\n",
    "    \"acetylation\" : 0.83,\n",
    "    \"deubiquitination\" :0.50,\n",
    "    \"methylation\" :.85,\n",
    "    \"phosphorylation\" : .98,\n",
    "    \"demethylation\" :0.0,\n",
    "    \"dephosphorylation\" :.85,\n",
    "    \"ubiquitination\":0.3\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 5min 30s, sys: 1min 17s, total: 6min 47s\n",
      "Wall time: 8min 52s\n"
     ]
    }
   ],
   "source": [
    "%%time \n",
    "\n",
    "full_df = None\n",
    "total_counts = {}\n",
    "for f in os.listdir(local_temp_pred_dir):\n",
    "    df = pd.read_json(os.path.join(local_temp_pred_dir, f), orient=\"records\", lines=True )\n",
    "    \n",
    "    count_dict = df.groupby([\"predicted\"])[\"predicted\"].count().to_dict()\n",
    "    min_dict = df.groupby([\"predicted\"])[\"predicted_confidence\"].min().to_dict()\n",
    "    \n",
    "    for k in count_dict:\n",
    "        if k not in total_counts:\n",
    "            total_counts[k] = {}\n",
    "        total_counts[k][\"count\"] = total_counts[k].get(\"count\", 0) + count_dict[k]\n",
    "        total_counts[k][\"min\"] = min(total_counts[k].get(\"min\", 1.0) , min_dict[k])\n",
    "\n",
    "\n",
    "\n",
    "    # Filter below threshold items\n",
    "    high_quality_frames = []\n",
    "    for k,t in threshold_config.items():\n",
    "        high_quality_frames.append(df.query(\"predicted == '{}' and predicted_confidence > {}\".format(k, t)))\n",
    "        \n",
    "    high_quality_df = pd.concat(high_quality_frames)\n",
    "    \n",
    "    \n",
    "    \n",
    "    if full_df is None:\n",
    "        full_df = high_quality_df\n",
    "    else:\n",
    "        full_df = pd.concat([high_quality_df, full_df])\n",
    "        \n",
    "    \n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_counts = pd.DataFrame(total_counts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>acetylation</th>\n",
       "      <th>dephosphorylation</th>\n",
       "      <th>deubiquitination</th>\n",
       "      <th>methylation</th>\n",
       "      <th>other</th>\n",
       "      <th>phosphorylation</th>\n",
       "      <th>ubiquitination</th>\n",
       "      <th>demethylation</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>2835.000000</td>\n",
       "      <td>17319.000000</td>\n",
       "      <td>1145.000000</td>\n",
       "      <td>13143.00000</td>\n",
       "      <td>9.854011e+06</td>\n",
       "      <td>404850.000000</td>\n",
       "      <td>354.000000</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>min</th>\n",
       "      <td>0.166537</td>\n",
       "      <td>0.168799</td>\n",
       "      <td>0.162342</td>\n",
       "      <td>0.17894</td>\n",
       "      <td>1.734827e-01</td>\n",
       "      <td>0.170796</td>\n",
       "      <td>0.181956</td>\n",
       "      <td>0.377643</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       acetylation  dephosphorylation  deubiquitination  methylation  \\\n",
       "count  2835.000000       17319.000000       1145.000000  13143.00000   \n",
       "min       0.166537           0.168799          0.162342      0.17894   \n",
       "\n",
       "              other  phosphorylation  ubiquitination  demethylation  \n",
       "count  9.854011e+06    404850.000000      354.000000       1.000000  \n",
       "min    1.734827e-01         0.170796        0.181956       0.377643  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_counts.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'acetylation': 16,\n",
       " 'demethylation': 1,\n",
       " 'dephosphorylation': 180,\n",
       " 'deubiquitination': 92,\n",
       " 'methylation': 1909,\n",
       " 'phosphorylation': 1252,\n",
       " 'ubiquitination': 36}"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.groupby([\"predicted\"])[\"predicted\"].count().to_dict()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>count</th>\n",
       "      <th>mean</th>\n",
       "      <th>std</th>\n",
       "      <th>min</th>\n",
       "      <th>25%</th>\n",
       "      <th>50%</th>\n",
       "      <th>75%</th>\n",
       "      <th>max</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>predicted</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>acetylation</th>\n",
       "      <td>16.0</td>\n",
       "      <td>0.853635</td>\n",
       "      <td>0.017940</td>\n",
       "      <td>0.834194</td>\n",
       "      <td>0.841396</td>\n",
       "      <td>0.846593</td>\n",
       "      <td>0.860633</td>\n",
       "      <td>0.906370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>demethylation</th>\n",
       "      <td>1.0</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dephosphorylation</th>\n",
       "      <td>180.0</td>\n",
       "      <td>0.898379</td>\n",
       "      <td>0.031899</td>\n",
       "      <td>0.850141</td>\n",
       "      <td>0.871749</td>\n",
       "      <td>0.892653</td>\n",
       "      <td>0.921433</td>\n",
       "      <td>0.962255</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deubiquitination</th>\n",
       "      <td>92.0</td>\n",
       "      <td>0.560631</td>\n",
       "      <td>0.044044</td>\n",
       "      <td>0.500080</td>\n",
       "      <td>0.522097</td>\n",
       "      <td>0.547255</td>\n",
       "      <td>0.592547</td>\n",
       "      <td>0.695426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>methylation</th>\n",
       "      <td>1909.0</td>\n",
       "      <td>0.888543</td>\n",
       "      <td>0.023966</td>\n",
       "      <td>0.850032</td>\n",
       "      <td>0.868216</td>\n",
       "      <td>0.886418</td>\n",
       "      <td>0.907715</td>\n",
       "      <td>0.946525</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phosphorylation</th>\n",
       "      <td>1252.0</td>\n",
       "      <td>0.984180</td>\n",
       "      <td>0.002718</td>\n",
       "      <td>0.980010</td>\n",
       "      <td>0.981914</td>\n",
       "      <td>0.983852</td>\n",
       "      <td>0.986243</td>\n",
       "      <td>0.991165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ubiquitination</th>\n",
       "      <td>36.0</td>\n",
       "      <td>0.315345</td>\n",
       "      <td>0.011990</td>\n",
       "      <td>0.300192</td>\n",
       "      <td>0.304990</td>\n",
       "      <td>0.313962</td>\n",
       "      <td>0.319787</td>\n",
       "      <td>0.347559</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    count      mean       std       min       25%       50%  \\\n",
       "predicted                                                                     \n",
       "acetylation          16.0  0.853635  0.017940  0.834194  0.841396  0.846593   \n",
       "demethylation         1.0  0.377643       NaN  0.377643  0.377643  0.377643   \n",
       "dephosphorylation   180.0  0.898379  0.031899  0.850141  0.871749  0.892653   \n",
       "deubiquitination     92.0  0.560631  0.044044  0.500080  0.522097  0.547255   \n",
       "methylation        1909.0  0.888543  0.023966  0.850032  0.868216  0.886418   \n",
       "phosphorylation    1252.0  0.984180  0.002718  0.980010  0.981914  0.983852   \n",
       "ubiquitination       36.0  0.315345  0.011990  0.300192  0.304990  0.313962   \n",
       "\n",
       "                        75%       max  \n",
       "predicted                              \n",
       "acetylation        0.860633  0.906370  \n",
       "demethylation      0.377643  0.377643  \n",
       "dephosphorylation  0.921433  0.962255  \n",
       "deubiquitination   0.592547  0.695426  \n",
       "methylation        0.907715  0.946525  \n",
       "phosphorylation    0.986243  0.991165  \n",
       "ubiquitination     0.319787  0.347559  "
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.groupby([\"predicted\"])[\"predicted_confidence\"].describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3486, 16)"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "download_file(s3_data, local_temp_wk_dir)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "data_file = os.path.join(local_temp_wk_dir, s3_data.split(\"/\")[-1])\n",
    "data_training_full_df = pd.read_json(data_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3381, 7)"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_training_full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3486, 16)"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>interactionId</th>\n",
       "      <th>interactionType</th>\n",
       "      <th>isNegative</th>\n",
       "      <th>participants</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>pubmedTitle</th>\n",
       "      <th>pubmedabstract</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1585513</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>false</td>\n",
       "      <td>[{'uniprotid': 'Q10728', 'alias': [['mypt1_rat...</td>\n",
       "      <td>17126281</td>\n",
       "      <td>None</td>\n",
       "      <td>Zipper-interacting protein kinase (ZIP kinase)...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1585516</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>false</td>\n",
       "      <td>[{'uniprotid': 'O43293-1', 'alias': [['o43293-...</td>\n",
       "      <td>17126281</td>\n",
       "      <td>None</td>\n",
       "      <td>Zipper-interacting protein kinase (ZIP kinase)...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   interactionId  interactionType isNegative  \\\n",
       "0        1585513  phosphorylation      false   \n",
       "1        1585516  phosphorylation      false   \n",
       "\n",
       "                                        participants  pubmedId pubmedTitle  \\\n",
       "0  [{'uniprotid': 'Q10728', 'alias': [['mypt1_rat...  17126281        None   \n",
       "1  [{'uniprotid': 'O43293-1', 'alias': [['o43293-...  17126281        None   \n",
       "\n",
       "                                      pubmedabstract  \n",
       "0  Zipper-interacting protein kinase (ZIP kinase)...  \n",
       "1  Zipper-interacting protein kinase (ZIP kinase)...  "
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_training_full_df.head(n=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>acetylation</th>\n",
       "      <th>confidence_scores</th>\n",
       "      <th>demethylation</th>\n",
       "      <th>dephosphorylation</th>\n",
       "      <th>deubiquitination</th>\n",
       "      <th>methylation</th>\n",
       "      <th>normalised_abstract</th>\n",
       "      <th>other</th>\n",
       "      <th>participant1Id</th>\n",
       "      <th>participant2Id</th>\n",
       "      <th>phosphorylation</th>\n",
       "      <th>predicted</th>\n",
       "      <th>predicted_confidence</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>ubiquitination</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>Fragile X syndrome is one of the most frequent...</td>\n",
       "      <td>0.804310</td>\n",
       "      <td>{'acetylation': -0.21777061220000002, 'demethy...</td>\n",
       "      <td>0.621809</td>\n",
       "      <td>0.245314</td>\n",
       "      <td>0.514101</td>\n",
       "      <td>33.977565</td>\n",
       "      <td>Fragile X syndrome is one of the most frequent...</td>\n",
       "      <td>1.754072</td>\n",
       "      <td>P00995</td>\n",
       "      <td>Q06787</td>\n",
       "      <td>0.446770</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.874934</td>\n",
       "      <td>8372665</td>\n",
       "      <td>0.470480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2783</th>\n",
       "      <td>The experiments reported here document that th...</td>\n",
       "      <td>0.286923</td>\n",
       "      <td>{'acetylation': -1.2485398501, 'demethylation'...</td>\n",
       "      <td>0.181673</td>\n",
       "      <td>0.359273</td>\n",
       "      <td>0.251972</td>\n",
       "      <td>0.231727</td>\n",
       "      <td>The experiments reported here document that th...</td>\n",
       "      <td>4.234165</td>\n",
       "      <td>P06400</td>\n",
       "      <td>P15172</td>\n",
       "      <td>308.552267</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.981568</td>\n",
       "      <td>8381715</td>\n",
       "      <td>0.248378</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               abstract  acetylation  \\\n",
       "395   Fragile X syndrome is one of the most frequent...     0.804310   \n",
       "2783  The experiments reported here document that th...     0.286923   \n",
       "\n",
       "                                      confidence_scores  demethylation  \\\n",
       "395   {'acetylation': -0.21777061220000002, 'demethy...       0.621809   \n",
       "2783  {'acetylation': -1.2485398501, 'demethylation'...       0.181673   \n",
       "\n",
       "      dephosphorylation  deubiquitination  methylation  \\\n",
       "395            0.245314          0.514101    33.977565   \n",
       "2783           0.359273          0.251972     0.231727   \n",
       "\n",
       "                                    normalised_abstract     other  \\\n",
       "395   Fragile X syndrome is one of the most frequent...  1.754072   \n",
       "2783  The experiments reported here document that th...  4.234165   \n",
       "\n",
       "     participant1Id participant2Id  phosphorylation        predicted  \\\n",
       "395          P00995         Q06787         0.446770      methylation   \n",
       "2783         P06400         P15172       308.552267  phosphorylation   \n",
       "\n",
       "      predicted_confidence  pubmedId  ubiquitination  \n",
       "395               0.874934   8372665        0.470480  \n",
       "2783              0.981568   8381715        0.248378  "
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.head(n=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3253, 16)"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df[~full_df.pubmedId.isin(data_training_full_df.pubmedId)].shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_df[\"PubmedInTrainingData\"] = full_df.pubmedId.isin(data_training_full_df.pubmedId)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>count</th>\n",
       "      <th>mean</th>\n",
       "      <th>std</th>\n",
       "      <th>min</th>\n",
       "      <th>25%</th>\n",
       "      <th>50%</th>\n",
       "      <th>75%</th>\n",
       "      <th>max</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>predicted</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>acetylation</th>\n",
       "      <td>14.0</td>\n",
       "      <td>0.848446</td>\n",
       "      <td>0.009895</td>\n",
       "      <td>0.834194</td>\n",
       "      <td>0.841258</td>\n",
       "      <td>0.845236</td>\n",
       "      <td>0.854201</td>\n",
       "      <td>0.869485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dephosphorylation</th>\n",
       "      <td>133.0</td>\n",
       "      <td>0.888944</td>\n",
       "      <td>0.023922</td>\n",
       "      <td>0.850141</td>\n",
       "      <td>0.870096</td>\n",
       "      <td>0.887317</td>\n",
       "      <td>0.902896</td>\n",
       "      <td>0.943967</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deubiquitination</th>\n",
       "      <td>88.0</td>\n",
       "      <td>0.557122</td>\n",
       "      <td>0.040684</td>\n",
       "      <td>0.500080</td>\n",
       "      <td>0.520021</td>\n",
       "      <td>0.545226</td>\n",
       "      <td>0.589662</td>\n",
       "      <td>0.659376</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>methylation</th>\n",
       "      <td>1888.0</td>\n",
       "      <td>0.888159</td>\n",
       "      <td>0.023650</td>\n",
       "      <td>0.850032</td>\n",
       "      <td>0.868155</td>\n",
       "      <td>0.885885</td>\n",
       "      <td>0.907304</td>\n",
       "      <td>0.944177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phosphorylation</th>\n",
       "      <td>1099.0</td>\n",
       "      <td>0.983993</td>\n",
       "      <td>0.002639</td>\n",
       "      <td>0.980010</td>\n",
       "      <td>0.981810</td>\n",
       "      <td>0.983620</td>\n",
       "      <td>0.986003</td>\n",
       "      <td>0.991165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ubiquitination</th>\n",
       "      <td>31.0</td>\n",
       "      <td>0.314249</td>\n",
       "      <td>0.011414</td>\n",
       "      <td>0.300192</td>\n",
       "      <td>0.304916</td>\n",
       "      <td>0.313602</td>\n",
       "      <td>0.317595</td>\n",
       "      <td>0.347559</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    count      mean       std       min       25%       50%  \\\n",
       "predicted                                                                     \n",
       "acetylation          14.0  0.848446  0.009895  0.834194  0.841258  0.845236   \n",
       "dephosphorylation   133.0  0.888944  0.023922  0.850141  0.870096  0.887317   \n",
       "deubiquitination     88.0  0.557122  0.040684  0.500080  0.520021  0.545226   \n",
       "methylation        1888.0  0.888159  0.023650  0.850032  0.868155  0.885885   \n",
       "phosphorylation    1099.0  0.983993  0.002639  0.980010  0.981810  0.983620   \n",
       "ubiquitination       31.0  0.314249  0.011414  0.300192  0.304916  0.313602   \n",
       "\n",
       "                        75%       max  \n",
       "predicted                              \n",
       "acetylation        0.854201  0.869485  \n",
       "dephosphorylation  0.902896  0.943967  \n",
       "deubiquitination   0.589662  0.659376  \n",
       "methylation        0.907304  0.944177  \n",
       "phosphorylation    0.986003  0.991165  \n",
       "ubiquitination     0.317595  0.347559  "
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.query(\"not PubmedInTrainingData\").groupby([\"predicted\"])[\"predicted_confidence\"].describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "pd.set_option('display.max_columns', None)\n",
    "pd.set_option('display.max_colwidth', 10000)\n",
    "pd.set_option('display.max_rows', 100)\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "plt.rcParams.update({'font.size': 12})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_df.to_csv(\"predictions.csv\", index=False, header=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>normalised_abstract</th>\n",
       "      <th>participant1Id</th>\n",
       "      <th>participant2Id</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>predicted</th>\n",
       "      <th>predicted_confidence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>7511</th>\n",
       "      <td>Gene activity is regulated by transcriptional and epigenetic mechanisms. A paper in 2005 by Zhang et al.(1) showed that STAT3 binds to the DNA methyl transferase, DNMT1 and their data indicated that STAT3 may cause epigenetic gene silencing by targeting DNMT1 to the PTPN6 promoter. Now, a paper by Lee et al.(2) has fleshed out the mechanism. They provide evidence that acetylation of STAT3 regulates the binding of DNMT1, CpG DNA methylation and regulation of several genes, including that encoding the estrogen receptor  (ESR1) in breast cancer cells.</td>\n",
       "      <td>Gene activity is regulated by transcriptional and epigenetic mechanisms. A paper in 2005 by Zhang et al.(1) showed that P40763 binds to the DNA methyl transferase, P26358 and their data indicated that P40763 may cause epigenetic gene silencing by targeting P26358 to the P29350 promoter. Now, a paper by Lee et al.(2) has fleshed out the mechanism. They provide evidence that acetylation of P40763 regulates the binding of P26358, CpG DNA methylation and regulation of several genes, including that encoding the P03372  (P03372) in breast cancer cells.</td>\n",
       "      <td>P26358</td>\n",
       "      <td>P40763</td>\n",
       "      <td>24058781</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.870079</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13650</th>\n",
       "      <td>The cells in the preimplantation mammalian embryo undergo several rounds of fast cell division. Whether the known DNA repair pathways are active during these early stages of development where cell division is of primary importance, has not been fully established. Because of the important role of phosphorylated H2A.X (gammaH2A.X) in the DNA damage response as well as its putative role in assembly of embryonic chromatin, we analysed its distribution in the preimplantation mouse embryo. We found that H2A.X is highly phosphorylated throughout preimplantation development in the absence of any induced DNA damage. Moreover, gammaH2A.X levels vary significantly throughout the cell cycle. Interestingly, after the 4-cell stage, we detected high levels of H2A.X phosphorylation in mitosis, where telomeres appeared focally enriched with gammaH2A.X. In contrast, 53BP1, which is known to be recruited to DNA damage sites, is undetectable at mitotic chromosomes at these stages and its localisation changes upon blastocyst formation from mainly nuclear to cytoplasmic. We also show that 53BP1 and gammaH2A.X rarely colocalise, suggesting that the high levels of phosphorylation of H2A.X in the embryo might not be directly linked to the DNA damage response in the embryo. Our data suggest that phosphorylation of H2A.X is an important event in the fast dividing cells of the early embryo in the absence of any induced DNA damage. We discuss the possible consequences of these findings on the genome-wide chromatin remodelling that ocurs in the preimplantation mammalian embryo.</td>\n",
       "      <td>The cells in the preimplantation mammalian embryo undergo several rounds of fast cell division. Whether the known DNA repair pathways are active during these early stages of development where cell division is of primary importance, has not been fully established. Because of the important role of phosphorylated P16104 (gammaH2A.X) in the DNA damage response as well as its putative role in assembly of embryonic chromatin, we analysed its distribution in the preimplantation mouse embryo. We found that P16104 is highly phosphorylated throughout preimplantation development in the absence of any induced DNA damage. Moreover, gammaH2A.X levels vary significantly throughout the cell cycle. Interestingly, after the 4-cell stage, we detected high levels of P16104 phosphorylation in mitosis, where telomeres appeared focally enriched with gammaH2A.X. In contrast, Q12888, which is known to be recruited to DNA damage sites, is undetectable at mitotic chromosomes at these stages and its localisation changes upon blastocyst formation from mainly nuclear to cytoplasmic. We also show that Q12888 and gammaH2A.X rarely colocalise, suggesting that the high levels of phosphorylation of P16104 in the embryo might not be directly linked to the DNA damage response in the embryo. Our data suggest that phosphorylation of P16104 is an important event in the fast dividing cells of the early embryo in the absence of any induced DNA damage. We discuss the possible consequences of these findings on the genome-wide chromatin remodelling that ocurs in the preimplantation mammalian embryo.</td>\n",
       "      <td>P16104</td>\n",
       "      <td>Q12888</td>\n",
       "      <td>19598117</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.981332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6593</th>\n",
       "      <td>The non-genomic membrane bound oestrogen receptor (mER) regulates intracellular signals through receptor-ligand interactions. The mER, along with G-protein coupled oestrogen receptor GPR 30 (GPER), induces diverse cell signalling pathways in murine lymphocytes. The mER isoform ER-alpha46 has recently been demonstrated in human B and T lymphocytes as an analogue receptor for chemokine CCL18, the signalling events of which are not clearly understood. Ligand-induced mER and GPER signalling events are shared with BCR, CD19 mediated intracellular signalling through phospholipase C, PIP2/IP3/PI3 mediated activation of Akt, MAP kinase, and mTOR. Oestrogen has the ability to induce CD40-mediated activation of B cells. The complete signalling pathways of mER, GPR30 and their interaction with other signals are targeted areas for novel drug development in B cells during infection, autoimmunity and cancer. Therefore, an in depth investigation is critical for determining shared signal outputs during B cell activation. Here, we focus on the mode of action of membrane bound ER in B cells as therapeutic checkpoints.</td>\n",
       "      <td>The non-genomic Q99527 (Q99527) regulates intracellular signals through receptor-ligand interactions. The Q99527, along with Q99527 (Q99527), induces diverse cell signalling pathways in murine lymphocytes. The Q99527 isoform P0337246 has recently been demonstrated in human B and T lymphocytes as an analogue receptor for chemokine P55774, the signalling events of which are not clearly understood. Ligand-induced Q99527 and Q99527 signalling events are shared with BCR, P15391 mediated intracellular signalling through phospholipase C, PIP2/P19957 mediated activation of Akt, MAP kinase, and P42345. Oestrogen has the ability to induce P25942-mediated activation of B cells. The complete signalling pathways of Q99527, Q99527 and their interaction with other signals are targeted areas for novel drug development in B cells during infection, autoimmunity and cancer. Therefore, an in depth investigation is critical for determining shared signal outputs during B cell activation. Here, we focus on the mode of action of membrane bound ER in B cells as therapeutic checkpoints.</td>\n",
       "      <td>P03372</td>\n",
       "      <td>Q99527</td>\n",
       "      <td>27189345</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.982047</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>895</th>\n",
       "      <td>The retinoblastoma binding protein KDM5A removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of KDM5A contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, KDM5A contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in KDM5A have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the PHD1 preferentially recognizes unmethylated H3K4 histone tail, product of KDM5A-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the PHD1 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in KDM5A--suggests a model for the spread of demethylation on chromatin.</td>\n",
       "      <td>The retinoblastoma binding protein P29375 removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of P29375 contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, P29375 contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in P29375 have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the Q96KS0 preferentially recognizes unmethylated H3K4 histone tail, product of P29375-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the Q96KS0 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in P29375--suggests a model for the spread of demethylation on chromatin.</td>\n",
       "      <td>P29375</td>\n",
       "      <td>Q96KS0</td>\n",
       "      <td>25686748</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.866325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15505</th>\n",
       "      <td>Aberrations in DNA methylation patterns promote changes in gene expression patterns and are invariably associated with neoplasia. DNA methylation is carried out and maintained by several DNA methyltransferases (DNMTs) among which DNMT1 functions as a maintenance methylase while DNMT3a and 3b serve as de novo enzymes. Although DNMT3b has been shown to preferentially target the methylation of DNA sequences residing in pericentric heterochromatin whether it is involved in gene specific methylation remains an open question. To address this issue, we have silenced the expression of DNMT3b in the prostate-derived PC3 cells through RNA interference and subsequently studied the accompanied cellular changes as well as the expression profiles of selected genes.</td>\n",
       "      <td>Aberrations in DNA methylation patterns promote changes in gene expression patterns and are invariably associated with neoplasia. DNA methylation is carried out and maintained by several DNA methyltransferases (DNMTs) among which P26358 functions as a maintenance methylase while 1788;1789 serve as de novo enzymes. Although Q9UBC3 has been shown to preferentially target the methylation of DNA sequences residing in pericentric heterochromatin whether it is involved in gene specific methylation remains an open question. To address this issue, we have silenced the expression of Q9UBC3 in the prostate-derived PC3 cells through RNA interference and subsequently studied the accompanied cellular changes as well as the expression profiles of selected genes.</td>\n",
       "      <td>P26358</td>\n",
       "      <td>Q9UBC3</td>\n",
       "      <td>18798999</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.939795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12473</th>\n",
       "      <td>Protein kinase C delta (PKC) is a serine (Ser)/threonine kinase, which regulates numerous cellular processes, including proliferation, differentiation, migration and apoptosis. In the current study, Chinese hamster ovary cells were transfected with either a constitutively activated PKC or a dominant negative PKC, phosphoprotein enrichment, two-dimensional difference gel electrophoresis and mass spectrometry was combined to globally identified candidates of PKC cascade. We found that Bcl-2 associated athanogene 3 (BAG3) was one of the targets of PKC cascade, and BAG3 interacted with PKC in vivo. In addition, we clarified that BAG3 was phosphorylate at Ser187 site in a PKC-dependent manner in vivo. BAG3 has been implicated in multiple cellular functions, including proliferation, differentiation, apoptosis, migration, invasion, macroautophagy and so on. We generated wild-type (WT)-, Ser187Ala (S187A)- or Ser187Asp (S187D)-BAG3 stably expressing FRO cells, and noticed that phosphorylation state of BAG3 influenced FRO morphology. Finally, for the first time, we showed that BAG3 was implicated in epithelial-mesenchymal transition (EMT) procedure, and phosphorylation state at Ser187 site had a critical role in EMT regulation by BAG3. Collectively, the current study indicates that BAG3 is a novel substrate of PKC, and PKC-mediated phosphorylation of BAG3 is implicated in EMT and invasiveness of thyroid cancer cells.</td>\n",
       "      <td>Q05655 (Q05655) is a serine (Ser)/threonine kinase, which regulates numerous cellular processes, including proliferation, differentiation, migration and apoptosis. In the current study, Chinese hamster ovary cells were transfected with either a constitutively activated Q05655 or a dominant negative Q05655, phosphoprotein enrichment, two-dimensional difference gel electrophoresis and mass spectrometry was combined to globally identified candidates of Q05655 cascade. We found that 9531;596 (O95817) was one of the targets of Q05655 cascade, and O95817 interacted with Q05655 in vivo. In addition, we clarified that O95817 was phosphorylate at Ser187 site in a Q05655-dependent manner in vivo. O95817 has been implicated in multiple cellular functions, including proliferation, differentiation, apoptosis, migration, invasion, macroautophagy and so on. We generated wild-type (WT)-, Ser187Ala (S187A)- or Ser187Asp (S187D)-O95817 stably expressing FRO cells, and noticed that phosphorylation state of O95817 influenced FRO morphology. Finally, for the first time, we showed that O95817 was implicated in epithelial-mesenchymal transition (EMT) procedure, and phosphorylation state at Ser187 site had a critical role in EMT regulation by O95817. Collectively, the current study indicates that O95817 is a novel substrate of Q05655, and Q05655-mediated phosphorylation of O95817 is implicated in EMT and invasiveness of thyroid cancer cells.</td>\n",
       "      <td>O95817</td>\n",
       "      <td>Q05655</td>\n",
       "      <td>23108398</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.983797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12965</th>\n",
       "      <td>The Dnmt3a DNA methyltransferase is responsible for establishing DNA methylation patterns during mammalian development. We show here that the mouse Dnmt3a DNA methyltransferase is able to transfer the methyl group from S-adenosyl-l-methionine (AdoMet) to a cysteine residue in its catalytic center. This reaction is irreversible and relatively slow. The yield of auto-methylation is increased by addition of Dnmt3L, which functions as a stimulator of Dnmt3a and enhances its AdoMet binding. Auto-methylation was observed in binary Dnmt3a AdoMet complexes. In the presence of CpG containing dsDNA, which is the natural substrate for Dnmt3a, the transfer of the methyl group from AdoMet to the flipped target base was preferred and auto-methylation was not detected. Therefore, this reaction might constitute a regulatory mechanism which could inactivate unused DNA methyltransferases in the cell, or it could simply be an aberrant side reaction caused by the high methyl group transfer potential of AdoMet.</td>\n",
       "      <td>The Q9Y6K1 DNA methyltransferase is responsible for establishing DNA methylation patterns during mammalian development. We show here that the mouse Q9Y6K1 DNA methyltransferase is able to transfer the methyl group from S-adenosyl-l-methionine (AdoMet) to a cysteine residue in its catalytic center. This reaction is irreversible and relatively slow. The yield of auto-methylation is increased by addition of Q9UJW3, which functions as a stimulator of Q9Y6K1 and enhances its AdoMet binding. Auto-methylation was observed in binary Q9Y6K1 AdoMet complexes. In the presence of CpG containing dsDNA, which is the natural substrate for Q9Y6K1, the transfer of the methyl group from AdoMet to the flipped target base was preferred and auto-methylation was not detected. Therefore, this reaction might constitute a regulatory mechanism which could inactivate unused DNA methyltransferases in the cell, or it could simply be an aberrant side reaction caused by the high methyl group transfer potential of AdoMet.</td>\n",
       "      <td>Q9UJW3</td>\n",
       "      <td>Q9Y6K1</td>\n",
       "      <td>21481189</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.933695</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10041</th>\n",
       "      <td>Proteins are frequently modified by post-translational methylation of lysine residues, catalyzed by S-adenosylmethionine-dependent lysine methyltransferases (KMTs). Lysine methylation of histone proteins has been extensively studied, but it has recently become evident that methylation of non-histone proteins is also abundant and important. The human methyltransferase METTL20 belongs to a group of 10 established and putative human KMTs. We here found METTL20 to be associated with mitochondria and determined that recombinant METTL20 methylated a single protein in extracts from human cells. Using an methyltransferase activity-based purification scheme, we identified the -subunit of the mitochondrially localized electron transfer flavoprotein (ETF) as the substrate of METTL20. Furthermore, METTL20 was found to specifically methylate two adjacent lysine residues, Lys(200) and Lys(203), in ETF both in vitro and in cells. Interestingly, the residues methylated by METTL20 partially overlap with the so-called \"recognition loop\" in ETF, which has been shown to mediate its interaction with various dehydrogenases. Accordingly, we found that METTL20-mediated methylation of ETF in vitro reduced its ability to receive electrons from the medium chain acyl-CoA dehydrogenase and the glutaryl-CoA dehydrogenase. In conclusion, the present study establishes METTL20 as the first human KMT localized to mitochondria and suggests that it may regulate cellular metabolism through modulating the interaction between its substrate ETF and dehydrogenases. Based on the previous naming of similar enzymes, we suggest the renaming of human METTL20 to ETF-KMT.</td>\n",
       "      <td>Proteins are frequently modified by post-translational methylation of lysine residues, catalyzed by S-adenosylmethionine-dependent lysine methyltransferases (KMTs). Lysine methylation of histone proteins has been extensively studied, but it has recently become evident that methylation of non-histone proteins is also abundant and important. The human methyltransferase Q8IXQ9 belongs to a group of 10 established and putative human KMTs. We here found Q8IXQ9 to be associated with mitochondria and determined that recombinant Q8IXQ9 methylated a single protein in extracts from human cells. Using an methyltransferase activity-based purification scheme, we identified the -subunit of the mitochondrially localized electron transfer flavoprotein (ETF) as the substrate of Q8IXQ9. Furthermore, Q8IXQ9 was found to specifically methylate two adjacent lysine residues, Lys(200) and Lys(203), in ETF both in vitro and in cells. Interestingly, the residues methylated by Q8IXQ9 partially overlap with the so-called \"recognition loop\" in ETF, which has been shown to mediate its interaction with various dehydrogenases. Accordingly, we found that Q8IXQ9-mediated methylation of ETF in vitro reduced its ability to receive electrons from the P11310 and the Q92947. In conclusion, the present study establishes Q8IXQ9 as the first human KMT localized to mitochondria and suggests that it may regulate cellular metabolism through modulating the interaction between its substrate ETF and dehydrogenases. Based on the previous naming of similar enzymes, we suggest the renaming of human Q8IXQ9 to ETF-KMT.</td>\n",
       "      <td>Q8IXQ9</td>\n",
       "      <td>Q92947</td>\n",
       "      <td>25416781</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.917449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12186</th>\n",
       "      <td>Dimethylation of histone H3 Arg2 (H3R2me2) maintains transcriptional silencing by inhibiting Set1 mediated trimethylation of H3K4. Here we demonstrate that Arg2 is also monomethylated (H3R2me1) in yeast but that its functional characteristics are distinct from H3R2me2: (i) H3R2me1 does not inhibit histone H3 Lys4 (H3K4) methylation; (ii) it is present throughout the coding region of genes; and (iii) it correlates with active transcription. Collectively, these results indicate that different H3R2 methylation states have defined roles in gene expression.</td>\n",
       "      <td>Dimethylation of histone H3 P78540 (H3R2me2) maintains transcriptional silencing by inhibiting O15047 mediated trimethylation of H3K4. Here we demonstrate that P78540 is also monomethylated (H3R2me1) in yeast but that its functional characteristics are distinct from H3R2me2: (i) H3R2me1 does not inhibit histone H3 Lys4 (H3K4) methylation; (ii) it is present throughout the coding region of genes; and (iii) it correlates with active transcription. Collectively, these results indicate that different H3R2 methylation states have defined roles in gene expression.</td>\n",
       "      <td>O15047</td>\n",
       "      <td>P78540</td>\n",
       "      <td>19270702</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.874299</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5037</th>\n",
       "      <td>RUNX1 (previously termed AML1) is a frequent target of human leukaemia-associated gene aberrations, and it encodes the DNA-binding subunit of the Core-Binding Factor transcription factor complex. RUNX1 expression is essential for the initiation of definitive haematopoiesis, for steady-state thrombopoiesis, and for normal lymphocytes development. Recent studies revealed that protein arginine methyltransferase 1 (PRMT1), which accounts for the majority of the type I PRMT activity in cells, methylates two arginine residues in RUNX1 (R206 and R210), and these modifications inhibit corepressor-binding to RUNX1 thereby enhancing its transcriptional activity. In order to elucidate the biological significance of these methylations, we established novel knock-in mouse lines with non-methylable, double arginine-to-lysine (RTAMR-to-KTAMK) mutations in RUNX1. Homozygous Runx1(KTAMK) (/) (KTAMK) mice are born alive and appear normal during adulthood. However, Runx1(KTAMK) (/) (KTAMK) mice showed a reduction in CD3(+) T lymphoid cells and a decrease in CD4(+) T cells in peripheral lymphoid organs, in comparison to their wild-type littermates, leading to a reduction in the CD4(+) to CD8(+) T-cell ratio. These findings suggest that arginine-methylation of RUNX1 in the RTAMR-motif is dispensable for the development of definitive haematopoiesis and for steady-state platelet production, however this modification affects the role of RUNX1 in the maintenance of the peripheral CD4(+) T-cell population.</td>\n",
       "      <td>Q01196 (previously termed Q01196) is a frequent target of human leukaemia-associated gene aberrations, and it encodes the DNA-binding subunit of the Core-Binding Factor transcription factor complex. Q01196 expression is essential for the initiation of definitive haematopoiesis, for steady-state thrombopoiesis, and for normal lymphocytes development. Recent studies revealed that Q99873 (Q99873), which accounts for the majority of the type I PRMT activity in cells, methylates two arginine residues in Q01196 (R206 and R210), and these modifications inhibit corepressor-binding to Q01196 thereby enhancing its transcriptional activity. In order to elucidate the biological significance of these methylations, we established novel knock-in mouse lines with non-methylable, double arginine-to-lysine (RTAMR-to-KTAMK) mutations in Q01196. Homozygous Q01196(KTAMK) (/) (KTAMK) mice are born alive and appear normal during adulthood. However, Q01196(KTAMK) (/) (KTAMK) mice showed a reduction in CD3(+) T lymphoid cells and a decrease in P01730(+) T cells in peripheral lymphoid organs, in comparison to their wild-type littermates, leading to a reduction in the P01730(+) to P01732(+) T-cell ratio. These findings suggest that arginine-methylation of Q01196 in the RTAMR-motif is dispensable for the development of definitive haematopoiesis and for steady-state platelet production, however this modification affects the role of Q01196 in the maintenance of the peripheral P01730(+) T-cell population.</td>\n",
       "      <td>Q01196</td>\n",
       "      <td>Q99873</td>\n",
       "      <td>26010396</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.929019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1346</th>\n",
       "      <td>The purified insulin receptor kinase catalyzed the phosphorylation of native tubulin and microtubule-associated proteins (MAPs; MAP2, tau) on tyrosine residues. Insulin (10(-7) M) stimulated the reaction by 4-10-fold by increasing Vmax with little change in Km. alpha-Tubulin was preferred as a substrate for the kinase compared to beta-tubulin. MAP2 was found to be the best substrate among the cytoskeletal proteins tested; in the presence of insulin, the Vmax for MAP2 was 6.3 nmol/min/mg, its Km was 5.1 microM, and 1.7 mol of phosphate were incorporated per mol of MAP2. Under the same conditions used for this phosphorylation of tubulin and MAPs, actin and tropomyosin were very poorly phosphorylated. These data, coupled with previous evidence for potential functional relationships between insulin action and microtubules, raise the possibility that microtubule proteins may be cellular targets for the insulin receptor kinase.</td>\n",
       "      <td>The purified insulin receptor kinase catalyzed the phosphorylation of native tubulin and microtubule-associated proteins (MAPs; P11137, tau) on tyrosine residues. Insulin (10(-7) M) stimulated the reaction by 4-10-fold by increasing Vmax with little change in Km. P68363 was preferred as a substrate for the kinase compared to beta-tubulin. P11137 was found to be the best substrate among the cytoskeletal proteins tested; in the presence of insulin, the Vmax for P11137 was 6.3 nmol/min/mg, its Km was 5.1 microM, and 1.7 mol of phosphate were incorporated per mol of P11137. Under the same conditions used for this phosphorylation of tubulin and MAPs, actin and tropomyosin were very poorly phosphorylated. These data, coupled with previous evidence for potential functional relationships between insulin action and microtubules, raise the possibility that microtubule proteins may be cellular targets for the insulin receptor kinase.</td>\n",
       "      <td>P11137</td>\n",
       "      <td>P68363</td>\n",
       "      <td>3920212</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.984268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11163</th>\n",
       "      <td>Neuroblastoma is a very heterogeneous tumor of childhood. The clinical spectra range from very aggressive metastatic disease to spontaneous regression, even without therapy. Aberrant DNA methylation pattern is a common feature of most cancers. For neuroblastoma, it has been demonstrated both for single genes as well as genome-wide, where a so-called methylator phenotype has been described. Here, we present a study using Illumina 450K methylation arrays on 60 neuroblastoma tumors. We show that aggressive tumors, characterized by International Neuroblastoma Risk Group (INRG) as stage M, are hypermethylated compared to low-grade tumors. On the contrary, INRG stage L tumors display more non-CpG methylation. The genes with the highest number of hypermethylated CpG sites in INRG M tumors are TERT, PCDHGA4, DLX5, and DLX6-AS1. Gene ontology analysis showed a representation of neuronal tumor relevant gene functions among the differentially methylated genes. For validation, we used a set of independent tumors previously analyzed with the Illumina 27K methylation arrays, which confirmed the differentially methylated sites. Top candidate genes with aberrant methylation were analyzed for altered gene expression through the R2 platform ( http://r2.amc.nl), and for correlations between methylation and gene expression in a public dataset. Altered expression in nonsurvivors was found for the genes B3GALT4 and KIAA1949, CLIC5, DLX6-AS, TERT, and PIRT, and strongest correlations were found for TRIM36, KIAA0513, and PIRT. Our data indicate that methylation profiling can be used for patient stratification and informs on epigenetically deregulated genes with the potential of increasing our knowledge about the underlying mechanisms of tumor development.</td>\n",
       "      <td>Neuroblastoma is a very heterogeneous tumor of childhood. The clinical spectra range from very aggressive metastatic disease to spontaneous regression, even without therapy. Aberrant DNA methylation pattern is a common feature of most cancers. For neuroblastoma, it has been demonstrated both for single genes as well as genome-wide, where a so-called methylator phenotype has been described. Here, we present a study using Illumina 450K methylation arrays on 60 neuroblastoma tumors. We show that aggressive tumors, characterized by International Neuroblastoma Risk Group (INRG) as stage M, are hypermethylated compared to low-grade tumors. On the contrary, INRG stage L tumors display more non-CpG methylation. The genes with the highest number of hypermethylated CpG sites in INRG M tumors are O14746, Q9Y5G9, P56178, and 285987. Gene ontology analysis showed a representation of neuronal tumor relevant gene functions among the differentially methylated genes. For validation, we used a set of independent tumors previously analyzed with the Illumina 27K methylation arrays, which confirmed the differentially methylated sites. Top candidate genes with aberrant methylation were analyzed for altered gene expression through the R2 platform ( http://r2.amc.nl), and for correlations between methylation and gene expression in a public dataset. Altered expression in nonsurvivors was found for the genes481737 and Q6NYC8, Q9NZA1, 285987, O14746, and P0C851, and strongest correlations were found for Q9NQ86, O60268, and P0C851. Our data indicate that methylation profiling can be used for patient stratification and informs on epigenetically deregulated genes with the potential of increasing our knowledge about the underlying mechanisms of tumor development.</td>\n",
       "      <td>O96024</td>\n",
       "      <td>Q6NYC8</td>\n",
       "      <td>26786290</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.880582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7659</th>\n",
       "      <td>The rising incidence of antimicrobial resistance (AMR) makes it imperative to understand the underlying mechanisms. Mycobacterium tuberculosis (Mtb) is the single leading cause of death from a bacterial pathogen and estimated to be the leading cause of death from AMR. A pyrido-benzimidazole, 14, was reported to have potent bactericidal activity against Mtb. Here, we isolated multiple Mtb clones resistant to 14. Each had mutations in the putative DNA-binding and dimerization domains of rv2887, a gene encoding a transcriptional repressor of the MarR family. The mutations in Rv2887 led to markedly increased expression of rv0560c. We characterized Rv0560c as an S-adenosyl-L-methionine-dependent methyltransferase that N-methylates 14, abolishing its mycobactericidal activity. An Mtb strain lacking rv0560c became resistant to 14 by mutating decaprenylphosphoryl--d-ribose 2-oxidase (DprE1), an essential enzyme in arabinogalactan synthesis; 14 proved to be a nanomolar inhibitor of DprE1, and methylation of 14 by Rv0560c abrogated this activity. Thus, 14 joins a growing list of DprE1 inhibitors that are potently mycobactericidal. Bacterial methylation of an antibacterial agent, 14, catalyzed by Rv0560c of Mtb, is a previously unreported mechanism of AMR.</td>\n",
       "      <td>The rising incidence of antimicrobial resistance (AMR) makes it imperative to understand the underlying mechanisms. Mycobacterium tuberculosis (Mtb) is the single leading cause of death from a bacterial pathogen and estimated to be the leading cause of death from AMR. A pyrido-benzimidazole, 14, was reported to have potent bactericidal activity against Mtb. Here, we isolated multiple Mtb clones resistant to 14. Each had mutations in the putative DNA-binding and dimerization domains of 888563, a gene encoding a transcriptional repressor of the MarR family. The mutations in 888563 led to markedly increased expression of 887637. We characterized 887637 as an S-adenosyl-L-methionine-dependent methyltransferase that N-methylates 14, abolishing its mycobactericidal activity. An Mtb strain lacking 887637 became resistant to 14 by mutating decaprenylphosphoryl--d-ribose 2-oxidase (DprE1), an essential enzyme in arabinogalactan synthesis; 14 proved to be a nanomolar inhibitor of DprE1, and methylation of 14 by 887637 abrogated this activity. Thus, 14 joins a growing list of DprE1 inhibitors that are potently mycobactericidal. Bacterial methylation of an antibacterial agent, 14, catalyzed by 887637 of Mtb, is a previously unreported mechanism of AMR.</td>\n",
       "      <td>887637</td>\n",
       "      <td>888563</td>\n",
       "      <td>27432954</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.902199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2810</th>\n",
       "      <td>14-3-3 sigma has been a major G2/M checkpoint control gene and has demonstrated that its inactivation in various cancers occurs mostly by epigenetic hypermethylation, not by genetic change. This study investigated the methylation status and expression of the 14-3-3 sigma gene in 46 oral squamous cell carcinomas by methylation-specific polymerase chain reaction, reverse transcriptase-polymerase chain reaction, Western blotting and immunohistochemistry. Exons of the p53 gene were examined for mutations by sequencing analysis and CyclinD1 by immunohistochemistry. Methylation of the 14-3-3 sigma gene was detected in 13% (6/46) of the oral tumours, but not in corresponding adjacent non-malignant and normal gingival tissues. Intratumoural heterogeneity was found in the tumour tissues including three 14-3-3 sigma-methylated samples. Methylation of 14-3-3 sigma was detected in 3 SCC with p53 mutations and 3 with wild-type p53. Our major findings are: (a) methylation of 14-3-3 gene promoter is a rare event in oral cancer; (b) it is not always associated with 14-3-3 protein levels and there is no clear relationship between its methylation and p53 mutation; (c) loss of 14-3-3 sigma expression is associated with reduced CyclinD1 gene expression.</td>\n",
       "      <td>P31947 has been a major G2/M checkpoint control gene and has demonstrated that its inactivation in various cancers occurs mostly by epigenetic hypermethylation, not by genetic change. This study investigated the methylation status and expression of the P31947 gene in 46 oral squamous cell carcinomas by methylation-specific polymerase chain reaction, reverse transcriptase-polymerase chain reaction, Western blotting and immunohistochemistry. Exons of the P04637 gene were examined for mutations by sequencing analysis and P24385 by immunohistochemistry. Methylation of the P31947 gene was detected in 13% (6/46) of the oral tumours, but not in corresponding adjacent non-malignant and normal gingival tissues. Intratumoural heterogeneity was found in the tumour tissues including three P31947-methylated samples. Methylation of P31947 was detected in 3 SCC with P04637 mutations and 3 with wild-type P04637. Our major findings are: (a) methylation of 14-3-3 gene promoter is a rare event in oral cancer; (b) it is not always associated with 14-3-3 protein levels and there is no clear relationship between its methylation and P04637 mutation; (c) loss of P31947 expression is associated with reduced P24385 gene expression.</td>\n",
       "      <td>P04637</td>\n",
       "      <td>P31947</td>\n",
       "      <td>17786341</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.860166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9242</th>\n",
       "      <td>Epithelial-to-mesenchymal transition (EMT) is a fundamental mechanism governing the switch of cells from an epithelial to a motile mesenchymal-like state. This transdifferentiation is regulated by key transcription factors, including Slug. The stability and function of Slug can be regulated by multiple mechanisms, including ubiquitin-mediated post-translational modifications. Here, by using a genome wide siRNA screen for human deubiquitinating enzymes (DUBs), we identified USP10 as a deubiquitinase for Slug in cancer cells. USP10 interacts with Slug and mediates its degradation by the proteasome. Importantly, USP10 is concomitantly highly expressed with Slug in cancer biopsies. Genetic knockdown of USP10 leads to suppressed Slug levels with a decreased expression of the mesenchymal marker Vimentin. Further, it reduces the migratory capacity of cancer cells. Reversely, overexpression of USP10 elevates the level of both Slug and Vimentin. Our study identifies USP10 as a regulator of the EMT-transcription factor Slug and cell migration.</td>\n",
       "      <td>Epithelial-to-mesenchymal transition (EMT) is a fundamental mechanism governing the switch of cells from an epithelial to a motile mesenchymal-like state. This transdifferentiation is regulated by key transcription factors, including O43623. The stability and function of O43623 can be regulated by multiple mechanisms, including ubiquitin-mediated post-translational modifications. Here, by using a genome wide siRNA screen for human deubiquitinating enzymes (DUBs), we identified Q14694 as a deubiquitinase for O43623 in cancer cells. Q14694 interacts with O43623 and mediates its degradation by the proteasome. Importantly, Q14694 is concomitantly highly expressed with O43623 in cancer biopsies. Genetic knockdown of Q14694 leads to suppressed O43623 levels with a decreased expression of the mesenchymal marker P08670. Further, it reduces the migratory capacity of cancer cells. Reversely, overexpression of Q14694 elevates the level of both O43623 and P08670. Our study identifies Q14694 as a regulator of the EMT-transcription factor O43623 and cell migration.</td>\n",
       "      <td>O43623</td>\n",
       "      <td>Q14694</td>\n",
       "      <td>29803676</td>\n",
       "      <td>deubiquitination</td>\n",
       "      <td>0.528337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17854</th>\n",
       "      <td>Ribosome-recycling factor (RRF) plays an essential role in the fourth step of protein synthesis in prokaryotes. RRF combined with elongation factor G (EF-G) disassembles the post-termination ribosome complex and recycles the protein synthesis machine for the next round of translation. A reductive-methylation-modified RRF from Thermoanaerobacter tengcongensis (TteRRF) has been crystallized using the vapour-diffusion method. The crystal grew in a condition consisting of 0.1 M citric acid pH 3.5, 3.0 M NaCl and 50 mg ml(-1) methylated protein solution at 289 K. A complete data set was collected from a crystal to 2.80  resolution using synchrotron radiation at 100 K. The crystal belonged to space group P6122/P6522 with unit-cell parameters a = b = 103.26, c = 89.17 . The asymmetric unit was estimated to contain one molecule of TteRRF.</td>\n",
       "      <td>Q96E11 (Q96E11) plays an essential role in the fourth step of protein synthesis in prokaryotes. Q96E11 combined with Q96RP9 (Q96RP9) disassembles the post-termination ribosome complex and recycles the protein synthesis machine for the next round of translation. A reductive-methylation-modified Q96E11 from Thermoanaerobacter tengcongensis (TteRRF) has been crystallized using the vapour-diffusion method. The crystal grew in a condition consisting of 0.1 M citric acid pH 3.5, 3.0 M NaCl and 50 mg ml(-1) methylated protein solution at 289 K. A complete data set was collected from a crystal to 2.80  resolution using synchrotron radiation at 100 K. The crystal belonged to space group P6122/P6522 with unit-cell parameters a = b = 103.26, c = 89.17 . The asymmetric unit was estimated to contain one molecule of TteRRF.</td>\n",
       "      <td>Q96E11</td>\n",
       "      <td>Q96RP9</td>\n",
       "      <td>24817715</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.854795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8660</th>\n",
       "      <td>Recent evidence points to a role of chromatin in regulation of alternative pre-mRNA splicing (AS). In order to identify novel chromatin regulators of AS, we screened an RNAi library of chromatin proteins using a cell-based high-throughput in vivo assay. We identified a set of chromatin proteins that regulate AS. Using simultaneous genome-wide expression and AS analysis, we demonstrate distinct and non-overlapping functions of these chromatin modifiers on transcription and AS. Detailed mechanistic characterization of one dual function chromatin modifier, the H3K9 methyltransferase EHMT2 (G9a), identified VEGFA as a major chromatin-mediated AS target. Silencing of EHMT2, or its heterodimer partner EHMT1, affects AS by promoting exclusion of VEGFA exon 6a, but does not alter total VEGFA mRNA levels. The epigenetic regulatory mechanism of AS by EHMT2 involves an adaptor system consisting of the chromatin modulator HP1, which binds methylated H3K9 and recruits splicing regulator SRSF1. The epigenetic regulation of VEGFA is physiologically relevant since EHMT2 is transcriptionally induced in response to hypoxia and triggers concomitant changes in AS of VEGFA. These results characterize a novel epigenetic regulatory mechanism of AS and they demonstrate separate roles of epigenetic modifiers in transcription and alternative splicing.</td>\n",
       "      <td>Recent evidence points to a role of chromatin in regulation of alternative pre-mRNA splicing (AS). In order to identify novel chromatin regulators of AS, we screened an RNAi library of chromatin proteins using a cell-based high-throughput in vivo assay. We identified a set of chromatin proteins that regulate AS. Using simultaneous genome-wide expression and AS analysis, we demonstrate distinct and non-overlapping functions of these chromatin modifiers on transcription and AS. Detailed mechanistic characterization of one dual function chromatin modifier, the H3K9 methyltransferase Q96KQ7 (Q96KQ7), identified P15692 as a major chromatin-mediated AS target. Silencing of Q96KQ7, or its heterodimer partner Q9H9B1, affects AS by promoting exclusion of P15692 exon 6a, but does not alter total P15692 mRNA levels. The epigenetic regulatory mechanism of AS by Q96KQ7 involves an adaptor system consisting of the chromatin modulator P59665, which binds methylated H3K9 and recruits splicing regulator Q07955. The epigenetic regulation of P15692 is physiologically relevant since Q96KQ7 is transcriptionally induced in response to hypoxia and triggers concomitant changes in AS of P15692. These results characterize a novel epigenetic regulatory mechanism of AS and they demonstrate separate roles of epigenetic modifiers in transcription and alternative splicing.</td>\n",
       "      <td>Q96KQ7</td>\n",
       "      <td>Q9H9B1</td>\n",
       "      <td>25414343</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.875132</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8369</th>\n",
       "      <td>Chemoresistance, due to inhibition of apoptotic response, is the major reason for the failure of anticancer therapies. HIPK2 regulates p53-apoptotic function via serine-46 (Ser46) phosphorylation and activation of p53 is a key determinant in ovarian cancer cell death. In this study we determined whether HIPK2 overexpression restored apoptotic response in chemoresistant cancer cells.</td>\n",
       "      <td>Chemoresistance, due to inhibition of apoptotic response, is the major reason for the failure of anticancer therapies. Q9H2X6 regulates P04637-apoptotic function via serine-46 (Ser46) phosphorylation and activation of P04637 is a key determinant in ovarian cancer cell death. In this study we determined whether Q9H2X6 overexpression restored apoptotic response in chemoresistant cancer cells.</td>\n",
       "      <td>P04637</td>\n",
       "      <td>Q9H2X6</td>\n",
       "      <td>18395248</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.983379</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14870</th>\n",
       "      <td>Bone morphogenetic proteins (BMPs) are an integral component of the TGFbeta superfamily, responsible for regulation of cell proliferation, differentiation, migration and programmed cell death in a variety of cell types. The BMPs transduce their signals directly through the SMAD family of proteins but they also have been reported to interact with the MAPK and Erk pathways. Inactivation of the BMP pathway genes has been implicated as important in several cancers. Recent work has shown that BMP3b is epigenetically inactivated in cancer and suggests that BMP6 can be epigenetically inactivated. We investigated whether BMP6 is epigenetically inactivated in cell lines and whether BMP3b and BMP6 are epigenetically inactivated in non-small-cell lung cancer (NSCLC). We also studied the relationship between BMP methylation and k-ras mutation. Here, we demonstrate that the BMP3b and BMP6 genes are common targets of epigenetic inactivation in NSCLC, and that they are significantly more likely to be concurrently inactivated (P=0.009). Furthermore, this coinactivation of BMP3b and BMP6 is significantly associated with mutation of k-ras codon 12 in lung cancer (P=0.003); those with a k-ras mutation were six times more likely to have concurrent methylation of these BMP loci. Hence, these data suggest that concurrent inactivation of the BMP and activation of the Ras signalling pathways are important in lung carcinogenesis.</td>\n",
       "      <td>Bone morphogenetic proteins (BMPs) are an integral component of the TGFbeta superfamily, responsible for regulation of cell proliferation, differentiation, migration and programmed cell death in a variety of cell types. The BMPs transduce their signals directly through the SMAD family of proteins but they also have been reported to interact with the MAPK and P28482 pathways. Inactivation of the P13497 pathway genes has been implicated as important in several cancers. Recent work has shown that P55107 is epigenetically inactivated in cancer and suggests that P22004 can be epigenetically inactivated. We investigated whether P22004 is epigenetically inactivated in cell lines and whether P55107 and P22004 are epigenetically inactivated in non-small-cell lung cancer (NSCLC). We also studied the relationship between P13497 methylation and k-ras mutation. Here, we demonstrate that the P55107 and P22004 genes are common targets of epigenetic inactivation in NSCLC, and that they are significantly more likely to be concurrently inactivated (P=0.009). Furthermore, this coinactivation of P55107 and P22004 is significantly associated with mutation of k-ras codon 12 in lung cancer (P=0.003); those with a k-ras mutation were six times more likely to have concurrent methylation of these P13497 loci. Hence, these data suggest that concurrent inactivation of the P13497 and activation of the Ras signalling pathways are important in lung carcinogenesis.</td>\n",
       "      <td>P22004</td>\n",
       "      <td>P55107</td>\n",
       "      <td>16175182</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.864911</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4909</th>\n",
       "      <td>Cytosine methylation of mammalian DNA is essential for the proper epigenetic regulation of gene expression and maintenance of genomic integrity. To define the mechanism through which demethylated cells die, and to establish a paradigm for identifying genes regulated by DNA methylation, we have generated mice with a conditional allele for the maintenance DNA methyltransferase gene Dnmt1. Cre-mediated deletion of Dnmt1 causes demethylation of cultured fibroblasts and a uniform p53-dependent cell death. Mutational inactivation of Trp53 partially rescues the demethylated fibroblasts for up to five population doublings in culture. Oligonucleotide microarray analysis showed that up to 10% of genes are aberrantly expressed in demethylated fibroblasts. Our results demonstrate that loss of Dnmt1 causes cell-type-specific changes in gene expression that impinge on several pathways, including expression of imprinted genes, cell-cycle control, growth factor/receptor signal transduction and mobilization of retroelements.</td>\n",
       "      <td>Cytosine methylation of mammalian DNA is essential for the proper epigenetic regulation of gene expression and maintenance of genomic integrity. To define the mechanism through which demethylated cells die, and to establish a paradigm for identifying genes regulated by DNA methylation, we have generated mice with a conditional allele for the maintenance DNA methyltransferase gene P26358. Cre-mediated deletion of P26358 causes demethylation of cultured fibroblasts and a uniform P04637-dependent cell death. Mutational inactivation of P04637 partially rescues the demethylated fibroblasts for up to five population doublings in culture. Oligonucleotide microarray analysis showed that up to 10% of genes are aberrantly expressed in demethylated fibroblasts. Our results demonstrate that loss of P26358 causes cell-type-specific changes in gene expression that impinge on several pathways, including expression of imprinted genes, cell-cycle control, growth factor/receptor signal transduction and mobilization of retroelements.</td>\n",
       "      <td>P04637</td>\n",
       "      <td>P26358</td>\n",
       "      <td>11137995</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.885095</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                abstract  \\\n",
       "7511                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Gene activity is regulated by transcriptional and epigenetic mechanisms. A paper in 2005 by Zhang et al.(1) showed that STAT3 binds to the DNA methyl transferase, DNMT1 and their data indicated that STAT3 may cause epigenetic gene silencing by targeting DNMT1 to the PTPN6 promoter. Now, a paper by Lee et al.(2) has fleshed out the mechanism. They provide evidence that acetylation of STAT3 regulates the binding of DNMT1, CpG DNA methylation and regulation of several genes, including that encoding the estrogen receptor  (ESR1) in breast cancer cells.    \n",
       "13650                                                                                                                                                                                             The cells in the preimplantation mammalian embryo undergo several rounds of fast cell division. Whether the known DNA repair pathways are active during these early stages of development where cell division is of primary importance, has not been fully established. Because of the important role of phosphorylated H2A.X (gammaH2A.X) in the DNA damage response as well as its putative role in assembly of embryonic chromatin, we analysed its distribution in the preimplantation mouse embryo. We found that H2A.X is highly phosphorylated throughout preimplantation development in the absence of any induced DNA damage. Moreover, gammaH2A.X levels vary significantly throughout the cell cycle. Interestingly, after the 4-cell stage, we detected high levels of H2A.X phosphorylation in mitosis, where telomeres appeared focally enriched with gammaH2A.X. In contrast, 53BP1, which is known to be recruited to DNA damage sites, is undetectable at mitotic chromosomes at these stages and its localisation changes upon blastocyst formation from mainly nuclear to cytoplasmic. We also show that 53BP1 and gammaH2A.X rarely colocalise, suggesting that the high levels of phosphorylation of H2A.X in the embryo might not be directly linked to the DNA damage response in the embryo. Our data suggest that phosphorylation of H2A.X is an important event in the fast dividing cells of the early embryo in the absence of any induced DNA damage. We discuss the possible consequences of these findings on the genome-wide chromatin remodelling that ocurs in the preimplantation mammalian embryo.   \n",
       "6593                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       The non-genomic membrane bound oestrogen receptor (mER) regulates intracellular signals through receptor-ligand interactions. The mER, along with G-protein coupled oestrogen receptor GPR 30 (GPER), induces diverse cell signalling pathways in murine lymphocytes. The mER isoform ER-alpha46 has recently been demonstrated in human B and T lymphocytes as an analogue receptor for chemokine CCL18, the signalling events of which are not clearly understood. Ligand-induced mER and GPER signalling events are shared with BCR, CD19 mediated intracellular signalling through phospholipase C, PIP2/IP3/PI3 mediated activation of Akt, MAP kinase, and mTOR. Oestrogen has the ability to induce CD40-mediated activation of B cells. The complete signalling pathways of mER, GPR30 and their interaction with other signals are targeted areas for novel drug development in B cells during infection, autoimmunity and cancer. Therefore, an in depth investigation is critical for determining shared signal outputs during B cell activation. Here, we focus on the mode of action of membrane bound ER in B cells as therapeutic checkpoints.   \n",
       "895                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       The retinoblastoma binding protein KDM5A removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of KDM5A contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, KDM5A contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in KDM5A have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the PHD1 preferentially recognizes unmethylated H3K4 histone tail, product of KDM5A-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the PHD1 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in KDM5A--suggests a model for the spread of demethylation on chromatin.   \n",
       "15505                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Aberrations in DNA methylation patterns promote changes in gene expression patterns and are invariably associated with neoplasia. DNA methylation is carried out and maintained by several DNA methyltransferases (DNMTs) among which DNMT1 functions as a maintenance methylase while DNMT3a and 3b serve as de novo enzymes. Although DNMT3b has been shown to preferentially target the methylation of DNA sequences residing in pericentric heterochromatin whether it is involved in gene specific methylation remains an open question. To address this issue, we have silenced the expression of DNMT3b in the prostate-derived PC3 cells through RNA interference and subsequently studied the accompanied cellular changes as well as the expression profiles of selected genes.   \n",
       "12473                                                                                                                                                                                                                                                                                                                                           Protein kinase C delta (PKC) is a serine (Ser)/threonine kinase, which regulates numerous cellular processes, including proliferation, differentiation, migration and apoptosis. In the current study, Chinese hamster ovary cells were transfected with either a constitutively activated PKC or a dominant negative PKC, phosphoprotein enrichment, two-dimensional difference gel electrophoresis and mass spectrometry was combined to globally identified candidates of PKC cascade. We found that Bcl-2 associated athanogene 3 (BAG3) was one of the targets of PKC cascade, and BAG3 interacted with PKC in vivo. In addition, we clarified that BAG3 was phosphorylate at Ser187 site in a PKC-dependent manner in vivo. BAG3 has been implicated in multiple cellular functions, including proliferation, differentiation, apoptosis, migration, invasion, macroautophagy and so on. We generated wild-type (WT)-, Ser187Ala (S187A)- or Ser187Asp (S187D)-BAG3 stably expressing FRO cells, and noticed that phosphorylation state of BAG3 influenced FRO morphology. Finally, for the first time, we showed that BAG3 was implicated in epithelial-mesenchymal transition (EMT) procedure, and phosphorylation state at Ser187 site had a critical role in EMT regulation by BAG3. Collectively, the current study indicates that BAG3 is a novel substrate of PKC, and PKC-mediated phosphorylation of BAG3 is implicated in EMT and invasiveness of thyroid cancer cells.    \n",
       "12965                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      The Dnmt3a DNA methyltransferase is responsible for establishing DNA methylation patterns during mammalian development. We show here that the mouse Dnmt3a DNA methyltransferase is able to transfer the methyl group from S-adenosyl-l-methionine (AdoMet) to a cysteine residue in its catalytic center. This reaction is irreversible and relatively slow. The yield of auto-methylation is increased by addition of Dnmt3L, which functions as a stimulator of Dnmt3a and enhances its AdoMet binding. Auto-methylation was observed in binary Dnmt3a AdoMet complexes. In the presence of CpG containing dsDNA, which is the natural substrate for Dnmt3a, the transfer of the methyl group from AdoMet to the flipped target base was preferred and auto-methylation was not detected. Therefore, this reaction might constitute a regulatory mechanism which could inactivate unused DNA methyltransferases in the cell, or it could simply be an aberrant side reaction caused by the high methyl group transfer potential of AdoMet.   \n",
       "10041                                                                                                              Proteins are frequently modified by post-translational methylation of lysine residues, catalyzed by S-adenosylmethionine-dependent lysine methyltransferases (KMTs). Lysine methylation of histone proteins has been extensively studied, but it has recently become evident that methylation of non-histone proteins is also abundant and important. The human methyltransferase METTL20 belongs to a group of 10 established and putative human KMTs. We here found METTL20 to be associated with mitochondria and determined that recombinant METTL20 methylated a single protein in extracts from human cells. Using an methyltransferase activity-based purification scheme, we identified the -subunit of the mitochondrially localized electron transfer flavoprotein (ETF) as the substrate of METTL20. Furthermore, METTL20 was found to specifically methylate two adjacent lysine residues, Lys(200) and Lys(203), in ETF both in vitro and in cells. Interestingly, the residues methylated by METTL20 partially overlap with the so-called \"recognition loop\" in ETF, which has been shown to mediate its interaction with various dehydrogenases. Accordingly, we found that METTL20-mediated methylation of ETF in vitro reduced its ability to receive electrons from the medium chain acyl-CoA dehydrogenase and the glutaryl-CoA dehydrogenase. In conclusion, the present study establishes METTL20 as the first human KMT localized to mitochondria and suggests that it may regulate cellular metabolism through modulating the interaction between its substrate ETF and dehydrogenases. Based on the previous naming of similar enzymes, we suggest the renaming of human METTL20 to ETF-KMT.    \n",
       "12186                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     Dimethylation of histone H3 Arg2 (H3R2me2) maintains transcriptional silencing by inhibiting Set1 mediated trimethylation of H3K4. Here we demonstrate that Arg2 is also monomethylated (H3R2me1) in yeast but that its functional characteristics are distinct from H3R2me2: (i) H3R2me1 does not inhibit histone H3 Lys4 (H3K4) methylation; (ii) it is present throughout the coding region of genes; and (iii) it correlates with active transcription. Collectively, these results indicate that different H3R2 methylation states have defined roles in gene expression.   \n",
       "5037                                                                                                                                                                                                                                                                  RUNX1 (previously termed AML1) is a frequent target of human leukaemia-associated gene aberrations, and it encodes the DNA-binding subunit of the Core-Binding Factor transcription factor complex. RUNX1 expression is essential for the initiation of definitive haematopoiesis, for steady-state thrombopoiesis, and for normal lymphocytes development. Recent studies revealed that protein arginine methyltransferase 1 (PRMT1), which accounts for the majority of the type I PRMT activity in cells, methylates two arginine residues in RUNX1 (R206 and R210), and these modifications inhibit corepressor-binding to RUNX1 thereby enhancing its transcriptional activity. In order to elucidate the biological significance of these methylations, we established novel knock-in mouse lines with non-methylable, double arginine-to-lysine (RTAMR-to-KTAMK) mutations in RUNX1. Homozygous Runx1(KTAMK) (/) (KTAMK) mice are born alive and appear normal during adulthood. However, Runx1(KTAMK) (/) (KTAMK) mice showed a reduction in CD3(+) T lymphoid cells and a decrease in CD4(+) T cells in peripheral lymphoid organs, in comparison to their wild-type littermates, leading to a reduction in the CD4(+) to CD8(+) T-cell ratio. These findings suggest that arginine-methylation of RUNX1 in the RTAMR-motif is dispensable for the development of definitive haematopoiesis and for steady-state platelet production, however this modification affects the role of RUNX1 in the maintenance of the peripheral CD4(+) T-cell population.    \n",
       "1346                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             The purified insulin receptor kinase catalyzed the phosphorylation of native tubulin and microtubule-associated proteins (MAPs; MAP2, tau) on tyrosine residues. Insulin (10(-7) M) stimulated the reaction by 4-10-fold by increasing Vmax with little change in Km. alpha-Tubulin was preferred as a substrate for the kinase compared to beta-tubulin. MAP2 was found to be the best substrate among the cytoskeletal proteins tested; in the presence of insulin, the Vmax for MAP2 was 6.3 nmol/min/mg, its Km was 5.1 microM, and 1.7 mol of phosphate were incorporated per mol of MAP2. Under the same conditions used for this phosphorylation of tubulin and MAPs, actin and tropomyosin were very poorly phosphorylated. These data, coupled with previous evidence for potential functional relationships between insulin action and microtubules, raise the possibility that microtubule proteins may be cellular targets for the insulin receptor kinase.   \n",
       "11163  Neuroblastoma is a very heterogeneous tumor of childhood. The clinical spectra range from very aggressive metastatic disease to spontaneous regression, even without therapy. Aberrant DNA methylation pattern is a common feature of most cancers. For neuroblastoma, it has been demonstrated both for single genes as well as genome-wide, where a so-called methylator phenotype has been described. Here, we present a study using Illumina 450K methylation arrays on 60 neuroblastoma tumors. We show that aggressive tumors, characterized by International Neuroblastoma Risk Group (INRG) as stage M, are hypermethylated compared to low-grade tumors. On the contrary, INRG stage L tumors display more non-CpG methylation. The genes with the highest number of hypermethylated CpG sites in INRG M tumors are TERT, PCDHGA4, DLX5, and DLX6-AS1. Gene ontology analysis showed a representation of neuronal tumor relevant gene functions among the differentially methylated genes. For validation, we used a set of independent tumors previously analyzed with the Illumina 27K methylation arrays, which confirmed the differentially methylated sites. Top candidate genes with aberrant methylation were analyzed for altered gene expression through the R2 platform ( http://r2.amc.nl), and for correlations between methylation and gene expression in a public dataset. Altered expression in nonsurvivors was found for the genes B3GALT4 and KIAA1949, CLIC5, DLX6-AS, TERT, and PIRT, and strongest correlations were found for TRIM36, KIAA0513, and PIRT. Our data indicate that methylation profiling can be used for patient stratification and informs on epigenetically deregulated genes with the potential of increasing our knowledge about the underlying mechanisms of tumor development.   \n",
       "7659                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   The rising incidence of antimicrobial resistance (AMR) makes it imperative to understand the underlying mechanisms. Mycobacterium tuberculosis (Mtb) is the single leading cause of death from a bacterial pathogen and estimated to be the leading cause of death from AMR. A pyrido-benzimidazole, 14, was reported to have potent bactericidal activity against Mtb. Here, we isolated multiple Mtb clones resistant to 14. Each had mutations in the putative DNA-binding and dimerization domains of rv2887, a gene encoding a transcriptional repressor of the MarR family. The mutations in Rv2887 led to markedly increased expression of rv0560c. We characterized Rv0560c as an S-adenosyl-L-methionine-dependent methyltransferase that N-methylates 14, abolishing its mycobactericidal activity. An Mtb strain lacking rv0560c became resistant to 14 by mutating decaprenylphosphoryl--d-ribose 2-oxidase (DprE1), an essential enzyme in arabinogalactan synthesis; 14 proved to be a nanomolar inhibitor of DprE1, and methylation of 14 by Rv0560c abrogated this activity. Thus, 14 joins a growing list of DprE1 inhibitors that are potently mycobactericidal. Bacterial methylation of an antibacterial agent, 14, catalyzed by Rv0560c of Mtb, is a previously unreported mechanism of AMR.   \n",
       "2810                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               14-3-3 sigma has been a major G2/M checkpoint control gene and has demonstrated that its inactivation in various cancers occurs mostly by epigenetic hypermethylation, not by genetic change. This study investigated the methylation status and expression of the 14-3-3 sigma gene in 46 oral squamous cell carcinomas by methylation-specific polymerase chain reaction, reverse transcriptase-polymerase chain reaction, Western blotting and immunohistochemistry. Exons of the p53 gene were examined for mutations by sequencing analysis and CyclinD1 by immunohistochemistry. Methylation of the 14-3-3 sigma gene was detected in 13% (6/46) of the oral tumours, but not in corresponding adjacent non-malignant and normal gingival tissues. Intratumoural heterogeneity was found in the tumour tissues including three 14-3-3 sigma-methylated samples. Methylation of 14-3-3 sigma was detected in 3 SCC with p53 mutations and 3 with wild-type p53. Our major findings are: (a) methylation of 14-3-3 gene promoter is a rare event in oral cancer; (b) it is not always associated with 14-3-3 protein levels and there is no clear relationship between its methylation and p53 mutation; (c) loss of 14-3-3 sigma expression is associated with reduced CyclinD1 gene expression.   \n",
       "9242                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           Epithelial-to-mesenchymal transition (EMT) is a fundamental mechanism governing the switch of cells from an epithelial to a motile mesenchymal-like state. This transdifferentiation is regulated by key transcription factors, including Slug. The stability and function of Slug can be regulated by multiple mechanisms, including ubiquitin-mediated post-translational modifications. Here, by using a genome wide siRNA screen for human deubiquitinating enzymes (DUBs), we identified USP10 as a deubiquitinase for Slug in cancer cells. USP10 interacts with Slug and mediates its degradation by the proteasome. Importantly, USP10 is concomitantly highly expressed with Slug in cancer biopsies. Genetic knockdown of USP10 leads to suppressed Slug levels with a decreased expression of the mesenchymal marker Vimentin. Further, it reduces the migratory capacity of cancer cells. Reversely, overexpression of USP10 elevates the level of both Slug and Vimentin. Our study identifies USP10 as a regulator of the EMT-transcription factor Slug and cell migration.   \n",
       "17854                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Ribosome-recycling factor (RRF) plays an essential role in the fourth step of protein synthesis in prokaryotes. RRF combined with elongation factor G (EF-G) disassembles the post-termination ribosome complex and recycles the protein synthesis machine for the next round of translation. A reductive-methylation-modified RRF from Thermoanaerobacter tengcongensis (TteRRF) has been crystallized using the vapour-diffusion method. The crystal grew in a condition consisting of 0.1 M citric acid pH 3.5, 3.0 M NaCl and 50 mg ml(-1) methylated protein solution at 289 K. A complete data set was collected from a crystal to 2.80  resolution using synchrotron radiation at 100 K. The crystal belonged to space group P6122/P6522 with unit-cell parameters a = b = 103.26, c = 89.17 . The asymmetric unit was estimated to contain one molecule of TteRRF.   \n",
       "8660                                                                                                                                                                                                                                                                                                                                                                                                                                Recent evidence points to a role of chromatin in regulation of alternative pre-mRNA splicing (AS). In order to identify novel chromatin regulators of AS, we screened an RNAi library of chromatin proteins using a cell-based high-throughput in vivo assay. We identified a set of chromatin proteins that regulate AS. Using simultaneous genome-wide expression and AS analysis, we demonstrate distinct and non-overlapping functions of these chromatin modifiers on transcription and AS. Detailed mechanistic characterization of one dual function chromatin modifier, the H3K9 methyltransferase EHMT2 (G9a), identified VEGFA as a major chromatin-mediated AS target. Silencing of EHMT2, or its heterodimer partner EHMT1, affects AS by promoting exclusion of VEGFA exon 6a, but does not alter total VEGFA mRNA levels. The epigenetic regulatory mechanism of AS by EHMT2 involves an adaptor system consisting of the chromatin modulator HP1, which binds methylated H3K9 and recruits splicing regulator SRSF1. The epigenetic regulation of VEGFA is physiologically relevant since EHMT2 is transcriptionally induced in response to hypoxia and triggers concomitant changes in AS of VEGFA. These results characterize a novel epigenetic regulatory mechanism of AS and they demonstrate separate roles of epigenetic modifiers in transcription and alternative splicing.    \n",
       "8369                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   Chemoresistance, due to inhibition of apoptotic response, is the major reason for the failure of anticancer therapies. HIPK2 regulates p53-apoptotic function via serine-46 (Ser46) phosphorylation and activation of p53 is a key determinant in ovarian cancer cell death. In this study we determined whether HIPK2 overexpression restored apoptotic response in chemoresistant cancer cells.   \n",
       "14870                                                                                                                                                                                                                                                                                                                                               Bone morphogenetic proteins (BMPs) are an integral component of the TGFbeta superfamily, responsible for regulation of cell proliferation, differentiation, migration and programmed cell death in a variety of cell types. The BMPs transduce their signals directly through the SMAD family of proteins but they also have been reported to interact with the MAPK and Erk pathways. Inactivation of the BMP pathway genes has been implicated as important in several cancers. Recent work has shown that BMP3b is epigenetically inactivated in cancer and suggests that BMP6 can be epigenetically inactivated. We investigated whether BMP6 is epigenetically inactivated in cell lines and whether BMP3b and BMP6 are epigenetically inactivated in non-small-cell lung cancer (NSCLC). We also studied the relationship between BMP methylation and k-ras mutation. Here, we demonstrate that the BMP3b and BMP6 genes are common targets of epigenetic inactivation in NSCLC, and that they are significantly more likely to be concurrently inactivated (P=0.009). Furthermore, this coinactivation of BMP3b and BMP6 is significantly associated with mutation of k-ras codon 12 in lung cancer (P=0.003); those with a k-ras mutation were six times more likely to have concurrent methylation of these BMP loci. Hence, these data suggest that concurrent inactivation of the BMP and activation of the Ras signalling pathways are important in lung carcinogenesis.   \n",
       "4909                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     Cytosine methylation of mammalian DNA is essential for the proper epigenetic regulation of gene expression and maintenance of genomic integrity. To define the mechanism through which demethylated cells die, and to establish a paradigm for identifying genes regulated by DNA methylation, we have generated mice with a conditional allele for the maintenance DNA methyltransferase gene Dnmt1. Cre-mediated deletion of Dnmt1 causes demethylation of cultured fibroblasts and a uniform p53-dependent cell death. Mutational inactivation of Trp53 partially rescues the demethylated fibroblasts for up to five population doublings in culture. Oligonucleotide microarray analysis showed that up to 10% of genes are aberrantly expressed in demethylated fibroblasts. Our results demonstrate that loss of Dnmt1 causes cell-type-specific changes in gene expression that impinge on several pathways, including expression of imprinted genes, cell-cycle control, growth factor/receptor signal transduction and mobilization of retroelements.   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      normalised_abstract  \\\n",
       "7511                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Gene activity is regulated by transcriptional and epigenetic mechanisms. A paper in 2005 by Zhang et al.(1) showed that P40763 binds to the DNA methyl transferase, P26358 and their data indicated that P40763 may cause epigenetic gene silencing by targeting P26358 to the P29350 promoter. Now, a paper by Lee et al.(2) has fleshed out the mechanism. They provide evidence that acetylation of P40763 regulates the binding of P26358, CpG DNA methylation and regulation of several genes, including that encoding the P03372  (P03372) in breast cancer cells.    \n",
       "13650                                                                                                                                                                                       The cells in the preimplantation mammalian embryo undergo several rounds of fast cell division. Whether the known DNA repair pathways are active during these early stages of development where cell division is of primary importance, has not been fully established. Because of the important role of phosphorylated P16104 (gammaH2A.X) in the DNA damage response as well as its putative role in assembly of embryonic chromatin, we analysed its distribution in the preimplantation mouse embryo. We found that P16104 is highly phosphorylated throughout preimplantation development in the absence of any induced DNA damage. Moreover, gammaH2A.X levels vary significantly throughout the cell cycle. Interestingly, after the 4-cell stage, we detected high levels of P16104 phosphorylation in mitosis, where telomeres appeared focally enriched with gammaH2A.X. In contrast, Q12888, which is known to be recruited to DNA damage sites, is undetectable at mitotic chromosomes at these stages and its localisation changes upon blastocyst formation from mainly nuclear to cytoplasmic. We also show that Q12888 and gammaH2A.X rarely colocalise, suggesting that the high levels of phosphorylation of P16104 in the embryo might not be directly linked to the DNA damage response in the embryo. Our data suggest that phosphorylation of P16104 is an important event in the fast dividing cells of the early embryo in the absence of any induced DNA damage. We discuss the possible consequences of these findings on the genome-wide chromatin remodelling that ocurs in the preimplantation mammalian embryo.   \n",
       "6593                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                The non-genomic Q99527 (Q99527) regulates intracellular signals through receptor-ligand interactions. The Q99527, along with Q99527 (Q99527), induces diverse cell signalling pathways in murine lymphocytes. The Q99527 isoform P0337246 has recently been demonstrated in human B and T lymphocytes as an analogue receptor for chemokine P55774, the signalling events of which are not clearly understood. Ligand-induced Q99527 and Q99527 signalling events are shared with BCR, P15391 mediated intracellular signalling through phospholipase C, PIP2/P19957 mediated activation of Akt, MAP kinase, and P42345. Oestrogen has the ability to induce P25942-mediated activation of B cells. The complete signalling pathways of Q99527, Q99527 and their interaction with other signals are targeted areas for novel drug development in B cells during infection, autoimmunity and cancer. Therefore, an in depth investigation is critical for determining shared signal outputs during B cell activation. Here, we focus on the mode of action of membrane bound ER in B cells as therapeutic checkpoints.   \n",
       "895                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              The retinoblastoma binding protein P29375 removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of P29375 contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, P29375 contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in P29375 have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the Q96KS0 preferentially recognizes unmethylated H3K4 histone tail, product of P29375-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the Q96KS0 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in P29375--suggests a model for the spread of demethylation on chromatin.   \n",
       "15505                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Aberrations in DNA methylation patterns promote changes in gene expression patterns and are invariably associated with neoplasia. DNA methylation is carried out and maintained by several DNA methyltransferases (DNMTs) among which P26358 functions as a maintenance methylase while 1788;1789 serve as de novo enzymes. Although Q9UBC3 has been shown to preferentially target the methylation of DNA sequences residing in pericentric heterochromatin whether it is involved in gene specific methylation remains an open question. To address this issue, we have silenced the expression of Q9UBC3 in the prostate-derived PC3 cells through RNA interference and subsequently studied the accompanied cellular changes as well as the expression profiles of selected genes.   \n",
       "12473                                                                                                                                                                                                                                                                                                                                  Q05655 (Q05655) is a serine (Ser)/threonine kinase, which regulates numerous cellular processes, including proliferation, differentiation, migration and apoptosis. In the current study, Chinese hamster ovary cells were transfected with either a constitutively activated Q05655 or a dominant negative Q05655, phosphoprotein enrichment, two-dimensional difference gel electrophoresis and mass spectrometry was combined to globally identified candidates of Q05655 cascade. We found that 9531;596 (O95817) was one of the targets of Q05655 cascade, and O95817 interacted with Q05655 in vivo. In addition, we clarified that O95817 was phosphorylate at Ser187 site in a Q05655-dependent manner in vivo. O95817 has been implicated in multiple cellular functions, including proliferation, differentiation, apoptosis, migration, invasion, macroautophagy and so on. We generated wild-type (WT)-, Ser187Ala (S187A)- or Ser187Asp (S187D)-O95817 stably expressing FRO cells, and noticed that phosphorylation state of O95817 influenced FRO morphology. Finally, for the first time, we showed that O95817 was implicated in epithelial-mesenchymal transition (EMT) procedure, and phosphorylation state at Ser187 site had a critical role in EMT regulation by O95817. Collectively, the current study indicates that O95817 is a novel substrate of Q05655, and Q05655-mediated phosphorylation of O95817 is implicated in EMT and invasiveness of thyroid cancer cells.    \n",
       "12965                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       The Q9Y6K1 DNA methyltransferase is responsible for establishing DNA methylation patterns during mammalian development. We show here that the mouse Q9Y6K1 DNA methyltransferase is able to transfer the methyl group from S-adenosyl-l-methionine (AdoMet) to a cysteine residue in its catalytic center. This reaction is irreversible and relatively slow. The yield of auto-methylation is increased by addition of Q9UJW3, which functions as a stimulator of Q9Y6K1 and enhances its AdoMet binding. Auto-methylation was observed in binary Q9Y6K1 AdoMet complexes. In the presence of CpG containing dsDNA, which is the natural substrate for Q9Y6K1, the transfer of the methyl group from AdoMet to the flipped target base was preferred and auto-methylation was not detected. Therefore, this reaction might constitute a regulatory mechanism which could inactivate unused DNA methyltransferases in the cell, or it could simply be an aberrant side reaction caused by the high methyl group transfer potential of AdoMet.   \n",
       "10041                                                                                                                                                                         Proteins are frequently modified by post-translational methylation of lysine residues, catalyzed by S-adenosylmethionine-dependent lysine methyltransferases (KMTs). Lysine methylation of histone proteins has been extensively studied, but it has recently become evident that methylation of non-histone proteins is also abundant and important. The human methyltransferase Q8IXQ9 belongs to a group of 10 established and putative human KMTs. We here found Q8IXQ9 to be associated with mitochondria and determined that recombinant Q8IXQ9 methylated a single protein in extracts from human cells. Using an methyltransferase activity-based purification scheme, we identified the -subunit of the mitochondrially localized electron transfer flavoprotein (ETF) as the substrate of Q8IXQ9. Furthermore, Q8IXQ9 was found to specifically methylate two adjacent lysine residues, Lys(200) and Lys(203), in ETF both in vitro and in cells. Interestingly, the residues methylated by Q8IXQ9 partially overlap with the so-called \"recognition loop\" in ETF, which has been shown to mediate its interaction with various dehydrogenases. Accordingly, we found that Q8IXQ9-mediated methylation of ETF in vitro reduced its ability to receive electrons from the P11310 and the Q92947. In conclusion, the present study establishes Q8IXQ9 as the first human KMT localized to mitochondria and suggests that it may regulate cellular metabolism through modulating the interaction between its substrate ETF and dehydrogenases. Based on the previous naming of similar enzymes, we suggest the renaming of human Q8IXQ9 to ETF-KMT.    \n",
       "12186                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                Dimethylation of histone H3 P78540 (H3R2me2) maintains transcriptional silencing by inhibiting O15047 mediated trimethylation of H3K4. Here we demonstrate that P78540 is also monomethylated (H3R2me1) in yeast but that its functional characteristics are distinct from H3R2me2: (i) H3R2me1 does not inhibit histone H3 Lys4 (H3K4) methylation; (ii) it is present throughout the coding region of genes; and (iii) it correlates with active transcription. Collectively, these results indicate that different H3R2 methylation states have defined roles in gene expression.   \n",
       "5037                                                                                                                                                                                                                                                                         Q01196 (previously termed Q01196) is a frequent target of human leukaemia-associated gene aberrations, and it encodes the DNA-binding subunit of the Core-Binding Factor transcription factor complex. Q01196 expression is essential for the initiation of definitive haematopoiesis, for steady-state thrombopoiesis, and for normal lymphocytes development. Recent studies revealed that Q99873 (Q99873), which accounts for the majority of the type I PRMT activity in cells, methylates two arginine residues in Q01196 (R206 and R210), and these modifications inhibit corepressor-binding to Q01196 thereby enhancing its transcriptional activity. In order to elucidate the biological significance of these methylations, we established novel knock-in mouse lines with non-methylable, double arginine-to-lysine (RTAMR-to-KTAMK) mutations in Q01196. Homozygous Q01196(KTAMK) (/) (KTAMK) mice are born alive and appear normal during adulthood. However, Q01196(KTAMK) (/) (KTAMK) mice showed a reduction in CD3(+) T lymphoid cells and a decrease in P01730(+) T cells in peripheral lymphoid organs, in comparison to their wild-type littermates, leading to a reduction in the P01730(+) to P01732(+) T-cell ratio. These findings suggest that arginine-methylation of Q01196 in the RTAMR-motif is dispensable for the development of definitive haematopoiesis and for steady-state platelet production, however this modification affects the role of Q01196 in the maintenance of the peripheral P01730(+) T-cell population.    \n",
       "1346                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             The purified insulin receptor kinase catalyzed the phosphorylation of native tubulin and microtubule-associated proteins (MAPs; P11137, tau) on tyrosine residues. Insulin (10(-7) M) stimulated the reaction by 4-10-fold by increasing Vmax with little change in Km. P68363 was preferred as a substrate for the kinase compared to beta-tubulin. P11137 was found to be the best substrate among the cytoskeletal proteins tested; in the presence of insulin, the Vmax for P11137 was 6.3 nmol/min/mg, its Km was 5.1 microM, and 1.7 mol of phosphate were incorporated per mol of P11137. Under the same conditions used for this phosphorylation of tubulin and MAPs, actin and tropomyosin were very poorly phosphorylated. These data, coupled with previous evidence for potential functional relationships between insulin action and microtubules, raise the possibility that microtubule proteins may be cellular targets for the insulin receptor kinase.   \n",
       "11163  Neuroblastoma is a very heterogeneous tumor of childhood. The clinical spectra range from very aggressive metastatic disease to spontaneous regression, even without therapy. Aberrant DNA methylation pattern is a common feature of most cancers. For neuroblastoma, it has been demonstrated both for single genes as well as genome-wide, where a so-called methylator phenotype has been described. Here, we present a study using Illumina 450K methylation arrays on 60 neuroblastoma tumors. We show that aggressive tumors, characterized by International Neuroblastoma Risk Group (INRG) as stage M, are hypermethylated compared to low-grade tumors. On the contrary, INRG stage L tumors display more non-CpG methylation. The genes with the highest number of hypermethylated CpG sites in INRG M tumors are O14746, Q9Y5G9, P56178, and 285987. Gene ontology analysis showed a representation of neuronal tumor relevant gene functions among the differentially methylated genes. For validation, we used a set of independent tumors previously analyzed with the Illumina 27K methylation arrays, which confirmed the differentially methylated sites. Top candidate genes with aberrant methylation were analyzed for altered gene expression through the R2 platform ( http://r2.amc.nl), and for correlations between methylation and gene expression in a public dataset. Altered expression in nonsurvivors was found for the genes481737 and Q6NYC8, Q9NZA1, 285987, O14746, and P0C851, and strongest correlations were found for Q9NQ86, O60268, and P0C851. Our data indicate that methylation profiling can be used for patient stratification and informs on epigenetically deregulated genes with the potential of increasing our knowledge about the underlying mechanisms of tumor development.   \n",
       "7659                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         The rising incidence of antimicrobial resistance (AMR) makes it imperative to understand the underlying mechanisms. Mycobacterium tuberculosis (Mtb) is the single leading cause of death from a bacterial pathogen and estimated to be the leading cause of death from AMR. A pyrido-benzimidazole, 14, was reported to have potent bactericidal activity against Mtb. Here, we isolated multiple Mtb clones resistant to 14. Each had mutations in the putative DNA-binding and dimerization domains of 888563, a gene encoding a transcriptional repressor of the MarR family. The mutations in 888563 led to markedly increased expression of 887637. We characterized 887637 as an S-adenosyl-L-methionine-dependent methyltransferase that N-methylates 14, abolishing its mycobactericidal activity. An Mtb strain lacking 887637 became resistant to 14 by mutating decaprenylphosphoryl--d-ribose 2-oxidase (DprE1), an essential enzyme in arabinogalactan synthesis; 14 proved to be a nanomolar inhibitor of DprE1, and methylation of 14 by 887637 abrogated this activity. Thus, 14 joins a growing list of DprE1 inhibitors that are potently mycobactericidal. Bacterial methylation of an antibacterial agent, 14, catalyzed by 887637 of Mtb, is a previously unreported mechanism of AMR.   \n",
       "2810                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            P31947 has been a major G2/M checkpoint control gene and has demonstrated that its inactivation in various cancers occurs mostly by epigenetic hypermethylation, not by genetic change. This study investigated the methylation status and expression of the P31947 gene in 46 oral squamous cell carcinomas by methylation-specific polymerase chain reaction, reverse transcriptase-polymerase chain reaction, Western blotting and immunohistochemistry. Exons of the P04637 gene were examined for mutations by sequencing analysis and P24385 by immunohistochemistry. Methylation of the P31947 gene was detected in 13% (6/46) of the oral tumours, but not in corresponding adjacent non-malignant and normal gingival tissues. Intratumoural heterogeneity was found in the tumour tissues including three P31947-methylated samples. Methylation of P31947 was detected in 3 SCC with P04637 mutations and 3 with wild-type P04637. Our major findings are: (a) methylation of 14-3-3 gene promoter is a rare event in oral cancer; (b) it is not always associated with 14-3-3 protein levels and there is no clear relationship between its methylation and P04637 mutation; (c) loss of P31947 expression is associated with reduced P24385 gene expression.   \n",
       "9242                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Epithelial-to-mesenchymal transition (EMT) is a fundamental mechanism governing the switch of cells from an epithelial to a motile mesenchymal-like state. This transdifferentiation is regulated by key transcription factors, including O43623. The stability and function of O43623 can be regulated by multiple mechanisms, including ubiquitin-mediated post-translational modifications. Here, by using a genome wide siRNA screen for human deubiquitinating enzymes (DUBs), we identified Q14694 as a deubiquitinase for O43623 in cancer cells. Q14694 interacts with O43623 and mediates its degradation by the proteasome. Importantly, Q14694 is concomitantly highly expressed with O43623 in cancer biopsies. Genetic knockdown of Q14694 leads to suppressed O43623 levels with a decreased expression of the mesenchymal marker P08670. Further, it reduces the migratory capacity of cancer cells. Reversely, overexpression of Q14694 elevates the level of both O43623 and P08670. Our study identifies Q14694 as a regulator of the EMT-transcription factor O43623 and cell migration.   \n",
       "17854                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Q96E11 (Q96E11) plays an essential role in the fourth step of protein synthesis in prokaryotes. Q96E11 combined with Q96RP9 (Q96RP9) disassembles the post-termination ribosome complex and recycles the protein synthesis machine for the next round of translation. A reductive-methylation-modified Q96E11 from Thermoanaerobacter tengcongensis (TteRRF) has been crystallized using the vapour-diffusion method. The crystal grew in a condition consisting of 0.1 M citric acid pH 3.5, 3.0 M NaCl and 50 mg ml(-1) methylated protein solution at 289 K. A complete data set was collected from a crystal to 2.80  resolution using synchrotron radiation at 100 K. The crystal belonged to space group P6122/P6522 with unit-cell parameters a = b = 103.26, c = 89.17 . The asymmetric unit was estimated to contain one molecule of TteRRF.   \n",
       "8660                                                                                                                                                                                                                                                                                                                                                                                                                Recent evidence points to a role of chromatin in regulation of alternative pre-mRNA splicing (AS). In order to identify novel chromatin regulators of AS, we screened an RNAi library of chromatin proteins using a cell-based high-throughput in vivo assay. We identified a set of chromatin proteins that regulate AS. Using simultaneous genome-wide expression and AS analysis, we demonstrate distinct and non-overlapping functions of these chromatin modifiers on transcription and AS. Detailed mechanistic characterization of one dual function chromatin modifier, the H3K9 methyltransferase Q96KQ7 (Q96KQ7), identified P15692 as a major chromatin-mediated AS target. Silencing of Q96KQ7, or its heterodimer partner Q9H9B1, affects AS by promoting exclusion of P15692 exon 6a, but does not alter total P15692 mRNA levels. The epigenetic regulatory mechanism of AS by Q96KQ7 involves an adaptor system consisting of the chromatin modulator P59665, which binds methylated H3K9 and recruits splicing regulator Q07955. The epigenetic regulation of P15692 is physiologically relevant since Q96KQ7 is transcriptionally induced in response to hypoxia and triggers concomitant changes in AS of P15692. These results characterize a novel epigenetic regulatory mechanism of AS and they demonstrate separate roles of epigenetic modifiers in transcription and alternative splicing.    \n",
       "8369                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Chemoresistance, due to inhibition of apoptotic response, is the major reason for the failure of anticancer therapies. Q9H2X6 regulates P04637-apoptotic function via serine-46 (Ser46) phosphorylation and activation of P04637 is a key determinant in ovarian cancer cell death. In this study we determined whether Q9H2X6 overexpression restored apoptotic response in chemoresistant cancer cells.   \n",
       "14870                                                                                                                                                                                                                                                                                                                   Bone morphogenetic proteins (BMPs) are an integral component of the TGFbeta superfamily, responsible for regulation of cell proliferation, differentiation, migration and programmed cell death in a variety of cell types. The BMPs transduce their signals directly through the SMAD family of proteins but they also have been reported to interact with the MAPK and P28482 pathways. Inactivation of the P13497 pathway genes has been implicated as important in several cancers. Recent work has shown that P55107 is epigenetically inactivated in cancer and suggests that P22004 can be epigenetically inactivated. We investigated whether P22004 is epigenetically inactivated in cell lines and whether P55107 and P22004 are epigenetically inactivated in non-small-cell lung cancer (NSCLC). We also studied the relationship between P13497 methylation and k-ras mutation. Here, we demonstrate that the P55107 and P22004 genes are common targets of epigenetic inactivation in NSCLC, and that they are significantly more likely to be concurrently inactivated (P=0.009). Furthermore, this coinactivation of P55107 and P22004 is significantly associated with mutation of k-ras codon 12 in lung cancer (P=0.003); those with a k-ras mutation were six times more likely to have concurrent methylation of these P13497 loci. Hence, these data suggest that concurrent inactivation of the P13497 and activation of the Ras signalling pathways are important in lung carcinogenesis.   \n",
       "4909                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Cytosine methylation of mammalian DNA is essential for the proper epigenetic regulation of gene expression and maintenance of genomic integrity. To define the mechanism through which demethylated cells die, and to establish a paradigm for identifying genes regulated by DNA methylation, we have generated mice with a conditional allele for the maintenance DNA methyltransferase gene P26358. Cre-mediated deletion of P26358 causes demethylation of cultured fibroblasts and a uniform P04637-dependent cell death. Mutational inactivation of P04637 partially rescues the demethylated fibroblasts for up to five population doublings in culture. Oligonucleotide microarray analysis showed that up to 10% of genes are aberrantly expressed in demethylated fibroblasts. Our results demonstrate that loss of P26358 causes cell-type-specific changes in gene expression that impinge on several pathways, including expression of imprinted genes, cell-cycle control, growth factor/receptor signal transduction and mobilization of retroelements.   \n",
       "\n",
       "      participant1Id participant2Id  pubmedId         predicted  \\\n",
       "7511          P26358         P40763  24058781       methylation   \n",
       "13650         P16104         Q12888  19598117   phosphorylation   \n",
       "6593          P03372         Q99527  27189345   phosphorylation   \n",
       "895           P29375         Q96KS0  25686748       methylation   \n",
       "15505         P26358         Q9UBC3  18798999       methylation   \n",
       "12473         O95817         Q05655  23108398   phosphorylation   \n",
       "12965         Q9UJW3         Q9Y6K1  21481189       methylation   \n",
       "10041         Q8IXQ9         Q92947  25416781       methylation   \n",
       "12186         O15047         P78540  19270702       methylation   \n",
       "5037          Q01196         Q99873  26010396       methylation   \n",
       "1346          P11137         P68363   3920212   phosphorylation   \n",
       "11163         O96024         Q6NYC8  26786290       methylation   \n",
       "7659          887637         888563  27432954       methylation   \n",
       "2810          P04637         P31947  17786341       methylation   \n",
       "9242          O43623         Q14694  29803676  deubiquitination   \n",
       "17854         Q96E11         Q96RP9  24817715       methylation   \n",
       "8660          Q96KQ7         Q9H9B1  25414343       methylation   \n",
       "8369          P04637         Q9H2X6  18395248   phosphorylation   \n",
       "14870         P22004         P55107  16175182       methylation   \n",
       "4909          P04637         P26358  11137995       methylation   \n",
       "\n",
       "       predicted_confidence  \n",
       "7511               0.870079  \n",
       "13650              0.981332  \n",
       "6593               0.982047  \n",
       "895                0.866325  \n",
       "15505              0.939795  \n",
       "12473              0.983797  \n",
       "12965              0.933695  \n",
       "10041              0.917449  \n",
       "12186              0.874299  \n",
       "5037               0.929019  \n",
       "1346               0.984268  \n",
       "11163              0.880582  \n",
       "7659               0.902199  \n",
       "2810               0.860166  \n",
       "9242               0.528337  \n",
       "17854              0.854795  \n",
       "8660               0.875132  \n",
       "8369               0.983379  \n",
       "14870              0.864911  \n",
       "4909               0.885095  "
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df[[\"abstract\", \"normalised_abstract\", \"participant1Id\",\"participant2Id\", \"pubmedId\", \"predicted\" ,\"predicted_confidence\" ]].sample(n=20)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "conda_pytorch_p36",
   "language": "python",
   "name": "conda_pytorch_p36"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
