{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "s3_prefix = \"s3://aegovan-data/pubmed_asbtract/predictions_multi_00/\"\n",
    "s3_data =\"s3://aegovan-data/human_output/human_interactions_ppi_v2.json\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "local_temp = \"temp\"\n",
    "local_temp_pred_dir = os.path.join( local_temp, \"pred_results\")\n",
    "local_temp_wk_dir = os.path.join( local_temp, \"wk\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!rm -rf $local_temp\n",
    "!mkdir -p $local_temp_pred_dir\n",
    "!mkdir -p $local_temp_wk_dir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!aws s3 cp s3://aegovan-data/pubmed_asbtract/predictions_multi_95/pubmed19n0538.json.txt.json.prediction.json ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "import boto3\n",
    "import glob\n",
    "from multiprocessing.dummy import Pool as ThreadPool\n",
    "import argparse\n",
    "import datetime \n",
    "import os\n",
    "\n",
    "\n",
    "def uploadfile(localpath, s3path):\n",
    "        \"\"\"\n",
    "Uploads a file to s3\n",
    "        :param localpath: The local path\n",
    "        :param s3path: The s3 path in format s3://mybucket/mydir/mysample.txt\n",
    "        \"\"\"\n",
    "\n",
    "        bucket, key = get_bucketname_key(s3path)\n",
    "\n",
    "        if key.endswith(\"/\"):\n",
    "            key = \"{}{}\".format(key, os.path.basename(localpath))\n",
    "        \n",
    "        s3 = boto3.client('s3')\n",
    "        \n",
    "        s3.upload_file(localpath, bucket, key)\n",
    "\n",
    "def get_bucketname_key(uripath):\n",
    "    assert uripath.startswith(\"s3://\")\n",
    "\n",
    "    path_without_scheme = uripath[5:]\n",
    "    bucket_end_index = path_without_scheme.find(\"/\")\n",
    "\n",
    "    bucket_name = path_without_scheme\n",
    "    key = \"/\"\n",
    "    if bucket_end_index > -1:\n",
    "        bucket_name = path_without_scheme[0:bucket_end_index]\n",
    "        key = path_without_scheme[bucket_end_index + 1:]\n",
    "\n",
    "    return bucket_name, key\n",
    "\n",
    "\n",
    "def download_file(s3path, local_dir):\n",
    "    bucket, key = get_bucketname_key(s3path)\n",
    "    \n",
    "    s3 = boto3.client('s3')\n",
    "    \n",
    "    local_file = os.path.join(local_dir, s3path.split(\"/\")[-1])\n",
    "    \n",
    "\n",
    "    s3.download_file(bucket, key, local_file)\n",
    "    \n",
    "def download_object(s3path):\n",
    "    bucket, key = get_bucketname_key(s3path)\n",
    "    \n",
    "    s3 = boto3.client('s3')    \n",
    "\n",
    "    s3_response_object = s3.get_object(Bucket=bucket, Key=key)\n",
    "    object_content = s3_response_object['Body'].read()\n",
    "    \n",
    "    return len(object_content)\n",
    "\n",
    "\n",
    "\n",
    "def list_files(s3path_prefix):\n",
    "    assert s3path_prefix.startswith(\"s3://\")\n",
    "    assert s3path_prefix.endswith(\"/\")\n",
    "    \n",
    "    bucket, key = get_bucketname_key(s3path_prefix)\n",
    "    \n",
    "   \n",
    "   \n",
    "    s3 = boto3.resource('s3')\n",
    "    \n",
    "    bucket = s3.Bucket(name=bucket)\n",
    "\n",
    "    return ( (o.bucket_name, o.key) for o in bucket.objects.filter(Prefix=key))\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "def upload_files(local_dir, s3_prefix, num_threads=20):    \n",
    "    input_tuples = ( (f,  s3_prefix) for f in glob.glob(\"{}/*\".format(local_dir)))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        pool.starmap(uploadfile, input_tuples)\n",
    "    \n",
    "\n",
    "\n",
    "def download_files(s3_prefix, local_dir, num_threads=20):    \n",
    "    input_tuples = ( (\"s3://{}/{}\".format(s3_bucket,s3_key),  local_dir) for s3_bucket, s3_key in list_files(s3_prefix))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.starmap(download_file, input_tuples)\n",
    "        \n",
    "        \n",
    "\n",
    "def download_objects(s3_prefix, num_threads=20):    \n",
    "    s3_files = ( \"s3://{}/{}\".format(s3_bucket,s3_key) for s3_bucket, s3_key in list_files(s3_prefix))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.map(download_object, s3_files)\n",
    "        \n",
    "    return sum(results)/1024\n",
    "        \n",
    "\n",
    "def get_directory_size(start_path):\n",
    "    total_size = 0\n",
    "    for dirpath, dirnames, filenames in os.walk(start_path):\n",
    "        for f in filenames:\n",
    "            fp = os.path.join(dirpath, f)\n",
    "            # skip if it is symbolic link\n",
    "            if not os.path.islink(fp):\n",
    "                total_size += os.path.getsize(fp)\n",
    "    return total_size\n",
    "\n",
    "def get_s3file_size(bucket, key):\n",
    "    s3 = boto3.client('s3')\n",
    "    response = s3.head_object(Bucket=bucket, Key=key)\n",
    "    size = response['ContentLength']\n",
    "    return size\n",
    "    \n",
    "def download_files_min_files(s3_prefix, local_dir, min_file_size=310, num_threads=20):    \n",
    "    input_tuples = ( (\"s3://{}/{}\".format(s3_bucket,s3_key),  local_dir) for s3_bucket, s3_key in list_files(s3_prefix) if get_s3file_size(s3_bucket, s3_key) > min_file_size )\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.starmap(download_file, input_tuples)\n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "\n",
    "download_files(s3_prefix, local_temp_pred_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "30\r\n"
     ]
    }
   ],
   "source": [
    "!ls -l $local_temp_dir | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "threshold_config = {\n",
    "    \"acetylation\" : 0.83,\n",
    "    \"deubiquitination\" :0.50,\n",
    "    \"methylation\" :.85,\n",
    "    \"phosphorylation\" : .98,\n",
    "    \"demethylation\" :0.0,\n",
    "    \"dephosphorylation\" :.85,\n",
    "    \"ubiquitination\":0.3\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 5min 48s, sys: 1min 25s, total: 7min 14s\n",
      "Wall time: 8min 44s\n"
     ]
    }
   ],
   "source": [
    "%%time \n",
    "\n",
    "full_df = None\n",
    "total_counts = {}\n",
    "for f in os.listdir(local_temp_pred_dir):\n",
    "    df = pd.read_json(os.path.join(local_temp_pred_dir, f), orient=\"records\", lines=True )\n",
    "    \n",
    "    count_dict = df.groupby([\"predicted\"])[\"predicted\"].count().to_dict()\n",
    "    min_dict = df.groupby([\"predicted\"])[\"predicted_confidence\"].min().to_dict()\n",
    "    \n",
    "    for k in count_dict:\n",
    "        if k not in total_counts:\n",
    "            total_counts[k] = {}\n",
    "        total_counts[k][\"count\"] = total_counts[k].get(\"count\", 0) + count_dict[k]\n",
    "        total_counts[k][\"min\"] = min(total_counts[k].get(\"min\", 1.0) , min_dict[k])\n",
    "\n",
    "\n",
    "\n",
    "    # Filter below threshold items\n",
    "    high_quality_frames = []\n",
    "    for k,t in threshold_config.items():\n",
    "        high_quality_frames.append(df.query(\"predicted == '{}' and predicted_confidence > {}\".format(k, t)))\n",
    "        \n",
    "    high_quality_df = pd.concat(high_quality_frames)\n",
    "    \n",
    "    \n",
    "    \n",
    "    if full_df is None:\n",
    "        full_df = high_quality_df\n",
    "    else:\n",
    "        full_df = pd.concat([high_quality_df, full_df])\n",
    "        \n",
    "    \n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "total_counts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'acetylation': 16,\n",
       " 'demethylation': 1,\n",
       " 'dephosphorylation': 180,\n",
       " 'deubiquitination': 92,\n",
       " 'methylation': 1909,\n",
       " 'phosphorylation': 1252,\n",
       " 'ubiquitination': 36}"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.groupby([\"predicted\"])[\"predicted\"].count().to_dict()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>count</th>\n",
       "      <th>mean</th>\n",
       "      <th>std</th>\n",
       "      <th>min</th>\n",
       "      <th>25%</th>\n",
       "      <th>50%</th>\n",
       "      <th>75%</th>\n",
       "      <th>max</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>predicted</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>acetylation</th>\n",
       "      <td>16.0</td>\n",
       "      <td>0.853635</td>\n",
       "      <td>0.017940</td>\n",
       "      <td>0.834194</td>\n",
       "      <td>0.841396</td>\n",
       "      <td>0.846593</td>\n",
       "      <td>0.860633</td>\n",
       "      <td>0.906370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>demethylation</th>\n",
       "      <td>1.0</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "      <td>0.377643</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>dephosphorylation</th>\n",
       "      <td>180.0</td>\n",
       "      <td>0.898379</td>\n",
       "      <td>0.031899</td>\n",
       "      <td>0.850141</td>\n",
       "      <td>0.871749</td>\n",
       "      <td>0.892653</td>\n",
       "      <td>0.921433</td>\n",
       "      <td>0.962255</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>deubiquitination</th>\n",
       "      <td>92.0</td>\n",
       "      <td>0.560631</td>\n",
       "      <td>0.044044</td>\n",
       "      <td>0.500080</td>\n",
       "      <td>0.522097</td>\n",
       "      <td>0.547255</td>\n",
       "      <td>0.592547</td>\n",
       "      <td>0.695426</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>methylation</th>\n",
       "      <td>1909.0</td>\n",
       "      <td>0.888543</td>\n",
       "      <td>0.023966</td>\n",
       "      <td>0.850032</td>\n",
       "      <td>0.868216</td>\n",
       "      <td>0.886418</td>\n",
       "      <td>0.907715</td>\n",
       "      <td>0.946525</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phosphorylation</th>\n",
       "      <td>1252.0</td>\n",
       "      <td>0.984180</td>\n",
       "      <td>0.002718</td>\n",
       "      <td>0.980010</td>\n",
       "      <td>0.981914</td>\n",
       "      <td>0.983852</td>\n",
       "      <td>0.986243</td>\n",
       "      <td>0.991165</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ubiquitination</th>\n",
       "      <td>36.0</td>\n",
       "      <td>0.315345</td>\n",
       "      <td>0.011990</td>\n",
       "      <td>0.300192</td>\n",
       "      <td>0.304990</td>\n",
       "      <td>0.313962</td>\n",
       "      <td>0.319787</td>\n",
       "      <td>0.347559</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    count      mean       std       min       25%       50%  \\\n",
       "predicted                                                                     \n",
       "acetylation          16.0  0.853635  0.017940  0.834194  0.841396  0.846593   \n",
       "demethylation         1.0  0.377643       NaN  0.377643  0.377643  0.377643   \n",
       "dephosphorylation   180.0  0.898379  0.031899  0.850141  0.871749  0.892653   \n",
       "deubiquitination     92.0  0.560631  0.044044  0.500080  0.522097  0.547255   \n",
       "methylation        1909.0  0.888543  0.023966  0.850032  0.868216  0.886418   \n",
       "phosphorylation    1252.0  0.984180  0.002718  0.980010  0.981914  0.983852   \n",
       "ubiquitination       36.0  0.315345  0.011990  0.300192  0.304990  0.313962   \n",
       "\n",
       "                        75%       max  \n",
       "predicted                              \n",
       "acetylation        0.860633  0.906370  \n",
       "demethylation      0.377643  0.377643  \n",
       "dephosphorylation  0.921433  0.962255  \n",
       "deubiquitination   0.592547  0.695426  \n",
       "methylation        0.907715  0.946525  \n",
       "phosphorylation    0.986243  0.991165  \n",
       "ubiquitination     0.319787  0.347559  "
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.groupby([\"predicted\"])[\"predicted_confidence\"].describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3486, 16)"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [],
   "source": [
    "download_file(s3_data, local_temp_wk_dir)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "data_file = os.path.join(local_temp_wk_dir, s3_data.split(\"/\")[-1])\n",
    "data_training_full_df = pd.read_json(data_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3381, 7)"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_training_full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3486, 16)"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>interactionId</th>\n",
       "      <th>interactionType</th>\n",
       "      <th>isNegative</th>\n",
       "      <th>participants</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>pubmedTitle</th>\n",
       "      <th>pubmedabstract</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1585513</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>false</td>\n",
       "      <td>[{'uniprotid': 'Q10728', 'alias': [['mypt1_rat'], ['Protein phosphatase 1 regulatory subunit 12A'], ['Ppp1r12a'], ['Mbs'], ['Mypt1'], ['Myosin phosphatase-targeting subunit 1'], ['Protein phosphatase myosin-binding subunit'], ['MBSP'], ['Serine/threonine protein phosphatase PP1 smooth muscle regulatory subunit M110'], ['Protein phosphatase subunit 1M']], 'alternative_uniprots': ['Q62937', 'Q9WU33']}, {'uniprotid': 'O43293-2', 'alias': [['o43293-2'], ['Death-associated protein kinase 3'], ['ZIPK-S'], ['DAPK3'], ['ZIPK'], ['DAP-like kinase'], ['MYPT1 kinase'], ['Zipper-interacting protein kinase']], 'alternative_uniprots': []}]</td>\n",
       "      <td>17126281</td>\n",
       "      <td>None</td>\n",
       "      <td>Zipper-interacting protein kinase (ZIP kinase) has been thought to be involved in apoptosis and the C-terminal leucine zipper motif is important for its function. Recent studies have revealed that ZIP kinase also plays a role in regulating myosin phosphorylation. Here, we found novel ZIP kinase isoform in which the C-terminal non-kinase domain containing a leucine zipper is eliminated (hZIPK-S). hZIPK-S binds to myosin phosphatase targeting subunit 1(MYPT1) similar to the long isoform (hZIPK-L). In addition, we found that hZIPK-S as well as hZIPK-L bind to myosin. These results indicate that a leucine zipper is not critical for the binding of ZIP kinase to MYPT1 and myosin. Consistently, hZIPK-S localized with stress-fibers where they co-localized with myosin. The residues 278-311, the C-terminal side of the kinase domain common to the both isoforms, is involved in the binding to MYPT1, while the myosin binding domain is within the kinase domain. These results suggest that the newly found hZIPK-S as well as the long isoform play an important role in the regulation of myosin phosphorylation.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1585516</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>false</td>\n",
       "      <td>[{'uniprotid': 'O43293-1', 'alias': [['o43293-1'], ['Death-associated protein kinase 3'], ['ZIPK-L'], ['DAPK3'], ['ZIPK'], ['DAP-like kinase'], ['MYPT1 kinase'], ['Zipper-interacting protein kinase']], 'alternative_uniprots': []}, {'uniprotid': 'Q10728', 'alias': [['mypt1_rat'], ['Protein phosphatase 1 regulatory subunit 12A'], ['Ppp1r12a'], ['Mbs'], ['Mypt1'], ['Myosin phosphatase-targeting subunit 1'], ['Protein phosphatase myosin-binding subunit'], ['MBSP'], ['Serine/threonine protein phosphatase PP1 smooth muscle regulatory subunit M110'], ['Protein phosphatase subunit 1M']], 'alternative_uniprots': ['Q62937', 'Q9WU33']}]</td>\n",
       "      <td>17126281</td>\n",
       "      <td>None</td>\n",
       "      <td>Zipper-interacting protein kinase (ZIP kinase) has been thought to be involved in apoptosis and the C-terminal leucine zipper motif is important for its function. Recent studies have revealed that ZIP kinase also plays a role in regulating myosin phosphorylation. Here, we found novel ZIP kinase isoform in which the C-terminal non-kinase domain containing a leucine zipper is eliminated (hZIPK-S). hZIPK-S binds to myosin phosphatase targeting subunit 1(MYPT1) similar to the long isoform (hZIPK-L). In addition, we found that hZIPK-S as well as hZIPK-L bind to myosin. These results indicate that a leucine zipper is not critical for the binding of ZIP kinase to MYPT1 and myosin. Consistently, hZIPK-S localized with stress-fibers where they co-localized with myosin. The residues 278-311, the C-terminal side of the kinase domain common to the both isoforms, is involved in the binding to MYPT1, while the myosin binding domain is within the kinase domain. These results suggest that the newly found hZIPK-S as well as the long isoform play an important role in the regulation of myosin phosphorylation.</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   interactionId  interactionType isNegative  \\\n",
       "0        1585513  phosphorylation      false   \n",
       "1        1585516  phosphorylation      false   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                participants  \\\n",
       "0  [{'uniprotid': 'Q10728', 'alias': [['mypt1_rat'], ['Protein phosphatase 1 regulatory subunit 12A'], ['Ppp1r12a'], ['Mbs'], ['Mypt1'], ['Myosin phosphatase-targeting subunit 1'], ['Protein phosphatase myosin-binding subunit'], ['MBSP'], ['Serine/threonine protein phosphatase PP1 smooth muscle regulatory subunit M110'], ['Protein phosphatase subunit 1M']], 'alternative_uniprots': ['Q62937', 'Q9WU33']}, {'uniprotid': 'O43293-2', 'alias': [['o43293-2'], ['Death-associated protein kinase 3'], ['ZIPK-S'], ['DAPK3'], ['ZIPK'], ['DAP-like kinase'], ['MYPT1 kinase'], ['Zipper-interacting protein kinase']], 'alternative_uniprots': []}]   \n",
       "1  [{'uniprotid': 'O43293-1', 'alias': [['o43293-1'], ['Death-associated protein kinase 3'], ['ZIPK-L'], ['DAPK3'], ['ZIPK'], ['DAP-like kinase'], ['MYPT1 kinase'], ['Zipper-interacting protein kinase']], 'alternative_uniprots': []}, {'uniprotid': 'Q10728', 'alias': [['mypt1_rat'], ['Protein phosphatase 1 regulatory subunit 12A'], ['Ppp1r12a'], ['Mbs'], ['Mypt1'], ['Myosin phosphatase-targeting subunit 1'], ['Protein phosphatase myosin-binding subunit'], ['MBSP'], ['Serine/threonine protein phosphatase PP1 smooth muscle regulatory subunit M110'], ['Protein phosphatase subunit 1M']], 'alternative_uniprots': ['Q62937', 'Q9WU33']}]   \n",
       "\n",
       "   pubmedId pubmedTitle  \\\n",
       "0  17126281        None   \n",
       "1  17126281        None   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        pubmedabstract  \n",
       "0  Zipper-interacting protein kinase (ZIP kinase) has been thought to be involved in apoptosis and the C-terminal leucine zipper motif is important for its function. Recent studies have revealed that ZIP kinase also plays a role in regulating myosin phosphorylation. Here, we found novel ZIP kinase isoform in which the C-terminal non-kinase domain containing a leucine zipper is eliminated (hZIPK-S). hZIPK-S binds to myosin phosphatase targeting subunit 1(MYPT1) similar to the long isoform (hZIPK-L). In addition, we found that hZIPK-S as well as hZIPK-L bind to myosin. These results indicate that a leucine zipper is not critical for the binding of ZIP kinase to MYPT1 and myosin. Consistently, hZIPK-S localized with stress-fibers where they co-localized with myosin. The residues 278-311, the C-terminal side of the kinase domain common to the both isoforms, is involved in the binding to MYPT1, while the myosin binding domain is within the kinase domain. These results suggest that the newly found hZIPK-S as well as the long isoform play an important role in the regulation of myosin phosphorylation.  \n",
       "1  Zipper-interacting protein kinase (ZIP kinase) has been thought to be involved in apoptosis and the C-terminal leucine zipper motif is important for its function. Recent studies have revealed that ZIP kinase also plays a role in regulating myosin phosphorylation. Here, we found novel ZIP kinase isoform in which the C-terminal non-kinase domain containing a leucine zipper is eliminated (hZIPK-S). hZIPK-S binds to myosin phosphatase targeting subunit 1(MYPT1) similar to the long isoform (hZIPK-L). In addition, we found that hZIPK-S as well as hZIPK-L bind to myosin. These results indicate that a leucine zipper is not critical for the binding of ZIP kinase to MYPT1 and myosin. Consistently, hZIPK-S localized with stress-fibers where they co-localized with myosin. The residues 278-311, the C-terminal side of the kinase domain common to the both isoforms, is involved in the binding to MYPT1, while the myosin binding domain is within the kinase domain. These results suggest that the newly found hZIPK-S as well as the long isoform play an important role in the regulation of myosin phosphorylation.  "
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_training_full_df.head(n=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>acetylation</th>\n",
       "      <th>confidence_scores</th>\n",
       "      <th>demethylation</th>\n",
       "      <th>dephosphorylation</th>\n",
       "      <th>deubiquitination</th>\n",
       "      <th>methylation</th>\n",
       "      <th>normalised_abstract</th>\n",
       "      <th>other</th>\n",
       "      <th>participant1Id</th>\n",
       "      <th>participant2Id</th>\n",
       "      <th>phosphorylation</th>\n",
       "      <th>predicted</th>\n",
       "      <th>predicted_confidence</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>ubiquitination</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>Fragile X syndrome is one of the most frequent causes of hereditary mental retardation. In the past, its diagnosis depended primarily on cytogenetic demonstration of chromosome fragile site Xq27.3. Recently, the gene FMR-1 has been found responsible for this disease. Here a combined method was used to study fragile X syndrome. A fragment (pP1fr) of DNA was subcloned from pE5.1 by polymerase chain reaction. With this probe, DNA samples from two cytogenetically proved families were analyzed by restriction fragment length polymorphisms. It was demonstrated that EcoRI polymorphism was an easy and accurate method for diagnosis of the fragile X syndrome. To study methylation status of patients, another methylation-sensitive enzyme, BssHII, could be used together with EcoRI. The PstI polymorphism of one family was also studied and showed one kb fragment as normal, and detected more precise changes in length. Prominent mosaicism necessary was characteristic in PstI polymorphism. The DNA diagnosis of fragile X syndrome was a reliable method.</td>\n",
       "      <td>0.804310</td>\n",
       "      <td>{'acetylation': -0.21777061220000002, 'demethylation': -0.4751224399, 'dephosphorylation': -1.4052172422, 'deubiquitination': -0.6653347515, 'methylation': 3.5257004499, 'other': 0.5619400702, 'phosphorylation': -0.8057110734, 'ubiquitination': -0.7540013894}</td>\n",
       "      <td>0.621809</td>\n",
       "      <td>0.245314</td>\n",
       "      <td>0.514101</td>\n",
       "      <td>33.977565</td>\n",
       "      <td>Fragile X syndrome is one of the most frequent causes of hereditary mental retardation. In the past, its diagnosis depended primarily on cytogenetic demonstration of chromosome fragile site Xq27.3. Recently, the gene Q06787 has been found responsible for this disease. Here a combined method was used to study fragile X syndrome. A fragment (pP1fr) of DNA was subcloned from pE5.1 by polymerase chain reaction. With this probe, DNA samples from two cytogenetically proved families were analyzed by restriction fragment length polymorphisms. It was demonstrated that EcoRI polymorphism was an easy and accurate method for diagnosis of the fragile X syndrome. To study methylation status of patients, another methylation-sensitive enzyme, BssHII, could be used together with EcoRI. The P00995 polymorphism of one family was also studied and showed one kb fragment as normal, and detected more precise changes in length. Prominent mosaicism necessary was characteristic in P00995 polymorphism. The DNA diagnosis of fragile X syndrome was a reliable method.</td>\n",
       "      <td>1.754072</td>\n",
       "      <td>P00995</td>\n",
       "      <td>Q06787</td>\n",
       "      <td>0.446770</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.874934</td>\n",
       "      <td>8372665</td>\n",
       "      <td>0.470480</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2783</th>\n",
       "      <td>The experiments reported here document that the tumor suppressor retinoblastoma protein (pRB) plays an important role in the production and maintenance of the terminally differentiated phenotype of muscle cells. We show that pRB inactivation, through either phosphorylation, binding to T antigen, or genetic alteration, inhibits myogenesis. Moreover, inactivation of pRB in terminally differentiated cells allows them to reenter the cell cycle. In addition to its involvement in the myogenic activities of MyoD, pRB is also required for the cell growth-inhibitory activity of this myogenic factor. We also show that pRB and MyoD directly bind to each other, both in vivo and in vitro, through a region that involves the pocket and the basic-helix-loop-helix domains, respectively. All the results obtained are consistent with the proposal that the effects of MyoD on the cell cycle and of pRB on the myogenic pathway result from the direct binding of the two molecules.</td>\n",
       "      <td>0.286923</td>\n",
       "      <td>{'acetylation': -1.2485398501, 'demethylation': -1.7055489540000002, 'dephosphorylation': -1.0236738801, 'deubiquitination': -1.378437835, 'methylation': -1.4621940672, 'other': 1.4431860685, 'phosphorylation': 5.7318912506, 'ubiquitination': -1.392804879}</td>\n",
       "      <td>0.181673</td>\n",
       "      <td>0.359273</td>\n",
       "      <td>0.251972</td>\n",
       "      <td>0.231727</td>\n",
       "      <td>The experiments reported here document that the tumor suppressor retinoblastoma protein (P06400) plays an important role in the production and maintenance of the terminally differentiated phenotype of muscle cells. We show that P06400 inactivation, through either phosphorylation, binding to T antigen, or genetic alteration, inhibits myogenesis. Moreover, inactivation of P06400 in terminally differentiated cells allows them to reenter the cell cycle. In addition to its involvement in the myogenic activities of P15172, P06400 is also required for the cell growth-inhibitory activity of this myogenic factor. We also show that P06400 and P15172 directly bind to each other, both in vivo and in vitro, through a region that involves the pocket and the basic-helix-loop-helix domains, respectively. All the results obtained are consistent with the proposal that the effects of P15172 on the cell cycle and of P06400 on the myogenic pathway result from the direct binding of the two molecules.</td>\n",
       "      <td>4.234165</td>\n",
       "      <td>P06400</td>\n",
       "      <td>P15172</td>\n",
       "      <td>308.552267</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.981568</td>\n",
       "      <td>8381715</td>\n",
       "      <td>0.248378</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      abstract  \\\n",
       "395   Fragile X syndrome is one of the most frequent causes of hereditary mental retardation. In the past, its diagnosis depended primarily on cytogenetic demonstration of chromosome fragile site Xq27.3. Recently, the gene FMR-1 has been found responsible for this disease. Here a combined method was used to study fragile X syndrome. A fragment (pP1fr) of DNA was subcloned from pE5.1 by polymerase chain reaction. With this probe, DNA samples from two cytogenetically proved families were analyzed by restriction fragment length polymorphisms. It was demonstrated that EcoRI polymorphism was an easy and accurate method for diagnosis of the fragile X syndrome. To study methylation status of patients, another methylation-sensitive enzyme, BssHII, could be used together with EcoRI. The PstI polymorphism of one family was also studied and showed one kb fragment as normal, and detected more precise changes in length. Prominent mosaicism necessary was characteristic in PstI polymorphism. The DNA diagnosis of fragile X syndrome was a reliable method.   \n",
       "2783                                                                                 The experiments reported here document that the tumor suppressor retinoblastoma protein (pRB) plays an important role in the production and maintenance of the terminally differentiated phenotype of muscle cells. We show that pRB inactivation, through either phosphorylation, binding to T antigen, or genetic alteration, inhibits myogenesis. Moreover, inactivation of pRB in terminally differentiated cells allows them to reenter the cell cycle. In addition to its involvement in the myogenic activities of MyoD, pRB is also required for the cell growth-inhibitory activity of this myogenic factor. We also show that pRB and MyoD directly bind to each other, both in vivo and in vitro, through a region that involves the pocket and the basic-helix-loop-helix domains, respectively. All the results obtained are consistent with the proposal that the effects of MyoD on the cell cycle and of pRB on the myogenic pathway result from the direct binding of the two molecules.   \n",
       "\n",
       "      acetylation  \\\n",
       "395      0.804310   \n",
       "2783     0.286923   \n",
       "\n",
       "                                                                                                                                                                                                                                                        confidence_scores  \\\n",
       "395   {'acetylation': -0.21777061220000002, 'demethylation': -0.4751224399, 'dephosphorylation': -1.4052172422, 'deubiquitination': -0.6653347515, 'methylation': 3.5257004499, 'other': 0.5619400702, 'phosphorylation': -0.8057110734, 'ubiquitination': -0.7540013894}   \n",
       "2783     {'acetylation': -1.2485398501, 'demethylation': -1.7055489540000002, 'dephosphorylation': -1.0236738801, 'deubiquitination': -1.378437835, 'methylation': -1.4621940672, 'other': 1.4431860685, 'phosphorylation': 5.7318912506, 'ubiquitination': -1.392804879}   \n",
       "\n",
       "      demethylation  dephosphorylation  deubiquitination  methylation  \\\n",
       "395        0.621809           0.245314          0.514101    33.977565   \n",
       "2783       0.181673           0.359273          0.251972     0.231727   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                normalised_abstract  \\\n",
       "395   Fragile X syndrome is one of the most frequent causes of hereditary mental retardation. In the past, its diagnosis depended primarily on cytogenetic demonstration of chromosome fragile site Xq27.3. Recently, the gene Q06787 has been found responsible for this disease. Here a combined method was used to study fragile X syndrome. A fragment (pP1fr) of DNA was subcloned from pE5.1 by polymerase chain reaction. With this probe, DNA samples from two cytogenetically proved families were analyzed by restriction fragment length polymorphisms. It was demonstrated that EcoRI polymorphism was an easy and accurate method for diagnosis of the fragile X syndrome. To study methylation status of patients, another methylation-sensitive enzyme, BssHII, could be used together with EcoRI. The P00995 polymorphism of one family was also studied and showed one kb fragment as normal, and detected more precise changes in length. Prominent mosaicism necessary was characteristic in P00995 polymorphism. The DNA diagnosis of fragile X syndrome was a reliable method.   \n",
       "2783                                                              The experiments reported here document that the tumor suppressor retinoblastoma protein (P06400) plays an important role in the production and maintenance of the terminally differentiated phenotype of muscle cells. We show that P06400 inactivation, through either phosphorylation, binding to T antigen, or genetic alteration, inhibits myogenesis. Moreover, inactivation of P06400 in terminally differentiated cells allows them to reenter the cell cycle. In addition to its involvement in the myogenic activities of P15172, P06400 is also required for the cell growth-inhibitory activity of this myogenic factor. We also show that P06400 and P15172 directly bind to each other, both in vivo and in vitro, through a region that involves the pocket and the basic-helix-loop-helix domains, respectively. All the results obtained are consistent with the proposal that the effects of P15172 on the cell cycle and of P06400 on the myogenic pathway result from the direct binding of the two molecules.   \n",
       "\n",
       "         other participant1Id participant2Id  phosphorylation  \\\n",
       "395   1.754072         P00995         Q06787         0.446770   \n",
       "2783  4.234165         P06400         P15172       308.552267   \n",
       "\n",
       "            predicted  predicted_confidence  pubmedId  ubiquitination  \n",
       "395       methylation              0.874934   8372665        0.470480  \n",
       "2783  phosphorylation              0.981568   8381715        0.248378  "
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.head(n=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3253, 16)"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df[~full_df.pubmedId.isin(data_training_full_df.pubmedId)].shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_df[\"PubmedInTrainingData\"] = full_df.pubmedId.isin(data_training_full_df.pubmedId)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "pd.set_option('display.max_columns', None)\n",
    "pd.set_option('display.max_colwidth', 10000)\n",
    "pd.set_option('display.max_rows', 100)\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "plt.rcParams.update({'font.size': 12})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_df.to_csv(\"predictions.csv\", index=False, header=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>normalised_abstract</th>\n",
       "      <th>participant1Id</th>\n",
       "      <th>participant2Id</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>predicted</th>\n",
       "      <th>predicted_confidence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>20286</th>\n",
       "      <td>The mitotic cell cycle can be described as an alternation between two states. During mitosis, MPF (mitosis promoting factor) is high and keeps inactive its numerous molecular antagonists. In interphase, MPF is inactivated, and the antagonists prevail. The transition between the two states is ensured by 'helper' molecules that favor one state over the other. It has long been assumed that active MPF (a dimer of cyclin B and cyclin-dependent kinase 1) induces exit from mitosis by activating APC:Cdc20, a ubiquitin ligase responsible for cyclin B degradation. The molecular details have not been fully worked out yet, but recent results show that MPF and the ubiquitin ligase are not involved in a simple negative feedback loop. While it is proven that MPF activates APC, new data suggest that MPF inhibits Cdc20, i.e., that MPF and Cdc20 are antagonists. We introduce this new idea into a published model for cell cycle regulation in Xenopus laevis, and study its dynamical behavior. We show that the new wiring permits oscillations with a simpler and smaller network than previously envisaged and that the antagonism between MPF and Cdc20 suggests a new interpretation of the spindle checkpoint.</td>\n",
       "      <td>The mitotic cell cycle can be described as an alternation between two states. During mitosis, Q13421 (Q13421) is high and keeps inactive its numerous molecular antagonists. In interphase, Q13421 is inactivated, and the antagonists prevail. The transition between the two states is ensured by 'helper' molecules that favor one state over the other. It has long been assumed that active Q13421 (a dimer of cyclin B and P06493) induces exit from mitosis by activating APC:Q12834, a ubiquitin ligase responsible for cyclin B degradation. The molecular details have not been fully worked out yet, but recent results show that Q13421 and the ubiquitin ligase are not involved in a simple negative feedback loop. While it is proven that Q13421 activates APC, new data suggest that Q13421 inhibits Q12834, i.e., that Q13421 and Q12834 are antagonists. We introduce this new idea into a published model for cell cycle regulation in Xenopus laevis, and study its dynamical behavior. We show that the new wiring permits oscillations with a simpler and smaller network than previously envisaged and that the antagonism between Q13421 and Q12834 suggests a new interpretation of the spindle checkpoint.</td>\n",
       "      <td>Q12834</td>\n",
       "      <td>Q13421</td>\n",
       "      <td>15970669</td>\n",
       "      <td>ubiquitination</td>\n",
       "      <td>0.305353</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>895</th>\n",
       "      <td>The retinoblastoma binding protein KDM5A removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of KDM5A contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, KDM5A contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in KDM5A have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the PHD1 preferentially recognizes unmethylated H3K4 histone tail, product of KDM5A-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the PHD1 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in KDM5A--suggests a model for the spread of demethylation on chromatin.</td>\n",
       "      <td>The retinoblastoma binding protein P29375 removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of P29375 contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, P29375 contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in P29375 have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the Q96KS0 preferentially recognizes unmethylated H3K4 histone tail, product of P29375-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the Q96KS0 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in P29375--suggests a model for the spread of demethylation on chromatin.</td>\n",
       "      <td>P29375</td>\n",
       "      <td>Q96KS0</td>\n",
       "      <td>25686748</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.866325</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1462</th>\n",
       "      <td>Protein arginine methylation has been recently identified as an important form of post-translational modification (PTM). It is carried out by the protein arginine methyltransferase (PRMT) family of enzymes, which in mammals consists of nine members. Among them, PRMT1 is the major arginine methyltransferase and participates in transcription, signal transduction, development and cancer. The function of PRMT1 in craniofacial development remains unclear. We generated Wnt1-Cre;Prmt1</td>\n",
       "      <td>Protein arginine methylation has been recently identified as an important form of post-translational modification (PTM). It is carried out by the protein arginine methyltransferase (PRMT) family of enzymes, which in mammals consists of nine members. Among them, Q99873 is the major arginine methyltransferase and participates in transcription, signal transduction, development and cancer. The function of Q99873 in craniofacial development remains unclear. We generated P04628-Cre;Prmt1</td>\n",
       "      <td>P04628</td>\n",
       "      <td>Q99873</td>\n",
       "      <td>29727702</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.909696</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2895</th>\n",
       "      <td>MDM2 is an E3 ubiquitin ligase which mediates ubiquitylation and proteasome-dependent degradation of the p53 tumor suppressor protein. Phosphorylation of MDM2 by the protein kinase AKT is thought to regulate MDM2 function in response to survival signals, but there has been uncertainty concerning the identity of the sites phosphorylated by AKT. In the present study, we identify Ser-166, a site previously reported as an AKT target, and Ser-188, a novel site which is the major site of phosphorylation of MDM2 by AKT in vitro. Analysis of MDM2 in cultured cells confirms that Ser-166 and Ser-188 are phosphorylated by AKT in a physiological context.</td>\n",
       "      <td>Q00987 is an E3 ubiquitin ligase which mediates ubiquitylation and proteasome-dependent degradation of the P04637 tumor suppressor protein. Phosphorylation of Q00987 by the protein kinase P31749 is thought to regulate Q00987 function in response to survival signals, but there has been uncertainty concerning the identity of the sites phosphorylated by P31749. In the present study, we identify Ser-166, a site previously reported as an P31749 target, and Ser-188, a novel site which is the major site of phosphorylation of Q00987 by P31749 in vitro. Analysis of Q00987 in cultured cells confirms that Ser-166 and Ser-188 are phosphorylated by P31749 in a physiological context.</td>\n",
       "      <td>P31749</td>\n",
       "      <td>Q00987</td>\n",
       "      <td>15527798</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.988527</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4929</th>\n",
       "      <td>Homeobox (HOX) transcript antisense RNA (HOTAIR), a long intergenic noncoding RNA (lincRNA), has been reported to play an oncogenic role in various cancers including small cell lung cancer (SCLC). However, it is not known whether HOTAIR can modulate chemoresistance in SCLC. The aim of this study is to investigate the roles of HOTAIR in chemoresistance of SCLC and its possible molecular mechanism. Knockdown of HOTAIR was carried out in SCLC multidrug-resistant cell lines (H69AR and H446AR) and the parental cell lines (H69 and H446) to assess its influence on chemoresistance. The results showed that downregulation of HOTAIR increased cell sensitivity to anticancer drugs through increasing cell apoptosis and cell cycle arrest, and suppressed tumor growth in vivo. Moreover, HOXA1 methylation increased in the resistant cells using bisulfite sequencing PCR. Depletion of HOTAIR reduced HOXA1 methylation by decreasing DNMT1 and DNMT3b expression. The interaction between HOTAIR and HOXA1 was validated by RNA immunoprecipitation. Taken together, our study suggested that HOTAIR mediates chemoresistance of SCLC by regulating HOXA1 methylation and could be utilized as a potential target for new adjuvant therapies against chemoresistance.</td>\n",
       "      <td>Homeobox (HOX) transcript antisense RNA (100124700), a long intergenic noncoding RNA (lincRNA), has been reported to play an oncogenic role in various cancers including small cell lung cancer (7864). However, it is not known whether 100124700 can modulate chemoresistance in 7864. The aim of this study is to investigate the roles of 100124700 in chemoresistance of 7864 and its possible molecular mechanism. Knockdown of 100124700 was carried out in 7864 multidrug-resistant cell lines (H69AR and H446AR) and the parental cell lines (H69 and H446) to assess its influence on chemoresistance. The results showed that downregulation of 100124700 increased cell sensitivity to anticancer drugs through increasing cell apoptosis and cell cycle arrest, and suppressed tumor growth in vivo. Moreover, P49639 methylation increased in the resistant cells using bisulfite sequencing PCR. Depletion of 100124700 reduced P49639 methylation by decreasing P26358 and Q9UBC3 expression. The interaction between 100124700 and P49639 was validated by RNA immunoprecipitation. Taken together, our study suggested that 100124700 mediates chemoresistance of 7864 by regulating P49639 methylation and could be utilized as a potential target for new adjuvant therapies against chemoresistance.</td>\n",
       "      <td>100124700</td>\n",
       "      <td>P49639</td>\n",
       "      <td>26707824</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.914474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12847</th>\n",
       "      <td>Here we demonstrate that the dual leucine zipper kinase (DLK) promotes robust regeneration of peripheral axons after nerve injury in mice. Peripheral axon regeneration is accelerated by prior injury; however, DLK KO neurons do not respond to a preconditioning lesion with enhanced regeneration invivo or invitro. Assays for activation of transcription factors in injury-induced proregenerative pathways reveal that loss of DLK abolishes upregulation of p-STAT3 and p-cJun in the cell body after axonal injury. DLK is not required for the phosphorylation of STAT3 at the site of nerve injury but is necessary for retrograde transport of p-STAT3 to the cell body. These data demonstrate that DLK enhances regeneration by promoting a retrograde injury signal that is required for the activation of the neuronal proregenerative program.</td>\n",
       "      <td>Here we demonstrate that the Q12852 (Q12852) promotes robust regeneration of peripheral axons after nerve injury in mice. Peripheral axon regeneration is accelerated by prior injury; however, Q12852 KO neurons do not respond to a preconditioning lesion with enhanced regeneration invivo or invitro. Assays for activation of transcription factors in injury-induced proregenerative pathways reveal that loss of Q12852 abolishes upregulation of p-P40763 and p-cJun in the cell body after axonal injury. Q12852 is not required for the phosphorylation of P40763 at the site of nerve injury but is necessary for retrograde transport of p-P40763 to the cell body. These data demonstrate that Q12852 enhances regeneration by promoting a retrograde injury signal that is required for the activation of the neuronal proregenerative program.</td>\n",
       "      <td>P40763</td>\n",
       "      <td>Q12852</td>\n",
       "      <td>22726832</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.983138</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2687</th>\n",
       "      <td>MTA2 is a member of metastasis associated family, which is highly expressed in several solid tumors and associated with tumor cells migration and invasion. Here, we report that MTA2 is acetylated at K152 and histone acetyltransferase p300 binds to and acetylates MTA2. Furthermore, mutation of the MTA2 acetylation site inhibits the growth of colorectal cancer cells and migration and invasion of Rat1 fibroblasts. These results reveal a novel post-translational regulation of MTA2 by the way of p300-dependent acetylation, which is important for tumor cells growth and migration and provides a potential target for clinical cancer research.</td>\n",
       "      <td>O94776 is a member of metastasis associated family, which is highly expressed in several solid tumors and associated with tumor cells migration and invasion. Here, we report that O94776 is acetylated at K152 and histone acetyltransferase Q09472 binds to and acetylates O94776. Furthermore, mutation of the O94776 acetylation site inhibits the growth of colorectal cancer cells and migration and invasion of Rat1 fibroblasts. These results reveal a novel post-translational regulation of O94776 by the way of Q09472-dependent acetylation, which is important for tumor cells growth and migration and provides a potential target for clinical cancer research.</td>\n",
       "      <td>O94776</td>\n",
       "      <td>Q09472</td>\n",
       "      <td>24468085</td>\n",
       "      <td>acetylation</td>\n",
       "      <td>0.834194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>Insulin has both short- and long-term effects on cellular metabolism. The short-term effects are known to involve the insulin receptor, a protein kinase capable of phosphorylating itself and other proteins. The role of the receptor was elucidated by studies of a mutant insulin receptor which lacked kinase activity and inhibited several actions of insulin. The long-term effects of insulin could be demonstrated by its growth-promoting effect on hepatoma cells, and by the suppression in transfected hepatoma cells of hepatitis B virus antigen production in a dose-dependent manner. The process whereby insulin appears to regulate gene expression is not clearly understood.</td>\n",
       "      <td>P01308 has both short- and long-term effects on cellular metabolism. The short-term effects are known to involve the P06213, a protein kinase capable of phosphorylating itself and other proteins. The role of the receptor was elucidated by studies of a mutant P06213 which lacked kinase activity and inhibited several actions of P01308. The long-term effects of P01308 could be demonstrated by its growth-promoting effect on hepatoma cells, and by the suppression in transfected hepatoma cells of hepatitis B virus antigen production in a dose-dependent manner. The process whereby P01308 appears to regulate gene expression is not clearly understood.</td>\n",
       "      <td>P01308</td>\n",
       "      <td>P06213</td>\n",
       "      <td>2184955</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.982965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11845</th>\n",
       "      <td>Protein arginine methyltransferase 5 (PRMT5) has been implicated as a key modulator of lymphomagenesis. Whether PRMT5 has overt oncogenic function in the context of leukemia/lymphoma and whether it represents a therapeutic target remains to be established. We demonstrate that inactivation of PRMT5 inhibits colony-forming activity by multiple oncogenic drivers, including cyclin D1, c-MYC, NOTCH1, and MLL-AF9. Furthermore, we demonstrate that PRMT5 overexpression specifically cooperates with cyclin D1 to drive lymphomagenesis in a mouse model, revealing inherent neoplastic activity. Molecular analysis of lymphomas revealed that arginine methylation of p53 selectively suppresses expression of crucial proapoptotic and antiproliferative target genes, thereby sustaining tumor cell self-renewal and proliferation and bypassing the need for the acquisition of inactivating p53 mutations. Critically, analysis of human tumor specimens reveals a strong correlation between cyclin D1 overexpression and p53 methylation, supporting the biomedical relevance of this pathway.</td>\n",
       "      <td>O14744 (O14744) has been implicated as a key modulator of lymphomagenesis. Whether O14744 has overt oncogenic function in the context of leukemia/lymphoma and whether it represents a therapeutic target remains to be established. We demonstrate that inactivation of O14744 inhibits colony-forming activity by multiple oncogenic drivers, including P24385, P01106, P46531, and Q03164. Furthermore, we demonstrate that O14744 overexpression specifically cooperates with P24385 to drive lymphomagenesis in a mouse model, revealing inherent neoplastic activity. Molecular analysis of lymphomas revealed that arginine methylation of P04637 selectively suppresses expression of crucial proapoptotic and antiproliferative target genes, thereby sustaining tumor cell self-renewal and proliferation and bypassing the need for the acquisition of inactivating P04637 mutations. Critically, analysis of human tumor specimens reveals a strong correlation between P24385 overexpression and P04637 methylation, supporting the biomedical relevance of this pathway.</td>\n",
       "      <td>O14744</td>\n",
       "      <td>P04637</td>\n",
       "      <td>25582697</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.901107</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1617</th>\n",
       "      <td>Gastric cancer (GC) is one of the most common malignant tumors in the world and microRNAs (miRNAs) play an important role in GC. In this study, we found miR497 played an important role and served as a novel biomarker in GC. Quantitative real-time PCR (qRT-PCR) was used to measure the miR497 expression in GC cell lines and 86paired GC samples and we also analyzed its correlation with GC clinicopathological parameters. A series of cellular function experiments were applied to validate the effects of miR497 on GC. In addition, methylation-specific PCR (MSP) was applied to detect the gene methylation status. Finally, the correlation between miR497 and the target gene was analyzed by western blotting assay. miR497 was reduced obviously in GC cells and tissues and significantly associated with the pathologic stage. Low expression of miR497 significantly inhibited the proliferation, invasion and migration of GC cell lines and accelerated apoptosis. Moreover, we found that the aberrant expression of miR497 may be ascribed to DNA methylation. microRNA.org and luciferase reporter assay suggested that RAF1 was a direct target of miR497 in GC. This study suggested that miR497 could serve as a tumor suppressor and a potential early diagnostic marker of GC by targeting Raf-1 proto-oncogene, serine/threonine kinase (RAF1).</td>\n",
       "      <td>Gastric cancer (GC) is one of the most common malignant tumors in the world and microRNAs (miRNAs) play an important role in GC. In this study, we found 574456 played an important role and served as a novel biomarker in GC. Quantitative real-time PCR (qRT-PCR) was used to measure the 574456 expression in GC cell lines and 86paired GC samples and we also analyzed its correlation with GC clinicopathological parameters. A series of cellular function experiments were applied to validate the effects of 574456 on GC. In addition, methylation-specific PCR (MSP) was applied to detect the gene methylation status. Finally, the correlation between 574456 and the target gene was analyzed by western blotting assay. 574456 was reduced obviously in GC cells and tissues and significantly associated with the pathologic stage. Low expression of 574456 significantly inhibited the proliferation, invasion and migration of GC cell lines and accelerated apoptosis. Moreover, we found that the aberrant expression of 574456 may be ascribed to DNA methylation. microRNA.org and luciferase reporter assay suggested that P04049 was a direct target of 574456 in GC. This study suggested that 574456 could serve as a tumor suppressor and a potential early diagnostic marker of GC by targeting P04049 proto-oncogene, serine/threonine kinase (P04049).</td>\n",
       "      <td>574456</td>\n",
       "      <td>P04049</td>\n",
       "      <td>28586056</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.885787</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>As molecular biologists, we are continuing to unravel the interactions by which DNA binding proteins mediate the expression of genes. The chromatin immunoprecipitation (ChIP) technique provides us with an exquisite tool to investigate the interplay between chromatin structure and its role in regulating transcription, replication, and recombination in vivo. We describe a robust assay used to identify the molecular determinants associated with chromatin. In this article we illustrate the ChIP technique and use the transcriptionally silent-hypermethylated multidrug resistance (MDR1) gene as the platform for methyl-CpG binding protein 2 (MeCP2) localization on chromatin. Driven by the hypothesis that repression is strongly dependent on the methylation profile of the endogenous promoter, we demonstrate that MDR1 is targeted by MeCP2. Methylated MDR1 chromatin is highly enriched with MeCP2 and is in striking contrast to localization observed in cells in which MDR1 is transcriptionally active. In a distinct model system we discuss experimental methods used to immunopurify the MeCP2 repressor complex on chromatin and quantify protein-DNA association by competitive PCR approach.</td>\n",
       "      <td>As molecular biologists, we are continuing to unravel the interactions by which DNA binding proteins mediate the expression of genes. The chromatin immunoprecipitation (ChIP) technique provides us with an exquisite tool to investigate the interplay between chromatin structure and its role in regulating transcription, replication, and recombination in vivo. We describe a robust assay used to identify the molecular determinants associated with chromatin. In this article we illustrate the ChIP technique and use the transcriptionally silent-hypermethylated multidrug resistance (P08183) gene as the platform for P51608 (P51608) localization on chromatin. Driven by the hypothesis that repression is strongly dependent on the methylation profile of the endogenous promoter, we demonstrate that P08183 is targeted by P51608. Methylated P08183 chromatin is highly enriched with P51608 and is in striking contrast to localization observed in cells in which P08183 is transcriptionally active. In a distinct model system we discuss experimental methods used to immunopurify the P51608 repressor complex on chromatin and quantify protein-DNA association by competitive PCR approach.</td>\n",
       "      <td>P08183</td>\n",
       "      <td>P51608</td>\n",
       "      <td>11726209</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.878734</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5042</th>\n",
       "      <td>In this study we analyzed the interaction of prion protein PrP(C) with components of glycosphingolipid-enriched microdomains in lymphoblastoid T cells. PrP(C) was distributed in small clusters on the plasma membrane, as revealed by immunoelectron microscopy. PrP(C) is present in microdomains, since it coimmunoprecipitates with GM3 and the raft marker GM1. A strict association between PrP(C) and Fyn was revealed by scanning confocal microscopy and coimmunoprecipitation experiments. The phosphorylation protein ZAP-70 was immunoprecipitated by anti-PrP after T cell activation. These results demonstrate that PrP(C) interacts with ZAP-70, suggesting that PrP(C) is a component of the multimolecular signaling complex within microdomains involved in T cell activation.</td>\n",
       "      <td>In this study we analyzed the interaction of prion protein F7VJQ1 with components of glycosphingolipid-enriched microdomains in lymphoblastoid T cells. F7VJQ1 was distributed in small clusters on the plasma membrane, as revealed by immunoelectron microscopy. F7VJQ1 is present in microdomains, since it coimmunoprecipitates with GM3 and the raft marker GM1. A strict association between F7VJQ1 and P06241 was revealed by scanning confocal microscopy and coimmunoprecipitation experiments. The phosphorylation protein P43403 was immunoprecipitated by anti-PrP after T cell activation. These results demonstrate that F7VJQ1 interacts with P43403, suggesting that F7VJQ1 is a component of the multimolecular signaling complex within microdomains involved in T cell activation.</td>\n",
       "      <td>F7VJQ1</td>\n",
       "      <td>P43403</td>\n",
       "      <td>14987990</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.983814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6620</th>\n",
       "      <td>Encephalopathy represents a common and serious manifestation of HIV-1 infection in children, but its pathogenesis is unclear. We demonstrated that gp120 activated human brain microvascular endothelial cells (HBMEC) derived from children in up-regulating ICAM-1 and VCAM-1 expression, IL-6 secretion and increased monocyte transmigration across monolayers. Another novel observation was our demonstration of CD4 in isolated HBMEC and on microvessels of children's brain cryosections. Gp120-induced monocyte migration was inhibited by anti-gp120 and anti-CD4 antibodies. This is the first demonstration that gp120 activates HBMEC via CD4, which may contribute to the development of HIV-1 encephalopathy in children.</td>\n",
       "      <td>Encephalopathy represents a common and serious manifestation of HIV-1 infection in children, but its pathogenesis is unclear. We demonstrated that Q14624 activated human brain microvascular endothelial cells (HBMEC) derived from children in up-regulating P05362 and P19320 expression, P05231 secretion and increased monocyte transmigration across monolayers. Another novel observation was our demonstration of P01730 in isolated HBMEC and on microvessels of children's brain cryosections. Q14624-induced monocyte migration was inhibited by anti-Q14624 and anti-P01730 antibodies. This is the first demonstration that Q14624 activates HBMEC via P01730, which may contribute to the development of HIV-1 encephalopathy in children.</td>\n",
       "      <td>P01730</td>\n",
       "      <td>Q14624</td>\n",
       "      <td>11517385</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.986022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16332</th>\n",
       "      <td>Native protein kinase C delta from porcine spleen is phosphorylated in vitro by the tyrosine kinase src and to a much smaller extent by fyn. The tyrosine phosphorylation of PKC delta is restricted to the activated state of the enzyme, i.e. it occurs only in the presence of an activator, such as TPA or bryostatin. Upon phosphorylation at tyrosine, the apparent molecular weight of PKC delta increases by 6 kDa. Phosphorylation by src induces a stimulation of PKC delta activity apparently exhibiting some substrate selectivity. Other PKC isoenzymes, such as cPKC (alpha, beta, gamma), are not phosphorylated by src or only to a very small extent. This phosphorylation is not dependent on TPA and does not cause an increase in activity and molecular weight of the enzyme.</td>\n",
       "      <td>Native protein kinase C delta from porcine spleen is phosphorylated in vitro by the tyrosine kinase P12931 and to a much smaller extent by fyn. The tyrosine phosphorylation of Q05655 is restricted to the activated state of the enzyme, i.e. it occurs only in the presence of an activator, such as TPA or bryostatin. Upon phosphorylation at tyrosine, the apparent molecular weight of Q05655 increases by 6 kDa. Phosphorylation by P12931 induces a stimulation of Q05655 activity apparently exhibiting some substrate selectivity. Other PKC isoenzymes, such as cPKC (alpha, beta, gamma), are not phosphorylated by P12931 or only to a very small extent. This phosphorylation is not dependent on TPA and does not cause an increase in activity and molecular weight of the enzyme.</td>\n",
       "      <td>P12931</td>\n",
       "      <td>Q05655</td>\n",
       "      <td>7516899</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.985279</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4863</th>\n",
       "      <td>Preeclampsia is a unique multiple system disorder during human pregnancy, which affects 5% to 8% of pregnancies. Its risks and complications have become the major causes of maternal and fetal morbidity and mortality. Although abnormal placentation to which DNA methylation dysregulation is always linked is speculated to be one of the reasons causing preeclampsia, the underlying mechanisms still remain elusive to date. Here we revealed that aberrant DNA methyltransferase 3A (DNMT3A) plays a critical role in preeclampsia. Our results show that the expression and localization of DNMT3A are dysregulated in preeclamptic placenta. Moreover, knockdown of DNMT3A obviously inhibits trophoblast cell migration and invasion. Mechanistically, IGFBP5 (insulin-like growth factor-binding protein 5), known as a suppressor, is upregulated by decreased DNMT3A because of promoter hypomethylation. Importantly, IGFBP5 downregulation can rescue the defects caused by DNMT3A knockdown, thereby, consolidating the significance of IGFBP5 in the downstream of DNMT3A in trophoblast. Furthermore, we detected low promoter methylation and high protein expression of IGFBP5 in the clinical samples of preeclamptic placenta. Collectively, our study suggests that dysregulation of DNMT3A and IGFBP5 is relevant to preeclampsia. Thus, we propose that DNMT3A and IGFBP5 can serve as potential markers and targets for the clinical diagnosis and therapy of preeclampsia.</td>\n",
       "      <td>Preeclampsia is a unique multiple system disorder during human pregnancy, which affects 5% to 8% of pregnancies. Its risks and complications have become the major causes of maternal and fetal morbidity and mortality. Although abnormal placentation to which DNA methylation dysregulation is always linked is speculated to be one of the reasons causing preeclampsia, the underlying mechanisms still remain elusive to date. Here we revealed that aberrant Q9Y6K1 (Q9Y6K1) plays a critical role in preeclampsia. Our results show that the expression and localization of Q9Y6K1 are dysregulated in preeclamptic placenta. Moreover, knockdown of Q9Y6K1 obviously inhibits trophoblast cell migration and invasion. Mechanistically, P24593 (P24593), known as a suppressor, is upregulated by decreased Q9Y6K1 because of promoter hypomethylation. Importantly, P24593 downregulation can rescue the defects caused by Q9Y6K1 knockdown, thereby, consolidating the significance of P24593 in the downstream of Q9Y6K1 in trophoblast. Furthermore, we detected low promoter methylation and high protein expression of P24593 in the clinical samples of preeclamptic placenta. Collectively, our study suggests that dysregulation of Q9Y6K1 and P24593 is relevant to preeclampsia. Thus, we propose that Q9Y6K1 and P24593 can serve as potential markers and targets for the clinical diagnosis and therapy of preeclampsia.</td>\n",
       "      <td>P24593</td>\n",
       "      <td>Q9Y6K1</td>\n",
       "      <td>28049695</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.885323</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6082</th>\n",
       "      <td>In mammals, the Golgi apparatus is disassembled at early mitosis and reassembled at the end of mitosis. For Golgi disassembly, membrane fusion needs to be blocked. Golgi biogenesis requires two distinct p97ATPase-mediated membrane fusion, the p97/p47 and p97/p37 pathways. We previously reported that p47 phosphorylation on Serine-140 by Cdc2 results in mitotic inhibition of the p97/p47 pathway [11]. In this study, we demonstrate that p37 is phosphorylated on Serine-56 and Threonine-59 by Cdc2 at mitosis, and this phosphorylated p37 does not bind to Golgi membranes. Using an in vitro Golgi reassembly assay, we show that mutated p37(S56D, T59D), which mimics mitotic phosphorylation, does not cause any cisternal regrowth, indicating that p37 phosphorylation inhibits the p97/p37 pathway. Our results demonstrate that p37 phosphorylation on Serine-56 and Threonine-59 is important for Golgi disassembly at mitosis.</td>\n",
       "      <td>In mammals, the Golgi apparatus is disassembled at early mitosis and reassembled at the end of mitosis. For Golgi disassembly, membrane fusion needs to be blocked. Golgi biogenesis requires two distinct p97ATPase-mediated membrane fusion, the P55072/Q9UNZ2 and P55072/Q8NFH4 pathways. We previously reported that Q9UNZ2 phosphorylation on Serine-140 by P06493 results in mitotic inhibition of the P55072/Q9UNZ2 pathway [11]. In this study, we demonstrate that Q8NFH4 is phosphorylated on Serine-56 and Threonine-59 by P06493 at mitosis, and this phosphorylated Q8NFH4 does not bind to Golgi membranes. Using an in vitro Golgi reassembly assay, we show that mutated Q8NFH4(S56D, T59D), which mimics mitotic phosphorylation, does not cause any cisternal regrowth, indicating that Q8NFH4 phosphorylation inhibits the P55072/Q8NFH4 pathway. Our results demonstrate that Q8NFH4 phosphorylation on Serine-56 and Threonine-59 is important for Golgi disassembly at mitosis.</td>\n",
       "      <td>P06493</td>\n",
       "      <td>Q8NFH4</td>\n",
       "      <td>20875789</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.986205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5958</th>\n",
       "      <td>POP2 protein of Saccharomyces cerevisiae is a component of a protein complex that regulates the transcription of many genes. We found that the 97th threonine residue (Thr 97) of Pop2p was phosphorylated upon glucose limitation. The Thr 97 phosphorylation occurred within 2 min after removing glucose and was reversed within 1 min after the readdition of glucose. The effects of hexokinase mutations and glucose analogs indicate that this phosphorylation is dependent on glucose phosphorylating activity. We purified a protein kinase that phosphorylates a peptide containing Thr 97 of Pop2p and identified it as Yak1p, a DYRK family kinase. Phosphorylation of Pop2p was barely detectable in a yak1Delta strain. We found that Yak1p interacted with Bmh1p and Bmh2p only in the presence of glucose. A GFP-Yak1p fusion protein shuttled rapidly between the nucleus and the cytoplasm in response to glucose. A strain with alanine substituted for Thr 97 in Pop2p showed overgrowth in the postdiauxic transition and failed to stop the cell cycle at G(1) phase in response to glucose deprivation. Thus, Yak1p and Pop2p are part of a novel glucose-sensing system in yeast that is involved in growth control in response to glucose availability.</td>\n",
       "      <td>Q56P42 protein of Saccharomyces cerevisiae is a component of a protein complex that regulates the transcription of many genes. We found that the 97th threonine residue (Thr 97) of Q56P42 was phosphorylated upon glucose limitation. The Thr 97 phosphorylation occurred within 2 min after removing glucose and was reversed within 1 min after the readdition of glucose. The effects of hexokinase mutations and glucose analogs indicate that this phosphorylation is dependent on glucose phosphorylating activity. We purified a protein kinase that phosphorylates a peptide containing Thr 97 of Q56P42 and identified it as Q9H422, a DYRK family kinase. Phosphorylation of Q56P42 was barely detectable in a yak1Delta strain. We found that Q9H422 interacted with Bmh1p and Bmh2p only in the presence of glucose. A GFP-Q9H422 fusion protein shuttled rapidly between the nucleus and the cytoplasm in response to glucose. A strain with alanine substituted for Thr 97 in Q56P42 showed overgrowth in the postdiauxic transition and failed to stop the cell cycle at G(1) phase in response to glucose deprivation. Thus, Q9H422 and Q56P42 are part of a novel glucose-sensing system in yeast that is involved in growth control in response to glucose availability.</td>\n",
       "      <td>Q56P42</td>\n",
       "      <td>Q9H422</td>\n",
       "      <td>11358866</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.985220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Many cellular proteins are methylated on lysine residues and this has been most intensively studied for histone proteins. Lysine methylations on non-histone proteins are also frequent, but in most cases the functional significance of the methylation event, as well as the identity of the responsible lysine (K) specific methyltransferase (KMT), remain unknown. Several recently discovered KMTs belong to the so-called seven--strand (7BS) class of MTases and we have here investigated an uncharacterized human 7BS MTase currently annotated as part of the endothelin converting enzyme 2, but which should be considered a separate enzyme. Combining in vitro enzymology and analyzes of knockout cells, we demonstrate that this MTase efficiently methylates K36 in eukaryotic translation elongation factor 1 alpha (eEF1A) in vitro and in vivo. We suggest that this novel KMT is named eEF1A-KMT4 (gene name EEF1AKMT4), in agreement with the recently established nomenclature. Furthermore, by ribosome profiling we show that the absence of K36 methylation affects translation dynamics and changes translation speed of distinct codons. Finally, we show that eEF1A-KMT4 is part of a novel family of human KMTs, defined by a shared sequence motif in the active site and we demonstrate the importance of this motif for catalytic activity.</td>\n",
       "      <td>Many cellular proteins are methylated on lysine residues and this has been most intensively studied for histone proteins. Lysine methylations on non-histone proteins are also frequent, but in most cases the functional significance of the methylation event, as well as the identity of the responsible lysine (K) specific methyltransferase (KMT), remain unknown. Several recently discovered KMTs belong to the so-called seven--strand (7BS) class of MTases and we have here investigated an uncharacterized human 7BS MTase currently annotated as part of the P0DPD6, but which should be considered a separate enzyme. Combining in vitro enzymology and analyzes of knockout cells, we demonstrate that this MTase efficiently methylates O76013 in eukaryotic translation elongation factor 1 alpha (P68104) in vitro and in vivo. We suggest that this novel KMT is named P68104-Q8TEK3 (gene name EEF1AKMT4), in agreement with the recently established nomenclature. Furthermore, by ribosome profiling we show that the absence of O76013 methylation affects translation dynamics and changes translation speed of distinct codons. Finally, we show that P68104-Q8TEK3 is part of a novel family of human KMTs, defined by a shared sequence motif in the active site and we demonstrate the importance of this motif for catalytic activity.</td>\n",
       "      <td>O76013</td>\n",
       "      <td>P68104</td>\n",
       "      <td>28520920</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.871510</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1837</th>\n",
       "      <td>Colorectal cancer (CRC) remains one of the most common cancers worldwide. Increasing evidence indicates that SPRY4 intronic transcript 1 (SPRY4-IT1) regulate cell growth, differentiation, apoptosis, and cancer progression. However, the expression and function of SPRY4-IT1 in the progression of CRC remains largely unknown. Here, we reported that SPRY4-IT1 was upregulated in CRC. Increased SPRY4-IT1 expression in CRC was associated with larger tumor size and higher clinical stage. In vitro experiments revealed that SPRY4-IT1 knockdown significantly inhibited CRC cell proliferation by causing G1 arrest and promoting apoptosis, whereas SPRY4-IT1 overexpression promoted cell proliferation. Further functional assays indicated that SPRY4-IT1 overexpression significantly promoted cell migration and invasion by regulate the epithelial-mesenchymal transition (EMT). Taken together, our study demonstrates that SPRY4-IT1 could act as a functional oncogene in CRC, as well as a potential therapeutic target to inhibit CRC metastasis.</td>\n",
       "      <td>Colorectal cancer (CRC) remains one of the most common cancers worldwide. Increasing evidence indicates that 100642175 (100642175;81848;79441) regulate cell growth, differentiation, apoptosis, and cancer progression. However, the expression and function of 100642175;81848;79441 in the progression of CRC remains largely unknown. Here, we reported that 100642175;81848;79441 was upregulated in CRC. Increased Q9C004-Q68CZ6 expression in CRC was associated with larger tumor size and higher clinical stage. In vitro experiments revealed that 100642175;81848;79441 knockdown significantly inhibited CRC cell proliferation by causing G1 arrest and promoting apoptosis, whereas 100642175;81848;79441 overexpression promoted cell proliferation. Further functional assays indicated that 100642175;81848;79441 overexpression significantly promoted cell migration and invasion by regulate the epithelial-mesenchymal transition (EMT). Taken together, our study demonstrates that 100642175;81848;79441 could act as a functional oncogene in CRC, as well as a potential therapeutic target to inhibit CRC metastasis.</td>\n",
       "      <td>100642175</td>\n",
       "      <td>100642175;81848;79441</td>\n",
       "      <td>27391336</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.986137</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10207</th>\n",
       "      <td>Although lysine methylation is classically known to regulate histone function, its role in modulating antiviral restriction factor activity remains uncharacterized. Interferon-induced transmembrane protein 3 (IFITM3) was found monomethylated on its lysine 88 residue (IFITM3-K88me1) to reduce its antiviral activity, mediated by the lysine methyltransferase SET7. Vesicular stomatitis virus and influenza A virus infection increased IFITM3-K88me1 levels by promoting the interaction between IFITM3 and SET7, suggesting that this pathway could be hijacked to support infection; conversely, IFN- reduced IFITM3-K88me1 levels. These findings may have important implications in the design of therapeutics targeting protein methylation against infectious diseases.</td>\n",
       "      <td>Although lysine methylation is classically known to regulate histone function, its role in modulating antiviral restriction factor activity remains uncharacterized. Q01628 (Q01628) was found monomethylated on its lysine 88 residue (Q01628-K88me1) to reduce its antiviral activity, mediated by the lysine methyltransferase 57108. Vesicular stomatitis virus and influenza A virus infection increased Q01628-K88me1 levels by promoting the interaction between Q01628 and 57108, suggesting that this pathway could be hijacked to support infection; conversely, P01562 reduced Q01628-K88me1 levels. These findings may have important implications in the design of therapeutics targeting protein methylation against infectious diseases.</td>\n",
       "      <td>P01562</td>\n",
       "      <td>Q01628</td>\n",
       "      <td>24129573</td>\n",
       "      <td>methylation</td>\n",
       "      <td>0.942059</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      abstract  \\\n",
       "20286                                                                                                                                                                                                                                                           The mitotic cell cycle can be described as an alternation between two states. During mitosis, MPF (mitosis promoting factor) is high and keeps inactive its numerous molecular antagonists. In interphase, MPF is inactivated, and the antagonists prevail. The transition between the two states is ensured by 'helper' molecules that favor one state over the other. It has long been assumed that active MPF (a dimer of cyclin B and cyclin-dependent kinase 1) induces exit from mitosis by activating APC:Cdc20, a ubiquitin ligase responsible for cyclin B degradation. The molecular details have not been fully worked out yet, but recent results show that MPF and the ubiquitin ligase are not involved in a simple negative feedback loop. While it is proven that MPF activates APC, new data suggest that MPF inhibits Cdc20, i.e., that MPF and Cdc20 are antagonists. We introduce this new idea into a published model for cell cycle regulation in Xenopus laevis, and study its dynamical behavior. We show that the new wiring permits oscillations with a simpler and smaller network than previously envisaged and that the antagonism between MPF and Cdc20 suggests a new interpretation of the spindle checkpoint.   \n",
       "895                                                                                                                                                                                                                                                                                                                                                                                                             The retinoblastoma binding protein KDM5A removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of KDM5A contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, KDM5A contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in KDM5A have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the PHD1 preferentially recognizes unmethylated H3K4 histone tail, product of KDM5A-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the PHD1 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in KDM5A--suggests a model for the spread of demethylation on chromatin.   \n",
       "1462                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Protein arginine methylation has been recently identified as an important form of post-translational modification (PTM). It is carried out by the protein arginine methyltransferase (PRMT) family of enzymes, which in mammals consists of nine members. Among them, PRMT1 is the major arginine methyltransferase and participates in transcription, signal transduction, development and cancer. The function of PRMT1 in craniofacial development remains unclear. We generated Wnt1-Cre;Prmt1   \n",
       "2895                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                MDM2 is an E3 ubiquitin ligase which mediates ubiquitylation and proteasome-dependent degradation of the p53 tumor suppressor protein. Phosphorylation of MDM2 by the protein kinase AKT is thought to regulate MDM2 function in response to survival signals, but there has been uncertainty concerning the identity of the sites phosphorylated by AKT. In the present study, we identify Ser-166, a site previously reported as an AKT target, and Ser-188, a novel site which is the major site of phosphorylation of MDM2 by AKT in vitro. Analysis of MDM2 in cultured cells confirms that Ser-166 and Ser-188 are phosphorylated by AKT in a physiological context.   \n",
       "4929                                                                                                                                                                                                             Homeobox (HOX) transcript antisense RNA (HOTAIR), a long intergenic noncoding RNA (lincRNA), has been reported to play an oncogenic role in various cancers including small cell lung cancer (SCLC). However, it is not known whether HOTAIR can modulate chemoresistance in SCLC. The aim of this study is to investigate the roles of HOTAIR in chemoresistance of SCLC and its possible molecular mechanism. Knockdown of HOTAIR was carried out in SCLC multidrug-resistant cell lines (H69AR and H446AR) and the parental cell lines (H69 and H446) to assess its influence on chemoresistance. The results showed that downregulation of HOTAIR increased cell sensitivity to anticancer drugs through increasing cell apoptosis and cell cycle arrest, and suppressed tumor growth in vivo. Moreover, HOXA1 methylation increased in the resistant cells using bisulfite sequencing PCR. Depletion of HOTAIR reduced HOXA1 methylation by decreasing DNMT1 and DNMT3b expression. The interaction between HOTAIR and HOXA1 was validated by RNA immunoprecipitation. Taken together, our study suggested that HOTAIR mediates chemoresistance of SCLC by regulating HOXA1 methylation and could be utilized as a potential target for new adjuvant therapies against chemoresistance.    \n",
       "12847                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Here we demonstrate that the dual leucine zipper kinase (DLK) promotes robust regeneration of peripheral axons after nerve injury in mice. Peripheral axon regeneration is accelerated by prior injury; however, DLK KO neurons do not respond to a preconditioning lesion with enhanced regeneration invivo or invitro. Assays for activation of transcription factors in injury-induced proregenerative pathways reveal that loss of DLK abolishes upregulation of p-STAT3 and p-cJun in the cell body after axonal injury. DLK is not required for the phosphorylation of STAT3 at the site of nerve injury but is necessary for retrograde transport of p-STAT3 to the cell body. These data demonstrate that DLK enhances regeneration by promoting a retrograde injury signal that is required for the activation of the neuronal proregenerative program.   \n",
       "2687                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        MTA2 is a member of metastasis associated family, which is highly expressed in several solid tumors and associated with tumor cells migration and invasion. Here, we report that MTA2 is acetylated at K152 and histone acetyltransferase p300 binds to and acetylates MTA2. Furthermore, mutation of the MTA2 acetylation site inhibits the growth of colorectal cancer cells and migration and invasion of Rat1 fibroblasts. These results reveal a novel post-translational regulation of MTA2 by the way of p300-dependent acetylation, which is important for tumor cells growth and migration and provides a potential target for clinical cancer research.    \n",
       "147                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Insulin has both short- and long-term effects on cellular metabolism. The short-term effects are known to involve the insulin receptor, a protein kinase capable of phosphorylating itself and other proteins. The role of the receptor was elucidated by studies of a mutant insulin receptor which lacked kinase activity and inhibited several actions of insulin. The long-term effects of insulin could be demonstrated by its growth-promoting effect on hepatoma cells, and by the suppression in transfected hepatoma cells of hepatitis B virus antigen production in a dose-dependent manner. The process whereby insulin appears to regulate gene expression is not clearly understood.   \n",
       "11845                                                                                                                                                                                                                                                                                                                                                                                         Protein arginine methyltransferase 5 (PRMT5) has been implicated as a key modulator of lymphomagenesis. Whether PRMT5 has overt oncogenic function in the context of leukemia/lymphoma and whether it represents a therapeutic target remains to be established. We demonstrate that inactivation of PRMT5 inhibits colony-forming activity by multiple oncogenic drivers, including cyclin D1, c-MYC, NOTCH1, and MLL-AF9. Furthermore, we demonstrate that PRMT5 overexpression specifically cooperates with cyclin D1 to drive lymphomagenesis in a mouse model, revealing inherent neoplastic activity. Molecular analysis of lymphomas revealed that arginine methylation of p53 selectively suppresses expression of crucial proapoptotic and antiproliferative target genes, thereby sustaining tumor cell self-renewal and proliferation and bypassing the need for the acquisition of inactivating p53 mutations. Critically, analysis of human tumor specimens reveals a strong correlation between cyclin D1 overexpression and p53 methylation, supporting the biomedical relevance of this pathway.   \n",
       "1617                                                                                                                         Gastric cancer (GC) is one of the most common malignant tumors in the world and microRNAs (miRNAs) play an important role in GC. In this study, we found miR497 played an important role and served as a novel biomarker in GC. Quantitative real-time PCR (qRT-PCR) was used to measure the miR497 expression in GC cell lines and 86paired GC samples and we also analyzed its correlation with GC clinicopathological parameters. A series of cellular function experiments were applied to validate the effects of miR497 on GC. In addition, methylation-specific PCR (MSP) was applied to detect the gene methylation status. Finally, the correlation between miR497 and the target gene was analyzed by western blotting assay. miR497 was reduced obviously in GC cells and tissues and significantly associated with the pathologic stage. Low expression of miR497 significantly inhibited the proliferation, invasion and migration of GC cell lines and accelerated apoptosis. Moreover, we found that the aberrant expression of miR497 may be ascribed to DNA methylation. microRNA.org and luciferase reporter assay suggested that RAF1 was a direct target of miR497 in GC. This study suggested that miR497 could serve as a tumor suppressor and a potential early diagnostic marker of GC by targeting Raf-1 proto-oncogene, serine/threonine kinase (RAF1).   \n",
       "185                                                                                                                                                                                                                                                                       As molecular biologists, we are continuing to unravel the interactions by which DNA binding proteins mediate the expression of genes. The chromatin immunoprecipitation (ChIP) technique provides us with an exquisite tool to investigate the interplay between chromatin structure and its role in regulating transcription, replication, and recombination in vivo. We describe a robust assay used to identify the molecular determinants associated with chromatin. In this article we illustrate the ChIP technique and use the transcriptionally silent-hypermethylated multidrug resistance (MDR1) gene as the platform for methyl-CpG binding protein 2 (MeCP2) localization on chromatin. Driven by the hypothesis that repression is strongly dependent on the methylation profile of the endogenous promoter, we demonstrate that MDR1 is targeted by MeCP2. Methylated MDR1 chromatin is highly enriched with MeCP2 and is in striking contrast to localization observed in cells in which MDR1 is transcriptionally active. In a distinct model system we discuss experimental methods used to immunopurify the MeCP2 repressor complex on chromatin and quantify protein-DNA association by competitive PCR approach.   \n",
       "5042                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        In this study we analyzed the interaction of prion protein PrP(C) with components of glycosphingolipid-enriched microdomains in lymphoblastoid T cells. PrP(C) was distributed in small clusters on the plasma membrane, as revealed by immunoelectron microscopy. PrP(C) is present in microdomains, since it coimmunoprecipitates with GM3 and the raft marker GM1. A strict association between PrP(C) and Fyn was revealed by scanning confocal microscopy and coimmunoprecipitation experiments. The phosphorylation protein ZAP-70 was immunoprecipitated by anti-PrP after T cell activation. These results demonstrate that PrP(C) interacts with ZAP-70, suggesting that PrP(C) is a component of the multimolecular signaling complex within microdomains involved in T cell activation.   \n",
       "6620                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Encephalopathy represents a common and serious manifestation of HIV-1 infection in children, but its pathogenesis is unclear. We demonstrated that gp120 activated human brain microvascular endothelial cells (HBMEC) derived from children in up-regulating ICAM-1 and VCAM-1 expression, IL-6 secretion and increased monocyte transmigration across monolayers. Another novel observation was our demonstration of CD4 in isolated HBMEC and on microvessels of children's brain cryosections. Gp120-induced monocyte migration was inhibited by anti-gp120 and anti-CD4 antibodies. This is the first demonstration that gp120 activates HBMEC via CD4, which may contribute to the development of HIV-1 encephalopathy in children.   \n",
       "16332                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      Native protein kinase C delta from porcine spleen is phosphorylated in vitro by the tyrosine kinase src and to a much smaller extent by fyn. The tyrosine phosphorylation of PKC delta is restricted to the activated state of the enzyme, i.e. it occurs only in the presence of an activator, such as TPA or bryostatin. Upon phosphorylation at tyrosine, the apparent molecular weight of PKC delta increases by 6 kDa. Phosphorylation by src induces a stimulation of PKC delta activity apparently exhibiting some substrate selectivity. Other PKC isoenzymes, such as cPKC (alpha, beta, gamma), are not phosphorylated by src or only to a very small extent. This phosphorylation is not dependent on TPA and does not cause an increase in activity and molecular weight of the enzyme.   \n",
       "4863   Preeclampsia is a unique multiple system disorder during human pregnancy, which affects 5% to 8% of pregnancies. Its risks and complications have become the major causes of maternal and fetal morbidity and mortality. Although abnormal placentation to which DNA methylation dysregulation is always linked is speculated to be one of the reasons causing preeclampsia, the underlying mechanisms still remain elusive to date. Here we revealed that aberrant DNA methyltransferase 3A (DNMT3A) plays a critical role in preeclampsia. Our results show that the expression and localization of DNMT3A are dysregulated in preeclamptic placenta. Moreover, knockdown of DNMT3A obviously inhibits trophoblast cell migration and invasion. Mechanistically, IGFBP5 (insulin-like growth factor-binding protein 5), known as a suppressor, is upregulated by decreased DNMT3A because of promoter hypomethylation. Importantly, IGFBP5 downregulation can rescue the defects caused by DNMT3A knockdown, thereby, consolidating the significance of IGFBP5 in the downstream of DNMT3A in trophoblast. Furthermore, we detected low promoter methylation and high protein expression of IGFBP5 in the clinical samples of preeclamptic placenta. Collectively, our study suggests that dysregulation of DNMT3A and IGFBP5 is relevant to preeclampsia. Thus, we propose that DNMT3A and IGFBP5 can serve as potential markers and targets for the clinical diagnosis and therapy of preeclampsia.   \n",
       "6082                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   In mammals, the Golgi apparatus is disassembled at early mitosis and reassembled at the end of mitosis. For Golgi disassembly, membrane fusion needs to be blocked. Golgi biogenesis requires two distinct p97ATPase-mediated membrane fusion, the p97/p47 and p97/p37 pathways. We previously reported that p47 phosphorylation on Serine-140 by Cdc2 results in mitotic inhibition of the p97/p47 pathway [11]. In this study, we demonstrate that p37 is phosphorylated on Serine-56 and Threonine-59 by Cdc2 at mitosis, and this phosphorylated p37 does not bind to Golgi membranes. Using an in vitro Golgi reassembly assay, we show that mutated p37(S56D, T59D), which mimics mitotic phosphorylation, does not cause any cisternal regrowth, indicating that p37 phosphorylation inhibits the p97/p37 pathway. Our results demonstrate that p37 phosphorylation on Serine-56 and Threonine-59 is important for Golgi disassembly at mitosis.   \n",
       "5958                                                                                                                                                                                                                          POP2 protein of Saccharomyces cerevisiae is a component of a protein complex that regulates the transcription of many genes. We found that the 97th threonine residue (Thr 97) of Pop2p was phosphorylated upon glucose limitation. The Thr 97 phosphorylation occurred within 2 min after removing glucose and was reversed within 1 min after the readdition of glucose. The effects of hexokinase mutations and glucose analogs indicate that this phosphorylation is dependent on glucose phosphorylating activity. We purified a protein kinase that phosphorylates a peptide containing Thr 97 of Pop2p and identified it as Yak1p, a DYRK family kinase. Phosphorylation of Pop2p was barely detectable in a yak1Delta strain. We found that Yak1p interacted with Bmh1p and Bmh2p only in the presence of glucose. A GFP-Yak1p fusion protein shuttled rapidly between the nucleus and the cytoplasm in response to glucose. A strain with alanine substituted for Thr 97 in Pop2p showed overgrowth in the postdiauxic transition and failed to stop the cell cycle at G(1) phase in response to glucose deprivation. Thus, Yak1p and Pop2p are part of a novel glucose-sensing system in yeast that is involved in growth control in response to glucose availability.   \n",
       "23                                                                                                                              Many cellular proteins are methylated on lysine residues and this has been most intensively studied for histone proteins. Lysine methylations on non-histone proteins are also frequent, but in most cases the functional significance of the methylation event, as well as the identity of the responsible lysine (K) specific methyltransferase (KMT), remain unknown. Several recently discovered KMTs belong to the so-called seven--strand (7BS) class of MTases and we have here investigated an uncharacterized human 7BS MTase currently annotated as part of the endothelin converting enzyme 2, but which should be considered a separate enzyme. Combining in vitro enzymology and analyzes of knockout cells, we demonstrate that this MTase efficiently methylates K36 in eukaryotic translation elongation factor 1 alpha (eEF1A) in vitro and in vivo. We suggest that this novel KMT is named eEF1A-KMT4 (gene name EEF1AKMT4), in agreement with the recently established nomenclature. Furthermore, by ribosome profiling we show that the absence of K36 methylation affects translation dynamics and changes translation speed of distinct codons. Finally, we show that eEF1A-KMT4 is part of a novel family of human KMTs, defined by a shared sequence motif in the active site and we demonstrate the importance of this motif for catalytic activity.   \n",
       "1837                                                                                                                                                                                                                                                                                                                                                                                                                                 Colorectal cancer (CRC) remains one of the most common cancers worldwide. Increasing evidence indicates that SPRY4 intronic transcript 1 (SPRY4-IT1) regulate cell growth, differentiation, apoptosis, and cancer progression. However, the expression and function of SPRY4-IT1 in the progression of CRC remains largely unknown. Here, we reported that SPRY4-IT1 was upregulated in CRC. Increased SPRY4-IT1 expression in CRC was associated with larger tumor size and higher clinical stage. In vitro experiments revealed that SPRY4-IT1 knockdown significantly inhibited CRC cell proliferation by causing G1 arrest and promoting apoptosis, whereas SPRY4-IT1 overexpression promoted cell proliferation. Further functional assays indicated that SPRY4-IT1 overexpression significantly promoted cell migration and invasion by regulate the epithelial-mesenchymal transition (EMT). Taken together, our study demonstrates that SPRY4-IT1 could act as a functional oncogene in CRC, as well as a potential therapeutic target to inhibit CRC metastasis.   \n",
       "10207                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Although lysine methylation is classically known to regulate histone function, its role in modulating antiviral restriction factor activity remains uncharacterized. Interferon-induced transmembrane protein 3 (IFITM3) was found monomethylated on its lysine 88 residue (IFITM3-K88me1) to reduce its antiviral activity, mediated by the lysine methyltransferase SET7. Vesicular stomatitis virus and influenza A virus infection increased IFITM3-K88me1 levels by promoting the interaction between IFITM3 and SET7, suggesting that this pathway could be hijacked to support infection; conversely, IFN- reduced IFITM3-K88me1 levels. These findings may have important implications in the design of therapeutics targeting protein methylation against infectious diseases.    \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   normalised_abstract  \\\n",
       "20286                                                                                                                                                                                                            The mitotic cell cycle can be described as an alternation between two states. During mitosis, Q13421 (Q13421) is high and keeps inactive its numerous molecular antagonists. In interphase, Q13421 is inactivated, and the antagonists prevail. The transition between the two states is ensured by 'helper' molecules that favor one state over the other. It has long been assumed that active Q13421 (a dimer of cyclin B and P06493) induces exit from mitosis by activating APC:Q12834, a ubiquitin ligase responsible for cyclin B degradation. The molecular details have not been fully worked out yet, but recent results show that Q13421 and the ubiquitin ligase are not involved in a simple negative feedback loop. While it is proven that Q13421 activates APC, new data suggest that Q13421 inhibits Q12834, i.e., that Q13421 and Q12834 are antagonists. We introduce this new idea into a published model for cell cycle regulation in Xenopus laevis, and study its dynamical behavior. We show that the new wiring permits oscillations with a simpler and smaller network than previously envisaged and that the antagonism between Q13421 and Q12834 suggests a new interpretation of the spindle checkpoint.   \n",
       "895                                                                                                                                                                                                                                                                                                                                           The retinoblastoma binding protein P29375 removes methyl marks from lysine 4 of histone H3 (H3K4). Misregulation of P29375 contributes to the pathogenesis of lung and gastric cancers. In addition to its catalytic jumonji C domain, P29375 contains three PHD reader domains, commonly recognized as chromatin recruitment modules. It is unknown whether any of these domains in P29375 have functions beyond recruitment and whether they regulate the catalytic activity of the demethylase. Here using biochemical and nuclear magnetic resonance (NMR)-based structural studies, we show that the Q96KS0 preferentially recognizes unmethylated H3K4 histone tail, product of P29375-mediated demethylation of tri-methylated H3K4 (H3K4me3). Binding of unmodified H3 peptide to the Q96KS0 stimulates catalytic domain-mediated removal of methyl marks from H3K4me3 peptide and nucleosome substrates. This positive-feedback mechanism--enabled by the functional coupling between a reader and a catalytic domain in P29375--suggests a model for the spread of demethylation on chromatin.   \n",
       "1462                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Protein arginine methylation has been recently identified as an important form of post-translational modification (PTM). It is carried out by the protein arginine methyltransferase (PRMT) family of enzymes, which in mammals consists of nine members. Among them, Q99873 is the major arginine methyltransferase and participates in transcription, signal transduction, development and cancer. The function of Q99873 in craniofacial development remains unclear. We generated P04628-Cre;Prmt1   \n",
       "2895                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Q00987 is an E3 ubiquitin ligase which mediates ubiquitylation and proteasome-dependent degradation of the P04637 tumor suppressor protein. Phosphorylation of Q00987 by the protein kinase P31749 is thought to regulate Q00987 function in response to survival signals, but there has been uncertainty concerning the identity of the sites phosphorylated by P31749. In the present study, we identify Ser-166, a site previously reported as an P31749 target, and Ser-188, a novel site which is the major site of phosphorylation of Q00987 by P31749 in vitro. Analysis of Q00987 in cultured cells confirms that Ser-166 and Ser-188 are phosphorylated by P31749 in a physiological context.   \n",
       "4929                                                                                                                        Homeobox (HOX) transcript antisense RNA (100124700), a long intergenic noncoding RNA (lincRNA), has been reported to play an oncogenic role in various cancers including small cell lung cancer (7864). However, it is not known whether 100124700 can modulate chemoresistance in 7864. The aim of this study is to investigate the roles of 100124700 in chemoresistance of 7864 and its possible molecular mechanism. Knockdown of 100124700 was carried out in 7864 multidrug-resistant cell lines (H69AR and H446AR) and the parental cell lines (H69 and H446) to assess its influence on chemoresistance. The results showed that downregulation of 100124700 increased cell sensitivity to anticancer drugs through increasing cell apoptosis and cell cycle arrest, and suppressed tumor growth in vivo. Moreover, P49639 methylation increased in the resistant cells using bisulfite sequencing PCR. Depletion of 100124700 reduced P49639 methylation by decreasing P26358 and Q9UBC3 expression. The interaction between 100124700 and P49639 was validated by RNA immunoprecipitation. Taken together, our study suggested that 100124700 mediates chemoresistance of 7864 by regulating P49639 methylation and could be utilized as a potential target for new adjuvant therapies against chemoresistance.    \n",
       "12847                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   Here we demonstrate that the Q12852 (Q12852) promotes robust regeneration of peripheral axons after nerve injury in mice. Peripheral axon regeneration is accelerated by prior injury; however, Q12852 KO neurons do not respond to a preconditioning lesion with enhanced regeneration invivo or invitro. Assays for activation of transcription factors in injury-induced proregenerative pathways reveal that loss of Q12852 abolishes upregulation of p-P40763 and p-cJun in the cell body after axonal injury. Q12852 is not required for the phosphorylation of P40763 at the site of nerve injury but is necessary for retrograde transport of p-P40763 to the cell body. These data demonstrate that Q12852 enhances regeneration by promoting a retrograde injury signal that is required for the activation of the neuronal proregenerative program.   \n",
       "2687                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  O94776 is a member of metastasis associated family, which is highly expressed in several solid tumors and associated with tumor cells migration and invasion. Here, we report that O94776 is acetylated at K152 and histone acetyltransferase Q09472 binds to and acetylates O94776. Furthermore, mutation of the O94776 acetylation site inhibits the growth of colorectal cancer cells and migration and invasion of Rat1 fibroblasts. These results reveal a novel post-translational regulation of O94776 by the way of Q09472-dependent acetylation, which is important for tumor cells growth and migration and provides a potential target for clinical cancer research.    \n",
       "147                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         P01308 has both short- and long-term effects on cellular metabolism. The short-term effects are known to involve the P06213, a protein kinase capable of phosphorylating itself and other proteins. The role of the receptor was elucidated by studies of a mutant P06213 which lacked kinase activity and inhibited several actions of P01308. The long-term effects of P01308 could be demonstrated by its growth-promoting effect on hepatoma cells, and by the suppression in transfected hepatoma cells of hepatitis B virus antigen production in a dose-dependent manner. The process whereby P01308 appears to regulate gene expression is not clearly understood.   \n",
       "11845                                                                                                                                                                                                                                                                                                                                                           O14744 (O14744) has been implicated as a key modulator of lymphomagenesis. Whether O14744 has overt oncogenic function in the context of leukemia/lymphoma and whether it represents a therapeutic target remains to be established. We demonstrate that inactivation of O14744 inhibits colony-forming activity by multiple oncogenic drivers, including P24385, P01106, P46531, and Q03164. Furthermore, we demonstrate that O14744 overexpression specifically cooperates with P24385 to drive lymphomagenesis in a mouse model, revealing inherent neoplastic activity. Molecular analysis of lymphomas revealed that arginine methylation of P04637 selectively suppresses expression of crucial proapoptotic and antiproliferative target genes, thereby sustaining tumor cell self-renewal and proliferation and bypassing the need for the acquisition of inactivating P04637 mutations. Critically, analysis of human tumor specimens reveals a strong correlation between P24385 overexpression and P04637 methylation, supporting the biomedical relevance of this pathway.   \n",
       "1617                                                            Gastric cancer (GC) is one of the most common malignant tumors in the world and microRNAs (miRNAs) play an important role in GC. In this study, we found 574456 played an important role and served as a novel biomarker in GC. Quantitative real-time PCR (qRT-PCR) was used to measure the 574456 expression in GC cell lines and 86paired GC samples and we also analyzed its correlation with GC clinicopathological parameters. A series of cellular function experiments were applied to validate the effects of 574456 on GC. In addition, methylation-specific PCR (MSP) was applied to detect the gene methylation status. Finally, the correlation between 574456 and the target gene was analyzed by western blotting assay. 574456 was reduced obviously in GC cells and tissues and significantly associated with the pathologic stage. Low expression of 574456 significantly inhibited the proliferation, invasion and migration of GC cell lines and accelerated apoptosis. Moreover, we found that the aberrant expression of 574456 may be ascribed to DNA methylation. microRNA.org and luciferase reporter assay suggested that P04049 was a direct target of 574456 in GC. This study suggested that 574456 could serve as a tumor suppressor and a potential early diagnostic marker of GC by targeting P04049 proto-oncogene, serine/threonine kinase (P04049).   \n",
       "185                                                                                                                                                                                                                         As molecular biologists, we are continuing to unravel the interactions by which DNA binding proteins mediate the expression of genes. The chromatin immunoprecipitation (ChIP) technique provides us with an exquisite tool to investigate the interplay between chromatin structure and its role in regulating transcription, replication, and recombination in vivo. We describe a robust assay used to identify the molecular determinants associated with chromatin. In this article we illustrate the ChIP technique and use the transcriptionally silent-hypermethylated multidrug resistance (P08183) gene as the platform for P51608 (P51608) localization on chromatin. Driven by the hypothesis that repression is strongly dependent on the methylation profile of the endogenous promoter, we demonstrate that P08183 is targeted by P51608. Methylated P08183 chromatin is highly enriched with P51608 and is in striking contrast to localization observed in cells in which P08183 is transcriptionally active. In a distinct model system we discuss experimental methods used to immunopurify the P51608 repressor complex on chromatin and quantify protein-DNA association by competitive PCR approach.   \n",
       "5042                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             In this study we analyzed the interaction of prion protein F7VJQ1 with components of glycosphingolipid-enriched microdomains in lymphoblastoid T cells. F7VJQ1 was distributed in small clusters on the plasma membrane, as revealed by immunoelectron microscopy. F7VJQ1 is present in microdomains, since it coimmunoprecipitates with GM3 and the raft marker GM1. A strict association between F7VJQ1 and P06241 was revealed by scanning confocal microscopy and coimmunoprecipitation experiments. The phosphorylation protein P43403 was immunoprecipitated by anti-PrP after T cell activation. These results demonstrate that F7VJQ1 interacts with P43403, suggesting that F7VJQ1 is a component of the multimolecular signaling complex within microdomains involved in T cell activation.   \n",
       "6620                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Encephalopathy represents a common and serious manifestation of HIV-1 infection in children, but its pathogenesis is unclear. We demonstrated that Q14624 activated human brain microvascular endothelial cells (HBMEC) derived from children in up-regulating P05362 and P19320 expression, P05231 secretion and increased monocyte transmigration across monolayers. Another novel observation was our demonstration of P01730 in isolated HBMEC and on microvessels of children's brain cryosections. Q14624-induced monocyte migration was inhibited by anti-Q14624 and anti-P01730 antibodies. This is the first demonstration that Q14624 activates HBMEC via P01730, which may contribute to the development of HIV-1 encephalopathy in children.   \n",
       "16332                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Native protein kinase C delta from porcine spleen is phosphorylated in vitro by the tyrosine kinase P12931 and to a much smaller extent by fyn. The tyrosine phosphorylation of Q05655 is restricted to the activated state of the enzyme, i.e. it occurs only in the presence of an activator, such as TPA or bryostatin. Upon phosphorylation at tyrosine, the apparent molecular weight of Q05655 increases by 6 kDa. Phosphorylation by P12931 induces a stimulation of Q05655 activity apparently exhibiting some substrate selectivity. Other PKC isoenzymes, such as cPKC (alpha, beta, gamma), are not phosphorylated by P12931 or only to a very small extent. This phosphorylation is not dependent on TPA and does not cause an increase in activity and molecular weight of the enzyme.   \n",
       "4863   Preeclampsia is a unique multiple system disorder during human pregnancy, which affects 5% to 8% of pregnancies. Its risks and complications have become the major causes of maternal and fetal morbidity and mortality. Although abnormal placentation to which DNA methylation dysregulation is always linked is speculated to be one of the reasons causing preeclampsia, the underlying mechanisms still remain elusive to date. Here we revealed that aberrant Q9Y6K1 (Q9Y6K1) plays a critical role in preeclampsia. Our results show that the expression and localization of Q9Y6K1 are dysregulated in preeclamptic placenta. Moreover, knockdown of Q9Y6K1 obviously inhibits trophoblast cell migration and invasion. Mechanistically, P24593 (P24593), known as a suppressor, is upregulated by decreased Q9Y6K1 because of promoter hypomethylation. Importantly, P24593 downregulation can rescue the defects caused by Q9Y6K1 knockdown, thereby, consolidating the significance of P24593 in the downstream of Q9Y6K1 in trophoblast. Furthermore, we detected low promoter methylation and high protein expression of P24593 in the clinical samples of preeclamptic placenta. Collectively, our study suggests that dysregulation of Q9Y6K1 and P24593 is relevant to preeclampsia. Thus, we propose that Q9Y6K1 and P24593 can serve as potential markers and targets for the clinical diagnosis and therapy of preeclampsia.   \n",
       "6082                                                                                                                                                                                                                                                                                                                                                                                                                                             In mammals, the Golgi apparatus is disassembled at early mitosis and reassembled at the end of mitosis. For Golgi disassembly, membrane fusion needs to be blocked. Golgi biogenesis requires two distinct p97ATPase-mediated membrane fusion, the P55072/Q9UNZ2 and P55072/Q8NFH4 pathways. We previously reported that Q9UNZ2 phosphorylation on Serine-140 by P06493 results in mitotic inhibition of the P55072/Q9UNZ2 pathway [11]. In this study, we demonstrate that Q8NFH4 is phosphorylated on Serine-56 and Threonine-59 by P06493 at mitosis, and this phosphorylated Q8NFH4 does not bind to Golgi membranes. Using an in vitro Golgi reassembly assay, we show that mutated Q8NFH4(S56D, T59D), which mimics mitotic phosphorylation, does not cause any cisternal regrowth, indicating that Q8NFH4 phosphorylation inhibits the P55072/Q8NFH4 pathway. Our results demonstrate that Q8NFH4 phosphorylation on Serine-56 and Threonine-59 is important for Golgi disassembly at mitosis.   \n",
       "5958                                                                                                                                                       Q56P42 protein of Saccharomyces cerevisiae is a component of a protein complex that regulates the transcription of many genes. We found that the 97th threonine residue (Thr 97) of Q56P42 was phosphorylated upon glucose limitation. The Thr 97 phosphorylation occurred within 2 min after removing glucose and was reversed within 1 min after the readdition of glucose. The effects of hexokinase mutations and glucose analogs indicate that this phosphorylation is dependent on glucose phosphorylating activity. We purified a protein kinase that phosphorylates a peptide containing Thr 97 of Q56P42 and identified it as Q9H422, a DYRK family kinase. Phosphorylation of Q56P42 was barely detectable in a yak1Delta strain. We found that Q9H422 interacted with Bmh1p and Bmh2p only in the presence of glucose. A GFP-Q9H422 fusion protein shuttled rapidly between the nucleus and the cytoplasm in response to glucose. A strain with alanine substituted for Thr 97 in Q56P42 showed overgrowth in the postdiauxic transition and failed to stop the cell cycle at G(1) phase in response to glucose deprivation. Thus, Q9H422 and Q56P42 are part of a novel glucose-sensing system in yeast that is involved in growth control in response to glucose availability.   \n",
       "23                                                                                 Many cellular proteins are methylated on lysine residues and this has been most intensively studied for histone proteins. Lysine methylations on non-histone proteins are also frequent, but in most cases the functional significance of the methylation event, as well as the identity of the responsible lysine (K) specific methyltransferase (KMT), remain unknown. Several recently discovered KMTs belong to the so-called seven--strand (7BS) class of MTases and we have here investigated an uncharacterized human 7BS MTase currently annotated as part of the P0DPD6, but which should be considered a separate enzyme. Combining in vitro enzymology and analyzes of knockout cells, we demonstrate that this MTase efficiently methylates O76013 in eukaryotic translation elongation factor 1 alpha (P68104) in vitro and in vivo. We suggest that this novel KMT is named P68104-Q8TEK3 (gene name EEF1AKMT4), in agreement with the recently established nomenclature. Furthermore, by ribosome profiling we show that the absence of O76013 methylation affects translation dynamics and changes translation speed of distinct codons. Finally, we show that P68104-Q8TEK3 is part of a novel family of human KMTs, defined by a shared sequence motif in the active site and we demonstrate the importance of this motif for catalytic activity.   \n",
       "1837                                                                                                                                                                                                                                                                                                   Colorectal cancer (CRC) remains one of the most common cancers worldwide. Increasing evidence indicates that 100642175 (100642175;81848;79441) regulate cell growth, differentiation, apoptosis, and cancer progression. However, the expression and function of 100642175;81848;79441 in the progression of CRC remains largely unknown. Here, we reported that 100642175;81848;79441 was upregulated in CRC. Increased Q9C004-Q68CZ6 expression in CRC was associated with larger tumor size and higher clinical stage. In vitro experiments revealed that 100642175;81848;79441 knockdown significantly inhibited CRC cell proliferation by causing G1 arrest and promoting apoptosis, whereas 100642175;81848;79441 overexpression promoted cell proliferation. Further functional assays indicated that 100642175;81848;79441 overexpression significantly promoted cell migration and invasion by regulate the epithelial-mesenchymal transition (EMT). Taken together, our study demonstrates that 100642175;81848;79441 could act as a functional oncogene in CRC, as well as a potential therapeutic target to inhibit CRC metastasis.   \n",
       "10207                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Although lysine methylation is classically known to regulate histone function, its role in modulating antiviral restriction factor activity remains uncharacterized. Q01628 (Q01628) was found monomethylated on its lysine 88 residue (Q01628-K88me1) to reduce its antiviral activity, mediated by the lysine methyltransferase 57108. Vesicular stomatitis virus and influenza A virus infection increased Q01628-K88me1 levels by promoting the interaction between Q01628 and 57108, suggesting that this pathway could be hijacked to support infection; conversely, P01562 reduced Q01628-K88me1 levels. These findings may have important implications in the design of therapeutics targeting protein methylation against infectious diseases.    \n",
       "\n",
       "      participant1Id         participant2Id  pubmedId        predicted  \\\n",
       "20286         Q12834                 Q13421  15970669   ubiquitination   \n",
       "895           P29375                 Q96KS0  25686748      methylation   \n",
       "1462          P04628                 Q99873  29727702      methylation   \n",
       "2895          P31749                 Q00987  15527798  phosphorylation   \n",
       "4929       100124700                 P49639  26707824      methylation   \n",
       "12847         P40763                 Q12852  22726832  phosphorylation   \n",
       "2687          O94776                 Q09472  24468085      acetylation   \n",
       "147           P01308                 P06213   2184955  phosphorylation   \n",
       "11845         O14744                 P04637  25582697      methylation   \n",
       "1617          574456                 P04049  28586056      methylation   \n",
       "185           P08183                 P51608  11726209      methylation   \n",
       "5042          F7VJQ1                 P43403  14987990  phosphorylation   \n",
       "6620          P01730                 Q14624  11517385  phosphorylation   \n",
       "16332         P12931                 Q05655   7516899  phosphorylation   \n",
       "4863          P24593                 Q9Y6K1  28049695      methylation   \n",
       "6082          P06493                 Q8NFH4  20875789  phosphorylation   \n",
       "5958          Q56P42                 Q9H422  11358866  phosphorylation   \n",
       "23            O76013                 P68104  28520920      methylation   \n",
       "1837       100642175  100642175;81848;79441  27391336  phosphorylation   \n",
       "10207         P01562                 Q01628  24129573      methylation   \n",
       "\n",
       "       predicted_confidence  \n",
       "20286              0.305353  \n",
       "895                0.866325  \n",
       "1462               0.909696  \n",
       "2895               0.988527  \n",
       "4929               0.914474  \n",
       "12847              0.983138  \n",
       "2687               0.834194  \n",
       "147                0.982965  \n",
       "11845              0.901107  \n",
       "1617               0.885787  \n",
       "185                0.878734  \n",
       "5042               0.983814  \n",
       "6620               0.986022  \n",
       "16332              0.985279  \n",
       "4863               0.885323  \n",
       "6082               0.986205  \n",
       "5958               0.985220  \n",
       "23                 0.871510  \n",
       "1837               0.986137  \n",
       "10207              0.942059  "
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df[[\"abstract\", \"normalised_abstract\", \"participant1Id\",\"participant2Id\", \"pubmedId\", \"predicted\" ,\"predicted_confidence\" ]].sample(n=20)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "conda_pytorch_p36",
   "language": "python",
   "name": "conda_pytorch_p36"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
