{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "s3_prefix = \"s3://aegovan-data/pubmed_asbtract/predictions_multi_80/\"\n",
    "s3_data =\"s3://aegovan-data/human_output/human_interactions_ppi_v2.json\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "local_temp = \"temp\"\n",
    "local_temp_pred_dir = os.path.join( local_temp, \"pred_results\")\n",
    "local_temp_wk_dir = os.path.join( local_temp, \"wk\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "!rm -rf $local_temp\n",
    "!mkdir -p $local_temp_pred_dir\n",
    "!mkdir -p $local_temp_wk_dir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!aws s3 cp s3://aegovan-data/pubmed_asbtract/predictions_multi_95/pubmed19n0538.json.txt.json.prediction.json ."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "import boto3\n",
    "import glob\n",
    "from multiprocessing.dummy import Pool as ThreadPool\n",
    "import argparse\n",
    "import datetime \n",
    "import os\n",
    "\n",
    "\n",
    "def uploadfile(localpath, s3path):\n",
    "        \"\"\"\n",
    "Uploads a file to s3\n",
    "        :param localpath: The local path\n",
    "        :param s3path: The s3 path in format s3://mybucket/mydir/mysample.txt\n",
    "        \"\"\"\n",
    "\n",
    "        bucket, key = get_bucketname_key(s3path)\n",
    "\n",
    "        if key.endswith(\"/\"):\n",
    "            key = \"{}{}\".format(key, os.path.basename(localpath))\n",
    "        \n",
    "        s3 = boto3.client('s3')\n",
    "        \n",
    "        s3.upload_file(localpath, bucket, key)\n",
    "\n",
    "def get_bucketname_key(uripath):\n",
    "    assert uripath.startswith(\"s3://\")\n",
    "\n",
    "    path_without_scheme = uripath[5:]\n",
    "    bucket_end_index = path_without_scheme.find(\"/\")\n",
    "\n",
    "    bucket_name = path_without_scheme\n",
    "    key = \"/\"\n",
    "    if bucket_end_index > -1:\n",
    "        bucket_name = path_without_scheme[0:bucket_end_index]\n",
    "        key = path_without_scheme[bucket_end_index + 1:]\n",
    "\n",
    "    return bucket_name, key\n",
    "\n",
    "\n",
    "def download_file(s3path, local_dir):\n",
    "    bucket, key = get_bucketname_key(s3path)\n",
    "    \n",
    "    s3 = boto3.client('s3')\n",
    "    \n",
    "    local_file = os.path.join(local_dir, s3path.split(\"/\")[-1])\n",
    "    \n",
    "\n",
    "    s3.download_file(bucket, key, local_file)\n",
    "    \n",
    "def download_object(s3path):\n",
    "    bucket, key = get_bucketname_key(s3path)\n",
    "    \n",
    "    s3 = boto3.client('s3')    \n",
    "\n",
    "    s3_response_object = s3.get_object(Bucket=bucket, Key=key)\n",
    "    object_content = s3_response_object['Body'].read()\n",
    "    \n",
    "    return len(object_content)\n",
    "\n",
    "\n",
    "\n",
    "def list_files(s3path_prefix):\n",
    "    assert s3path_prefix.startswith(\"s3://\")\n",
    "    assert s3path_prefix.endswith(\"/\")\n",
    "    \n",
    "    bucket, key = get_bucketname_key(s3path_prefix)\n",
    "    \n",
    "   \n",
    "   \n",
    "    s3 = boto3.resource('s3')\n",
    "    \n",
    "    bucket = s3.Bucket(name=bucket)\n",
    "\n",
    "    return ( (o.bucket_name, o.key) for o in bucket.objects.filter(Prefix=key))\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "def upload_files(local_dir, s3_prefix, num_threads=20):    \n",
    "    input_tuples = ( (f,  s3_prefix) for f in glob.glob(\"{}/*\".format(local_dir)))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        pool.starmap(uploadfile, input_tuples)\n",
    "    \n",
    "\n",
    "\n",
    "def download_files(s3_prefix, local_dir, num_threads=20):    \n",
    "    input_tuples = ( (\"s3://{}/{}\".format(s3_bucket,s3_key),  local_dir) for s3_bucket, s3_key in list_files(s3_prefix))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.starmap(download_file, input_tuples)\n",
    "        \n",
    "        \n",
    "\n",
    "def download_objects(s3_prefix, num_threads=20):    \n",
    "    s3_files = ( \"s3://{}/{}\".format(s3_bucket,s3_key) for s3_bucket, s3_key in list_files(s3_prefix))\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.map(download_object, s3_files)\n",
    "        \n",
    "    return sum(results)/1024\n",
    "        \n",
    "\n",
    "def get_directory_size(start_path):\n",
    "    total_size = 0\n",
    "    for dirpath, dirnames, filenames in os.walk(start_path):\n",
    "        for f in filenames:\n",
    "            fp = os.path.join(dirpath, f)\n",
    "            # skip if it is symbolic link\n",
    "            if not os.path.islink(fp):\n",
    "                total_size += os.path.getsize(fp)\n",
    "    return total_size\n",
    "\n",
    "def get_s3file_size(bucket, key):\n",
    "    s3 = boto3.client('s3')\n",
    "    response = s3.head_object(Bucket=bucket, Key=key)\n",
    "    size = response['ContentLength']\n",
    "    return size\n",
    "    \n",
    "def download_files_min_files(s3_prefix, local_dir, min_file_size=310, num_threads=20):    \n",
    "    input_tuples = ( (\"s3://{}/{}\".format(s3_bucket,s3_key),  local_dir) for s3_bucket, s3_key in list_files(s3_prefix) if get_s3file_size(s3_bucket, s3_key) > min_file_size )\n",
    "    \n",
    "    with ThreadPool(num_threads) as pool:\n",
    "        results = pool.starmap(download_file, input_tuples)\n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 29.4 s, sys: 6.38 s, total: 35.7 s\n",
      "Wall time: 2min 38s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "download_files(s3_prefix, local_temp_pred_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     150\r\n"
     ]
    }
   ],
   "source": [
    "!ls -l $local_temp_dir | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "total = 0\n",
    "largest_df = None\n",
    "full_df = None\n",
    "value_dict={}\n",
    "for f in os.listdir(local_temp_pred_dir):\n",
    "    df = pd.read_json(os.path.join(local_temp_pred_dir, f), orient=\"records\", lines=True )\n",
    "    \n",
    "    if largest_df is None:\n",
    "        largest_df = df\n",
    "    \n",
    "    if df.shape[0] > largest_df.shape[0]:\n",
    "        largest_df=df\n",
    "    \n",
    "    if full_df is None:\n",
    "        full_df = df\n",
    "    else:\n",
    "        full_df = pd.concat([df, full_df])\n",
    "        \n",
    "    prediction_counts = df[\"predicted\"].value_counts().to_dict()\n",
    "    for k,v in prediction_counts.items():      \n",
    "        value_dict [k] =  v +  value_dict.get(k, 0)\n",
    "    \n",
    "    #total records\n",
    "    total += df.shape[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'phosphorylation': 86137,\n",
       " 'methylation': 3174,\n",
       " 'dephosphorylation': 316,\n",
       " 'acetylation': 52}"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "value_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "predicted\n",
       "acetylation          0.906370\n",
       "dephosphorylation    0.962255\n",
       "methylation          0.946525\n",
       "phosphorylation      0.991165\n",
       "Name: predicted_confidence, dtype: float64"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.groupby([\"predicted\"])[\"predicted_confidence\"].max()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "89679"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "total"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "download_file(s3_data, local_temp_wk_dir)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "data_file = os.path.join(local_temp_wk_dir, s3_data.split(\"/\")[-1])\n",
    "data_training_full_df = pd.read_json(data_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3381, 7)"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_training_full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(89679, 16)"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>interactionId</th>\n",
       "      <th>interactionType</th>\n",
       "      <th>isNegative</th>\n",
       "      <th>participants</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>pubmedTitle</th>\n",
       "      <th>pubmedabstract</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1585513</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>false</td>\n",
       "      <td>[{'uniprotid': 'Q10728', 'alias': [['mypt1_rat...</td>\n",
       "      <td>17126281</td>\n",
       "      <td>None</td>\n",
       "      <td>Zipper-interacting protein kinase (ZIP kinase)...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1585516</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>false</td>\n",
       "      <td>[{'uniprotid': 'O43293-1', 'alias': [['o43293-...</td>\n",
       "      <td>17126281</td>\n",
       "      <td>None</td>\n",
       "      <td>Zipper-interacting protein kinase (ZIP kinase)...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   interactionId  interactionType isNegative  \\\n",
       "0        1585513  phosphorylation      false   \n",
       "1        1585516  phosphorylation      false   \n",
       "\n",
       "                                        participants  pubmedId pubmedTitle  \\\n",
       "0  [{'uniprotid': 'Q10728', 'alias': [['mypt1_rat...  17126281        None   \n",
       "1  [{'uniprotid': 'O43293-1', 'alias': [['o43293-...  17126281        None   \n",
       "\n",
       "                                      pubmedabstract  \n",
       "0  Zipper-interacting protein kinase (ZIP kinase)...  \n",
       "1  Zipper-interacting protein kinase (ZIP kinase)...  "
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_training_full_df.head(n=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>acetylation</th>\n",
       "      <th>confidence_scores</th>\n",
       "      <th>demethylation</th>\n",
       "      <th>dephosphorylation</th>\n",
       "      <th>deubiquitination</th>\n",
       "      <th>methylation</th>\n",
       "      <th>normalised_abstract</th>\n",
       "      <th>other</th>\n",
       "      <th>participant1Id</th>\n",
       "      <th>participant2Id</th>\n",
       "      <th>phosphorylation</th>\n",
       "      <th>predicted</th>\n",
       "      <th>predicted_confidence</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>ubiquitination</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>For the past century, the renin-angiotensin sy...</td>\n",
       "      <td>0.345884</td>\n",
       "      <td>{'acetylation': -1.0616518736, 'demethylation'...</td>\n",
       "      <td>0.165448</td>\n",
       "      <td>0.678281</td>\n",
       "      <td>0.272083</td>\n",
       "      <td>0.206545</td>\n",
       "      <td>For the past century, the renin-angiotensin sy...</td>\n",
       "      <td>4.597788</td>\n",
       "      <td>P01019</td>\n",
       "      <td>P23946</td>\n",
       "      <td>86.943618</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.929718</td>\n",
       "      <td>23176219</td>\n",
       "      <td>0.306509</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Bone formation is a rarely encountered finding...</td>\n",
       "      <td>0.557860</td>\n",
       "      <td>{'acetylation': -0.5836479068, 'demethylation'...</td>\n",
       "      <td>0.209011</td>\n",
       "      <td>0.807035</td>\n",
       "      <td>0.421113</td>\n",
       "      <td>0.248991</td>\n",
       "      <td>Bone formation is a rarely encountered finding...</td>\n",
       "      <td>2.604435</td>\n",
       "      <td>P37023</td>\n",
       "      <td>Q9UK05</td>\n",
       "      <td>32.053825</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.857942</td>\n",
       "      <td>23177617</td>\n",
       "      <td>0.459018</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            abstract  acetylation  \\\n",
       "0  For the past century, the renin-angiotensin sy...     0.345884   \n",
       "1  Bone formation is a rarely encountered finding...     0.557860   \n",
       "\n",
       "                                   confidence_scores  demethylation  \\\n",
       "0  {'acetylation': -1.0616518736, 'demethylation'...       0.165448   \n",
       "1  {'acetylation': -0.5836479068, 'demethylation'...       0.209011   \n",
       "\n",
       "   dephosphorylation  deubiquitination  methylation  \\\n",
       "0           0.678281          0.272083     0.206545   \n",
       "1           0.807035          0.421113     0.248991   \n",
       "\n",
       "                                 normalised_abstract     other participant1Id  \\\n",
       "0  For the past century, the renin-angiotensin sy...  4.597788         P01019   \n",
       "1  Bone formation is a rarely encountered finding...  2.604435         P37023   \n",
       "\n",
       "  participant2Id  phosphorylation        predicted  predicted_confidence  \\\n",
       "0         P23946        86.943618  phosphorylation              0.929718   \n",
       "1         Q9UK05        32.053825  phosphorylation              0.857942   \n",
       "\n",
       "   pubmedId  ubiquitination  \n",
       "0  23176219        0.306509  \n",
       "1  23177617        0.459018  "
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df.head(n=2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(89210, 16)"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df[~full_df.pubmedId.isin(data_training_full_df.pubmedId)].shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_df[\"PubmedInTrainingData\"] = full_df.pubmedId.isin(data_training_full_df.pubmedId)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "pd.set_option('display.max_columns', None)\n",
    "pd.set_option('display.max_colwidth', 10000)\n",
    "pd.set_option('display.max_rows', 100)\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "plt.rcParams.update({'font.size': 12})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_df.to_csv(\"predictions.csv\", index=False, header=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>normalised_abstract</th>\n",
       "      <th>participant1Id</th>\n",
       "      <th>participant2Id</th>\n",
       "      <th>pubmedId</th>\n",
       "      <th>predicted</th>\n",
       "      <th>predicted_confidence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>Previous genetic linkage studies identified a locus for susceptibility to prostate cancer called HPCX at Xq27. The candidate region contains two clusters of SPANX genes. The first cluster called SPANX-A/D includes SPANX-A1, SPANX-A2, SPANX-B, SPANX-C, and SPANX-D; the second cluster called SPANX-N includes SPANX-N1, SPANX-N2, SPANX-N3, and SPANX-N4. The SPANX genes encode cancer-testis (CT) specific antigens. Previous studies identified SPANX-B and SPANX-D variants produced by gene conversion events, none of which are associated with X-linked prostate cancer.</td>\n",
       "      <td>Previous genetic linkage studies identified a locus for susceptibility to prostate cancer called 9566 at Xq27. The candidate region contains two clusters of Q9NS26 genes. The first cluster called 30014;64648 includes Q9NS26, Q9NS26, Q9NS25, Q9NY87, and Q9BXN6; the second cluster called Q9NS26-N includes Q5VSR9, Q5MJ10, Q5MJ09, and Q5MJ08. The Q9NS26 genes encode cancer-testis (CT) specific antigens. Previous studies identified Q9NS25 and Q9BXN6 variants produced by gene conversion events, none of which are associated with X-linked prostate cancer.</td>\n",
       "      <td>30014;64648</td>\n",
       "      <td>Q9NS26</td>\n",
       "      <td>17373721</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.803549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>SCA6 is a slowly progressive, late-onset cerebellar ataxia due to a trinucleotide expansion in the CACNA1A gene. We describe two unrelated cases that presented with Parkinsonism and cerebellar ataxia. One case was L-dopa-responsive with a pattern of (18)F-dopa uptake similar to Parkinson's disease, and the second case was not L-dopa-responsive and had an atypical pattern of nigrostriatal dysfunction. We suggest that SCA6, in common with SCA2 and SCA3, may be associated with Parkinsonism attributable to nigral loss and dopaminergic dysfunction. Moreover, isolated cases may be confused with multiple system atrophy.</td>\n",
       "      <td>O00555 is a slowly progressive, late-onset cerebellar ataxia due to a trinucleotide expansion in the O00555 gene. We describe two unrelated cases that presented with Parkinsonism and cerebellar ataxia. One case was L-dopa-responsive with a pattern of (18)F-dopa uptake similar to Parkinson's disease, and the second case was not L-dopa-responsive and had an atypical pattern of nigrostriatal dysfunction. We suggest that O00555, in common with Q99700 and P54252, may be associated with Parkinsonism attributable to nigral loss and dopaminergic dysfunction. Moreover, isolated cases may be confused with multiple system atrophy.</td>\n",
       "      <td>O00555</td>\n",
       "      <td>Q99700</td>\n",
       "      <td>15954136</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.891431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>Induction of drug-metabolizing microsomal cytochromes p450 (p450s) results in a striking proliferation of the smooth endoplasmic reticulum (ER). Overexpression of P450s in yeast and cultured cells produces a similar response. The signals mediating this process are not known but probably involve signal transduction pathways involved in the unfolded protein response (UPR) or the ER overload response (EOR). We have examined the temporal response of specific genes in these pathways and genes globally to overexpression of p450 in cultured cells. Activity of NFkappaB, an EOR component, was substantially increased by overexpression of full-length p450 2C2 or a chimera with the 28-amino acid signal anchor sequence of p450 2C2 in HepG2 cells, and the activation correlated temporally with the accumulation of p450 in the cells. In the UPR pathway, activation of the transcription factor XBP1 by IRE1 also correlated with the accumulation of p450 in the cells, and in contrast, maximum activation of the BiP/grp78 promoter preceded the accumulation. Differential effects of expression of p450 on apoptosis were observed in nonhepatic COS1 and hepatic HepG2 cells. In COS1 cells, apoptosis was induced, and this correlated with sustained activation of the pro-apoptotic JNK pathway, induction of CHOP, and an absence of the increased NFkappaB activity. In HepG2 cells, JNK was only transiently activated, and CHOP expression was not induced. As assessed by DNA microarray analysis, up-regulation of signaling genes was predominant including those involved in anti-apoptosis and ER stress. These results suggest that both the EOR and UPR pathways are involved in the cellular response to induction of p450 expression and that in hepatic cells genes are also induced to block apoptosis, which may be a physiologically relevant response to prevent cell death during xenobiotic induced expression of p450 in the liver.</td>\n",
       "      <td>Induction of drug-metabolizing microsomal cytochromes P20813 (p450s) results in a striking proliferation of the smooth endoplasmic reticulum (ER). Overexpression of P450s in yeast and cultured cells produces a similar response. The signals mediating this process are not known but probably involve signal transduction pathways involved in the unfolded protein response (UPR) or the ER overload response (EOR). We have examined the temporal response of specific genes in these pathways and genes globally to overexpression of P20813 in cultured cells. Activity of P19838, an EOR component, was substantially increased by overexpression of full-length P20813 2C2 or a chimera with the 28-amino acid signal anchor sequence of P20813 2C2 in HepG2 cells, and the activation correlated temporally with the accumulation of P20813 in the cells. In the UPR pathway, activation of the transcription factor P17861 by O75460 also correlated with the accumulation of P20813 in the cells, and in contrast, maximum activation of the P11021/P11021 promoter preceded the accumulation. Differential effects of expression of P20813 on apoptosis were observed in nonhepatic COS1 and hepatic HepG2 cells. In COS1 cells, apoptosis was induced, and this correlated with sustained activation of the pro-apoptotic P45983 pathway, induction of P35638, and an absence of the increased P19838 activity. In HepG2 cells, P45983 was only transiently activated, and P35638 expression was not induced. As assessed by DNA microarray analysis, up-regulation of signaling genes was predominant including those involved in anti-apoptosis and ER stress. These results suggest that both the EOR and UPR pathways are involved in the cellular response to induction of P20813 expression and that in hepatic cells genes are also induced to block apoptosis, which may be a physiologically relevant response to prevent cell death during xenobiotic induced expression of P20813 in the liver.</td>\n",
       "      <td>O75460</td>\n",
       "      <td>P20813</td>\n",
       "      <td>14718536</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.842936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>Ag-dependent activation of naive T cells induces dramatic changes in cellular metabolism that are essential for cell growth, division, and differentiation. In recent years, the serine/threonine kinase mechanistic target of rapamycin (mTOR) has emerged as a key integrator of signaling pathways that regulate these metabolic processes. However, the role of specific downstream effectors of mTOR function in T cells is poorly understood. Ribosomal protein S6 (rpS6) is an essential component of the ribosome and is inducibly phosphorylated following mTOR activation in eukaryotic cells. In the current work, we addressed the role of phosphorylation of rpS6 as an effector of mTOR function in T cell development, growth, proliferation, and differentiation using knockin and TCR transgenic mice. Surprisingly, we demonstrate that rpS6 phosphorylation is not required for any of these processes either in vitro or in vivo. Indeed, rpS6 knockin mice are completely sensitive to the inhibitory effects of rapamycin and an S6 kinase 1 (S6K1)-specific inhibitor on T cell activation and proliferation. These results place the mTOR complex 1-S6K1 axis as a crucial determinant of T cell activation independently of its ability to regulate rpS6 phosphorylation.</td>\n",
       "      <td>Ag-dependent activation of naive T cells induces dramatic changes in cellular metabolism that are essential for cell growth, division, and differentiation. In recent years, the serine/threonine kinase P42345 (P42345) has emerged as a key integrator of signaling pathways that regulate these metabolic processes. However, the role of specific downstream effectors of P42345 function in T cells is poorly understood. P62753 (P62753) is an essential component of the ribosome and is inducibly phosphorylated following P42345 activation in eukaryotic cells. In the current work, we addressed the role of phosphorylation of P62753 as an effector of P42345 function in T cell development, growth, proliferation, and differentiation using knockin and 6962 transgenic mice. Surprisingly, we demonstrate that P62753 phosphorylation is not required for any of these processes either in vitro or in vivo. Indeed, P62753 knockin mice are completely sensitive to the inhibitory effects of rapamycin and an P23443 (P23443)-specific inhibitor on T cell activation and proliferation. These results place the P42345 complex 1-P23443 axis as a crucial determinant of T cell activation independently of its ability to regulate P62753 phosphorylation.</td>\n",
       "      <td>P42345</td>\n",
       "      <td>P62753</td>\n",
       "      <td>26453749</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.959844</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>Kaposi's sarcoma-associated herpesvirus (KSHV) uses several strategies to counteract the interferon (IFN) system. In this study, the relationship of the protein LANA2 from KSHV to the IFN-activated protein kinase (PKR) and 2-5A system was analysed. It was found that LANA2 could not abrogate apoptosis or RNA degradation mediated by the 2-5A system. However, expression of LANA2 inhibited apoptosis triggered by PKR. LANA2 also counteracted the PKR-mediated inhibition of protein synthesis and partially blocked PKR-induced phosphorylation of eIF-2alpha. Analysis of PKR-induced activation of caspases 3 and 9 revealed that LANA2 abrogated activation of caspase 3 but not of caspase 9. These findings show that LANA2 is able to interfere with downstream events triggered by PKR. Hence, LANA2 should be considered as a KSHV defence protein against IFN.</td>\n",
       "      <td>Kaposi's sarcoma-associated herpesvirus (KSHV) uses several strategies to counteract the interferon (IFN) system. In this study, the relationship of the protein LANA2 from KSHV to the IFN-activated protein kinase (P19525) and 2-5A system was analysed. It was found that LANA2 could not abrogate apoptosis or RNA degradation mediated by the 2-5A system. However, expression of LANA2 inhibited apoptosis triggered by P19525. LANA2 also counteracted the P19525-mediated inhibition of protein synthesis and partially blocked P19525-induced phosphorylation of Q9BY44. Analysis of P19525-induced activation of caspases 3 and 9 revealed that LANA2 abrogated activation of P42574 but not of P55211. These findings show that LANA2 is able to interfere with downstream events triggered by P19525. Hence, LANA2 should be considered as a KSHV defence protein against IFN.</td>\n",
       "      <td>P19525</td>\n",
       "      <td>Q9BY44</td>\n",
       "      <td>12771415</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.862227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>The eukaryotic transcript elongation factor TFIIS is encoded by a nonessential gene, PPR2, in Saccharomyces cerevisiae. Disruptions of PPR2 are lethal in conjunction with a disruption in the nonessential gene TAF14/TFG3. While investigating which of the Taf14p-containing complexes may be responsible for the synthetic lethality between ppr2Delta and taf14Delta, we discovered genetic interactions between PPR2 and both TFG1 and TFG2 encoding the two larger subunits of the TFIIF complex that also contains Taf14p. Mutant alleles of tfg1 or tfg2 that render cells cold sensitive have improved growth at low temperature in the absence of TFIIS. Remarkably, the amino-terminal 130 amino acids of TFIIS, which are dispensable for the known in vitro and in vivo activities of TFIIS, are required to complement the lethality in taf14Delta ppr2Delta cells. Analyses of deletion and chimeric gene constructs of PPR2 implicate contributions by different regions of this N-terminal domain. No strong common phenotypes were identified for the ppr2Delta and taf14Delta strains, implying that the proteins are not functionally redundant. Instead, the absence of Taf14p in the cell appears to create a dependence on an undefined function of TFIIS mediated by its N-terminal region. This region of TFIIS is also at least in part responsible for the deleterious effect of TFIIS on tfg1 or tfg2 cold-sensitive cells. Together, these results suggest a physiologically relevant functional connection between TFIIS and TFIIF.</td>\n",
       "      <td>The eukaryotic transcript elongation factor Q15560 is encoded by a nonessential gene, 780910, in Saccharomyces cerevisiae. Disruptions of 780910 are lethal in conjunction with a disruption in the nonessential gene TAF14/TFG3. While investigating which of the Taf14p-containing complexes may be responsible for the synthetic lethality between ppr2Delta and taf14Delta, we discovered genetic interactions between 780910 and both TFG1 and TFG2 encoding the two larger subunits of the TFIIF complex that also contains Taf14p. Mutant alleles of tfg1 or tfg2 that render cells cold sensitive have improved growth at low temperature in the absence of Q15560. Remarkably, the amino-terminal 130 amino acids of Q15560, which are dispensable for the known in vitro and in vivo activities of Q15560, are required to complement the lethality in taf14Delta ppr2Delta cells. Analyses of deletion and chimeric gene constructs of 780910 implicate contributions by different regions of this N-terminal domain. No strong common phenotypes were identified for the 780910Delta and taf14Delta strains, implying that the proteins are not functionally redundant. Instead, the absence of Taf14p in the cell appears to create a dependence on an undefined function of Q15560 mediated by its N-terminal region. This region of Q15560 is also at least in part responsible for the deleterious effect of Q15560 on tfg1 or tfg2 cold-sensitive cells. Together, these results suggest a physiologically relevant functional connection between Q15560 and TFIIF.</td>\n",
       "      <td>780910</td>\n",
       "      <td>Q15560</td>\n",
       "      <td>16648643</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.919070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>Axin2 is a negative regulator of Wnt/beta-catenin signalling with roles in early development and tumour suppression. Axin2 is induced by E2F1 and therefore acts as a point of cross-talk between the pRb/E2F and Wnt/beta-catenin pathways: two of the most frequently deregulated pathways in human cancers. In this study, we show that E2F1 up-regulates axin2 by two independent mechanisms. The human axin2 gene allows transcription of messages with three different 5' untranslated regions and in the first mechanism E2F1 directly activates the transcription of only one of these species by acting at canonical E2F binding sites. Second, E2F1 induces stabilisation of axin2 mRNAs. We discuss this regulation with respect to other known E2F targets.</td>\n",
       "      <td>Q9Y2T1 is a negative regulator of Wnt/P35222 signalling with roles in early development and tumour suppression. Q9Y2T1 is induced by Q01094 and therefore acts as a point of cross-talk between the P06400/E2F and Wnt/P35222 pathways: two of the most frequently deregulated pathways in human cancers. In this study, we show that Q01094 up-regulates Q9Y2T1 by two independent mechanisms. The human Q9Y2T1 gene allows transcription of messages with three different 5' untranslated regions and in the first mechanism Q01094 directly activates the transcription of only one of these species by acting at canonical E2F binding sites. Second, Q01094 induces stabilisation of Q9Y2T1 mRNAs. We discuss this regulation with respect to other known E2F targets.</td>\n",
       "      <td>P06400</td>\n",
       "      <td>Q01094</td>\n",
       "      <td>15766563</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.897457</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Retinitis pigmentosa (RP) is a group of genetically and clinically heterogeneous retinopathies, some of which have been shown to result from mutations in two different known retinal genes, rhodopsin (3q) and peripherin-rds (6p). Three additional anonymous loci at 7p, 7q and pericentric 8 have been implicated by linkage studies. There are still, however, a few families in which all known loci have been excluded. In this report we present data indicating a location, on the short arm of chromosome 17, for the autosomal dominant RP (ADRP) locus in a large South African (SA) family of British ancestry. Positive two-point lod scores have been obtained for nine markers (D17S938, Z = 5.43; D17S796, Z = 4.82; D17S849, Z = 3.6; D17S786, Z = 3.55; TP53, Z = 3.55; D17S578, Z = 3.29; D17S960, Z = 3.16; D17S926, Z = 1.51; D17S804, Z = 0.47 all at theta = 0.10 except D17S804 and D17S926, theta = 0.20). These data provide definitive evidence for the localization of an ADRP gene on chromosome 17p. The human recoverin gene has been localized to 17p13.1 and was consequently a prime candidate for ADRP in the family studied. However, mutation screening of the three exons of this gene failed to produce any evidence of recoverin being the gene involved in the pathogenesis of ADRP in this SA family.</td>\n",
       "      <td>Retinitis pigmentosa (RP) is a group of genetically and clinically heterogeneous retinopathies, some of which have been shown to result from mutations in two different known retinal genes, rhodopsin (3q) and peripherin-rds (6p). Three additional anonymous loci at 7p, 7q and pericentric 8 have been implicated by linkage studies. There are still, however, a few families in which all known loci have been excluded. In this report we present data indicating a location, on the short arm of chromosome 17, for the Q99541 (Q99541) locus in a large South African (SA) family of British ancestry. Positive two-point lod scores have been obtained for nine markers (D17S938, Z = 5.43; D17S796, Z = 4.82; D17S849, Z = 3.6; D17S786, Z = 3.55; P04637, Z = 3.55; D17S578, Z = 3.29; D17S960, Z = 3.16; D17S926, Z = 1.51; D17S804, Z = 0.47 all at theta = 0.10 except D17S804 and D17S926, theta = 0.20). These data provide definitive evidence for the localization of an Q99541 gene on chromosome 17p. The human P35243 gene has been localized to 17p13.1 and was consequently a prime candidate for Q99541 in the family studied. However, mutation screening of the three exons of this gene failed to produce any evidence of P35243 being the gene involved in the pathogenesis of Q99541 in this SA family.</td>\n",
       "      <td>P35243</td>\n",
       "      <td>Q99541</td>\n",
       "      <td>7951236</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.900106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>The zinc-finger protein A20 has crucial physiological functions as a dual inhibitor of nuclear factor-B (NF-B) activation and apoptosis in tumor necrosis factor (TNF) receptor 1 signaling pathway. Although the molecular basis for the anti-NF-B function of A20 has been well elucidated, the anti-apoptotic function of A20 is largely unknown. Here, we report a novel mechanism underlying the anti-apoptotic function of A20: A20 blocks TNF-induced apoptosis through suppression of c-jun N-terminal kinase (JNK) by targeting apoptosis signal-regulating kinase1 (ASK1). First, the ectopic expression of A20 drastically inhibits TNF-induced JNK activation and apoptosis in multiple cell types including those deficient of NF-B activation. Unexpectedly, the blunting effect of A20 on TNF-induced JNK activation is not mediated by affecting the TNFR1 signaling complex formation. Instead, A20 interacts with ASK1, an important MAPKK kinase in the JNK signaling cascade. More importantly, overexpression of wild-type A20, but not of mutant A20 (ZnF4; C624A, C627A), promotes degradation of the ASK1 through the ubiquitin-proteasome system. Taken together, the results from this study reveal a novel anti-apoptotic mechanism of A20 in TNF signaling pathway: A20 binds to ASK1 and mediates ASK1 degradation, leading to suppression of JNK activation and eventually blockage of apoptosis.</td>\n",
       "      <td>The P21580 has crucial physiological functions as a dual inhibitor of nuclear factor-B (NF-B) activation and apoptosis in tumor necrosis factor (P01375) receptor 1 signaling pathway. Although the molecular basis for the anti-NF-B function of P21580 has been well elucidated, the anti-apoptotic function of P21580 is largely unknown. Here, we report a novel mechanism underlying the anti-apoptotic function of P21580: P21580 blocks P01375-induced apoptosis through suppression of P45983 (P45983) by targeting Q99683 (Q99683). First, the ectopic expression of P21580 drastically inhibits P01375-induced P45983 activation and apoptosis in multiple cell types including those deficient of NF-B activation. Unexpectedly, the blunting effect of P21580 on P01375-induced P45983 activation is not mediated by affecting the P19438 signaling complex formation. Instead, P21580 interacts with Q99683, an important MAPKK kinase in the P45983 signaling cascade. More importantly, overexpression of wild-type P21580, but not of mutant P21580 (7897; C624A, C627A), promotes degradation of the Q99683 through the ubiquitin-proteasome system. Taken together, the results from this study reveal a novel anti-apoptotic mechanism of P21580 in P01375 signaling pathway: P21580 binds to Q99683 and mediates Q99683 degradation, leading to suppression of P45983 activation and eventually blockage of apoptosis.</td>\n",
       "      <td>P21580</td>\n",
       "      <td>Q99683</td>\n",
       "      <td>20448643</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.928017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>Beta-amyloid accumulates around neurons in Alzheimer's disease and is thought to contribute to the neurodegenerative process. This study examined the role of the tumour suppressor protein, p53, in the neurodegenerative pathway, with focus on the interaction of p53 with the lysosomal system. beta-Amyloid increased expression of p53 and its transcription target, Bax, in cultured cortical neurons. In addition, A beta increased the association of phospho-p53(ser15) with the lysosomal compartment and this correlated with destabilization of the lysosomal membrane and a concomitant increase in cytosolic cathepsin-L activity. These effects of beta-amyloid were abolished by the p53 inhibitor, pifithrin-alpha, and siRNA-mediated knockdown of p53, demonstrating that p53 is a critical regulator of lysosomal integrity and the induction of cathepsin-L protease activity. In addition, activation of the apoptotic cascade was abolished by pifithrin-alpha. We conclude that p53 associates with the lysosome to regulate a lysosomal branch of the apoptotic cascade which contributes to beta-amyloid-mediated neurodegeneration.</td>\n",
       "      <td>Beta-amyloid accumulates around neurons in Alzheimer's disease and is thought to contribute to the neurodegenerative process. This study examined the role of the tumour suppressor protein, P04637, in the neurodegenerative pathway, with focus on the interaction of P04637 with the lysosomal system. beta-Amyloid increased expression of P04637 and its transcription target, Q07812, in cultured cortical neurons. In addition, Q9P2R7 increased the association of phospho-P04637(3190208) with the lysosomal compartment and this correlated with destabilization of the lysosomal membrane and a concomitant increase in cytosolic P07711 activity. These effects of beta-amyloid were abolished by the P04637 inhibitor, pifithrin-alpha, and siRNA-mediated knockdown of P04637, demonstrating that P04637 is a critical regulator of lysosomal integrity and the induction of P07711 protease activity. In addition, activation of the apoptotic cascade was abolished by pifithrin-alpha. We conclude that P04637 associates with the lysosome to regulate a lysosomal branch of the apoptotic cascade which contributes to beta-amyloid-mediated neurodegeneration.</td>\n",
       "      <td>P04637</td>\n",
       "      <td>Q9P2R7</td>\n",
       "      <td>19059678</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.905761</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>Dysadherin is a recently discovered cancer-related cell membrane glycoprotein that has an important role in tumor progression and metastasis. We are focusing on the role of dysadherin in E-cadherin downregulation, the different expression patterns of the molecule in cancer of the head and neck and its potential role as a molecular target for future applications in diagnosis, prognosis, and management of the disease.</td>\n",
       "      <td>Q96DB9 is a recently discovered cancer-related cell membrane glycoprotein that has an important role in tumor progression and metastasis. We are focusing on the role of Q96DB9 in P12830 downregulation, the different expression patterns of the molecule in cancer of the head and neck and its potential role as a molecular target for future applications in diagnosis, prognosis, and management of the disease.</td>\n",
       "      <td>P12830</td>\n",
       "      <td>Q96DB9</td>\n",
       "      <td>22331710</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.884789</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>We have demonstrated that caspase-1 is a mediator of both cisplatin-induced acute kidney injury (AKI) and ischemic AKI. As caspase-1 is activated in the inflammasome, we investigated the inflammasome in cisplatin-induced and ischemic AKI. Mice were injected with cisplatin or subjected to bilateral renal pedicle clamping. Immunoblot analysis of whole kidney after cisplatin-induced AKI revealed: 1) an increase in apoptosis-associated Speck-like protein containing a caspase recruitment domain (ASC), the major protein that complexes with nucleotide-binding oligomerization domain, leucine-rich repeat and pyrin domain containing proteins (NLRP) 1 or 3 to form the inflammasome; 2) an increase in caspase-1 activity, caspase-5, and NLRP1, components of the NLRP1 inflammasome; and 3) a trend toward increased NLRP3. To determine whether the NLRP3 inflammasome plays an injurious role in cisplatin-induced AKI, we studied NLRP knockout (NLRP3(-/-)) mice. In cisplatin-induced AKI, the blood urea nitrogen, serum creatinine, acute tubular necrosis score, and tubular apoptosis score were not significantly decreased in NALP3(-/-) mice compared with wild-type mice. We have previously demonstrated the injurious role of caspase-1 in ischemic AKI. NLRP3, but not ASC or NLRP1, is increased in ischemic AKI. NLRP3(-/-) mice with ischemic AKI had significantly lower blood urea nitrogen, serum creatinine, and acute tubular necrosis and apoptosis scores than the wild-type controls. The difference in protection against cisplatin-induced AKI compared with ischemic AKI in NLRP3(-/-) mice was not explained by the differences in proinflammatory cytokines interleukin (IL)-1, IL-6, chemokine (C-X-C motif) ligand 1, or tumor necrosis factor . NLRP3 inflammasome is a mediator of ischemic AKI but not cisplatin-induced AKI, and further investigation of the NLRP1 inflammasome in cisplatin-induced AKI should prove interesting.</td>\n",
       "      <td>We have demonstrated that P29466 is a mediator of both cisplatin-induced acute kidney injury (AKI) and ischemic AKI. As P29466 is activated in the inflammasome, we investigated the inflammasome in cisplatin-induced and ischemic AKI. Mice were injected with cisplatin or subjected to bilateral renal pedicle clamping. Immunoblot analysis of whole kidney after cisplatin-induced AKI revealed: 1) an increase in apoptosis-associated Speck-like protein containing a caspase recruitment domain (ASC), the major protein that complexes with nucleotide-binding oligomerization domain, leucine-rich repeat and pyrin domain containing proteins (NLRP) 1 or 3 to form the inflammasome; 2) an increase in P29466 activity, P51878, and Q9C000, components of the Q9C000 inflammasome; and 3) a trend toward increased Q96P20. To determine whether the Q96P20 inflammasome plays an injurious role in cisplatin-induced AKI, we studied NLRP knockout (Q96P20(-/-)) mice. In cisplatin-induced AKI, the blood urea nitrogen, serum creatinine, acute tubular necrosis score, and tubular apoptosis score were not significantly decreased in Q96P20(-/-) mice compared with wild-type mice. We have previously demonstrated the injurious role of P29466 in ischemic AKI. Q96P20, but not ASC or Q9C000, is increased in ischemic AKI. Q96P20(-/-) mice with ischemic AKI had significantly lower blood urea nitrogen, serum creatinine, and acute tubular necrosis and apoptosis scores than the wild-type controls. The difference in protection against cisplatin-induced AKI compared with ischemic AKI in Q96P20(-/-) mice was not explained by the differences in proinflammatory cytokines interleukin (IL)-1, P05231, P01375 . Q96P20 inflammasome is a mediator of ischemic AKI but not cisplatin-induced AKI, and further investigation of the Q9C000 inflammasome in cisplatin-induced AKI should prove interesting.</td>\n",
       "      <td>P01375</td>\n",
       "      <td>Q96P20</td>\n",
       "      <td>23833276</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.885537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>The transcription factor EB (TFEB) plays a pivotal role in the regulation of basic cellular processes, such as lysosomal biogenesis and autophagy. The subcellular localization and activity of TFEB are regulated by mechanistic target of rapamycin (mTOR)-mediated phosphorylation, which occurs at the lysosomal surface. Phosphorylated TFEB is retained in the cytoplasm, whereas dephosphorylated TFEB translocates to the nucleus to induce the transcription of target genes. Thus, a lysosome-to-nucleus signaling pathway regulates cellular energy metabolism through TFEB. Recently, in vivo studies have revealed that TFEB is also involved in physiological processes, such as lipid catabolism. TFEB has attracted a lot of attention owing to its ability to induce the intracellular clearance of pathogenic factors in a variety of murine models of disease, such as Parkinson's and Alzheimer's, suggesting that novel therapeutic strategies could be based on the modulation of TFEB activity. In this Cell Science at a Glance article and accompanying poster, we present an overview of the latest research on TFEB function and its implication in human diseases.</td>\n",
       "      <td>The P19484 (P19484) plays a pivotal role in the regulation of basic cellular processes, such as lysosomal biogenesis and autophagy. The subcellular localization and activity of P19484 are regulated by P42345 (P42345)-mediated phosphorylation, which occurs at the lysosomal surface. Phosphorylated P19484 is retained in the cytoplasm, whereas dephosphorylated P19484 translocates to the nucleus to induce the transcription of target genes. Thus, a lysosome-to-nucleus signaling pathway regulates cellular energy metabolism through P19484. Recently, in vivo studies have revealed that P19484 is also involved in physiological processes, such as lipid catabolism. P19484 has attracted a lot of attention owing to its ability to induce the intracellular clearance of pathogenic factors in a variety of murine models of disease, such as Parkinson's and Alzheimer's, suggesting that novel therapeutic strategies could be based on the modulation of P19484 activity. In this Cell Science at a Glance article and accompanying poster, we present an overview of the latest research on P19484 function and its implication in human diseases.</td>\n",
       "      <td>P19484</td>\n",
       "      <td>P42345</td>\n",
       "      <td>27252382</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.835544</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>The targeted therapy for triple-negative breast cancer (TNBC) is a great challenge due to our poor understanding on its molecular etiology. In the present study, our clinical data showed that the expression of G-protein coupled estrogen receptor (GPER) is negatively associated with lymph node metastasis, high-grade tumor and fibronectin (FN) expression while positively associated with the favorable outcome in 135 TNBC patients. In our experimental studies, both the invitro migration and invasion of TNBC cells were inhibited by GPER specific agonist G-1, through the suppression of the epithelial mesenchymal transition (EMT). The G-1 treatment also reduced the phosphorylation, nuclear localization, and transcriptional activities of NF-B. While over expression of NF-B attenuated the action of G-1 in suppressing EMT. Our data further illustrated that the phosphorylation of GSK-3 by PI3K/Akt and ERK1/2 mediated, at least partially, the inhibitory effect of G-1 on NF-B activities. It was further confirmed in a study of MDA-MB-231 tumor xenografts in nude mice. The data showed that G-1 inhibited the invivo growth and invasive potential of TNBC via suppression of EMT. Our present study demonstrated that an activation of GPER pathway elicits tumor suppressive actions on TNBC, and supports the use of G-1 therapeutics for TNBC metastasis.</td>\n",
       "      <td>The targeted therapy for triple-negative breast cancer (TNBC) is a great challenge due to our poor understanding on its molecular etiology. In the present study, our clinical data showed that the expression of Q99527 (Q99527) is negatively associated with lymph node metastasis, high-grade tumor and fibronectin (FN) expression while positively associated with the favorable outcome in 135 TNBC patients. In our experimental studies, both the invitro migration and invasion of TNBC cells were inhibited by Q99527 specific agonist A0A0G2JNB4, through the suppression of the epithelial mesenchymal transition (EMT). The A0A0G2JNB4 treatment also reduced the phosphorylation, nuclear localization, and transcriptional activities of NF-B. While over expression of NF-B attenuated the action of A0A0G2JNB4 in suppressing EMT. Our data further illustrated that the phosphorylation of GSK-3 by PI3K/P31749 and 5595;5594 mediated, at least partially, the inhibitory effect of A0A0G2JNB4 on NF-B activities. It was further confirmed in a study of MDA-MB-231 tumor xenografts in nude mice. The data showed that A0A0G2JNB4 inhibited the invivo growth and invasive potential of TNBC via suppression of EMT. Our present study demonstrated that an activation of Q99527 pathway elicits tumor suppressive actions on TNBC, and supports the use of A0A0G2JNB4 therapeutics for TNBC metastasis.</td>\n",
       "      <td>A0A0G2JNB4</td>\n",
       "      <td>Q99527</td>\n",
       "      <td>26842883</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.892953</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>Lung cancer is the most common cancer as well as the leading cause of cancer-related mortalities worldwide. Macrophages are the most abundant immune cells in primary and metastatic tumors, and contribute to tumor initiation, progression and metastasis. Macrophages have been shown to demonstrate a high level of plasticity, with the ability to undergo dynamic transition between M1 and M2 polarized phenotypes. In the present study, we investigated a pivotal role of miR-130a in macrophage polarization and whether it was associated with poor prognosis in non-small cell lung cancer (NSCLC), using RT-qPCR and western blot analyses. The invitro experiments showed that miRNA-130a was expressed at a higher level in M1 compared to M2 macrophages. The enforced expression of miR-130a in macrophages resulted in a significantly increased production of proinflammatory cytokines, whereas deletion of miR-130a impaired the M2associated gene expression and led to an M1-biased response. Mechanistically, the bioinformatics analysis revealed that proliferator-activated receptor (PPAR) is a potential target of miR-130a. Additionally, the luciferase assay confirmed that PPAR translation was suppressed by miR-130a through the interaction with the 3'UTR of PPAR mRNA. A subsequent analysis revaled that the induction of miR-130a suppressed PPAR protein expression. In NSCLC patients, the results showed that miR-130a downregulation exhibited clinical relevance as it was correlated with poor prognosis and increased tumor stage and metastasis. In addition, miR130a was inversely correlated with the macrophage marker, CD163, and target gene, PPAR. Taken together, the results established miR-130a as a molecular switch during macrophage development and as a potential target for the treatment of NSCLC.</td>\n",
       "      <td>Lung cancer is the most common cancer as well as the leading cause of cancer-related mortalities worldwide. Macrophages are the most abundant immune cells in primary and metastatic tumors, and contribute to tumor initiation, progression and metastasis. Macrophages have been shown to demonstrate a high level of plasticity, with the ability to undergo dynamic transition between M1 and M2 polarized phenotypes. In the present study, we investigated a pivotal role of 406919 in macrophage polarization and whether it was associated with poor prognosis in non-small cell lung cancer (NSCLC), using RT-qPCR and western blot analyses. The invitro experiments showed that 406919 was expressed at a higher level in M1 compared to M2 macrophages. The enforced expression of 406919 in macrophages resulted in a significantly increased production of proinflammatory cytokines, whereas deletion of 406919 impaired the M2associated gene expression and led to an M1-biased response. Mechanistically, the bioinformatics analysis revealed that proliferator-activated receptor (Q07869) is a potential target of 406919. Additionally, the luciferase assay confirmed that Q07869 translation was suppressed by 406919 through the interaction with the 3'UTR of Q07869 mRNA. A subsequent analysis revaled that the induction of 406919 suppressed Q07869 protein expression. In NSCLC patients, the results showed that 406919 downregulation exhibited clinical relevance as it was correlated with poor prognosis and increased tumor stage and metastasis. In addition, 406919 was inversely correlated with the macrophage marker, Q86VB7, and target gene, Q07869. Taken together, the results established 406919 as a molecular switch during macrophage development and as a potential target for the treatment of NSCLC.</td>\n",
       "      <td>406919</td>\n",
       "      <td>Q07869</td>\n",
       "      <td>26398698</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.808635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>Cystinuria is a frequently inherited metabolic disorder in the Czech population (frequency 1/5,600) caused by a defect in the renal transport of cystine and dibasic amino acids (arginine, lysine and ornithine). The disease is characterized by increased urinary excretion of the amino acids and often leads to recurrent nephrolithiasis. Cystinuria is classified into two subtypes (type I and type non-I). Type I is caused predominantly by mutations in the SLC3A1 gene (2p16.3), encoding heavy subunit (rBAT) of the heterodimeric transporter. Cystinuria non-I type is caused by mutations in the SLC7A9 gene (19q13.1). In this study, we present results of molecular genetic analysis of the SLC3A1 and the SLC7A9 genes in 24 unrelated cystinuria families. Individual exons of the SLC3A1 and SLC7A9 genes were analyzed by direct sequencing. We found ten different mutations in the SLC3A1 gene including six novel ones: three missense mutations (G140R), D179Y and R365P), one splice site mutation (1137-2A&gt;G), one deletion (1515_1516delAA), and one nonsense mutation (Q119X). The most frequent mutation, M467T; was detected in 36% of all type I classified alleles. In the SLC7A9 gene we found six mutations including three new ones: one missense mutation (G319R), one insertion (611_612insA) and one deletion (205_206delTG). One patient was compound heterozygote for one SLC3A1 and one SLC7A9 mutation. Our results confirm that cystinuria is a heterogeneous disorder at the molecular level, and contribute to the understanding of the distribution and frequency of mutations causing cystinuria in the Caucasian population.</td>\n",
       "      <td>Cystinuria is a frequently inherited metabolic disorder in the Czech population (frequency 1/5,600) caused by a defect in the renal transport of cystine and dibasic amino acids (arginine, lysine and ornithine). The disease is characterized by increased urinary excretion of the amino acids and often leads to recurrent nephrolithiasis. Cystinuria is classified into two subtypes (type I and type non-I). Type I is caused predominantly by mutations in the Q07837 gene (2p16.3), encoding heavy subunit (rBAT) of the heterodimeric transporter. Cystinuria non-I type is caused by mutations in the P82251 gene (19q13.1). In this study, we present results of molecular genetic analysis of the Q07837 and the P82251 genes in 24 unrelated cystinuria families. Individual exons of the Q07837 and P82251 genes were analyzed by direct sequencing. We found ten different mutations in the Q07837 gene including six novel ones: three missense mutations (G140R), D179Y and R365P), one splice site mutation (1137-2A&gt;G), one deletion (1515_1516delAA), and one nonsense mutation (Q119X). The most frequent mutation, M467T; was detected in 36% of all type I classified alleles. In the P82251 gene we found six mutations including three new ones: one missense mutation (G319R), one insertion (611_612insA) and one deletion (205_206delTG). One patient was compound heterozygote for one Q07837 and one P82251 mutation. Our results confirm that cystinuria is a heterogeneous disorder at the molecular level, and contribute to the understanding of the distribution and frequency of mutations causing cystinuria in the Caucasian population.</td>\n",
       "      <td>P82251</td>\n",
       "      <td>Q07837</td>\n",
       "      <td>16138908</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.852938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>The t(11;14)(q13;q32) translocation with cyclin D1 overexpression commonly is found in multiple myeloma (MM) and in mantle cell lymphoma (MCL). Several reports have shown that p53 mutations in MCL lead to blastoid transformation and a worse prognosis; however, the role of p53 mutations in MM with t(11;14) is unclear.</td>\n",
       "      <td>The t(11;14)(q13;q32) translocation with P24385 overexpression commonly is found in multiple myeloma (MM) and in mantle cell lymphoma (MCL). Several reports have shown that P04637 mutations in MCL lead to blastoid transformation and a worse prognosis; however, the role of P04637 mutations in MM with t(11;14) is unclear.</td>\n",
       "      <td>P04637</td>\n",
       "      <td>P24385</td>\n",
       "      <td>10223569</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.808685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Understanding the signalling mechanisms controlling inflammatory cytokine production is pivotal to the research of both acute and chronic immune disorders. Tyrosine phosphorylation is one of the earliest events to occur in response to an immune challenge yet the role of specific tyrosine kinases in inflammatory cytokine production has been difficult to ascribe due to conflicting literature. Here we show that the pyrazolo pyrimidine compound PP2, a selective inhibitor of Src family kinases (SFK), can inhibit LPS-induced TNF production as well as a number of other inflammatory cytokines. In addition, we show similar effects of PP2 on cytokine production when induced by other TLRs, (1, 2 and 5-8), indicating that SFK are important common regulators of TLR signalling. PP2 suppressed the activity of both TNF and IL-10 driven reporter genes, suggesting that this activity is mediated at the level of transcription. Interestingly, however, PP2 had no significant effect on the activation of NF-kappaB, or on p42/44 ERK, p46/54 JNK or p38 MAPK phosphorylation. In contrast, PP2 did inhibit AP-1 nuclear accumulation in response to LPS. Taken together, these findings show that the Src kinases are able to control inflammatory cytokine production at the transcriptional level independently of NF-kappaB, and highlight the role of the AP-1 family of transcription factors as downstream mediators of Src kinase action.</td>\n",
       "      <td>Understanding the signalling mechanisms controlling inflammatory cytokine production is pivotal to the research of both acute and chronic immune disorders. Tyrosine phosphorylation is one of the earliest events to occur in response to an immune challenge yet the role of specific tyrosine kinases in inflammatory cytokine production has been difficult to ascribe due to conflicting literature. Here we show that the pyrazolo pyrimidine compound 4888, a selective inhibitor of Src family kinases (SFK), can inhibit LPS-induced P01375 production as well as a number of other inflammatory cytokines. In addition, we show similar effects of 4888 on cytokine production when induced by other TLRs, (1, 2 and 5-8), indicating that SFK are important common regulators of TLR signalling. 4888 suppressed the activity of both P01375 and P22301 driven reporter genes, suggesting that this activity is mediated at the level of transcription. Interestingly, however, 4888 had no significant effect on the activation of P19838, or on Q00532/44 ERK, p46/54 JNK or P28482 MAPK phosphorylation. In contrast, 4888 did inhibit AP-1 nuclear accumulation in response to LPS. Taken together, these findings show that the Src kinases are able to control inflammatory cytokine production at the transcriptional level independently of P19838, and highlight the role of the AP-1 family of transcription factors as downstream mediators of Src kinase action.</td>\n",
       "      <td>4888</td>\n",
       "      <td>P19838</td>\n",
       "      <td>17875324</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.812231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>We investigated the effects of the expression of myosin light chain (MLC) isoforms on the Ca2+ sensitivity of isometric force production of demembranated (skinned) fibers of papillary muscle from the left ventricle of three groups: patients with ischemic cardiomyopathy, patients with dilated cardiomyopathy (NYHA IV) and normal human hearts. Expression and phosphorylation of the phosphorylatable MLC isoforms (MLC-2) was equal within all three groups. However, 72% of the patients investigated in this study expressed the atrial essential MLC (ALC-1) in addition to the essential ventricular MLC (VLC-1) ranging between 2.4% and 10.3%. Using fibers from failing hearts, we observed a significant positive correlation between ALC-1 and Ca2+ sensitivity in that the higher the ALC-1 expression the higher the Ca2(+)-sensitivity: pCa50 (Ca2+ required for half-maximal force production) was 5.87 without ALC-1 and 6.08 with 10.3% ALC-1. Fibers from a normal heart (no ALC-1) revealed a pCa50 of 5.85. Isoform and phosphorylation patterns of tropomyosin and troponin I remained unchanged in the patients and normal hearts. Our results suggest that Ca2+ responsiveness and force development of the human heart is regulated by the expression of different MLC-1 isoforms.</td>\n",
       "      <td>We investigated the effects of the expression of Q15049 (Q15049) isoforms on the Ca2+ sensitivity of isometric force production of demembranated (skinned) fibers of papillary muscle from the left ventricle of three groups: patients with ischemic cardiomyopathy, patients with dilated cardiomyopathy (NYHA IV) and normal human hearts. Expression and phosphorylation of the phosphorylatable Q15049 isoforms (P10916) was equal within all three groups. However, 72% of the patients investigated in this study expressed the atrial essential Q15049 (P12829) in addition to the essential ventricular Q15049 (P08590) ranging between 2.4% and 10.3%. Using fibers from failing hearts, we observed a significant positive correlation between P12829 and Ca2+ sensitivity in that the higher the P12829 expression the higher the Ca2(+)-sensitivity: pCa50 (Ca2+ required for half-maximal force production) was 5.87 without P12829.08 with 10.3% P12829. Fibers from a normal heart (no P12829) revealed a pCa50 of 5.85. Isoform and phosphorylation patterns of tropomyosin and troponin I remained unchanged in the patients and normal hearts. Our results suggest that Ca2+ responsiveness and force development of the human heart is regulated by the expression of different Q15049 isoforms.</td>\n",
       "      <td>P12829</td>\n",
       "      <td>Q15049</td>\n",
       "      <td>9160869</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.882644</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>Identification of biological markers that may predict response to chemotherapy would allow the individualization of treatment by enabling selection of patients most likely to benefit from chemotherapy. The aims of this study were to determine whether p53 mutation status and p53 and p33(ING1b) protein expression can predict which patients with Dukes' C colorectal cancer following curative surgical resection respond to adjuvant chemotherapy with 5-fluorouracil (5-FU).</td>\n",
       "      <td>Identification of biological markers that may predict response to chemotherapy would allow the individualization of treatment by enabling selection of patients most likely to benefit from chemotherapy. The aims of this study were to determine whether P04637 mutation status and P04637 and Q9UK53(ING1b) protein expression can predict which patients with Dukes' C colorectal cancer following curative surgical resection respond to adjuvant chemotherapy with 5-fluorouracil (5-FU).</td>\n",
       "      <td>P04637</td>\n",
       "      <td>Q9UK53</td>\n",
       "      <td>17949449</td>\n",
       "      <td>phosphorylation</td>\n",
       "      <td>0.837153</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            abstract  \\\n",
       "45                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Previous genetic linkage studies identified a locus for susceptibility to prostate cancer called HPCX at Xq27. The candidate region contains two clusters of SPANX genes. The first cluster called SPANX-A/D includes SPANX-A1, SPANX-A2, SPANX-B, SPANX-C, and SPANX-D; the second cluster called SPANX-N includes SPANX-N1, SPANX-N2, SPANX-N3, and SPANX-N4. The SPANX genes encode cancer-testis (CT) specific antigens. Previous studies identified SPANX-B and SPANX-D variants produced by gene conversion events, none of which are associated with X-linked prostate cancer.   \n",
       "72                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      SCA6 is a slowly progressive, late-onset cerebellar ataxia due to a trinucleotide expansion in the CACNA1A gene. We describe two unrelated cases that presented with Parkinsonism and cerebellar ataxia. One case was L-dopa-responsive with a pattern of (18)F-dopa uptake similar to Parkinson's disease, and the second case was not L-dopa-responsive and had an atypical pattern of nigrostriatal dysfunction. We suggest that SCA6, in common with SCA2 and SCA3, may be associated with Parkinsonism attributable to nigral loss and dopaminergic dysfunction. Moreover, isolated cases may be confused with multiple system atrophy.   \n",
       "96         Induction of drug-metabolizing microsomal cytochromes p450 (p450s) results in a striking proliferation of the smooth endoplasmic reticulum (ER). Overexpression of P450s in yeast and cultured cells produces a similar response. The signals mediating this process are not known but probably involve signal transduction pathways involved in the unfolded protein response (UPR) or the ER overload response (EOR). We have examined the temporal response of specific genes in these pathways and genes globally to overexpression of p450 in cultured cells. Activity of NFkappaB, an EOR component, was substantially increased by overexpression of full-length p450 2C2 or a chimera with the 28-amino acid signal anchor sequence of p450 2C2 in HepG2 cells, and the activation correlated temporally with the accumulation of p450 in the cells. In the UPR pathway, activation of the transcription factor XBP1 by IRE1 also correlated with the accumulation of p450 in the cells, and in contrast, maximum activation of the BiP/grp78 promoter preceded the accumulation. Differential effects of expression of p450 on apoptosis were observed in nonhepatic COS1 and hepatic HepG2 cells. In COS1 cells, apoptosis was induced, and this correlated with sustained activation of the pro-apoptotic JNK pathway, induction of CHOP, and an absence of the increased NFkappaB activity. In HepG2 cells, JNK was only transiently activated, and CHOP expression was not induced. As assessed by DNA microarray analysis, up-regulation of signaling genes was predominant including those involved in anti-apoptosis and ER stress. These results suggest that both the EOR and UPR pathways are involved in the cellular response to induction of p450 expression and that in hepatic cells genes are also induced to block apoptosis, which may be a physiologically relevant response to prevent cell death during xenobiotic induced expression of p450 in the liver.   \n",
       "35                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Ag-dependent activation of naive T cells induces dramatic changes in cellular metabolism that are essential for cell growth, division, and differentiation. In recent years, the serine/threonine kinase mechanistic target of rapamycin (mTOR) has emerged as a key integrator of signaling pathways that regulate these metabolic processes. However, the role of specific downstream effectors of mTOR function in T cells is poorly understood. Ribosomal protein S6 (rpS6) is an essential component of the ribosome and is inducibly phosphorylated following mTOR activation in eukaryotic cells. In the current work, we addressed the role of phosphorylation of rpS6 as an effector of mTOR function in T cell development, growth, proliferation, and differentiation using knockin and TCR transgenic mice. Surprisingly, we demonstrate that rpS6 phosphorylation is not required for any of these processes either in vitro or in vivo. Indeed, rpS6 knockin mice are completely sensitive to the inhibitory effects of rapamycin and an S6 kinase 1 (S6K1)-specific inhibitor on T cell activation and proliferation. These results place the mTOR complex 1-S6K1 axis as a crucial determinant of T cell activation independently of its ability to regulate rpS6 phosphorylation.    \n",
       "78                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Kaposi's sarcoma-associated herpesvirus (KSHV) uses several strategies to counteract the interferon (IFN) system. In this study, the relationship of the protein LANA2 from KSHV to the IFN-activated protein kinase (PKR) and 2-5A system was analysed. It was found that LANA2 could not abrogate apoptosis or RNA degradation mediated by the 2-5A system. However, expression of LANA2 inhibited apoptosis triggered by PKR. LANA2 also counteracted the PKR-mediated inhibition of protein synthesis and partially blocked PKR-induced phosphorylation of eIF-2alpha. Analysis of PKR-induced activation of caspases 3 and 9 revealed that LANA2 abrogated activation of caspase 3 but not of caspase 9. These findings show that LANA2 is able to interfere with downstream events triggered by PKR. Hence, LANA2 should be considered as a KSHV defence protein against IFN.   \n",
       "98                                                                                                                                                                                                                                                                                                                                                                                                                                The eukaryotic transcript elongation factor TFIIS is encoded by a nonessential gene, PPR2, in Saccharomyces cerevisiae. Disruptions of PPR2 are lethal in conjunction with a disruption in the nonessential gene TAF14/TFG3. While investigating which of the Taf14p-containing complexes may be responsible for the synthetic lethality between ppr2Delta and taf14Delta, we discovered genetic interactions between PPR2 and both TFG1 and TFG2 encoding the two larger subunits of the TFIIF complex that also contains Taf14p. Mutant alleles of tfg1 or tfg2 that render cells cold sensitive have improved growth at low temperature in the absence of TFIIS. Remarkably, the amino-terminal 130 amino acids of TFIIS, which are dispensable for the known in vitro and in vivo activities of TFIIS, are required to complement the lethality in taf14Delta ppr2Delta cells. Analyses of deletion and chimeric gene constructs of PPR2 implicate contributions by different regions of this N-terminal domain. No strong common phenotypes were identified for the ppr2Delta and taf14Delta strains, implying that the proteins are not functionally redundant. Instead, the absence of Taf14p in the cell appears to create a dependence on an undefined function of TFIIS mediated by its N-terminal region. This region of TFIIS is also at least in part responsible for the deleterious effect of TFIIS on tfg1 or tfg2 cold-sensitive cells. Together, these results suggest a physiologically relevant functional connection between TFIIS and TFIIF.   \n",
       "107                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Axin2 is a negative regulator of Wnt/beta-catenin signalling with roles in early development and tumour suppression. Axin2 is induced by E2F1 and therefore acts as a point of cross-talk between the pRb/E2F and Wnt/beta-catenin pathways: two of the most frequently deregulated pathways in human cancers. In this study, we show that E2F1 up-regulates axin2 by two independent mechanisms. The human axin2 gene allows transcription of messages with three different 5' untranslated regions and in the first mechanism E2F1 directly activates the transcription of only one of these species by acting at canonical E2F binding sites. Second, E2F1 induces stabilisation of axin2 mRNAs. We discuss this regulation with respect to other known E2F targets.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   Retinitis pigmentosa (RP) is a group of genetically and clinically heterogeneous retinopathies, some of which have been shown to result from mutations in two different known retinal genes, rhodopsin (3q) and peripherin-rds (6p). Three additional anonymous loci at 7p, 7q and pericentric 8 have been implicated by linkage studies. There are still, however, a few families in which all known loci have been excluded. In this report we present data indicating a location, on the short arm of chromosome 17, for the autosomal dominant RP (ADRP) locus in a large South African (SA) family of British ancestry. Positive two-point lod scores have been obtained for nine markers (D17S938, Z = 5.43; D17S796, Z = 4.82; D17S849, Z = 3.6; D17S786, Z = 3.55; TP53, Z = 3.55; D17S578, Z = 3.29; D17S960, Z = 3.16; D17S926, Z = 1.51; D17S804, Z = 0.47 all at theta = 0.10 except D17S804 and D17S926, theta = 0.20). These data provide definitive evidence for the localization of an ADRP gene on chromosome 17p. The human recoverin gene has been localized to 17p13.1 and was consequently a prime candidate for ADRP in the family studied. However, mutation screening of the three exons of this gene failed to produce any evidence of recoverin being the gene involved in the pathogenesis of ADRP in this SA family.   \n",
       "21                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   The zinc-finger protein A20 has crucial physiological functions as a dual inhibitor of nuclear factor-B (NF-B) activation and apoptosis in tumor necrosis factor (TNF) receptor 1 signaling pathway. Although the molecular basis for the anti-NF-B function of A20 has been well elucidated, the anti-apoptotic function of A20 is largely unknown. Here, we report a novel mechanism underlying the anti-apoptotic function of A20: A20 blocks TNF-induced apoptosis through suppression of c-jun N-terminal kinase (JNK) by targeting apoptosis signal-regulating kinase1 (ASK1). First, the ectopic expression of A20 drastically inhibits TNF-induced JNK activation and apoptosis in multiple cell types including those deficient of NF-B activation. Unexpectedly, the blunting effect of A20 on TNF-induced JNK activation is not mediated by affecting the TNFR1 signaling complex formation. Instead, A20 interacts with ASK1, an important MAPKK kinase in the JNK signaling cascade. More importantly, overexpression of wild-type A20, but not of mutant A20 (ZnF4; C624A, C627A), promotes degradation of the ASK1 through the ubiquitin-proteasome system. Taken together, the results from this study reveal a novel anti-apoptotic mechanism of A20 in TNF signaling pathway: A20 binds to ASK1 and mediates ASK1 degradation, leading to suppression of JNK activation and eventually blockage of apoptosis.   \n",
       "26                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   Beta-amyloid accumulates around neurons in Alzheimer's disease and is thought to contribute to the neurodegenerative process. This study examined the role of the tumour suppressor protein, p53, in the neurodegenerative pathway, with focus on the interaction of p53 with the lysosomal system. beta-Amyloid increased expression of p53 and its transcription target, Bax, in cultured cortical neurons. In addition, A beta increased the association of phospho-p53(ser15) with the lysosomal compartment and this correlated with destabilization of the lysosomal membrane and a concomitant increase in cytosolic cathepsin-L activity. These effects of beta-amyloid were abolished by the p53 inhibitor, pifithrin-alpha, and siRNA-mediated knockdown of p53, demonstrating that p53 is a critical regulator of lysosomal integrity and the induction of cathepsin-L protease activity. In addition, activation of the apoptotic cascade was abolished by pifithrin-alpha. We conclude that p53 associates with the lysosome to regulate a lysosomal branch of the apoptotic cascade which contributes to beta-amyloid-mediated neurodegeneration.   \n",
       "31                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Dysadherin is a recently discovered cancer-related cell membrane glycoprotein that has an important role in tumor progression and metastasis. We are focusing on the role of dysadherin in E-cadherin downregulation, the different expression patterns of the molecule in cancer of the head and neck and its potential role as a molecular target for future applications in diagnosis, prognosis, and management of the disease.   \n",
       "71   We have demonstrated that caspase-1 is a mediator of both cisplatin-induced acute kidney injury (AKI) and ischemic AKI. As caspase-1 is activated in the inflammasome, we investigated the inflammasome in cisplatin-induced and ischemic AKI. Mice were injected with cisplatin or subjected to bilateral renal pedicle clamping. Immunoblot analysis of whole kidney after cisplatin-induced AKI revealed: 1) an increase in apoptosis-associated Speck-like protein containing a caspase recruitment domain (ASC), the major protein that complexes with nucleotide-binding oligomerization domain, leucine-rich repeat and pyrin domain containing proteins (NLRP) 1 or 3 to form the inflammasome; 2) an increase in caspase-1 activity, caspase-5, and NLRP1, components of the NLRP1 inflammasome; and 3) a trend toward increased NLRP3. To determine whether the NLRP3 inflammasome plays an injurious role in cisplatin-induced AKI, we studied NLRP knockout (NLRP3(-/-)) mice. In cisplatin-induced AKI, the blood urea nitrogen, serum creatinine, acute tubular necrosis score, and tubular apoptosis score were not significantly decreased in NALP3(-/-) mice compared with wild-type mice. We have previously demonstrated the injurious role of caspase-1 in ischemic AKI. NLRP3, but not ASC or NLRP1, is increased in ischemic AKI. NLRP3(-/-) mice with ischemic AKI had significantly lower blood urea nitrogen, serum creatinine, and acute tubular necrosis and apoptosis scores than the wild-type controls. The difference in protection against cisplatin-induced AKI compared with ischemic AKI in NLRP3(-/-) mice was not explained by the differences in proinflammatory cytokines interleukin (IL)-1, IL-6, chemokine (C-X-C motif) ligand 1, or tumor necrosis factor . NLRP3 inflammasome is a mediator of ischemic AKI but not cisplatin-induced AKI, and further investigation of the NLRP1 inflammasome in cisplatin-induced AKI should prove interesting.    \n",
       "13                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    The transcription factor EB (TFEB) plays a pivotal role in the regulation of basic cellular processes, such as lysosomal biogenesis and autophagy. The subcellular localization and activity of TFEB are regulated by mechanistic target of rapamycin (mTOR)-mediated phosphorylation, which occurs at the lysosomal surface. Phosphorylated TFEB is retained in the cytoplasm, whereas dephosphorylated TFEB translocates to the nucleus to induce the transcription of target genes. Thus, a lysosome-to-nucleus signaling pathway regulates cellular energy metabolism through TFEB. Recently, in vivo studies have revealed that TFEB is also involved in physiological processes, such as lipid catabolism. TFEB has attracted a lot of attention owing to its ability to induce the intracellular clearance of pathogenic factors in a variety of murine models of disease, such as Parkinson's and Alzheimer's, suggesting that novel therapeutic strategies could be based on the modulation of TFEB activity. In this Cell Science at a Glance article and accompanying poster, we present an overview of the latest research on TFEB function and its implication in human diseases.   \n",
       "38                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             The targeted therapy for triple-negative breast cancer (TNBC) is a great challenge due to our poor understanding on its molecular etiology. In the present study, our clinical data showed that the expression of G-protein coupled estrogen receptor (GPER) is negatively associated with lymph node metastasis, high-grade tumor and fibronectin (FN) expression while positively associated with the favorable outcome in 135 TNBC patients. In our experimental studies, both the invitro migration and invasion of TNBC cells were inhibited by GPER specific agonist G-1, through the suppression of the epithelial mesenchymal transition (EMT). The G-1 treatment also reduced the phosphorylation, nuclear localization, and transcriptional activities of NF-B. While over expression of NF-B attenuated the action of G-1 in suppressing EMT. Our data further illustrated that the phosphorylation of GSK-3 by PI3K/Akt and ERK1/2 mediated, at least partially, the inhibitory effect of G-1 on NF-B activities. It was further confirmed in a study of MDA-MB-231 tumor xenografts in nude mice. The data showed that G-1 inhibited the invivo growth and invasive potential of TNBC via suppression of EMT. Our present study demonstrated that an activation of GPER pathway elicits tumor suppressive actions on TNBC, and supports the use of G-1 therapeutics for TNBC metastasis.   \n",
       "74                                                                                                                              Lung cancer is the most common cancer as well as the leading cause of cancer-related mortalities worldwide. Macrophages are the most abundant immune cells in primary and metastatic tumors, and contribute to tumor initiation, progression and metastasis. Macrophages have been shown to demonstrate a high level of plasticity, with the ability to undergo dynamic transition between M1 and M2 polarized phenotypes. In the present study, we investigated a pivotal role of miR-130a in macrophage polarization and whether it was associated with poor prognosis in non-small cell lung cancer (NSCLC), using RT-qPCR and western blot analyses. The invitro experiments showed that miRNA-130a was expressed at a higher level in M1 compared to M2 macrophages. The enforced expression of miR-130a in macrophages resulted in a significantly increased production of proinflammatory cytokines, whereas deletion of miR-130a impaired the M2associated gene expression and led to an M1-biased response. Mechanistically, the bioinformatics analysis revealed that proliferator-activated receptor (PPAR) is a potential target of miR-130a. Additionally, the luciferase assay confirmed that PPAR translation was suppressed by miR-130a through the interaction with the 3'UTR of PPAR mRNA. A subsequent analysis revaled that the induction of miR-130a suppressed PPAR protein expression. In NSCLC patients, the results showed that miR-130a downregulation exhibited clinical relevance as it was correlated with poor prognosis and increased tumor stage and metastasis. In addition, miR130a was inversely correlated with the macrophage marker, CD163, and target gene, PPAR. Taken together, the results established miR-130a as a molecular switch during macrophage development and as a potential target for the treatment of NSCLC.    \n",
       "101                                                                                                                                                                                                                                                                                                                  Cystinuria is a frequently inherited metabolic disorder in the Czech population (frequency 1/5,600) caused by a defect in the renal transport of cystine and dibasic amino acids (arginine, lysine and ornithine). The disease is characterized by increased urinary excretion of the amino acids and often leads to recurrent nephrolithiasis. Cystinuria is classified into two subtypes (type I and type non-I). Type I is caused predominantly by mutations in the SLC3A1 gene (2p16.3), encoding heavy subunit (rBAT) of the heterodimeric transporter. Cystinuria non-I type is caused by mutations in the SLC7A9 gene (19q13.1). In this study, we present results of molecular genetic analysis of the SLC3A1 and the SLC7A9 genes in 24 unrelated cystinuria families. Individual exons of the SLC3A1 and SLC7A9 genes were analyzed by direct sequencing. We found ten different mutations in the SLC3A1 gene including six novel ones: three missense mutations (G140R), D179Y and R365P), one splice site mutation (1137-2A>G), one deletion (1515_1516delAA), and one nonsense mutation (Q119X). The most frequent mutation, M467T; was detected in 36% of all type I classified alleles. In the SLC7A9 gene we found six mutations including three new ones: one missense mutation (G319R), one insertion (611_612insA) and one deletion (205_206delTG). One patient was compound heterozygote for one SLC3A1 and one SLC7A9 mutation. Our results confirm that cystinuria is a heterogeneous disorder at the molecular level, and contribute to the understanding of the distribution and frequency of mutations causing cystinuria in the Caucasian population.   \n",
       "54                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    The t(11;14)(q13;q32) translocation with cyclin D1 overexpression commonly is found in multiple myeloma (MM) and in mantle cell lymphoma (MCL). Several reports have shown that p53 mutations in MCL lead to blastoid transformation and a worse prognosis; however, the role of p53 mutations in MM with t(11;14) is unclear.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Understanding the signalling mechanisms controlling inflammatory cytokine production is pivotal to the research of both acute and chronic immune disorders. Tyrosine phosphorylation is one of the earliest events to occur in response to an immune challenge yet the role of specific tyrosine kinases in inflammatory cytokine production has been difficult to ascribe due to conflicting literature. Here we show that the pyrazolo pyrimidine compound PP2, a selective inhibitor of Src family kinases (SFK), can inhibit LPS-induced TNF production as well as a number of other inflammatory cytokines. In addition, we show similar effects of PP2 on cytokine production when induced by other TLRs, (1, 2 and 5-8), indicating that SFK are important common regulators of TLR signalling. PP2 suppressed the activity of both TNF and IL-10 driven reporter genes, suggesting that this activity is mediated at the level of transcription. Interestingly, however, PP2 had no significant effect on the activation of NF-kappaB, or on p42/44 ERK, p46/54 JNK or p38 MAPK phosphorylation. In contrast, PP2 did inhibit AP-1 nuclear accumulation in response to LPS. Taken together, these findings show that the Src kinases are able to control inflammatory cytokine production at the transcriptional level independently of NF-kappaB, and highlight the role of the AP-1 family of transcription factors as downstream mediators of Src kinase action.   \n",
       "67                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 We investigated the effects of the expression of myosin light chain (MLC) isoforms on the Ca2+ sensitivity of isometric force production of demembranated (skinned) fibers of papillary muscle from the left ventricle of three groups: patients with ischemic cardiomyopathy, patients with dilated cardiomyopathy (NYHA IV) and normal human hearts. Expression and phosphorylation of the phosphorylatable MLC isoforms (MLC-2) was equal within all three groups. However, 72% of the patients investigated in this study expressed the atrial essential MLC (ALC-1) in addition to the essential ventricular MLC (VLC-1) ranging between 2.4% and 10.3%. Using fibers from failing hearts, we observed a significant positive correlation between ALC-1 and Ca2+ sensitivity in that the higher the ALC-1 expression the higher the Ca2(+)-sensitivity: pCa50 (Ca2+ required for half-maximal force production) was 5.87 without ALC-1 and 6.08 with 10.3% ALC-1. Fibers from a normal heart (no ALC-1) revealed a pCa50 of 5.85. Isoform and phosphorylation patterns of tropomyosin and troponin I remained unchanged in the patients and normal hearts. Our results suggest that Ca2+ responsiveness and force development of the human heart is regulated by the expression of different MLC-1 isoforms.   \n",
       "56                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Identification of biological markers that may predict response to chemotherapy would allow the individualization of treatment by enabling selection of patients most likely to benefit from chemotherapy. The aims of this study were to determine whether p53 mutation status and p53 and p33(ING1b) protein expression can predict which patients with Dukes' C colorectal cancer following curative surgical resection respond to adjuvant chemotherapy with 5-fluorouracil (5-FU).   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           normalised_abstract  \\\n",
       "45                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   Previous genetic linkage studies identified a locus for susceptibility to prostate cancer called 9566 at Xq27. The candidate region contains two clusters of Q9NS26 genes. The first cluster called 30014;64648 includes Q9NS26, Q9NS26, Q9NS25, Q9NY87, and Q9BXN6; the second cluster called Q9NS26-N includes Q5VSR9, Q5MJ10, Q5MJ09, and Q5MJ08. The Q9NS26 genes encode cancer-testis (CT) specific antigens. Previous studies identified Q9NS25 and Q9BXN6 variants produced by gene conversion events, none of which are associated with X-linked prostate cancer.   \n",
       "72                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         O00555 is a slowly progressive, late-onset cerebellar ataxia due to a trinucleotide expansion in the O00555 gene. We describe two unrelated cases that presented with Parkinsonism and cerebellar ataxia. One case was L-dopa-responsive with a pattern of (18)F-dopa uptake similar to Parkinson's disease, and the second case was not L-dopa-responsive and had an atypical pattern of nigrostriatal dysfunction. We suggest that O00555, in common with Q99700 and P54252, may be associated with Parkinsonism attributable to nigral loss and dopaminergic dysfunction. Moreover, isolated cases may be confused with multiple system atrophy.   \n",
       "96   Induction of drug-metabolizing microsomal cytochromes P20813 (p450s) results in a striking proliferation of the smooth endoplasmic reticulum (ER). Overexpression of P450s in yeast and cultured cells produces a similar response. The signals mediating this process are not known but probably involve signal transduction pathways involved in the unfolded protein response (UPR) or the ER overload response (EOR). We have examined the temporal response of specific genes in these pathways and genes globally to overexpression of P20813 in cultured cells. Activity of P19838, an EOR component, was substantially increased by overexpression of full-length P20813 2C2 or a chimera with the 28-amino acid signal anchor sequence of P20813 2C2 in HepG2 cells, and the activation correlated temporally with the accumulation of P20813 in the cells. In the UPR pathway, activation of the transcription factor P17861 by O75460 also correlated with the accumulation of P20813 in the cells, and in contrast, maximum activation of the P11021/P11021 promoter preceded the accumulation. Differential effects of expression of P20813 on apoptosis were observed in nonhepatic COS1 and hepatic HepG2 cells. In COS1 cells, apoptosis was induced, and this correlated with sustained activation of the pro-apoptotic P45983 pathway, induction of P35638, and an absence of the increased P19838 activity. In HepG2 cells, P45983 was only transiently activated, and P35638 expression was not induced. As assessed by DNA microarray analysis, up-regulation of signaling genes was predominant including those involved in anti-apoptosis and ER stress. These results suggest that both the EOR and UPR pathways are involved in the cellular response to induction of P20813 expression and that in hepatic cells genes are also induced to block apoptosis, which may be a physiologically relevant response to prevent cell death during xenobiotic induced expression of P20813 in the liver.   \n",
       "35                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Ag-dependent activation of naive T cells induces dramatic changes in cellular metabolism that are essential for cell growth, division, and differentiation. In recent years, the serine/threonine kinase P42345 (P42345) has emerged as a key integrator of signaling pathways that regulate these metabolic processes. However, the role of specific downstream effectors of P42345 function in T cells is poorly understood. P62753 (P62753) is an essential component of the ribosome and is inducibly phosphorylated following P42345 activation in eukaryotic cells. In the current work, we addressed the role of phosphorylation of P62753 as an effector of P42345 function in T cell development, growth, proliferation, and differentiation using knockin and 6962 transgenic mice. Surprisingly, we demonstrate that P62753 phosphorylation is not required for any of these processes either in vitro or in vivo. Indeed, P62753 knockin mice are completely sensitive to the inhibitory effects of rapamycin and an P23443 (P23443)-specific inhibitor on T cell activation and proliferation. These results place the P42345 complex 1-P23443 axis as a crucial determinant of T cell activation independently of its ability to regulate P62753 phosphorylation.    \n",
       "78                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Kaposi's sarcoma-associated herpesvirus (KSHV) uses several strategies to counteract the interferon (IFN) system. In this study, the relationship of the protein LANA2 from KSHV to the IFN-activated protein kinase (P19525) and 2-5A system was analysed. It was found that LANA2 could not abrogate apoptosis or RNA degradation mediated by the 2-5A system. However, expression of LANA2 inhibited apoptosis triggered by P19525. LANA2 also counteracted the P19525-mediated inhibition of protein synthesis and partially blocked P19525-induced phosphorylation of Q9BY44. Analysis of P19525-induced activation of caspases 3 and 9 revealed that LANA2 abrogated activation of P42574 but not of P55211. These findings show that LANA2 is able to interfere with downstream events triggered by P19525. Hence, LANA2 should be considered as a KSHV defence protein against IFN.   \n",
       "98                                                                                                                                                                                                                                                                                                                                                                                                                                        The eukaryotic transcript elongation factor Q15560 is encoded by a nonessential gene, 780910, in Saccharomyces cerevisiae. Disruptions of 780910 are lethal in conjunction with a disruption in the nonessential gene TAF14/TFG3. While investigating which of the Taf14p-containing complexes may be responsible for the synthetic lethality between ppr2Delta and taf14Delta, we discovered genetic interactions between 780910 and both TFG1 and TFG2 encoding the two larger subunits of the TFIIF complex that also contains Taf14p. Mutant alleles of tfg1 or tfg2 that render cells cold sensitive have improved growth at low temperature in the absence of Q15560. Remarkably, the amino-terminal 130 amino acids of Q15560, which are dispensable for the known in vitro and in vivo activities of Q15560, are required to complement the lethality in taf14Delta ppr2Delta cells. Analyses of deletion and chimeric gene constructs of 780910 implicate contributions by different regions of this N-terminal domain. No strong common phenotypes were identified for the 780910Delta and taf14Delta strains, implying that the proteins are not functionally redundant. Instead, the absence of Taf14p in the cell appears to create a dependence on an undefined function of Q15560 mediated by its N-terminal region. This region of Q15560 is also at least in part responsible for the deleterious effect of Q15560 on tfg1 or tfg2 cold-sensitive cells. Together, these results suggest a physiologically relevant functional connection between Q15560 and TFIIF.   \n",
       "107                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                Q9Y2T1 is a negative regulator of Wnt/P35222 signalling with roles in early development and tumour suppression. Q9Y2T1 is induced by Q01094 and therefore acts as a point of cross-talk between the P06400/E2F and Wnt/P35222 pathways: two of the most frequently deregulated pathways in human cancers. In this study, we show that Q01094 up-regulates Q9Y2T1 by two independent mechanisms. The human Q9Y2T1 gene allows transcription of messages with three different 5' untranslated regions and in the first mechanism Q01094 directly activates the transcription of only one of these species by acting at canonical E2F binding sites. Second, Q01094 induces stabilisation of Q9Y2T1 mRNAs. We discuss this regulation with respect to other known E2F targets.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Retinitis pigmentosa (RP) is a group of genetically and clinically heterogeneous retinopathies, some of which have been shown to result from mutations in two different known retinal genes, rhodopsin (3q) and peripherin-rds (6p). Three additional anonymous loci at 7p, 7q and pericentric 8 have been implicated by linkage studies. There are still, however, a few families in which all known loci have been excluded. In this report we present data indicating a location, on the short arm of chromosome 17, for the Q99541 (Q99541) locus in a large South African (SA) family of British ancestry. Positive two-point lod scores have been obtained for nine markers (D17S938, Z = 5.43; D17S796, Z = 4.82; D17S849, Z = 3.6; D17S786, Z = 3.55; P04637, Z = 3.55; D17S578, Z = 3.29; D17S960, Z = 3.16; D17S926, Z = 1.51; D17S804, Z = 0.47 all at theta = 0.10 except D17S804 and D17S926, theta = 0.20). These data provide definitive evidence for the localization of an Q99541 gene on chromosome 17p. The human P35243 gene has been localized to 17p13.1 and was consequently a prime candidate for Q99541 in the family studied. However, mutation screening of the three exons of this gene failed to produce any evidence of P35243 being the gene involved in the pathogenesis of Q99541 in this SA family.   \n",
       "21                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  The P21580 has crucial physiological functions as a dual inhibitor of nuclear factor-B (NF-B) activation and apoptosis in tumor necrosis factor (P01375) receptor 1 signaling pathway. Although the molecular basis for the anti-NF-B function of P21580 has been well elucidated, the anti-apoptotic function of P21580 is largely unknown. Here, we report a novel mechanism underlying the anti-apoptotic function of P21580: P21580 blocks P01375-induced apoptosis through suppression of P45983 (P45983) by targeting Q99683 (Q99683). First, the ectopic expression of P21580 drastically inhibits P01375-induced P45983 activation and apoptosis in multiple cell types including those deficient of NF-B activation. Unexpectedly, the blunting effect of P21580 on P01375-induced P45983 activation is not mediated by affecting the P19438 signaling complex formation. Instead, P21580 interacts with Q99683, an important MAPKK kinase in the P45983 signaling cascade. More importantly, overexpression of wild-type P21580, but not of mutant P21580 (7897; C624A, C627A), promotes degradation of the Q99683 through the ubiquitin-proteasome system. Taken together, the results from this study reveal a novel anti-apoptotic mechanism of P21580 in P01375 signaling pathway: P21580 binds to Q99683 and mediates Q99683 degradation, leading to suppression of P45983 activation and eventually blockage of apoptosis.   \n",
       "26                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Beta-amyloid accumulates around neurons in Alzheimer's disease and is thought to contribute to the neurodegenerative process. This study examined the role of the tumour suppressor protein, P04637, in the neurodegenerative pathway, with focus on the interaction of P04637 with the lysosomal system. beta-Amyloid increased expression of P04637 and its transcription target, Q07812, in cultured cortical neurons. In addition, Q9P2R7 increased the association of phospho-P04637(3190208) with the lysosomal compartment and this correlated with destabilization of the lysosomal membrane and a concomitant increase in cytosolic P07711 activity. These effects of beta-amyloid were abolished by the P04637 inhibitor, pifithrin-alpha, and siRNA-mediated knockdown of P04637, demonstrating that P04637 is a critical regulator of lysosomal integrity and the induction of P07711 protease activity. In addition, activation of the apoptotic cascade was abolished by pifithrin-alpha. We conclude that P04637 associates with the lysosome to regulate a lysosomal branch of the apoptotic cascade which contributes to beta-amyloid-mediated neurodegeneration.   \n",
       "31                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     Q96DB9 is a recently discovered cancer-related cell membrane glycoprotein that has an important role in tumor progression and metastasis. We are focusing on the role of Q96DB9 in P12830 downregulation, the different expression patterns of the molecule in cancer of the head and neck and its potential role as a molecular target for future applications in diagnosis, prognosis, and management of the disease.   \n",
       "71                                                                                  We have demonstrated that P29466 is a mediator of both cisplatin-induced acute kidney injury (AKI) and ischemic AKI. As P29466 is activated in the inflammasome, we investigated the inflammasome in cisplatin-induced and ischemic AKI. Mice were injected with cisplatin or subjected to bilateral renal pedicle clamping. Immunoblot analysis of whole kidney after cisplatin-induced AKI revealed: 1) an increase in apoptosis-associated Speck-like protein containing a caspase recruitment domain (ASC), the major protein that complexes with nucleotide-binding oligomerization domain, leucine-rich repeat and pyrin domain containing proteins (NLRP) 1 or 3 to form the inflammasome; 2) an increase in P29466 activity, P51878, and Q9C000, components of the Q9C000 inflammasome; and 3) a trend toward increased Q96P20. To determine whether the Q96P20 inflammasome plays an injurious role in cisplatin-induced AKI, we studied NLRP knockout (Q96P20(-/-)) mice. In cisplatin-induced AKI, the blood urea nitrogen, serum creatinine, acute tubular necrosis score, and tubular apoptosis score were not significantly decreased in Q96P20(-/-) mice compared with wild-type mice. We have previously demonstrated the injurious role of P29466 in ischemic AKI. Q96P20, but not ASC or Q9C000, is increased in ischemic AKI. Q96P20(-/-) mice with ischemic AKI had significantly lower blood urea nitrogen, serum creatinine, and acute tubular necrosis and apoptosis scores than the wild-type controls. The difference in protection against cisplatin-induced AKI compared with ischemic AKI in Q96P20(-/-) mice was not explained by the differences in proinflammatory cytokines interleukin (IL)-1, P05231, P01375 . Q96P20 inflammasome is a mediator of ischemic AKI but not cisplatin-induced AKI, and further investigation of the Q9C000 inflammasome in cisplatin-induced AKI should prove interesting.    \n",
       "13                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    The P19484 (P19484) plays a pivotal role in the regulation of basic cellular processes, such as lysosomal biogenesis and autophagy. The subcellular localization and activity of P19484 are regulated by P42345 (P42345)-mediated phosphorylation, which occurs at the lysosomal surface. Phosphorylated P19484 is retained in the cytoplasm, whereas dephosphorylated P19484 translocates to the nucleus to induce the transcription of target genes. Thus, a lysosome-to-nucleus signaling pathway regulates cellular energy metabolism through P19484. Recently, in vivo studies have revealed that P19484 is also involved in physiological processes, such as lipid catabolism. P19484 has attracted a lot of attention owing to its ability to induce the intracellular clearance of pathogenic factors in a variety of murine models of disease, such as Parkinson's and Alzheimer's, suggesting that novel therapeutic strategies could be based on the modulation of P19484 activity. In this Cell Science at a Glance article and accompanying poster, we present an overview of the latest research on P19484 function and its implication in human diseases.   \n",
       "38                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              The targeted therapy for triple-negative breast cancer (TNBC) is a great challenge due to our poor understanding on its molecular etiology. In the present study, our clinical data showed that the expression of Q99527 (Q99527) is negatively associated with lymph node metastasis, high-grade tumor and fibronectin (FN) expression while positively associated with the favorable outcome in 135 TNBC patients. In our experimental studies, both the invitro migration and invasion of TNBC cells were inhibited by Q99527 specific agonist A0A0G2JNB4, through the suppression of the epithelial mesenchymal transition (EMT). The A0A0G2JNB4 treatment also reduced the phosphorylation, nuclear localization, and transcriptional activities of NF-B. While over expression of NF-B attenuated the action of A0A0G2JNB4 in suppressing EMT. Our data further illustrated that the phosphorylation of GSK-3 by PI3K/P31749 and 5595;5594 mediated, at least partially, the inhibitory effect of A0A0G2JNB4 on NF-B activities. It was further confirmed in a study of MDA-MB-231 tumor xenografts in nude mice. The data showed that A0A0G2JNB4 inhibited the invivo growth and invasive potential of TNBC via suppression of EMT. Our present study demonstrated that an activation of Q99527 pathway elicits tumor suppressive actions on TNBC, and supports the use of A0A0G2JNB4 therapeutics for TNBC metastasis.   \n",
       "74                                                                                                                                                                  Lung cancer is the most common cancer as well as the leading cause of cancer-related mortalities worldwide. Macrophages are the most abundant immune cells in primary and metastatic tumors, and contribute to tumor initiation, progression and metastasis. Macrophages have been shown to demonstrate a high level of plasticity, with the ability to undergo dynamic transition between M1 and M2 polarized phenotypes. In the present study, we investigated a pivotal role of 406919 in macrophage polarization and whether it was associated with poor prognosis in non-small cell lung cancer (NSCLC), using RT-qPCR and western blot analyses. The invitro experiments showed that 406919 was expressed at a higher level in M1 compared to M2 macrophages. The enforced expression of 406919 in macrophages resulted in a significantly increased production of proinflammatory cytokines, whereas deletion of 406919 impaired the M2associated gene expression and led to an M1-biased response. Mechanistically, the bioinformatics analysis revealed that proliferator-activated receptor (Q07869) is a potential target of 406919. Additionally, the luciferase assay confirmed that Q07869 translation was suppressed by 406919 through the interaction with the 3'UTR of Q07869 mRNA. A subsequent analysis revaled that the induction of 406919 suppressed Q07869 protein expression. In NSCLC patients, the results showed that 406919 downregulation exhibited clinical relevance as it was correlated with poor prognosis and increased tumor stage and metastasis. In addition, 406919 was inversely correlated with the macrophage marker, Q86VB7, and target gene, Q07869. Taken together, the results established 406919 as a molecular switch during macrophage development and as a potential target for the treatment of NSCLC.    \n",
       "101                                                                                                                                                                                                                                                                                                                                            Cystinuria is a frequently inherited metabolic disorder in the Czech population (frequency 1/5,600) caused by a defect in the renal transport of cystine and dibasic amino acids (arginine, lysine and ornithine). The disease is characterized by increased urinary excretion of the amino acids and often leads to recurrent nephrolithiasis. Cystinuria is classified into two subtypes (type I and type non-I). Type I is caused predominantly by mutations in the Q07837 gene (2p16.3), encoding heavy subunit (rBAT) of the heterodimeric transporter. Cystinuria non-I type is caused by mutations in the P82251 gene (19q13.1). In this study, we present results of molecular genetic analysis of the Q07837 and the P82251 genes in 24 unrelated cystinuria families. Individual exons of the Q07837 and P82251 genes were analyzed by direct sequencing. We found ten different mutations in the Q07837 gene including six novel ones: three missense mutations (G140R), D179Y and R365P), one splice site mutation (1137-2A>G), one deletion (1515_1516delAA), and one nonsense mutation (Q119X). The most frequent mutation, M467T; was detected in 36% of all type I classified alleles. In the P82251 gene we found six mutations including three new ones: one missense mutation (G319R), one insertion (611_612insA) and one deletion (205_206delTG). One patient was compound heterozygote for one Q07837 and one P82251 mutation. Our results confirm that cystinuria is a heterogeneous disorder at the molecular level, and contribute to the understanding of the distribution and frequency of mutations causing cystinuria in the Caucasian population.   \n",
       "54                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           The t(11;14)(q13;q32) translocation with P24385 overexpression commonly is found in multiple myeloma (MM) and in mantle cell lymphoma (MCL). Several reports have shown that P04637 mutations in MCL lead to blastoid transformation and a worse prognosis; however, the role of P04637 mutations in MM with t(11;14) is unclear.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      Understanding the signalling mechanisms controlling inflammatory cytokine production is pivotal to the research of both acute and chronic immune disorders. Tyrosine phosphorylation is one of the earliest events to occur in response to an immune challenge yet the role of specific tyrosine kinases in inflammatory cytokine production has been difficult to ascribe due to conflicting literature. Here we show that the pyrazolo pyrimidine compound 4888, a selective inhibitor of Src family kinases (SFK), can inhibit LPS-induced P01375 production as well as a number of other inflammatory cytokines. In addition, we show similar effects of 4888 on cytokine production when induced by other TLRs, (1, 2 and 5-8), indicating that SFK are important common regulators of TLR signalling. 4888 suppressed the activity of both P01375 and P22301 driven reporter genes, suggesting that this activity is mediated at the level of transcription. Interestingly, however, 4888 had no significant effect on the activation of P19838, or on Q00532/44 ERK, p46/54 JNK or P28482 MAPK phosphorylation. In contrast, 4888 did inhibit AP-1 nuclear accumulation in response to LPS. Taken together, these findings show that the Src kinases are able to control inflammatory cytokine production at the transcriptional level independently of P19838, and highlight the role of the AP-1 family of transcription factors as downstream mediators of Src kinase action.   \n",
       "67                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        We investigated the effects of the expression of Q15049 (Q15049) isoforms on the Ca2+ sensitivity of isometric force production of demembranated (skinned) fibers of papillary muscle from the left ventricle of three groups: patients with ischemic cardiomyopathy, patients with dilated cardiomyopathy (NYHA IV) and normal human hearts. Expression and phosphorylation of the phosphorylatable Q15049 isoforms (P10916) was equal within all three groups. However, 72% of the patients investigated in this study expressed the atrial essential Q15049 (P12829) in addition to the essential ventricular Q15049 (P08590) ranging between 2.4% and 10.3%. Using fibers from failing hearts, we observed a significant positive correlation between P12829 and Ca2+ sensitivity in that the higher the P12829 expression the higher the Ca2(+)-sensitivity: pCa50 (Ca2+ required for half-maximal force production) was 5.87 without P12829.08 with 10.3% P12829. Fibers from a normal heart (no P12829) revealed a pCa50 of 5.85. Isoform and phosphorylation patterns of tropomyosin and troponin I remained unchanged in the patients and normal hearts. Our results suggest that Ca2+ responsiveness and force development of the human heart is regulated by the expression of different Q15049 isoforms.   \n",
       "56                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Identification of biological markers that may predict response to chemotherapy would allow the individualization of treatment by enabling selection of patients most likely to benefit from chemotherapy. The aims of this study were to determine whether P04637 mutation status and P04637 and Q9UK53(ING1b) protein expression can predict which patients with Dukes' C colorectal cancer following curative surgical resection respond to adjuvant chemotherapy with 5-fluorouracil (5-FU).   \n",
       "\n",
       "    participant1Id participant2Id  pubmedId        predicted  \\\n",
       "45     30014;64648         Q9NS26  17373721  phosphorylation   \n",
       "72          O00555         Q99700  15954136  phosphorylation   \n",
       "96          O75460         P20813  14718536  phosphorylation   \n",
       "35          P42345         P62753  26453749  phosphorylation   \n",
       "78          P19525         Q9BY44  12771415  phosphorylation   \n",
       "98          780910         Q15560  16648643  phosphorylation   \n",
       "107         P06400         Q01094  15766563  phosphorylation   \n",
       "1           P35243         Q99541   7951236  phosphorylation   \n",
       "21          P21580         Q99683  20448643  phosphorylation   \n",
       "26          P04637         Q9P2R7  19059678  phosphorylation   \n",
       "31          P12830         Q96DB9  22331710  phosphorylation   \n",
       "71          P01375         Q96P20  23833276  phosphorylation   \n",
       "13          P19484         P42345  27252382  phosphorylation   \n",
       "38      A0A0G2JNB4         Q99527  26842883  phosphorylation   \n",
       "74          406919         Q07869  26398698  phosphorylation   \n",
       "101         P82251         Q07837  16138908  phosphorylation   \n",
       "54          P04637         P24385  10223569  phosphorylation   \n",
       "1             4888         P19838  17875324  phosphorylation   \n",
       "67          P12829         Q15049   9160869  phosphorylation   \n",
       "56          P04637         Q9UK53  17949449  phosphorylation   \n",
       "\n",
       "     predicted_confidence  \n",
       "45               0.803549  \n",
       "72               0.891431  \n",
       "96               0.842936  \n",
       "35               0.959844  \n",
       "78               0.862227  \n",
       "98               0.919070  \n",
       "107              0.897457  \n",
       "1                0.900106  \n",
       "21               0.928017  \n",
       "26               0.905761  \n",
       "31               0.884789  \n",
       "71               0.885537  \n",
       "13               0.835544  \n",
       "38               0.892953  \n",
       "74               0.808635  \n",
       "101              0.852938  \n",
       "54               0.808685  \n",
       "1                0.812231  \n",
       "67               0.882644  \n",
       "56               0.837153  "
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "full_df[[\"abstract\", \"normalised_abstract\", \"participant1Id\",\"participant2Id\", \"pubmedId\", \"predicted\" ,\"predicted_confidence\" ]].sample(n=20)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
