{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Set up"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 1. Set  up  accounts and role"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!pip install sagemaker==1.39.0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sagemaker\n",
    "import boto3\n",
    "\n",
    "sagemaker_session = sagemaker.Session()\n",
    "account_id =  boto3.client('sts').get_caller_identity().get('Account')\n",
    "region = boto3.session.Session().region_name\n",
    "\n",
    "\n",
    "#role = sagemaker.get_execution_role()\n",
    "role=\"arn:aws:iam::{}:role/service-role/AmazonSageMaker-ExecutionRole-20190118T115449\".format(account_id)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 2. Setup image and instance type"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "pytorch_custom_image_name=\"ppi-extractor:gpu-1.0.0-201910130520\"\n",
    "instance_type = \"ml.p3.8xlarge\" "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "docker_repo = \"{}.dkr.ecr.{}.amazonaws.com/{}\".format(account_id, region, pytorch_custom_image_name)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 3. Configure train/ test and validation datasets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "bucket = \"aegovan-data\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "trainfile = \"s3://{}/aimed/AIMedtrain.json\".format(bucket)\n",
    "#trainfile = \"s3://{}/aimed/AIMedFull.json\".format(bucket)\n",
    "\n",
    "\n",
    "valfile=\"s3://{}/aimed/AIMedval.json\".format(bucket)\n",
    "# trainfile = \"s3://{}/aimed/AIMedtrain_pubmedoverlap.json\".format(bucket)\n",
    "# valfile=\"s3://{}/aimed/AIMedval_pubmedoverlap.json\".format(bucket)\n",
    "#embeddingfile=\"s3://{}/embeddings/PubMed-and-PMC-w2v.bin.txt\".format(bucket)\n",
    "#embeddingfile=\"s3://{}/embeddings/bio_nlp_vec/PubMed-shuffle-win-30.bin.txt\".format(bucket)\n",
    "pretrained_bert=\"s3://{}/embeddings/bert/\".format(bucket)\n",
    "\n",
    "\n",
    "s3_output_path= \"s3://{}/results/\".format(bucket)\n",
    "s3_code_path= \"s3://{}/aimed_bert_code\".format(bucket)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Start training"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "pub_inputs = {\n",
    "    \"train\" : trainfile,\n",
    "    \"val\" :valfile,\n",
    "    \"PRETRAINED_BIOBERT\" : pretrained_bert\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "BertNetworkFactoryhyperparameters = {\n",
    "    \"dataset\":\"PpiAimedDatasetFactory\",\n",
    "    \"network\" :\"RelationExtractorBioBertFactory\",\n",
    "    \"trainfile\":trainfile.split(\"/\")[-1],\n",
    "     \"valfile\":valfile.split(\"/\")[-1],\n",
    "    \"batchsize\": \"8\",\n",
    "    \"epochs\" : \"1000\",   \n",
    "    \"log-level\" : \"INFO\",\n",
    "    \"learningrate\":.00001,\n",
    "    \"earlystoppingpatience\":20\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "metric_definitions = [{\"Name\": \"TrainLoss\",\n",
    "                     \"Regex\": \"###score: train_loss### (\\d*[.]?\\d*)\"}\n",
    "                    ,{\"Name\": \"ValidationLoss\",\n",
    "                     \"Regex\": \"###score: val_loss### (\\d*[.]?\\d*)\"}\n",
    "                    ,{\"Name\": \"TrainFScore\",\n",
    "                     \"Regex\": \"###score: train_fscore### (\\d*[.]?\\d*)\"}\n",
    "                   ,{\"Name\": \"ValidationFScore\",\n",
    "                     \"Regex\": \"###score: val_fscore### (\\d*[.]?\\d*)\"}\n",
    "                    ]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "commit d271cb3afe738bfab69c71df3ce803ca2531f8f8\n",
      "    Fix gpu\n"
     ]
    }
   ],
   "source": [
    "!git log -1 | head -1\n",
    "!git log -1 | head -5 | tail -1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "git_config = {'repo': 'https://github.com/elangovana/PPI-typed-relation-extractor.git',\n",
    "              'branch': 'master',\n",
    "            #  'commit': '58a09e154935248667062a36fdae7d86b86b477c'\n",
    "             }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "hyperparameters = BertNetworkFactoryhyperparameters\n",
    "inputs = pub_inputs \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'batchsize': '8',\n",
       " 'dataset': 'PpiAimedDatasetFactory',\n",
       " 'earlystoppingpatience': 20,\n",
       " 'epochs': '1000',\n",
       " 'learningrate': 1e-05,\n",
       " 'log-level': 'INFO',\n",
       " 'network': 'RelationExtractorBioBertFactory',\n",
       " 'trainfile': 'AIMedtrain.json',\n",
       " 'valfile': 'AIMedval.json'}"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "hyperparameters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'PRETRAINED_BIOBERT': 's3://aegovan-data/embeddings/bert/',\n",
       " 'train': 's3://aegovan-data/aimed/AIMedtrain.json',\n",
       " 'val': 's3://aegovan-data/aimed/AIMedval.json'}"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "inputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sagemaker.pytorch import PyTorch\n",
    "\n",
    "estimator = PyTorch(\n",
    "     #entry_point='main_train_k_fold.py',\n",
    "    entry_point='main_train_bert.py',\n",
    "                    source_dir = 'source/algorithms',\n",
    "                    dependencies =['source/algorithms', 'source/datasets', 'source/preprocessor', 'source/modelnetworks','source/trainpipelinesbuilders'],\n",
    "                    role=role,\n",
    "                    framework_version =\"1.0.0\",\n",
    "                    py_version='py3',\n",
    "                    git_config= git_config,\n",
    "                    image_name= docker_repo,\n",
    "                    train_instance_count=1,\n",
    "                    train_instance_type=instance_type,\n",
    "                    hyperparameters =hyperparameters,\n",
    "                    output_path=s3_output_path,\n",
    "                    metric_definitions=metric_definitions,\n",
    "                    #train_use_spot_instances = True\n",
    "                    train_volume_size=30,\n",
    "                    code_location=s3_code_path,\n",
    "                    base_job_name =\"aimed-ppi-bert-extractor\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2019-10-13 06:14:12 Starting - Starting the training job...\n",
      "2019-10-13 06:14:14 Starting - Launching requested ML instances...\n",
      "2019-10-13 06:15:10 Starting - Preparing the instances for training......\n",
      "2019-10-13 06:16:27 Downloading - Downloading input data\n",
      "2019-10-13 06:16:27 Training - Downloading the training image............\n",
      "2019-10-13 06:18:44 Training - Training image download completed. Training in progress.\u001b[31mbash: cannot set terminal process group (-1): Inappropriate ioctl for device\u001b[0m\n",
      "\u001b[31mbash: no job control in this shell\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:46,188 sagemaker-containers INFO     Imported framework sagemaker_pytorch_container.training\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:46,232 sagemaker_pytorch_container.training INFO     Block until all host DNS lookups succeed.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:49,249 sagemaker_pytorch_container.training INFO     Invoking user training script.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:49,691 sagemaker-containers INFO     Module main_train_bert does not provide a setup.py. \u001b[0m\n",
      "\u001b[31mGenerating setup.py\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:49,692 sagemaker-containers INFO     Generating setup.cfg\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:49,692 sagemaker-containers INFO     Generating MANIFEST.in\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:49,692 sagemaker-containers INFO     Installing module with the following command:\u001b[0m\n",
      "\u001b[31m/usr/bin/python -m pip install -U . \u001b[0m\n",
      "\u001b[31mProcessing /opt/ml/code\u001b[0m\n",
      "\u001b[31mBuilding wheels for collected packages: main-train-bert\n",
      "  Running setup.py bdist_wheel for main-train-bert: started\n",
      "  Running setup.py bdist_wheel for main-train-bert: finished with status 'done'\n",
      "  Stored in directory: /tmp/pip-ephem-wheel-cache-mbc0vrrr/wheels/35/24/16/37574d11bf9bde50616c67372a334f94fa8356bc7164af8ca3\u001b[0m\n",
      "\u001b[31mSuccessfully built main-train-bert\u001b[0m\n",
      "\u001b[31mInstalling collected packages: main-train-bert\u001b[0m\n",
      "\u001b[31mSuccessfully installed main-train-bert-1.0.0\u001b[0m\n",
      "\u001b[31mYou are using pip version 18.1, however version 19.2.3 is available.\u001b[0m\n",
      "\u001b[31mYou should consider upgrading via the 'pip install --upgrade pip' command.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:51,887 sagemaker-containers INFO     Invoking user script\n",
      "\u001b[0m\n",
      "\u001b[31mTraining Env:\n",
      "\u001b[0m\n",
      "\u001b[31m{\n",
      "    \"additional_framework_parameters\": {},\n",
      "    \"channel_input_dirs\": {\n",
      "        \"val\": \"/opt/ml/input/data/val\",\n",
      "        \"PRETRAINED_BIOBERT\": \"/opt/ml/input/data/PRETRAINED_BIOBERT\",\n",
      "        \"train\": \"/opt/ml/input/data/train\"\n",
      "    },\n",
      "    \"current_host\": \"algo-1\",\n",
      "    \"framework_module\": \"sagemaker_pytorch_container.training:main\",\n",
      "    \"hosts\": [\n",
      "        \"algo-1\"\n",
      "    ],\n",
      "    \"hyperparameters\": {\n",
      "        \"learningrate\": 1e-05,\n",
      "        \"trainfile\": \"AIMedtrain.json\",\n",
      "        \"batchsize\": \"8\",\n",
      "        \"network\": \"RelationExtractorBioBertFactory\",\n",
      "        \"log-level\": \"INFO\",\n",
      "        \"valfile\": \"AIMedval.json\",\n",
      "        \"dataset\": \"PpiAimedDatasetFactory\",\n",
      "        \"epochs\": \"1000\",\n",
      "        \"earlystoppingpatience\": 20\n",
      "    },\n",
      "    \"input_config_dir\": \"/opt/ml/input/config\",\n",
      "    \"input_data_config\": {\n",
      "        \"val\": {\n",
      "            \"TrainingInputMode\": \"File\",\n",
      "            \"S3DistributionType\": \"FullyReplicated\",\n",
      "            \"RecordWrapperType\": \"None\"\n",
      "        },\n",
      "        \"PRETRAINED_BIOBERT\": {\n",
      "            \"TrainingInputMode\": \"File\",\n",
      "            \"S3DistributionType\": \"FullyReplicated\",\n",
      "            \"RecordWrapperType\": \"None\"\n",
      "        },\n",
      "        \"train\": {\n",
      "            \"TrainingInputMode\": \"File\",\n",
      "            \"S3DistributionType\": \"FullyReplicated\",\n",
      "            \"RecordWrapperType\": \"None\"\n",
      "        }\n",
      "    },\n",
      "    \"input_dir\": \"/opt/ml/input\",\n",
      "    \"is_master\": true,\n",
      "    \"job_name\": \"aimed-ppi-bert-extractor-2019-10-13-06-13-45-856\",\n",
      "    \"log_level\": 20,\n",
      "    \"master_hostname\": \"algo-1\",\n",
      "    \"model_dir\": \"/opt/ml/model\",\n",
      "    \"module_dir\": \"s3://aegovan-data/aimed_bert_code/aimed-ppi-bert-extractor-2019-10-13-06-13-45-856/source/sourcedir.tar.gz\",\n",
      "    \"module_name\": \"main_train_bert\",\n",
      "    \"network_interface_name\": \"eth0\",\n",
      "    \"num_cpus\": 32,\n",
      "    \"num_gpus\": 4,\n",
      "    \"output_data_dir\": \"/opt/ml/output/data\",\n",
      "    \"output_dir\": \"/opt/ml/output\",\n",
      "    \"output_intermediate_dir\": \"/opt/ml/output/intermediate\",\n",
      "    \"resource_config\": {\n",
      "        \"current_host\": \"algo-1\",\n",
      "        \"hosts\": [\n",
      "            \"algo-1\"\n",
      "        ],\n",
      "        \"network_interface_name\": \"eth0\"\n",
      "    },\n",
      "    \"user_entry_point\": \"main_train_bert.py\"\u001b[0m\n",
      "\u001b[31m}\n",
      "\u001b[0m\n",
      "\u001b[31mEnvironment variables:\n",
      "\u001b[0m\n",
      "\u001b[31mSM_HOSTS=[\"algo-1\"]\u001b[0m\n",
      "\u001b[31mSM_NETWORK_INTERFACE_NAME=eth0\u001b[0m\n",
      "\u001b[31mSM_HPS={\"batchsize\":\"8\",\"dataset\":\"PpiAimedDatasetFactory\",\"earlystoppingpatience\":20,\"epochs\":\"1000\",\"learningrate\":1e-05,\"log-level\":\"INFO\",\"network\":\"RelationExtractorBioBertFactory\",\"trainfile\":\"AIMedtrain.json\",\"valfile\":\"AIMedval.json\"}\u001b[0m\n",
      "\u001b[31mSM_USER_ENTRY_POINT=main_train_bert.py\u001b[0m\n",
      "\u001b[31mSM_FRAMEWORK_PARAMS={}\u001b[0m\n",
      "\u001b[31mSM_RESOURCE_CONFIG={\"current_host\":\"algo-1\",\"hosts\":[\"algo-1\"],\"network_interface_name\":\"eth0\"}\u001b[0m\n",
      "\u001b[31mSM_INPUT_DATA_CONFIG={\"PRETRAINED_BIOBERT\":{\"RecordWrapperType\":\"None\",\"S3DistributionType\":\"FullyReplicated\",\"TrainingInputMode\":\"File\"},\"train\":{\"RecordWrapperType\":\"None\",\"S3DistributionType\":\"FullyReplicated\",\"TrainingInputMode\":\"File\"},\"val\":{\"RecordWrapperType\":\"None\",\"S3DistributionType\":\"FullyReplicated\",\"TrainingInputMode\":\"File\"}}\u001b[0m\n",
      "\u001b[31mSM_OUTPUT_DATA_DIR=/opt/ml/output/data\u001b[0m\n",
      "\u001b[31mSM_CHANNELS=[\"PRETRAINED_BIOBERT\",\"train\",\"val\"]\u001b[0m\n",
      "\u001b[31mSM_CURRENT_HOST=algo-1\u001b[0m\n",
      "\u001b[31mSM_MODULE_NAME=main_train_bert\u001b[0m\n",
      "\u001b[31mSM_LOG_LEVEL=20\u001b[0m\n",
      "\u001b[31mSM_FRAMEWORK_MODULE=sagemaker_pytorch_container.training:main\u001b[0m\n",
      "\u001b[31mSM_INPUT_DIR=/opt/ml/input\u001b[0m\n",
      "\u001b[31mSM_INPUT_CONFIG_DIR=/opt/ml/input/config\u001b[0m\n",
      "\u001b[31mSM_OUTPUT_DIR=/opt/ml/output\u001b[0m\n",
      "\u001b[31mSM_NUM_CPUS=32\u001b[0m\n",
      "\u001b[31mSM_NUM_GPUS=4\u001b[0m\n",
      "\u001b[31mSM_MODEL_DIR=/opt/ml/model\u001b[0m\n",
      "\u001b[31mSM_MODULE_DIR=s3://aegovan-data/aimed_bert_code/aimed-ppi-bert-extractor-2019-10-13-06-13-45-856/source/sourcedir.tar.gz\u001b[0m\n",
      "\u001b[31mSM_TRAINING_ENV={\"additional_framework_parameters\":{},\"channel_input_dirs\":{\"PRETRAINED_BIOBERT\":\"/opt/ml/input/data/PRETRAINED_BIOBERT\",\"train\":\"/opt/ml/input/data/train\",\"val\":\"/opt/ml/input/data/val\"},\"current_host\":\"algo-1\",\"framework_module\":\"sagemaker_pytorch_container.training:main\",\"hosts\":[\"algo-1\"],\"hyperparameters\":{\"batchsize\":\"8\",\"dataset\":\"PpiAimedDatasetFactory\",\"earlystoppingpatience\":20,\"epochs\":\"1000\",\"learningrate\":1e-05,\"log-level\":\"INFO\",\"network\":\"RelationExtractorBioBertFactory\",\"trainfile\":\"AIMedtrain.json\",\"valfile\":\"AIMedval.json\"},\"input_config_dir\":\"/opt/ml/input/config\",\"input_data_config\":{\"PRETRAINED_BIOBERT\":{\"RecordWrapperType\":\"None\",\"S3DistributionType\":\"FullyReplicated\",\"TrainingInputMode\":\"File\"},\"train\":{\"RecordWrapperType\":\"None\",\"S3DistributionType\":\"FullyReplicated\",\"TrainingInputMode\":\"File\"},\"val\":{\"RecordWrapperType\":\"None\",\"S3DistributionType\":\"FullyReplicated\",\"TrainingInputMode\":\"File\"}},\"input_dir\":\"/opt/ml/input\",\"is_master\":true,\"job_name\":\"aimed-ppi-bert-extractor-2019-10-13-06-13-45-856\",\"log_level\":20,\"master_hostname\":\"algo-1\",\"model_dir\":\"/opt/ml/model\",\"module_dir\":\"s3://aegovan-data/aimed_bert_code/aimed-ppi-bert-extractor-2019-10-13-06-13-45-856/source/sourcedir.tar.gz\",\"module_name\":\"main_train_bert\",\"network_interface_name\":\"eth0\",\"num_cpus\":32,\"num_gpus\":4,\"output_data_dir\":\"/opt/ml/output/data\",\"output_dir\":\"/opt/ml/output\",\"output_intermediate_dir\":\"/opt/ml/output/intermediate\",\"resource_config\":{\"current_host\":\"algo-1\",\"hosts\":[\"algo-1\"],\"network_interface_name\":\"eth0\"},\"user_entry_point\":\"main_train_bert.py\"}\u001b[0m\n",
      "\u001b[31mSM_USER_ARGS=[\"--batchsize\",\"8\",\"--dataset\",\"PpiAimedDatasetFactory\",\"--earlystoppingpatience\",\"20\",\"--epochs\",\"1000\",\"--learningrate\",\"1e-05\",\"--log-level\",\"INFO\",\"--network\",\"RelationExtractorBioBertFactory\",\"--trainfile\",\"AIMedtrain.json\",\"--valfile\",\"AIMedval.json\"]\u001b[0m\n",
      "\u001b[31mSM_OUTPUT_INTERMEDIATE_DIR=/opt/ml/output/intermediate\u001b[0m\n",
      "\u001b[31mSM_CHANNEL_VAL=/opt/ml/input/data/val\u001b[0m\n",
      "\u001b[31mSM_CHANNEL_PRETRAINED_BIOBERT=/opt/ml/input/data/PRETRAINED_BIOBERT\u001b[0m\n",
      "\u001b[31mSM_CHANNEL_TRAIN=/opt/ml/input/data/train\u001b[0m\n",
      "\u001b[31mSM_HP_LEARNINGRATE=1e-05\u001b[0m\n",
      "\u001b[31mSM_HP_TRAINFILE=AIMedtrain.json\u001b[0m\n",
      "\u001b[31mSM_HP_BATCHSIZE=8\u001b[0m\n",
      "\u001b[31mSM_HP_NETWORK=RelationExtractorBioBertFactory\u001b[0m\n",
      "\u001b[31mSM_HP_LOG-LEVEL=INFO\u001b[0m\n",
      "\u001b[31mSM_HP_VALFILE=AIMedval.json\u001b[0m\n",
      "\u001b[31mSM_HP_DATASET=PpiAimedDatasetFactory\u001b[0m\n",
      "\u001b[31mSM_HP_EPOCHS=1000\u001b[0m\n",
      "\u001b[31mSM_HP_EARLYSTOPPINGPATIENCE=20\u001b[0m\n",
      "\u001b[31mPYTHONPATH=/usr/local/bin:/usr/lib/python36.zip:/usr/lib/python3.6:/usr/lib/python3.6/lib-dynload:/usr/local/lib/python3.6/dist-packages:/usr/lib/python3/dist-packages\n",
      "\u001b[0m\n",
      "\u001b[31mInvoking script with the following command:\n",
      "\u001b[0m\n",
      "\u001b[31m/usr/bin/python -m main_train_bert --batchsize 8 --dataset PpiAimedDatasetFactory --earlystoppingpatience 20 --epochs 1000 --learningrate 1e-05 --log-level INFO --network RelationExtractorBioBertFactory --trainfile AIMedtrain.json --valfile AIMedval.json\n",
      "\n",
      "\u001b[0m\n",
      "\u001b[31m['--batchsize', '8', '--learningrate', '1e-05']\u001b[0m\n",
      "\u001b[31m{'dataset': 'PpiAimedDatasetFactory', 'network': 'RelationExtractorBioBertFactory', 'trainfile': 'AIMedtrain.json', 'traindir': '/opt/ml/input/data/train', 'valfile': 'AIMedval.json', 'valdir': '/opt/ml/input/data/val', 'pretrained_biobert_dir': '/opt/ml/input/data/PRETRAINED_BIOBERT', 'outdir': '/opt/ml/output/data', 'modeldir': '/opt/ml/model', 'epochs': 1000, 'earlystoppingpatience': 20, 'interaction_type': None, 'log_level': 'INFO'}\u001b[0m\n",
      "\u001b[31m{'batchsize': '8', 'learningrate': '1e-05'}\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:54,482 - trainpipelinesbuilders.BertTrainInferenceBuilder - INFO - Retrieving key batchsize with default 32, found 8\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:54,482 - trainpipelinesbuilders.BertTrainInferenceBuilder - INFO - Retrieving key pretrained_biobert_dir with default None, found /opt/ml/input/data/PRETRAINED_BIOBERT\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:54,483 - pytorch_pretrained_bert.tokenization - INFO - loading vocabulary file /opt/ml/input/data/PRETRAINED_BIOBERT/vocab.txt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:54,536 - pytorch_pretrained_bert.tokenization - INFO - loading vocabulary file /opt/ml/input/data/PRETRAINED_BIOBERT/vocab.txt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:54,576 - modelnetworks.RelationExtractorBioBertFactory - INFO - Retrieving key pretrained_biobert_dir with default None, found /opt/ml/input/data/PRETRAINED_BIOBERT\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:54,576 - pytorch_pretrained_bert.modeling - INFO - loading archive file /opt/ml/input/data/PRETRAINED_BIOBERT\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:54,577 - pytorch_pretrained_bert.modeling - INFO - Model config {\n",
      "  \"attention_probs_dropout_prob\": 0.1,\n",
      "  \"hidden_act\": \"gelu\",\n",
      "  \"hidden_dropout_prob\": 0.1,\n",
      "  \"hidden_size\": 768,\n",
      "  \"initializer_range\": 0.02,\n",
      "  \"intermediate_size\": 3072,\n",
      "  \"max_position_embeddings\": 512,\n",
      "  \"num_attention_heads\": 12,\n",
      "  \"num_hidden_layers\": 12,\n",
      "  \"type_vocab_size\": 2,\n",
      "  \"vocab_size\": 28996\u001b[0m\n",
      "\u001b[31m}\n",
      "\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,332 - pytorch_pretrained_bert.modeling - INFO - Weights of BertForSequenceClassification not initialized from pretrained model: ['classifier.weight', 'classifier.bias']\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,332 - pytorch_pretrained_bert.modeling - INFO - Weights from pretrained model not used in BertForSequenceClassification: ['cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.seq_relationship.weight', 'cls.seq_relationship.bias']\u001b[0m\n",
      "\u001b[31mBertForSequenceClassification(\n",
      "  (bert): BertModel(\n",
      "    (embeddings): BertEmbeddings(\n",
      "      (word_embeddings): Embedding(28996, 768, padding_idx=0)\n",
      "      (position_embeddings): Embedding(512, 768)\n",
      "      (token_type_embeddings): Embedding(2, 768)\n",
      "      (LayerNorm): BertLayerNorm()\n",
      "      (dropout): Dropout(p=0.1)\n",
      "    )\n",
      "    (encoder): BertEncoder(\n",
      "      (layer): ModuleList(\n",
      "        (0): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (1): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (2): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (3): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (4): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (5): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (6): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (7): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (8): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (9): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (10): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "        (11): BertLayer(\n",
      "          (attention): BertAttention(\n",
      "            (self): BertSelfAttention(\n",
      "              (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "            (output): BertSelfOutput(\n",
      "              (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (intermediate): BertIntermediate(\n",
      "            (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "          )\n",
      "          (output): BertOutput(\n",
      "            (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "            (LayerNorm): BertLayerNorm()\n",
      "            (dropout): Dropout(p=0.1)\n",
      "          )\n",
      "        )\n",
      "      )\n",
      "    )\n",
      "    (pooler): BertPooler(\n",
      "      (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "      (activation): Tanh()\n",
      "    )\n",
      "  )\n",
      "  (dropout): Dropout(p=0.1)\n",
      "  (classifier): Linear(in_features=768, out_features=2, bias=True)\u001b[0m\n",
      "\u001b[31m)\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,334 - trainpipelinesbuilders.BertTrainInferenceBuilder - INFO - Using model <class 'modelnetworks.RelationExtractorBioBert.RelationExtractorBioBert'>\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,335 - trainpipelinesbuilders.BertTrainInferenceBuilder - INFO - \u001b[0m\n",
      "\u001b[31mRelationExtractorBioBert(\n",
      "  (model): BertForSequenceClassification(\n",
      "    (bert): BertModel(\n",
      "      (embeddings): BertEmbeddings(\n",
      "        (word_embeddings): Embedding(28996, 768, padding_idx=0)\n",
      "        (position_embeddings): Embedding(512, 768)\n",
      "        (token_type_embeddings): Embedding(2, 768)\n",
      "        (LayerNorm): BertLayerNorm()\n",
      "        (dropout): Dropout(p=0.1)\n",
      "      )\n",
      "      (encoder): BertEncoder(\n",
      "        (layer): ModuleList(\n",
      "          (0): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (1): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (2): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (3): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (4): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (5): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (6): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (7): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (8): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (9): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (10): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "          (11): BertLayer(\n",
      "            (attention): BertAttention(\n",
      "              (self): BertSelfAttention(\n",
      "                (query): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (key): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (value): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "              (output): BertSelfOutput(\n",
      "                (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "                (LayerNorm): BertLayerNorm()\n",
      "                (dropout): Dropout(p=0.1)\n",
      "              )\n",
      "            )\n",
      "            (intermediate): BertIntermediate(\n",
      "              (dense): Linear(in_features=768, out_features=3072, bias=True)\n",
      "            )\n",
      "            (output): BertOutput(\n",
      "              (dense): Linear(in_features=3072, out_features=768, bias=True)\n",
      "              (LayerNorm): BertLayerNorm()\n",
      "              (dropout): Dropout(p=0.1)\n",
      "            )\n",
      "          )\n",
      "        )\n",
      "      )\n",
      "      (pooler): BertPooler(\n",
      "        (dense): Linear(in_features=768, out_features=768, bias=True)\n",
      "        (activation): Tanh()\n",
      "      )\n",
      "    )\n",
      "    (dropout): Dropout(p=0.1)\n",
      "    (classifier): Linear(in_features=768, out_features=2, bias=True)\n",
      "  )\u001b[0m\n",
      "\u001b[31m)\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,335 - trainpipelinesbuilders.BertTrainInferenceBuilder - INFO - Using loss function <class 'torch.nn.modules.loss.CrossEntropyLoss'>\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,501 - algorithms.BertTrainInferencePipeline - INFO - Train set has 4857 records, val has 365\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,501 - algorithms.transform_berttext_tokenise - INFO - Transforming TransformBertTextTokenise\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,601 - preprocessor.ProteinMasker - WARNING - The text at offset_start 50 must match entity 'interleukin-1beta converting enzyme', but found 'iated by PROTEIN1 converting enzyme' for text \u001b[0m\n",
      "\u001b[31mActivation of interferon-gamma inducing factor mediated by PROTEIN1 converting enzyme.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,602 - preprocessor.ProteinMasker - WARNING - The text at offset_start 3 must match entity 'interleukin 2 receptor', but found 'uble PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mSoluble PROTEIN1 receptor and interleukin 1alpha in toxic epidermal necrolysis: a comparative analysis of serum and blister fluid samples.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,618 - preprocessor.ProteinMasker - WARNING - The text at offset_start 62 must match entity 'IGF-1R beta', but found 'OTEIN1 beta' for text \u001b[0m\n",
      "\u001b[31mInteraction of p85 and N+C-SH2 with IR beta (delta C-43) or PROTEIN1 beta(delta C-43) in which the C-terminal 43 amino acids (including the YXXM motif) were deleted, persisted.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,641 - preprocessor.ProteinMasker - WARNING - The text at offset_start 49 must match entity 'IFN-gamma', but found 'a SC1) th' for text \u001b[0m\n",
      "\u001b[31mA mutant form of human interferon-gamma (IFN-gamma SC1) that binds one PROTEIN1 (IFN-gamma R alpha) has been designed and characterized.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,651 - preprocessor.ProteinMasker - WARNING - The text at offset_start 69 must match entity 'insulin', but found ' phosph' for text \u001b[0m\n",
      "\u001b[31mIn the present study, we demonstrate that gastrin stimulates tyrosine phosphorylation of PROTEIN1 (IRS-1), the major cytoplasmic substrate of the insulin receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,686 - preprocessor.ProteinMasker - WARNING - The text at offset_start 152 must match entity 'EPO binding protein', but found 'IN1 binding protein' for text \u001b[0m\n",
      "\u001b[31mWe employed both random and site-specific mutagenesis to determine the function of amino acid residues in the extracellular domain (referred to as PROTEIN1 binding protein, EBP) of the EPOR.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,688 - preprocessor.ProteinMasker - WARNING - The text at offset_start -15 must match entity 'IL-1beta', but found 'atory cy' for text \u001b[0m\n",
      "\u001b[31mThe PROTEIN2 (ICE) processes the inactive IL-1beta precursor to the proinflammatory cytokine.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,703 - preprocessor.ProteinMasker - WARNING - The text at offset_start 156 must match entity 'IL-2 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mTwo proteins have been described as constituents of the IL-4 receptor, a approximately 140-kDa glycoprotein (IL-4R) and the gamma chain (gammac) of the PROTEIN1 receptor, but neither of these proteins binds IL-13.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,811 - preprocessor.ProteinMasker - WARNING - The text at offset_start 17 must match entity 'FGFR3 IIIc', but found 'TEIN1 IIIc' for text \u001b[0m\n",
      "\u001b[31mExpression of PROTEIN1 IIIc in stably transfected L6 cells conferred growth responses to several members of the FGF family including FGF-1, -2, -4, and -6, while FGFR3 IIIb-expressing cells responded only to FGF-1.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,812 - preprocessor.ProteinMasker - WARNING - The text at offset_start 118 must match entity 'IL-1-receptor', but found 'EIN1-receptor' for text \u001b[0m\n",
      "\u001b[31mThree naturally occurring ligands for the IL-1 receptor (IL1R) exist: the agonists IL-1alpha and IL-1beta and the PROTEIN1-receptor antagonist IL1RA (ref. 2).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,825 - preprocessor.ProteinMasker - WARNING - The text at offset_start 69 must match entity 'dystrophin related protein', but found 'n PROTEIN1 related protein' for text \u001b[0m\n",
      "\u001b[31mIn this recombinant expression system, the dystrophin relatives, human PROTEIN1 related protein (DRP or utrophin) and the 87K postsynaptic protein from Torpedo electric organ, also bind to translated beta 1-syntrophin.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,831 - preprocessor.ProteinMasker - WARNING - The text at offset_start 32 must match entity 'erythropoietin (EPO) receptor', but found 'f the PROTEIN1 (EPO) receptor' for text \u001b[0m\n",
      "\u001b[31mShared and unique determinants of the PROTEIN1 (EPO) receptor are important for binding EPO and EPO mimetic peptide.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,844 - preprocessor.ProteinMasker - WARNING - The text at offset_start 53 must match entity 'Bcl-2-binding protein', but found 'TEIN1-binding protein' for text \u001b[0m\n",
      "\u001b[31mCloning and functional analysis of BAG-1: a novel PROTEIN1-binding protein with anti-cell death activity.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,852 - preprocessor.ProteinMasker - WARNING - The text at offset_start 41 must match entity 'interleukin 13 receptor alpha chain', but found 'human PROTEIN1 receptor alpha chain' for text \u001b[0m\n",
      "\u001b[31mcDNA cloning and characterization of the human PROTEIN1 receptor alpha chain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,870 - preprocessor.ProteinMasker - WARNING - The text at offset_start 112 must match entity 'interleukin 10 (IL-10) receptor', but found 'o the PROTEIN1 (IL-10) receptor' for text \u001b[0m\n",
      "\u001b[31mTo define the mechanism underlying signal transducer and activator of transcription (STAT) protein recruitment to the PROTEIN1 (IL-10) receptor, the STAT proteins activated by IL-10 in different cell populations were first defined using electrophoretic mobility shift assays.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,894 - preprocessor.ProteinMasker - WARNING - The text at offset_start 41 must match entity 'interleukin 13 receptor', but found 'human PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mcDNA cloning and characterization of the human PROTEIN1 receptor alpha chain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,902 - preprocessor.ProteinMasker - WARNING - The text at offset_start 81 must match entity 'MAP kinase kinase', but found 'd PROTEIN1 kinase' for text \u001b[0m\n",
      "\u001b[31mThe v-Raf proteins purified from cells infected with EC12 or 22W viruses activated PROTEIN1 kinase from skeletal muscle in vitro.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,931 - preprocessor.ProteinMasker - WARNING - The text at offset_start 63 must match entity 'erythropoietin receptor', but found 'human PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mIdentification of a critical ligand binding determinant of the human PROTEIN1 receptor. Evidence for common ligand binding motifs in the cytokine receptor family.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,932 - preprocessor.ProteinMasker - WARNING - The text at offset_start 111 must match entity 'IL-10', but found 'to th' for text \u001b[0m\n",
      "\u001b[31mTo define the mechanism underlying signal transducer and activator of transcription (STAT) protein recruitment to the PROTEIN1, the STAT proteins activated by IL-10 in different cell populations were first defined using electrophoretic mobility shift assays.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,990 - preprocessor.ProteinMasker - WARNING - The text at offset_start 208 must match entity 'FGFR3 IIIc', but found 'TEIN1 IIIc' for text \u001b[0m\n",
      "\u001b[31mIn order to characterize the signaling pathway through which FGFR3 regulates cell growth, L6 cells lacking any endogenous FGFR were stably transfected with the two different human isoforms, FGFR3 IIIb and PROTEIN1 IIIc, that result from alternative splicing of exon III of the FGFR3 gene encoding the ligand binding domain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:18:59,991 - preprocessor.ProteinMasker - WARNING - The text at offset_start 59 must match entity 'p75', but found 'e r' for text \u001b[0m\n",
      "\u001b[31mNerve growth factor binds independently to two transmembrane receptors, the PROTEIN1 and the p140trk (trkA) tyrosine kinase receptor, which are both co-expressed in the majority of neuronal cells that respond to NGF.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,058 - preprocessor.ProteinMasker - WARNING - The text at offset_start -5 must match entity 'IFN-gamma', but found '' for text \u001b[0m\n",
      "\u001b[31mPROTEIN1 was derived by linking the two peptide chains of the IFN-gamma dimer by a seven-residue linker and changing His111 in the first chain to an aspartic acid residue.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,125 - preprocessor.ProteinMasker - WARNING - The text at offset_start 82 must match entity 'IL-1 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mA new cytokine-receptor binding mode revealed by the crystal structure of the PROTEIN1 receptor with an antagonist.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,146 - preprocessor.ProteinMasker - WARNING - The text at offset_start 27 must match entity 'erythropoietin receptor', but found 'human PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mMutagenesis studies of the human PROTEIN1 receptor. Establishment of structure-function relationships.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,165 - preprocessor.ProteinMasker - WARNING - The text at offset_start 151 must match entity 'CSB', but found 'and' for text \u001b[0m\n",
      "\u001b[31mInteractions involving the human RNA polymerase II transcription/nucleotide excision repair complex TFIIH, the nucleotide excision repair protein XPG, and PROTEIN1.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,202 - preprocessor.ProteinMasker - WARNING - The text at offset_start 118 must match entity 'IL-1-receptor antagonist', but found 'EIN1-receptor antagonist' for text \u001b[0m\n",
      "\u001b[31mThree naturally occurring ligands for the IL-1 receptor (IL1R) exist: the agonists IL-1alpha and IL-1beta and the PROTEIN1-receptor antagonist IL1RA (ref. 2).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,214 - preprocessor.ProteinMasker - WARNING - The text at offset_start 5 must match entity 'IFN-gamma R alpha', but found ' PROTEIN1 R alpha' for text \u001b[0m\n",
      "\u001b[31mThus, PROTEIN1 R alpha binding does not result in a large conformational change in the AB loop as previously suggested.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,225 - preprocessor.ProteinMasker - WARNING - The text at offset_start 82 must match entity 'IFN-gamma', but found 'pha and P' for text \u001b[0m\n",
      "\u001b[31mThis suggests that one domain of IFN-gamma is sufficient to recruit IFN-gamma R alpha and PROTEIN1 into a complex competent for eliciting biological activity.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,225 - preprocessor.ProteinMasker - WARNING - The text at offset_start 167 must match entity 'insulin receptor', but found 'ROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mIn this study, we used the two-hybrid assay of protein-protein interaction in the yeast Saccharomyces cerevisiae to study the interaction between human IRS-1 and the PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,248 - preprocessor.ProteinMasker - WARNING - The text at offset_start 13 must match entity 'erythropoietin receptor', but found 'f the PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mMutagenesis of the PROTEIN1 receptor (EPOR) permits analysis of the contribution that individual amino acid residues make to erythropoietin (EPO) binding.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,249 - preprocessor.ProteinMasker - WARNING - The text at offset_start 66 must match entity 'GITR ligand', but found 'EIN1 ligand' for text \u001b[0m\n",
      "\u001b[31mWe have identified a new TNF-related ligand, designated human PROTEIN1 ligand (hGITRL), and its human receptor (hGITR), an ortholog of the recently discovered murine glucocorticoid-induced TNFR-related (mGITR) protein [4].\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,250 - preprocessor.ProteinMasker - WARNING - The text at offset_start 19 must match entity 'TcR zeta', but found ' the PRO' for text \u001b[0m\n",
      "\u001b[31mCD5 associates with the PROTEIN1 complex and is rapidly phosphosphorylated on tyrosine residues as a result of TcR zeta/CD3 ligation.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,275 - preprocessor.ProteinMasker - WARNING - The text at offset_start 226 must match entity 'insulin receptor', but found 'ROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mTaken together, our results suggest that IRS-1 may serve as a converging target in the signaling pathways stimulated by receptors that belong to different families, such as the gastrin/CCKB G protein-coupled receptor and the PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,299 - preprocessor.ProteinMasker - WARNING - The text at offset_start 34 must match entity 'CD30 ligand', but found 'EIN1 ligand' for text \u001b[0m\n",
      "\u001b[31mCD30 and its counter-receptor PROTEIN1 ligand (CD30L) are members of the TNF-receptor/TNFalpha superfamily and function to regulate lymphocyte survival and differentiation.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,304 - preprocessor.ProteinMasker - WARNING - The text at offset_start 196 must match entity 'IL-2 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mMonoclonal antibodies to the beta chain of the IL-2 receptor inhibited the biological activity of IL-15, and IL-15 competed for binding with IL-2, indicating that IL-15 uses components of the PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,325 - preprocessor.ProteinMasker - WARNING - The text at offset_start 50 must match entity 'interferon alpha (IFNalpha) receptor', but found 'e human PROTEIN1 (IFNalpha) receptor' for text \u001b[0m\n",
      "\u001b[31mCoexpression of the alpha and betaL subunits of the human PROTEIN1 (IFNalpha) receptor is required for the induction of an antiviral state by human IFNalpha.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,354 - preprocessor.ProteinMasker - WARNING - The text at offset_start 83 must match entity 'IFN-gamma', but found 'n of PROT' for text \u001b[0m\n",
      "\u001b[31mComparison of the wild-type receptor-binding domain and the Asp111-containing domain of PROTEIN1 show that they are structurally equivalent but have very different electrostatic surface potentials.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,355 - preprocessor.ProteinMasker - WARNING - The text at offset_start -26 must match entity 'Vascular endothelial growth factor-related protein', but found '' for text \u001b[0m\n",
      "\u001b[31mPROTEIN1-related protein: a ligand and specific activator of the tyrosine kinase receptor Flt4.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,389 - preprocessor.ProteinMasker - WARNING - The text at offset_start 119 must match entity 'RII alpha', but found 'IN1 alpha' for text \u001b[0m\n",
      "\u001b[31mMutation of Ala-522 to proline, which is located outside the predicted amphipathic helix region, had no effect on PROTEIN1 alpha binding.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,391 - preprocessor.ProteinMasker - WARNING - The text at offset_start 67 must match entity 'IFN-gamma R alpha', but found ' PROTEIN1 R alpha' for text \u001b[0m\n",
      "\u001b[31mThis suggests that one domain of IFN-gamma is sufficient to recruit PROTEIN1 R alpha and IFN-gamma R beta into a complex competent for eliciting biological activity.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,418 - preprocessor.ProteinMasker - WARNING - The text at offset_start 236 must match entity 'dystrophin related protein', but found 'n/PROTEIN1 related protein' for text \u001b[0m\n",
      "\u001b[31mFinally, we used in vitro translated proteins in an immunoprecipitation assay to show that, like beta 1-syntrophin, both beta 2- and alpha 1-syntrophin interact with peptides encoding the syntrophin-binding region of dystrophin, utrophin/PROTEIN1 related protein, and the Torpedo 87K protein.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,467 - preprocessor.ProteinMasker - WARNING - The text at offset_start 129 must match entity 'RII alpha', but found 'IN1 alpha' for text \u001b[0m\n",
      "\u001b[31mA bacterially expressed 318-amino acid fragment, Ht 31 (418-736), containing the amphipathic helix region, was able to bind PROTEIN1 alpha.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,473 - preprocessor.ProteinMasker - WARNING - The text at offset_start 96 must match entity 'IR beta', but found 'N1 beta' for text \u001b[0m\n",
      "\u001b[31mThe results of his3 activation indicated that p85, N + C-SH2 and C-SH2 interact with both PROTEIN1 beta and IGF-1R beta, whereas N-SH2 and SH3 + N-SH2 interact only with IR beta.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,483 - preprocessor.ProteinMasker - WARNING - The text at offset_start 31 must match entity 'IGF-1R beta', but found 'OTEIN1 beta' for text \u001b[0m\n",
      "\u001b[31mThis activation with p85 and PROTEIN1 beta was 4-fold less as compared to IR beta and was somewhat increased (2-fold) for IGF-1R beta (delta C-43).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,485 - preprocessor.ProteinMasker - WARNING - The text at offset_start 94 must match entity 'IGF-I', but found 'th th' for text \u001b[0m\n",
      "\u001b[31mEvidence for a differential interaction of SHC and the insulin receptor substrate-1 (IRS-1) with the PROTEIN1 in the yeast two-hybrid system.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,569 - preprocessor.ProteinMasker - WARNING - The text at offset_start 134 must match entity 'IL-8 receptor subtypes A', but found 'EIN1 receptor subtypes A' for text \u001b[0m\n",
      "\u001b[31mThis work shows that single and double Ala substitutions of His18 and Phe21 in IL-8 reduced up to 77-fold the binding affinity to PROTEIN1 receptor subtypes A (CXCR1) and B (CXCR2) and to the Duffy antigen.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,576 - preprocessor.ProteinMasker - WARNING - The text at offset_start 42 must match entity 'keratinocyte growth factor', but found ' the FGFR2 IIIb isoform or' for text \u001b[0m\n",
      "\u001b[31mFGF-7 recognizes one FGFR isoform known as the FGFR2 IIIb isoform or PROTEIN1 (KGFR), whereas FGF-2 binds well to FGFR1, FGFR2, and FGFR4 but interacts poorly with KGFR.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,583 - preprocessor.ProteinMasker - WARNING - The text at offset_start 189 must match entity 'CCKB G protein-coupled receptor', but found 'EIN1 G protein-coupled receptor' for text \u001b[0m\n",
      "\u001b[31mTaken together, our results suggest that IRS-1 may serve as a converging target in the signaling pathways stimulated by receptors that belong to different families, such as the gastrin/PROTEIN1 G protein-coupled receptor and the insulin receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,684 - preprocessor.ProteinMasker - WARNING - The text at offset_start 33 must match entity 'TRKB', but found 'uman' for text \u001b[0m\n",
      "\u001b[31mThe extracellular domain of the human PROTEIN1 expressed in Chinese hamster ovary cells is a highly glycosylated protein, possessing binding ability for brain-derived neurotrophic factor (BDNF).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,684 - preprocessor.ProteinMasker - WARNING - The text at offset_start 136 must match entity 'IL-10 receptor', but found 'TEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mUsing a structure-function mutagenesis approach, two tyrosine residues (Tyr427 and Tyr477) in the intracellular domain of the murine PROTEIN1 receptor were found to be redundantly required for receptor function and for activation of Stat3 but not for Stat1 or Stat5.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,707 - preprocessor.ProteinMasker - WARNING - The text at offset_start 29 must match entity 'ECK receptor protein-tyrosine kinase', but found 'IN1 receptor protein-tyrosine kinase' for text \u001b[0m\n",
      "\u001b[31mB61 is a ligand for the PROTEIN1 receptor protein-tyrosine kinase.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,733 - preprocessor.ProteinMasker - WARNING - The text at offset_start 93 must match entity 'Fas', but found 'n P' for text \u001b[0m\n",
      "\u001b[31mThe death-inducing receptor Fas is activated when cross-linked by the type II membrane protein PROTEIN1 (FasL).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,739 - preprocessor.ProteinMasker - WARNING - The text at offset_start 65 must match entity 'erythropoietin (EPO) receptor', but found 'f the PROTEIN1 (EPO) receptor' for text \u001b[0m\n",
      "\u001b[31mWe have shown previously that Phe93 in the extracellular domain of the PROTEIN1 (EPO) receptor (EPOR) is crucial for binding EPO.Substitution of Phe93 with alanine resulted in a dramatic decrease in EPO binding to the Escherichia coli-expressed extracellular domain of the EPOR (EPO-binding protein or EBP) and no detectable binding to full-length mutant receptor expressed in COS cells.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,781 - preprocessor.ProteinMasker - WARNING - The text at offset_start 170 must match entity 'IR beta', but found 'N1 beta' for text \u001b[0m\n",
      "\u001b[31mThe results of his3 activation indicated that p85, N + C-SH2 and C-SH2 interact with both IR beta and IGF-1R beta, whereas N-SH2 and SH3 + N-SH2 interact only with PROTEIN1 beta.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,794 - preprocessor.ProteinMasker - WARNING - The text at offset_start 48 must match entity 'insulin receptor substrate 1', but found 'ROTEIN1 receptor substrate 1' for text \u001b[0m\n",
      "\u001b[31mGastrin stimulates tyrosine phosphorylation of PROTEIN1 receptor substrate 1 and its association with Grb2 and the phosphatidylinositol 3-kinase.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,797 - preprocessor.ProteinMasker - WARNING - The text at offset_start 43 must match entity 'IFN-gamma SC1', but found ' PROTEIN1 SC1' for text \u001b[0m\n",
      "\u001b[31mIsothermal titration calorimetry shows that PROTEIN1 SC1 forms a 1:1 complex with its high-affinity receptor (IFN-gamma R alpha) with an affinity of 27(+/- 9) nM.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,823 - preprocessor.ProteinMasker - WARNING - The text at offset_start 87 must match entity 'BMP receptor IA extracellular domain', but found 'finity PROTEIN1 extracellular domain' for text \u001b[0m\n",
      "\u001b[31mHere we report the crystal structure of human dimeric BMP-2 in complex with two high affinity PROTEIN1 extracellular domains(BRIAec).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,849 - preprocessor.ProteinMasker - WARNING - The text at offset_start 67 must match entity 'Rb-associated protein', but found 'N1-associated protein' for text \u001b[0m\n",
      "\u001b[31mWe report here the identification of one such human cellular PROTEIN1-associated protein of relative molecular mass 46,000 (46K) (RbAP46).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,862 - preprocessor.ProteinMasker - WARNING - The text at offset_start 122 must match entity 'IGF-1R beta', but found 'OTEIN1 beta' for text \u001b[0m\n",
      "\u001b[31mThis activation with p85 and IGF-1R beta was 4-fold less as compared to IR beta and was somewhat increased (2-fold) for PROTEIN1 beta (delta C-43).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,960 - preprocessor.ProteinMasker - WARNING - The text at offset_start 43 must match entity 'Fas ligand', but found 'IN1 ligand' for text \u001b[0m\n",
      "\u001b[31mCharacterization of Fas (Apo-1, CD95)-PROTEIN1 ligand interaction.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:00,999 - preprocessor.ProteinMasker - WARNING - The text at offset_start 33 must match entity 'EPO', but found ' th' for text \u001b[0m\n",
      "\u001b[31mShared and unique determinants of the PROTEIN1 are important for binding EPO and EPO mimetic peptide.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,030 - preprocessor.ProteinMasker - WARNING - The text at offset_start 52 must match entity 'IL-7 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mCharacterization of a novel high affinity human PROTEIN1 receptor. Expression on T cells and association with IL-7 driven proliferation.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,043 - preprocessor.ProteinMasker - WARNING - The text at offset_start 40 must match entity 'IFN-gamma SC1', but found '(PROTEIN1 SC1' for text \u001b[0m\n",
      "\u001b[31mA mutant form of human interferon-gamma (PROTEIN1 SC1) that binds one IFN-gamma receptor alpha chain (IFN-gamma R alpha) has been designed and characterized.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,049 - preprocessor.ProteinMasker - WARNING - The text at offset_start -2 must match entity 'erythropoietin receptor', but found '' for text \u001b[0m\n",
      "\u001b[31mThe PROTEIN1 receptor (EPOR) is a member of a family of cytokine and growth factor receptors that share conserved features in their extracellular and cytoplasmic domains.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,071 - preprocessor.ProteinMasker - WARNING - The text at offset_start 46 must match entity 'IL-1 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mThree naturally occurring ligands for the PROTEIN1 receptor (IL1R) exist: the agonists IL-1alpha and IL-1beta and the IL-1-receptor antagonist IL1RA (ref. 2).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,076 - preprocessor.ProteinMasker - WARNING - The text at offset_start 76 must match entity 'JAK2 kinase', but found 'EIN1 kinase' for text \u001b[0m\n",
      "\u001b[31mInterferon-gamma and several other cytokines that are known to activate PROTEIN1 kinase were also found to stimulate Raf-1 kinase activity toward MEK-1 in mammalian cells.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,084 - preprocessor.ProteinMasker - WARNING - The text at offset_start 111 must match entity 'IL-5 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mCloning and characterization of a specific interleukin (IL)-13 binding protein structurally related to the PROTEIN1 receptor alpha chain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,096 - preprocessor.ProteinMasker - WARNING - The text at offset_start 155 must match entity 'E1B', but found 'IN1' for text \u001b[0m\n",
      "\u001b[31mThe acidic amino terminus of the p53 protein has been shown to contain this trans-activation activity as well as the domains for mdm-2 and adenovirus PROTEIN1 55-kD protein binding.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,114 - preprocessor.ProteinMasker - WARNING - The text at offset_start 116 must match entity 'Raf-1 kinase', but found 'TEIN1 kinase' for text \u001b[0m\n",
      "\u001b[31mInterferon-gamma and several other cytokines that are known to activate JAK2 kinase were also found to stimulate PROTEIN1 kinase activity toward MEK-1 in mammalian cells.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,122 - preprocessor.ProteinMasker - WARNING - The text at offset_start 155 must match entity 'IL-6 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mThe high affinity interleukin-6 (IL-6) signaling complex consists of IL-6 and two membrane-associated receptor components: a low affinity but specific PROTEIN1 receptor and the affinity converter/signal transducing protein gp130.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,210 - preprocessor.ProteinMasker - WARNING - The text at offset_start 28 must match entity 'interferon alpha receptor', but found ' of the PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mA region of the beta subunit of the PROTEIN1 receptor different from box 1 interacts with Jak1 and is sufficient to activate the Jak-Stat pathway and induce an antiviral state.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,211 - preprocessor.ProteinMasker - WARNING - The text at offset_start 89 must match entity 'interleukin-10 receptor', but found 'n the PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mStat3 recruitment by two distinct ligand-induced, tyrosine-phosphorylated docking sites in the PROTEIN1 receptor intracellular domain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,237 - preprocessor.ProteinMasker - WARNING - The text at offset_start 48 must match entity 'IFNalpha', but found 'the huma' for text \u001b[0m\n",
      "\u001b[31mCoexpression of the alpha and betaL subunits of the human PROTEIN1 is required for the induction of an antiviral state by human IFNalpha.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,287 - preprocessor.ProteinMasker - WARNING - The text at offset_start 167 must match entity 'insulin receptor', but found 'ROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mIn the present study, we demonstrate that gastrin stimulates tyrosine phosphorylation of insulin receptor substrate 1 (IRS-1), the major cytoplasmic substrate of the PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,321 - preprocessor.ProteinMasker - WARNING - The text at offset_start 155 must match entity 'TRKB receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mThese results, obtained from a variety of experimental techniques, highlight the importance of two distinct regions of the extracellular domain of the PROTEIN1 receptor in binding BDNF.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,327 - preprocessor.ProteinMasker - WARNING - The text at offset_start 156 must match entity 'mGITR', but found 'urine' for text \u001b[0m\n",
      "\u001b[31mWe have identified a new TNF-related ligand, designated human GITR ligand (hGITRL), and its human receptor (hGITR), an ortholog of the recently discovered murine PROTEIN1 [4].\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,328 - preprocessor.ProteinMasker - WARNING - The text at offset_start 106 must match entity 'MAP kinase kinase', but found 'e PROTEIN1 kinase' for text \u001b[0m\n",
      "\u001b[31mThese findings suggest that one function of c-Raf-1 in mitogenic signaling is to phosphorylate and activate PROTEIN1 kinase.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,351 - preprocessor.ProteinMasker - WARNING - The text at offset_start 21 must match entity 'Ln-5 gamma2', but found 'EIN1 gamma2' for text \u001b[0m\n",
      "\u001b[31mMMP2 cleaved the PROTEIN1 gamma2 subunit at residue 587, exposing a putative cryptic promigratory site on Ln-5 that triggers cell motility.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,352 - preprocessor.ProteinMasker - WARNING - The text at offset_start 6 must match entity 'interferon-gamma inducing factor', but found 'tion of PROTEIN1 inducing factor' for text \u001b[0m\n",
      "\u001b[31mActivation of PROTEIN1 inducing factor mediated by interleukin-1beta converting enzyme.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,373 - preprocessor.ProteinMasker - WARNING - The text at offset_start 29 must match entity 'CD3', but found 'IN1' for text \u001b[0m\n",
      "\u001b[31mCD5 associates with the PROTEIN1 complex and is rapidly phosphosphorylated on tyrosine residues as a result of TcR zeta/CD3 ligation.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,398 - preprocessor.ProteinMasker - WARNING - The text at offset_start 11 must match entity 'interleukin-1', but found 'ucture of the' for text \u001b[0m\n",
      "\u001b[31mCrystal structure of the PROTEIN1 complexed with interleukin-1beta.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,422 - preprocessor.ProteinMasker - WARNING - The text at offset_start 18 must match entity 'serine-threonine phosphatase 2A', but found 'he protein HOX11 interacted wit' for text \u001b[0m\n",
      "\u001b[31mWe observed that the protein HOX11 interacted with protein PROTEIN1 (PP2AC), as well as protein phosphatase 1 (PP1C) in mammalian cells.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,492 - preprocessor.ProteinMasker - WARNING - The text at offset_start 102 must match entity 'lymphotoxin beta receptor', but found 'ss both PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mIt has been shown that LIGHT triggers apoptosis of various tumor cells including HT29 cells that express both PROTEIN1 receptor ( LTbetaR ) and HVEM / TR2 receptors .\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,499 - preprocessor.ProteinMasker - WARNING - The text at offset_start 77 must match entity 'transferrin receptor', but found 'th PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mCrystal structure of the hereditary haemochromatosis protein HFE complexed with PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,499 - preprocessor.ProteinMasker - WARNING - The text at offset_start 106 must match entity 'IL-10 receptor', but found 'TEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mThese data demonstrate that Stat3 but not Stat1 or Stat5 is directly recruited to the ligand-activated PROTEIN1 receptor by binding to specific but redundant receptor intracellular domain sequences containing phosphotyrosine.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,500 - preprocessor.ProteinMasker - WARNING - The text at offset_start 66 must match entity 'EPO', but found ' th' for text \u001b[0m\n",
      "\u001b[31mWe have shown previously that Phe93 in the extracellular domain of the PROTEIN1 (EPOR) is crucial for binding EPO.Substitution of Phe93 with alanine resulted in a dramatic decrease in EPO binding to the Escherichia coli-expressed extracellular domain of the EPOR (EPO-binding protein or EBP) and no detectable binding to full-length mutant receptor expressed in COS cells.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,531 - preprocessor.ProteinMasker - WARNING - The text at offset_start 18 must match entity 'interleukin-2 (IL-2) receptor', but found ' the PROTEIN1 (IL-2) receptor' for text \u001b[0m\n",
      "\u001b[31mThe gamma chain of the PROTEIN1 (IL-2) receptor is shared with the functional IL-4 receptor and is causatively related to X-linked severe combined immunodeficiency (XSCID), which is ascribed to a profound T cell defect.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,609 - preprocessor.ProteinMasker - WARNING - The text at offset_start 29 must match entity 'VEGF-related protein', but found 'EIN1-related protein' for text \u001b[0m\n",
      "\u001b[31mThis protein, designated PROTEIN1-related protein (VRP), specifically binds to the extracellular domain of Flt4, stimulates the tyrosine phosphorylation of Flt4 expressed in mammalian cells, and promotes the mitogenesis of human lung endothelial cells.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,660 - preprocessor.ProteinMasker - WARNING - The text at offset_start -5 must match entity 'interleukin-1beta (IL-1beta) converting enzyme', but found '' for text \u001b[0m\n",
      "\u001b[31mThe PROTEIN1 (IL-1beta) converting enzyme (ICE) processes the inactive IL-1beta precursor to the proinflammatory cytokine.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,694 - preprocessor.ProteinMasker - WARNING - The text at offset_start 62 must match entity 'RII alpha', but found 'IN1 alpha' for text \u001b[0m\n",
      "\u001b[31mThese data suggest that anchoring proteins interact with PROTEIN1 alpha via an amphipathic helix binding motif.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,728 - preprocessor.ProteinMasker - WARNING - The text at offset_start 39 must match entity 'insulin receptor substrate 1', but found 'tion of PROTEIN1 substrate 1' for text \u001b[0m\n",
      "\u001b[31mGastrin stimulates tyrosine phosphorylation of PROTEIN1 substrate 1 and its association with Grb2 and the phosphatidylinositol 3-kinase.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,746 - preprocessor.ProteinMasker - WARNING - The text at offset_start 151 must match entity 'E1B', but found 'IN1' for text \u001b[0m\n",
      "\u001b[31mSeveral hydrophobic amino acids in the p53 amino-terminal domain are required for transcriptional activation, binding to mdm-2 and the adenovirus PROTEIN1 55-kD protein.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,759 - preprocessor.ProteinMasker - WARNING - The text at offset_start 12 must match entity 'CDC2-HS', but found 'EIN1-HS' for text \u001b[0m\n",
      "\u001b[31mOne was PROTEIN1-HS, and the second was the human homolog of Xenopus CDK2.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,773 - preprocessor.ProteinMasker - WARNING - The text at offset_start 17 must match entity 'IL-2', but found 'f th' for text \u001b[0m\n",
      "\u001b[31mThe gamma chain of the PROTEIN1 is shared with the functional IL-4 receptor and is causatively related to X-linked severe combined immunodeficiency (XSCID), which is ascribed to a profound T cell defect.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,788 - preprocessor.ProteinMasker - WARNING - The text at offset_start -10 must match entity 'mitogen-activated protein kinase kinase', but found '' for text \u001b[0m\n",
      "\u001b[31mActivation of PROTEIN1 kinase by v-Raf in NIH 3T3 cells and in vitro.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,827 - preprocessor.ProteinMasker - WARNING - The text at offset_start 70 must match entity 'IFN-gamma receptor', but found ' PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mA mutant form of human interferon-gamma (IFN-gamma SC1) that binds one PROTEIN1 receptor alpha chain (IFN-gamma R alpha) has been designed and characterized.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,847 - preprocessor.ProteinMasker - WARNING - The text at offset_start 37 must match entity 'interleukin-6 receptor', but found 'uman PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mAnalysis of the human interleukin-6/human PROTEIN1 receptor binding interface at the amino acid level: proposed mechanism of interaction.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,912 - preprocessor.ProteinMasker - WARNING - The text at offset_start 142 must match entity 'IFN-gamma R alpha', but found '/PROTEIN1 R alpha' for text \u001b[0m\n",
      "\u001b[31mThe AB loops of IFN-gamma SC1 adopt conformations similar to the ordered loops of IFN-gamma observed in the crystal structure of the IFN-gamma/PROTEIN1 R alpha complex.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:01,913 - preprocessor.ProteinMasker - WARNING - The text at offset_start 33 must match entity 'interleukin 13 receptor alpha chain', but found 'g to the human PROTEIN1 alpha chain' for text \u001b[0m\n",
      "\u001b[31mWe have cloned cDNAs corresponding to the human PROTEIN1 alpha chain (IL-13Ralpha).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,001 - preprocessor.ProteinMasker - WARNING - The text at offset_start 54 must match entity 'IL-10 receptor', but found 'TEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mExpression cloning and characterization of a human PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,011 - preprocessor.ProteinMasker - WARNING - The text at offset_start 287 must match entity 'IL-13 receptor', but found 'TEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mOnly weak IL-13 binding activity was found in cells transfected with only IL-13Ralpha; however, the combination of both IL-13Ralpha and IL-4Ralpha resulted in substantial binding activity, with a Kd of approximately 400 pM, indicating that both chains are essential components of the PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,059 - preprocessor.ProteinMasker - WARNING - The text at offset_start 100 must match entity 'IL1R', but found 'EIN2' for text \u001b[0m\n",
      "\u001b[31mHere we describe the crystal structure at 2.7 A resolution of the soluble extracellular part of PROTEIN2 complexed with IL1RA.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,087 - preprocessor.ProteinMasker - WARNING - The text at offset_start 205 must match entity 'NF-kappaB-inducing kinase', but found ' PROTEIN1-inducing kinase' for text \u001b[0m\n",
      "\u001b[31mCotransfection of hGITRL and hGITR in embryonic kidney 293 cells activated the anti-apoptotic transcription factor NF-kappaB, via a pathway that appeared to involve TNFR-associated factor 2 (TRAF2) [7] and PROTEIN1-inducing kinase (NIK) [8].\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,121 - preprocessor.ProteinMasker - WARNING - The text at offset_start 45 must match entity 'RAS binding protein', but found 'IN1 binding protein' for text \u001b[0m\n",
      "\u001b[31mHuman RIN1 was first characterized as a PROTEIN1 binding protein based on the properties of its carboxyl-terminal domain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,157 - preprocessor.ProteinMasker - WARNING - The text at offset_start 133 must match entity 'Fas ligand', but found 'IN1 ligand' for text \u001b[0m\n",
      "\u001b[31mHere , we demonstrate that TR6 specifically binds two cellular ligands , LIGHT ( herpes virus entry mediator ( HVEM ) - L ) and PROTEIN1 ligand ( FasL / CD95L ) .\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,215 - preprocessor.ProteinMasker - WARNING - The text at offset_start 68 must match entity 'IL-2 receptor beta', but found 'EIN1 receptor beta' for text \u001b[0m\n",
      "\u001b[31mInterleukin-2 (IL-2) signaling requires the dimerization of the PROTEIN1 receptor beta.(IL-2R beta) and common gamma (gamma c) chains.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,234 - preprocessor.ProteinMasker - WARNING - The text at offset_start 42 must match entity 'TFIID beta', but found 'TEIN1 beta' for text \u001b[0m\n",
      "\u001b[31mIn contrast to hTAFII28, hTAFII18 is a PROTEIN1 beta-specific hTAFII. hTAFII20 is the homologue of p22, an alternatively spliced form of dTAFII30 alpha (p32).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,235 - preprocessor.ProteinMasker - WARNING - The text at offset_start 102 must match entity 'lymphotoxin beta receptor', but found 'ss both PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mIt has been shown that LIGHT triggers apoptosis of various tumor cells including HT29 cells that express both PROTEIN1 receptor (LTbetaR) and HVEM/TR2 receptors.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,247 - preprocessor.ProteinMasker - WARNING - The text at offset_start 162 must match entity 'FGFR3 IIIb', but found 'TEIN1 IIIb' for text \u001b[0m\n",
      "\u001b[31mExpression of FGFR3 IIIc in stably transfected L6 cells conferred growth responses to several members of the FGF family including FGF-1, -2, -4, and -6, while PROTEIN1 IIIb-expressing cells responded only to FGF-1.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,348 - preprocessor.ProteinMasker - WARNING - The text at offset_start 137 must match entity 'IL-6 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mMonomeric (IL-6M) and dimeric (IL-6D) forms of Escherichia coli-derived human IL-6 and the extracellular (\"soluble\") portions of the PROTEIN1 receptor (sIL-6R) and gp130 have been purified in order to investigate the effect of IL-6 dimerization on binding to the receptor complex.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,365 - preprocessor.ProteinMasker - WARNING - The text at offset_start 78 must match entity 'IL-10 receptor', but found 'TEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mIL-10 also activated Stat5 in Ba/F3 cells that stably expressed the murine PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,365 - preprocessor.ProteinMasker - WARNING - The text at offset_start 318 must match entity 'hTFIID beta', but found 'OTEIN1 beta' for text \u001b[0m\n",
      "\u001b[31mWe have cloned cDNAs encoding three novel TAFIIs [TATA-binding protein (TBP)-associated factors] from the human (h) HeLa cell TFIID complexes hTAFII28, hTAFII20 and hTAFII18. hTAFII28 is a core hTAFII present in both of the previously described hTFIID species which either lack or contain hTAFII30 (hTFIID alpha and PROTEIN1 beta respectively), and is the homologue of Drosophila (d)TAFII30 beta. hTAFII18 is a novel hTAFII which shows homology to the N-terminal region of the yeast TAFIISPT3, but has no known Drosophila counterpart.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,373 - preprocessor.ProteinMasker - WARNING - The text at offset_start 80 must match entity 'RII alpha', but found 'IN1 alpha' for text \u001b[0m\n",
      "\u001b[31mSpecifically, substitution of proline for Ala-498 significantly diminished PROTEIN1 alpha binding, and similar mutation of Ile-502 or Ile-507 abolished interaction.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,445 - preprocessor.ProteinMasker - WARNING - The text at offset_start 152 must match entity 'epidermal growth factor receptor', but found 'gulator of the PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mThe precise function of c-Cbl in these pathways is not clear, although a genetic analysis in Caenorhabditis elegans suggests that c-Cbl is a negative regulator of the PROTEIN1 receptor.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,469 - preprocessor.ProteinMasker - WARNING - The text at offset_start 290 must match entity 'EPO-binding protein', but found 'IN1-binding protein' for text \u001b[0m\n",
      "\u001b[31mWe have shown previously that Phe93 in the extracellular domain of the erythropoietin (EPO) receptor (EPOR) is crucial for binding EPO.Substitution of Phe93 with alanine resulted in a dramatic decrease in EPO binding to the Escherichia coli-expressed extracellular domain of the EPOR (PROTEIN1-binding protein or EBP) and no detectable binding to full-length mutant receptor expressed in COS cells.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,474 - preprocessor.ProteinMasker - WARNING - The text at offset_start 65 must match entity 'IL-7 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mFunctional participation of the IL-2 receptor gamma chain in PROTEIN1 receptor complexes.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,494 - preprocessor.ProteinMasker - WARNING - The text at offset_start 42 must match entity 'interleukin 13 receptor', but found 'human PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mWe have cloned cDNAs corresponding to the human PROTEIN1 receptor alpha chain (IL-13Ralpha).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,496 - preprocessor.ProteinMasker - WARNING - The text at offset_start 138 must match entity 'interleukin-1 beta-converting enzyme', but found 'or of the PROTEIN1-converting enzyme' for text \u001b[0m\n",
      "\u001b[31mOverexpression of FADD in MCF7 and BJAB cells induces apoptosis, which, like Fas-induced apoptosis, is blocked by CrmA, a specific inhibitor of the PROTEIN1-converting enzyme.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,509 - preprocessor.ProteinMasker - WARNING - The text at offset_start 85 must match entity 'beta-galactosidase', but found 'd express PROTEIN2' for text \u001b[0m\n",
      "\u001b[31mTemporally following this growth arrest, the cells develop a senescence morphology and express PROTEIN2 (SA-beta-gal).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,535 - preprocessor.ProteinMasker - WARNING - The text at offset_start 36 must match entity 'IL-2 receptor gamma', but found 'EIN1 receptor gamma' for text \u001b[0m\n",
      "\u001b[31mFunctional participation of the PROTEIN1 receptor gamma chain in IL-7 receptor complexes.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,562 - preprocessor.ProteinMasker - WARNING - The text at offset_start 111 must match entity 'p140trk (trkA) tyrosine kinase receptor', but found 'ROTEIN1 (trkA) tyrosine kinase receptor' for text \u001b[0m\n",
      "\u001b[31mNerve growth factor binds independently to two transmembrane receptors, the p75 neurotrophin receptor and the PROTEIN1 (trkA) tyrosine kinase receptor, which are both co-expressed in the majority of neuronal cells that respond to NGF.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,634 - preprocessor.ProteinMasker - WARNING - The text at offset_start 86 must match entity 'IL-6', but found 'eric' for text \u001b[0m\n",
      "\u001b[31mInfluence of interleukin-6 (IL-6) dimerization on formation of the high affinity hexameric PROTEIN1 complex.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,646 - preprocessor.ProteinMasker - WARNING - The text at offset_start 27 must match entity 'IFN-gamma R alpha', but found ' PROTEIN1 R alpha' for text \u001b[0m\n",
      "\u001b[31mDespite binding to only one PROTEIN1 R alpha, IFN-gamma SC1 is biologically active in cell proliferation, MHC class I induction, and anti-viral assays.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,658 - preprocessor.ProteinMasker - WARNING - The text at offset_start 88 must match entity 'trkA', but found 'phin' for text \u001b[0m\n",
      "\u001b[31mNerve growth factor binds independently to two transmembrane receptors, the p75 neurotrophin receptor and the PROTEIN1, which are both co-expressed in the majority of neuronal cells that respond to NGF.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,744 - preprocessor.ProteinMasker - WARNING - The text at offset_start 110 must match entity 'IFN-gamma R alpha', but found '(PROTEIN1 R alpha' for text \u001b[0m\n",
      "\u001b[31mIsothermal titration calorimetry shows that IFN-gamma SC1 forms a 1:1 complex with its high-affinity receptor (PROTEIN1 R alpha) with an affinity of 27(+/- 9) nM.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,768 - preprocessor.ProteinMasker - WARNING - The text at offset_start 42 must match entity 'interleukin 13 receptor alpha chain', but found 'human PROTEIN1 receptor alpha chain' for text \u001b[0m\n",
      "\u001b[31mWe have cloned cDNAs corresponding to the human PROTEIN1 receptor alpha chain (IL-13Ralpha).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,839 - preprocessor.ProteinMasker - WARNING - The text at offset_start 109 must match entity 'IL-1-receptor antagonist', but found ' the PROTEIN1 antagonist' for text \u001b[0m\n",
      "\u001b[31mThree naturally occurring ligands for the IL-1 receptor (IL1R) exist: the agonists IL-1alpha and IL-1beta and the PROTEIN1 antagonist IL1RA (ref. 2).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,869 - preprocessor.ProteinMasker - WARNING - The text at offset_start 74 must match entity 'Bcl-2', but found 'code ' for text \u001b[0m\n",
      "\u001b[31mUsing a protein interaction cloning technique, we identified cDNAs that encode a novel PROTEIN1, termed BAG-1.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,873 - preprocessor.ProteinMasker - WARNING - The text at offset_start 88 must match entity 'IGF-1 receptor', but found 'TEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mInteraction of the p85 subunit of PI 3-kinase with the insulin receptor (IR) and the PROTEIN1 receptor (IGF-1R) was investigated using the two-hybrid system by assessing for his3 and lacZ activation in S. cerevisiae.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,886 - preprocessor.ProteinMasker - WARNING - The text at offset_start 102 must match entity 'IFN-gamma R alpha', but found '(PROTEIN1 R alpha' for text \u001b[0m\n",
      "\u001b[31mA mutant form of human interferon-gamma (IFN-gamma SC1) that binds one IFN-gamma receptor alpha chain (PROTEIN1 R alpha) has been designed and characterized.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,899 - preprocessor.ProteinMasker - WARNING - The text at offset_start 54 must match entity 'Raf-1 cysteine-rich domain', but found 'TEIN1 cysteine-rich domain' for text \u001b[0m\n",
      "\u001b[31mWe recently reported that Ras interaction with the PROTEIN1 cysteine-rich domain (Raf-CRD, residues 139-184) may be required for Raf-1 activation.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,900 - preprocessor.ProteinMasker - WARNING - The text at offset_start 6 must match entity 'T-cell receptor zeta chain (TcR zeta)/CD3', but found 'sociates with the PROTEIN1 (TcR zeta)/CD3' for text \u001b[0m\n",
      "\u001b[31mCD5 associates with the PROTEIN1 (TcR zeta)/CD3 complex and is rapidly phosphosphorylated on tyrosine residues as a result of TcR zeta/CD3 ligation.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,942 - preprocessor.ProteinMasker - WARNING - The text at offset_start 14 must match entity 'transferrin receptor', but found 'he PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mHFE binds to the PROTEIN1 receptor (TfR), a receptor by which cells acquire iron-loaded transferrin.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:02,943 - preprocessor.ProteinMasker - WARNING - The text at offset_start 24 must match entity 'IFN-gamma SC1', but found ' PROTEIN1 SC1' for text \u001b[0m\n",
      "\u001b[31mThe crystal structure of PROTEIN1 SC1 has been determined at 2.9 A resolution from crystals grown in 1.4 M citrate solutions at pH 7.6.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,021 - preprocessor.ProteinMasker - WARNING - The text at offset_start 53 must match entity 'IL-6 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mThe interaction between interleukin-6 (IL-6) and PROTEIN1 receptor (IL-6R) is the initial and most specific step in the IL-6 signaling pathway.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,033 - preprocessor.ProteinMasker - WARNING - The text at offset_start 56 must match entity 'insulin receptor', but found 'ROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mInteraction of the p85 subunit of PI 3-kinase with the PROTEIN1 receptor (IR) and the IGF-1 receptor (IGF-1R) was investigated using the two-hybrid system by assessing for his3 and lacZ activation in S. cerevisiae.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,034 - preprocessor.ProteinMasker - WARNING - The text at offset_start 107 must match entity 'EPO mimetic peptide', but found 'IN1 mimetic peptide' for text \u001b[0m\n",
      "\u001b[31mShared and unique determinants of the erythropoietin (EPO) receptor are important for binding EPO and PROTEIN1 mimetic peptide.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,034 - preprocessor.ProteinMasker - WARNING - The text at offset_start 301 must match entity 'hTFIID alpha', but found 'OTEIN1 alpha' for text \u001b[0m\n",
      "\u001b[31mWe have cloned cDNAs encoding three novel TAFIIs [TATA-binding protein (TBP)-associated factors] from the human (h) HeLa cell TFIID complexes hTAFII28, hTAFII20 and hTAFII18. hTAFII28 is a core hTAFII present in both of the previously described hTFIID species which either lack or contain hTAFII30 (PROTEIN1 alpha and hTFIID beta respectively), and is the homologue of Drosophila (d)TAFII30 beta. hTAFII18 is a novel hTAFII which shows homology to the N-terminal region of the yeast TAFIISPT3, but has no known Drosophila counterpart.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,053 - preprocessor.ProteinMasker - WARNING - The text at offset_start 87 must match entity 'IL-4 receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mThe gamma chain of the interleukin-2 (IL-2) receptor is shared with the functional PROTEIN1 receptor and is causatively related to X-linked severe combined immunodeficiency (XSCID), which is ascribed to a profound T cell defect.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,094 - preprocessor.ProteinMasker - WARNING - The text at offset_start 38 must match entity 'IL-1 receptor-associated kinase', but found 'EIN1 receptor-associated kinase' for text \u001b[0m\n",
      "\u001b[31mA protein kinase designated IRAK (PROTEIN1 receptor-associated kinase) was purified, and its complementary DNA was molecularly cloned.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,120 - preprocessor.ProteinMasker - WARNING - The text at offset_start 49 must match entity 'IFN-gamma receptor', but found 'a SC1) that binds ' for text \u001b[0m\n",
      "\u001b[31mA mutant form of human interferon-gamma (IFN-gamma SC1) that binds one PROTEIN1 (IFN-gamma R alpha) has been designed and characterized.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,140 - preprocessor.ProteinMasker - WARNING - The text at offset_start 263 must match entity 'insulin receptor substrate-1', but found 'and the PROTEIN1 substrate-1' for text \u001b[0m\n",
      "\u001b[31mUsing the yeast two-hybrid system, a genetic assay for studying protein-protein interactions, we have examined and compared the interaction of the insulin-like growth factor-I receptor (IGF-IR) and the insulin receptor (IR) with their two known substrates p52Shc and the PROTEIN1 substrate-1 (IRS-1).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,240 - preprocessor.ProteinMasker - WARNING - The text at offset_start 48 must match entity 'insulin receptor', but found 'ROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mAlthough IRS-1 is thought to interact with the PROTEIN1 receptor, the nature of the interaction has not been defined.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,240 - preprocessor.ProteinMasker - WARNING - The text at offset_start 32 must match entity 'interleukin 13 receptor alpha chain', but found 'n of the human PROTEIN1 alpha chain' for text \u001b[0m\n",
      "\u001b[31mcDNA cloning and characterization of the human PROTEIN1 alpha chain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,262 - preprocessor.ProteinMasker - WARNING - The text at offset_start 32 must match entity 'IL-1', but found ' the' for text \u001b[0m\n",
      "\u001b[31mEach of these molecules binds to the PROTEIN1 (IL1R).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,263 - preprocessor.ProteinMasker - WARNING - The text at offset_start 67 must match entity 'TRKB receptor', but found 'EIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mInteractions between brain-derived neurotrophic factor and the PROTEIN1 receptor. Identification of two ligand binding domains in soluble TRKB by affinity separation and chemical cross-linking.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,268 - preprocessor.ProteinMasker - WARNING - The text at offset_start 81 must match entity 'insulin-like growth factor-I (IGF-I) receptor', but found '-1 (IRS-1) with the PROTEIN1 (IGF-I) receptor' for text \u001b[0m\n",
      "\u001b[31mEvidence for a differential interaction of SHC and the insulin receptor substrate-1 (IRS-1) with the PROTEIN1 (IGF-I) receptor in the yeast two-hybrid system.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,290 - preprocessor.ProteinMasker - WARNING - The text at offset_start 104 must match entity 'IGF-1R beta', but found 'OTEIN1 beta' for text \u001b[0m\n",
      "\u001b[31mThe results of his3 activation indicated that p85, N + C-SH2 and C-SH2 interact with both IR beta and PROTEIN1 beta, whereas N-SH2 and SH3 + N-SH2 interact only with IR beta.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,291 - preprocessor.ProteinMasker - WARNING - The text at offset_start 128 must match entity 'PTF alpha', but found 'IN1 alpha' for text \u001b[0m\n",
      "\u001b[31mWe previously have shown that PTF purified from human HeLa cells is a multisubunit complex of four polypeptides designated PROTEIN1 alpha, -beta, -gamma, and -delta.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,334 - preprocessor.ProteinMasker - WARNING - The text at offset_start 48 must match entity 'insulin receptor', but found 'ROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mGastrin stimulates tyrosine phosphorylation of PROTEIN1 receptor substrate 1 and its association with Grb2 and the phosphatidylinositol 3-kinase.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,335 - preprocessor.ProteinMasker - WARNING - The text at offset_start 127 must match entity 'insulin-like growth factor-I receptor', but found ' interaction of the PROTEIN1 receptor' for text \u001b[0m\n",
      "\u001b[31mUsing the yeast two-hybrid system, a genetic assay for studying protein-protein interactions, we have examined and compared the interaction of the PROTEIN1 receptor (IGF-IR) and the insulin receptor (IR) with their two known substrates p52Shc and the insulin receptor substrate-1 (IRS-1).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,341 - preprocessor.ProteinMasker - WARNING - The text at offset_start 131 must match entity 'NGF-receptor', but found 'IN1-receptor' for text \u001b[0m\n",
      "\u001b[31mAlthough multimeric complexes of trkA and p75 were not detected by affinity crosslinking, immunoprecipitation of cross-linked PROTEIN1-receptor complexes with trk-specific antibodies resulted in selective immunoprecipitation of crosslinked p75.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,347 - preprocessor.ProteinMasker - WARNING - The text at offset_start 118 must match entity 'EPO-mimetic peptide', but found 'IN1-mimetic peptide' for text \u001b[0m\n",
      "\u001b[31mRemarkably, Phe93 forms extensive contacts with a peptide ligand in the crystal structure of the EBP bound to an PROTEIN1-mimetic peptide (EMP1), suggesting that Phe93 is also important for EMP1 binding.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,371 - preprocessor.ProteinMasker - WARNING - The text at offset_start 193 must match entity 'FGFR3 IIIb', but found 'TEIN1 IIIb' for text \u001b[0m\n",
      "\u001b[31mIn order to characterize the signaling pathway through which FGFR3 regulates cell growth, L6 cells lacking any endogenous FGFR were stably transfected with the two different human isoforms, PROTEIN1 IIIb and FGFR3 IIIc, that result from alternative splicing of exon III of the FGFR3 gene encoding the ligand binding domain.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,427 - preprocessor.ProteinMasker - WARNING - The text at offset_start 79 must match entity 'HVEM', but found 'ROTE' for text \u001b[0m\n",
      "\u001b[31mHere, we demonstrate that TR6 specifically binds two cellular ligands, LIGHT (PROTEIN2) and Fas ligand (FasL/CD95L).\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,439 - preprocessor.ProteinMasker - WARNING - The text at offset_start 106 must match entity 'MAP kinase kinase', but found 'd PROTEIN1 kinase' for text \u001b[0m\n",
      "\u001b[31mFurthermore, a bacterially expressed v-Raf fusion protein (glutathione S-transferase-p3722W) also activated PROTEIN1 kinase in vitro.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,440 - preprocessor.ProteinMasker - WARNING - The text at offset_start 38 must match entity 'interferon-gamma inducing factor', but found 'rsor of PROTEIN1 inducing factor' for text \u001b[0m\n",
      "\u001b[31mICE was also shown to cleave the precursor of PROTEIN1 inducing factor (IGIF) at the authentic processing site with high efficiency, thereby activating IGIF and facilitating its export.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,440 - preprocessor.ProteinMasker - WARNING - The text at offset_start 74 must match entity 'IFN-gamma SC1', but found ' PROTEIN1 SC1' for text \u001b[0m\n",
      "\u001b[31mThe structure also reveals the final six C-terminal amino acid residues of PROTEIN1 SC1 (residues 253-258) that have not been observed in any other reported IFN-gamma structures.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,446 - preprocessor.ProteinMasker - WARNING - The text at offset_start 30 must match entity 'ECK receptor protein-tyrosine kinase', but found 'IN1 receptor protein-tyrosine kinase' for text \u001b[0m\n",
      "\u001b[31mA protein ligand for the PROTEIN1 receptor protein-tyrosine kinase has been isolated by using the extracellular domain (ECK-X) of the receptor as an affinity reagent.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,594 - algorithms.transform_berttext_tokenise - INFO - Completed TransformBertTextTokenise\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:03,594 - algorithms.transform_berttext_token_to_index - INFO - Transforming TransformBertTextTokenToIndex\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m2019-10-13 06:19:04,042 - algorithms.transform_berttext_token_to_index - INFO - Completed TransformBertTextTokenToIndex\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,055 - algorithms.transform_berttext_tokenise - INFO - Transforming TransformBertTextTokenise\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,128 - preprocessor.ProteinMasker - WARNING - The text at offset_start 45 must match entity 'TATA-binding protein', but found 't binds directly to ' for text \u001b[0m\n",
      "\u001b[31mThe retinoblastoma-susceptibility gene product binds directly to the human PROTEIN1 TAFII250.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,129 - preprocessor.ProteinMasker - WARNING - The text at offset_start 48 must match entity 'TATA-binding protein-associated factor', but found 'tion in the PROTEIN1-associated factor' for text \u001b[0m\n",
      "\u001b[31mThese cells contain a temperature-sensitive mutation in the PROTEIN1-associated factor TAFII250, first identified as the cell cycle regulatory protein CCG1.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,257 - preprocessor.ProteinMasker - WARNING - The text at offset_start 21 must match entity 'PDGF', but found 'h th' for text \u001b[0m\n",
      "\u001b[31mSHPTP2 associates with the PROTEIN1 after ligand stimulation, and binding of SHPTP2 to this receptor promotes tyrosine phosphorylation of SHPTP2.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,258 - preprocessor.ProteinMasker - WARNING - The text at offset_start 5 must match entity 'platelet-derived growth factor (PDGF) receptor', but found '2 associates with the PROTEIN1 (PDGF) receptor' for text \u001b[0m\n",
      "\u001b[31mSHPTP2 associates with the PROTEIN1 (PDGF) receptor after ligand stimulation, and binding of SHPTP2 to this receptor promotes tyrosine phosphorylation of SHPTP2.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,304 - preprocessor.ProteinMasker - WARNING - The text at offset_start 79 must match entity 'CREB-binding protein', but found 'EIN1-binding protein' for text \u001b[0m\n",
      "\u001b[31mSREBP transcriptional activity is mediated through an interaction with the PROTEIN1-binding protein.\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,421 - algorithms.transform_berttext_tokenise - INFO - Completed TransformBertTextTokenise\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,421 - algorithms.transform_berttext_token_to_index - INFO - Transforming TransformBertTextTokenToIndex\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,455 - algorithms.transform_berttext_token_to_index - INFO - Completed TransformBertTextTokenToIndex\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,456 - algorithms.transform_label_encoder - INFO - Running TransformLabelEncoder\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,458 - algorithms.transform_label_encoder - INFO - Complete TransformLabelEncoder\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,500 - algorithms.BertTrainInferencePipeline - INFO - Retrieving key learningrate with default .01, found 1e-05\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,500 - algorithms.BertTrainInferencePipeline - INFO - Retrieving key weight_decay with default .0001, found .0001\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,500 - pytorch_pretrained_bert.optimization - WARNING - t_total value of -1 results in schedule not being applied\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,501 - algorithms.BertTrainInferencePipeline - INFO - Using optimiser <class 'pytorch_pretrained_bert.optimization.BertAdam'>\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,502 - algorithms.transform_label_rehaper - INFO - Loading int 1 to tensor 1\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,502 - algorithms.BertTrainInferencePipeline - INFO - Positive label True is 1\u001b[0m\n",
      "\u001b[31m2019-10-13 06:19:04,530 - algorithms.BertTrain - INFO - using score : <class 'algorithms.result_scorer_f1.ResultScorerF1'>\u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:40,752 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:40,789 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_d59647d3-5700-40d0-ac6a-bbadff5e32fa_20191013_062340.csv: \u001b[0m\n",
      "\u001b[31m[[3928    0]\n",
      " [ 929    0]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:40,797 - algorithms.BertTrain - INFO - Train set result details: 0.0\u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:40,797 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:45,101 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_74894d4e-e0c6-4e17-915d-447a1bf02e4e_20191013_062345.csv: \u001b[0m\n",
      "\u001b[31m[[304   0]\n",
      " [ 61   0]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:45,102 - algorithms.BertTrain - INFO - Validation set result details: 0.0 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:45,102 - algorithms.BertTrain - INFO - Snapshotting because the current score 0.0 is greater than None \u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:45,102 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/model/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:23:45,542 - algorithms.BertTrain - INFO - Run    281     0       608     2/608         0% 297.467044 20.982365       0.0000       0.0000\u001b[0m\n",
      "\u001b[31m###score: train_loss### 297.46704365313053\u001b[0m\n",
      "\u001b[31m###score: val_loss### 20.982364550232887\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.0\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.0\u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:19,859 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:19,866 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_e59195e6-8447-43cd-949a-79718ec78e8b_20191013_062819.csv: \u001b[0m\n",
      "\u001b[31m[[3928    0]\n",
      " [ 929    0]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:19,872 - algorithms.BertTrain - INFO - Train set result details: 0.0\u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:19,872 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:24,156 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_e42ac294-951a-4e28-8f52-ce84424fb99e_20191013_062824.csv: \u001b[0m\n",
      "\u001b[31m[[304   0]\n",
      " [ 61   0]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:24,157 - algorithms.BertTrain - INFO - Validation set result details: 0.0 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:24,157 - algorithms.BertTrain - INFO - Snapshotting because the current loss 19.604102976620197 is lower than 20.982364550232887 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:24,157 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/output/data/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:28:24,579 - algorithms.BertTrain - INFO - Run    560     1      1216     2/608         0% 279.313289 19.604103       0.0000       0.0000\u001b[0m\n",
      "\u001b[31m###score: train_loss### 279.31328853964806\u001b[0m\n",
      "\u001b[31m###score: val_loss### 19.604102976620197\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.0\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.0\u001b[0m\n",
      "\u001b[31m2019-10-13 06:32:56,998 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:32:57,005 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_3a9d8b6d-2ae9-4150-bcda-31bfc829251d_20191013_063257.csv: \u001b[0m\n",
      "\u001b[31m[[3928    0]\n",
      " [ 929    0]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:32:57,011 - algorithms.BertTrain - INFO - Train set result details: 0.0\u001b[0m\n",
      "\u001b[31m2019-10-13 06:32:57,011 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:33:01,286 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_c1981716-1f66-4e68-ac65-5b369aeaa241_20191013_063301.csv: \u001b[0m\n",
      "\u001b[31m[[304   0]\n",
      " [ 61   0]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:33:01,287 - algorithms.BertTrain - INFO - Validation set result details: 0.0 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:33:01,287 - algorithms.BertTrain - INFO - Snapshotting because the current loss 17.030347734689713 is lower than 19.604102976620197 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:33:01,287 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/output/data/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:33:01,812 - algorithms.BertTrain - INFO - Run    837     2      1824     2/608         0% 235.975859 17.030348       0.0000       0.0000\u001b[0m\n",
      "\u001b[31m###score: train_loss### 235.9758588820696\u001b[0m\n",
      "\u001b[31m###score: val_loss### 17.030347734689713\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.0\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.0\u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:34,205 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:34,213 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_0b517276-6ff4-41cf-a5d6-991963e13d63_20191013_063734.csv: \u001b[0m\n",
      "\u001b[31m[[3740  188]\n",
      " [ 121  808]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:34,219 - algorithms.BertTrain - INFO - Train set result details: 0.8394805194805195\u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:34,219 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:38,497 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_25eaea0e-69e0-4776-85c5-9c69f00429c3_20191013_063738.csv: \u001b[0m\n",
      "\u001b[31m[[277  27]\n",
      " [ 17  44]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:38,498 - algorithms.BertTrain - INFO - Validation set result details: 0.6666666666666666 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:38,498 - algorithms.BertTrain - INFO - Snapshotting because the current score 0.6666666666666666 is greater than 0.0 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:38,498 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/model/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:37:39,036 - algorithms.BertTrain - INFO - Run   1114     3      2432     2/608         0% 105.420494 12.744195       0.8395       0.6667\u001b[0m\n",
      "\u001b[31m###score: train_loss### 105.4204942509532\u001b[0m\n",
      "\u001b[31m###score: val_loss### 12.74419455602765\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.8394805194805195\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.6666666666666666\u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:11,618 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:11,625 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_a41db3e4-b00e-44b9-a8eb-bca39418516e_20191013_064211.csv: \u001b[0m\n",
      "\u001b[31m[[3839   89]\n",
      " [  61  868]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:11,631 - algorithms.BertTrain - INFO - Train set result details: 0.9204665959703076\u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:11,631 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:15,912 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_c920f9a6-5881-4776-bdf6-abfcfa3e5423_20191013_064215.csv: \u001b[0m\n",
      "\u001b[31m[[276  28]\n",
      " [ 15  46]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:15,913 - algorithms.BertTrain - INFO - Validation set result details: 0.6814814814814815 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:15,913 - algorithms.BertTrain - INFO - Snapshotting because the current score 0.6814814814814815 is greater than 0.6666666666666666 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:15,913 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/model/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:42:16,420 - algorithms.BertTrain - INFO - Run   1391     4      3040     2/608         0% 56.131028 14.111387       0.9205       0.6815\u001b[0m\n",
      "\u001b[31m###score: train_loss### 56.13102777302265\u001b[0m\n",
      "\u001b[31m###score: val_loss### 14.111386666074395\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.9204665959703076\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.6814814814814815\u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:48,736 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:48,743 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_e4e038d3-3b93-4f5e-8a43-2d9ac388876b_20191013_064648.csv: \u001b[0m\n",
      "\u001b[31m[[3892   36]\n",
      " [  72  857]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:48,750 - algorithms.BertTrain - INFO - Train set result details: 0.9407244785949506\u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:48,750 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:53,027 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_69b949c6-5d6f-4225-994f-7cf0480c7eaf_20191013_064653.csv: \u001b[0m\n",
      "\u001b[31m[[285  19]\n",
      " [ 19  42]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:53,029 - algorithms.BertTrain - INFO - Validation set result details: 0.6885245901639344 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:53,029 - algorithms.BertTrain - INFO - Snapshotting because the current score 0.6885245901639344 is greater than 0.6814814814814815 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:53,029 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/model/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:46:53,535 - algorithms.BertTrain - INFO - Run   1669     5      3648     2/608         0% 41.106530 14.376156       0.9407       0.6885\u001b[0m\n",
      "\u001b[31m###score: train_loss### 41.10653046146035\u001b[0m\n",
      "\u001b[31m###score: val_loss### 14.376156121492386\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.9407244785949506\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.6885245901639344\u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:26,920 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:26,928 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_a55a8ee9-c829-42d6-86c0-272ecc01da08_20191013_065126.csv: \u001b[0m\n",
      "\u001b[31m[[3859   69]\n",
      " [  28  901]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:26,934 - algorithms.BertTrain - INFO - Train set result details: 0.9489204844655083\u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:26,934 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:31,214 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_efb3445a-9150-4583-b6f7-acd999bae4a8_20191013_065131.csv: \u001b[0m\n",
      "\u001b[31m[[278  26]\n",
      " [ 15  46]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:31,215 - algorithms.BertTrain - INFO - Validation set result details: 0.6917293233082706 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:31,216 - algorithms.BertTrain - INFO - Snapshotting because the current score 0.6917293233082706 is greater than 0.6885245901639344 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:31,216 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/model/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:51:31,725 - algorithms.BertTrain - INFO - Run   1947     6      4256     2/608         0% 35.046889 17.277316       0.9489       0.6917\u001b[0m\n",
      "\u001b[31m###score: train_loss### 35.04688894748688\u001b[0m\n",
      "\u001b[31m###score: val_loss### 17.277315854094923\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.9489204844655083\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.6917293233082706\u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:05,343 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:05,350 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_07dfc9d3-b23c-4f91-8ae0-4a38f146b130_20191013_065605.csv: \u001b[0m\n",
      "\u001b[31m[[3909   19]\n",
      " [  35  894]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:05,357 - algorithms.BertTrain - INFO - Train set result details: 0.9706840390879479\u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:05,357 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:09,640 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_918da9c2-4dc4-4597-bb03-189891984a6a_20191013_065609.csv: \u001b[0m\n",
      "\u001b[31m[[289  15]\n",
      " [ 18  43]]\u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:09,641 - algorithms.BertTrain - INFO - Validation set result details: 0.7226890756302522 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:09,642 - algorithms.BertTrain - INFO - Snapshotting because the current score 0.7226890756302522 is greater than 0.6917293233082706 \u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:09,642 - algorithms.ModelSnapshotCallback - INFO - Snappshotting model to /opt/ml/model/best_snaphsotmodel.pt\u001b[0m\n",
      "\u001b[31m2019-10-13 06:56:10,152 - algorithms.BertTrain - INFO - Run   2225     7      4864     2/608         0% 30.415745 20.696409       0.9707       0.7227\u001b[0m\n",
      "\u001b[31m###score: train_loss### 30.415745295584202\u001b[0m\n",
      "\u001b[31m###score: val_loss### 20.696409022435546\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.9706840390879479\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.7226890756302522\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m2019-10-13 07:00:43,766 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 07:00:43,773 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_79c3f392-1e63-44ab-aca7-4264aa5c4529_20191013_070043.csv: \u001b[0m\n",
      "\u001b[31m[[3903   25]\n",
      " [   8  921]]\u001b[0m\n",
      "\u001b[31m2019-10-13 07:00:43,780 - algorithms.BertTrain - INFO - Train set result details: 0.9824\u001b[0m\n",
      "\u001b[31m2019-10-13 07:00:43,780 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 07:00:48,062 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_250163d1-a531-4ecb-b123-afb09caff9bd_20191013_070048.csv: \u001b[0m\n",
      "\u001b[31m[[282  22]\n",
      " [ 16  45]]\u001b[0m\n",
      "\u001b[31m2019-10-13 07:00:48,064 - algorithms.BertTrain - INFO - Validation set result details: 0.703125 \u001b[0m\n",
      "\u001b[31m2019-10-13 07:00:48,064 - algorithms.BertTrain - INFO - Run   2503     8      5472     2/608         0% 15.340616 20.660026       0.9824       0.7031\u001b[0m\n",
      "\u001b[31m###score: train_loss### 15.340615972876549\u001b[0m\n",
      "\u001b[31m###score: val_loss### 20.660025525139645\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.9824\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.703125\u001b[0m\n",
      "\u001b[31m2019-10-13 07:05:21,517 - algorithms.BertTrain - INFO - Train set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 07:05:21,524 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_3825f74a-6f0c-46eb-af6f-8c6a81ed181e_20191013_070521.csv: \u001b[0m\n",
      "\u001b[31m[[3890   38]\n",
      " [   6  923]]\u001b[0m\n",
      "\u001b[31m2019-10-13 07:05:21,531 - algorithms.BertTrain - INFO - Train set result details: 0.9767195767195767\u001b[0m\n",
      "\u001b[31m2019-10-13 07:05:21,531 - algorithms.BertTrain - INFO - Validation set result details:\u001b[0m\n",
      "\u001b[31m2019-10-13 07:05:25,814 - algorithms.result_writer - INFO - Confusion matrix, full output in /opt/ml/output/data/predictedvsactual_b0672129-0b1b-416c-99d3-faa74a0ed062_20191013_070525.csv: \u001b[0m\n",
      "\u001b[31m[[280  24]\n",
      " [ 13  48]]\u001b[0m\n",
      "\u001b[31m2019-10-13 07:05:25,815 - algorithms.BertTrain - INFO - Validation set result details: 0.7218045112781954 \u001b[0m\n",
      "\u001b[31m2019-10-13 07:05:25,815 - algorithms.BertTrain - INFO - Run   2781     9      6080     2/608         0% 17.274194 17.087270       0.9767       0.7218\u001b[0m\n",
      "\u001b[31m###score: train_loss### 17.27419427037239\u001b[0m\n",
      "\u001b[31m###score: val_loss### 17.087270230054855\u001b[0m\n",
      "\u001b[31m###score: train_fscore### 0.9767195767195767\u001b[0m\n",
      "\u001b[31m###score: val_fscore### 0.7218045112781954\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "estimator.fit(inputs)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
